<!DOCTYPE html >
<html id="a0FcBivOl8w80_RPClwx6xgJ4JHe-PMC2964648" data-origid="PMC2964648" class="anndoc" data-anndoc-version="2.0" lang="" xml:lang="" xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="UTF-8"/>
    <meta name="generator" content="com.jmcejuela.anndoc.v2.bio.NcbiJournalArticleParser_v0_4$"/>
    <meta name="dcterms.source" content="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2964648/"/>
    <title>a0FcBivOl8w80_RPClwx6xgJ4JHe-PMC2964648</title>
  </head>
  <body>
    <article>
      <section data-type="title">
        <h2 id="s1h1">Mutational spectrum of the SPG4 (SPAST) and SPG3A (ATL1) genes in Spanish patients with hereditary spastic paraplegia</h2>
      </section>
      <section data-type="abstract">
        <h2 id="s2h1">Abstract</h2>
        <div class="subsections">
          <section data-type="">
            <h3 id="s2s1h1">Background</h3>
            <div class="content">
              <p id="s2s1p1">Hereditary Spastic Paraplegias (HSP) are characterized by progressive spasticity and weakness of the lower limbs. At least 45 loci have been identified in families with autosomal dominant (AD), autosomal recessive (AR), or X-linked hereditary patterns. Mutations in the SPAST (SPG4) and ATL1 (SPG3A) genes would account for about 50% of the ADHSP cases.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s2s2h1">Methods</h3>
            <div class="content">
              <p id="s2s2p1">We defined the SPAST and ATL1 mutational spectrum in a total of 370 unrelated HSP index cases from Spain (83% with a pure phenotype).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s2s3h1">Results</h3>
            <div class="content">
              <p id="s2s3p1">We found 50 SPAST mutations (including two large deletions) in 54 patients and 7 ATL1 mutations in 11 patients. A total of 33 of the SPAST and 3 of the ATL1 were new mutations. A total of 141 (31%) were familial cases, and we found a higher frequency of mutation carriers among these compared to apparently sporadic cases (38% vs. 5%). Five of the SPAST mutations were predicted to affect the pre-mRNA splicing, and in 4 of them we demonstrated this effect at the cDNA level. In addition to large deletions, splicing, frameshifting, and missense mutations, we also found a nucleotide change in the stop codon that would result in a larger ORF.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s2s4h1">Conclusions</h3>
            <div class="content">
              <p id="s2s4p1">In a large cohort of Spanish patients with spastic paraplegia, SPAST and ATL1 mutations were found in 15% of the cases. These mutations were more frequent in familial cases (compared to sporadic), and were associated with heterogeneous clinical manifestations.</p>
            </div>
          </section>
        </div>
      </section>
      <section data-type="">
        <h2 id="s3h1">Background</h2>
        <div class="content">
          <p id="s3p1">The hereditary spastic paraplegias (HSP) are characterized by progressive spasticity and weakness of the lower limbs due to axonal degeneration in the pyramidal tract. The disease is classified as &quot;pure&quot; when spasticity is the only clinical finding, and as &quot;complicated&quot; when other clinical features (dementia, cerebellar ataxia, epilepsy, peripheral neuropathy) are also present [1]. HSP is frequently familial and at least 45 loci have been identified in families with autosomal dominant (AD), autosomal recessive (AR), or X-linked inheritance [2-4]. This genetic heterogeneity partly explains the differences in disease severity, age at onset, rate of progression, and degree of disability between families. However, intrafamilial heterogeneity is also frequent.</p>
          <p id="s3p2">SPG4 (OMIM#604277) is the most common form of HSP, accounting for approximately 40% of the familial and 6-15% of the sporadic cases. The SPAST/SPG4 gene encodes spastin, a member of the AAA (ATPase associated with various cellular activities) family of proteins, implicated in the remodeling of protein complexes upon ATP hydrolysis and the coordination of axonal microtubule interactions with the tubular endoplasmic reticulum network [5,6]. To date, &gt;240 SPAST mutations have been reported (http://www.hgmd.cf.ac.uk/ac/index.php; date of consultation August 2, 2010), mainly in patients with pure HSP [7-12]. Most of the SPAST mutations are single-nucleotide changes or small deletions/insertions, but large deletions and duplications have also been reported [13-16]. This suggested that both, haploinsufficiency and &quot;toxic&quot; gain of function could explain the pathogenic mechanism of SPAST mutations [5,17-19].</p>
          <p id="s3p3">SPG3A (OMIM#606439) is the second most frequent form of ADHSP. The ATL1/SPG3A gene encodes atlastin, a protein localized in the endoplasmic reticulum and the Golgi and implicated in vesicle trafficking and neurite outgrowth [20,21]. To date, &gt;25 ATL1 mutations have been reported (http://www.hgmd.cf.ac.uk/ac/index.php; date of consultation August 2, 2010), mainly missense changes that supported a gain of function pathogenic mechanism. ATL1 mutations accounted for approximately 10% of the ADHSP families, and have been mainly found in pure HSP [22,23]. ATL1 mutations are also frequent in early onset (childhood or adolescence) cases [24-26].</p>
          <p id="s3p4">This is the first report of the mutational spectrum of the SPAST and ATL1 genes in a large cohort of unrelated HSP patients from Spain. Few studies on cohorts &gt;200 patients have been published, and the parallel screening of both genes was rarely reported.</p>
        </div>
      </section>
      <section data-type="methods">
        <h2 id="s4h1">Methods</h2>
        <div class="subsections">
          <section data-type="">
            <h3 id="s4s1h1">Patients</h3>
            <div class="content">
              <p id="s4s1p1">This study was approved by the Ethical Committee of Hospital Universitario Central Asturias (HUCA), and all the participants (patients and controls) signed an informed consent. A total of 370 non-related patients (index cases) were recruited through the Neurology Departments of several Hospitals from Spain. HSP was diagnosed by qualified neurologists on the basis of Harding's criteria. Based on the clinical, radiological, and biochemical findings, cases with diseases that mimicked spastic paraplegia were excluded [27]. A total of 141 patients (38%) had a family history of HSP with a dominant inheritance pattern, and were thus classified as ADHSP cases. The absence of family history of the disease was established in 229 patients (62%) after interview on their first and second degree relatives. These cases were classified as &quot;apparently&quot; sporadic or with an uncertain inheritance pattern. In 177 of these the two parents were alive and did not have symptoms consistent with HSP, while in 52 the inheritance pattern could not be established because no clinical data were available from relatives.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s2h1">SPAST and ATL1 sequencing</h3>
            <div class="content">
              <p id="s4s2p1">DNA was obtained from blood leukocytes and the 17 coding exons of SPAST (plus at least 50 bp of the flanking intronic sequences) were polymerase chain reaction (PCR) amplified (Additional file 1, Table S1). PCR fragments were purified and the two strands were sequenced using Big Dye chemistry in an ABI3130 system (Applied Biosystems, Ca, USA). These sequences were compared with the SPAST reference sequence (ENSG00000021574 for the genomic; ENST00000315285 for the transcript; http://www.ensembl.org). In patients who were negative for SPAST mutations and had ADHSP (n = 88) or patients without a family history of HSP and onset age ≤20 years (n = 99), we amplified and sequenced the 13 coding exons of ATL1 (reference sequences ENSG00000198513 for the genomic, and ENST00000358385 for the transcript; http://www.ensembl.org). All the DNA sequence variants were named following the guidelines of the Human Genome Variation Society (http://www.hgvs.org/mutnomen).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s3h1">Controls screening</h3>
            <div class="content">
              <p id="s4s3p1">All the new putative mutations (not previously reported as SPAST or ATL1 mutations or polymorphisms) were screened in 400 controls. These were healthy individuals aged 21-65 years recruited through the Blood Bank of HUCA. The new nucleotide changes were genotyped through PCR-RFLP, single strand conformation analysis (SSCA), or denaturing high performance liquid chromatography (DHPLC), as reported [28]. Each PCR fragment containing a putative mutation gave a characteristic RFLP, SSCA or DHPLC pattern, and we could thus determine its presence/absence in the controls. When a new nucleotide change was not found among the 400 controls we determined, when it was possible, the carrier status of all the available affected relatives to confirm the segregation of the mutation with the disease.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s4h1">MLPA analysis</h3>
            <div class="content">
              <p id="s4s4p1">The multiplex ligation dependent probe amplification (MLPA) assay with the Salsa Kit P165 HPA (MRC Holland, Amsterdam) was used to determine the existence of genome copy number aberrations in the SPAST and ATL1 genes in ADHSP patients who were negative for sequencing mutations [13]. To investigate the consequences of large SPAST deletions at the transcript level, we isolated the mRNA from leukocytes in 10 ml of blood (Trizol reagent, Invitrogen, Carlsbad, CA, USA) and synthesised the cDNA (Quantitec Reverse Transcripcion kit, Quiagen, Hilden, Germany). The cDNA was amplified with primers that matched exons flanking the deletion (conditions available upon request), and the PCR products were purified and sequenced.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s5h1">Analysis of SPAST transcripts</h3>
            <div class="content">
              <p id="s4s5p1">To determine the effect of some SPAST mutations on mRNA transcripts, the cDNA synthesised from leukocytes was amplified with primers that matched the exons flanking the mutation (additional file 1, Table S2), and the PCR fragments were purified from agarose gels and sequenced.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s6h1">Statistical analysis</h3>
            <div class="content">
              <p id="s4s6p1">The analysis of variance (SPSS 17.0 software) was used to compare the mean onset age and disease duration between patients with mutations in SPAST and ATL1 and patients without mutations.</p>
            </div>
          </section>
        </div>
      </section>
      <section data-type="">
        <h2 id="s5h1">Results</h2>
        <div class="subsections">
          <section data-type="">
            <h3 id="s5s1h1">Patients characteristics</h3>
            <div class="content">
              <p id="s5s1p1">We studied a total of 370 unrelated HSP index cases, with a mean onset age of 28 (± 21) years (range 1-77 years), and a mean disease duration of 17 (± 15) years. A total of 321 patients (87%) had a pure HSP, while 49 (13%) were complicated cases with peripheral neuropathy (the most frequent finding), cerebellar or cerebral atrophy, mental retardation, nystagmus, or dysarthia.</p>
              <p id="s5s1p2">A total of 44 of the 141 patients (31%) with ADHSP had a SPAST mutation, and 10 mutations were found among the 229 (5%) cases apparently sporadic or with an uncertain inheritance pattern. However, in three of these patients the mutation (p.Leu426Val, p.Lys503insArg, and p.Met390Val) was de novo because none of the two parents was mutation carrier (the paternity was confirmed). We found 10 ATL1 mutation carriers among 88 ADHSP cases (11%), and the only apparently sporadic proband had a de novo mutation (p.Gln154Glu) (Additional file 2, Figure S1).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s2h1">SPAST mutations</h3>
            <div class="content">
              <p id="s5s2p1">We found 48 SPAST mutations in 52 patients (Table 1). These were missense changes in amino acids conserved between species (n = 22), nonsense mutations (n = 7), small frameshifting insertions/deletions (n = 13), nucleotide changes in the intronic splicing consensus sequence (n = 5), and a sense mutation in the stop codon (Figure 1). A total of 15 mutations had been previously reported, while 33 (69%) were novel.</p>
              <figure class="table" data-origid="T1">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2964648/#T1">Table 1</a>
                <figcaption>
                  <p id="s5s2f1p1">Mutations identied in the SPAST gene in the Spanish HSP cohort.</p>
                </figcaption>
              </figure>
              <figure data-origid="F1">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2964648/#F1">Figure 1</a>
                <figcaption>
                  <p id="s5s2f2p1">Schematic diagram of SPAST and ATL1 domains showing the location of the mutations. The one letter code was used to identify the amino acid changes. Novel mutations are in bold.</p>
                </figcaption>
              </figure>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s3h1">SPAST missense mutations out of the AAA domain</h3>
            <div class="content">
              <p id="s5s3p1">Most of the missense mutations were in the AAA domain. Two novel missense changes (p. Pro293Leu, and p. Asp613Ala) were outside of the AAA domain, where missense mutations have been rarely found. The two amino acids were evolutionary conserved. The p.Pro293Leu proband was a 51-year-old man with progressive gait disorder starting at the age of 31, who has been confined to a wheelchair due to his spasticity (pure form) for the last two years. His 46-year-old sister, and one of his daughters (18 years old) also carried this mutation. They both presented brisk reflexes and Babinski's sign, but no spasticity and an almost normal gait. The p.Asp613Ala was found in a 45 years old man with a pure phenotype affecting only the lower limbs and the first symptoms at the age of 36 years. He did not report a family history of the disease, although no relatives were studied.</p>
              <p id="s5s3p2">We also found a new SPAST missense variant (p.Ile328Leu) in a 15 years old male with complicated spastic paraplegia starting at the age of 18 months. This putative mutation was also found in his father's grandmother (asymptomatic), but the proband's father was not available for study. Although this variant was not found among the controls and involved a conserved amino acid, a bioinformatics analysis (SIFT v2. program, Sorting Intolerant From Tolerant; http://sift.jcvi.org/) indicated a non significant change in the protein function, and was thus classified as a change of uncertain pathogenic effect.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s4h1">SPAST c.583C&gt;G (exon 3) affected splicing</h3>
            <div class="content">
              <p id="s5s4p1">The c.583C&gt;G (p.Leu195Val) was found in a 19 years old male with an onset at the age of 2 years and a phenotype complicated with neuropathy. He was from a family with ADSPH in which we confirmed the segregation of the disease with this mutation. This has been previously reported as a missense mutation, and the bioinformatic analysis (SIFT v2. program) indicated that this change would affect the protein function. However, the c.583C&gt;G was four nucleotides from the 3' end of exon 3, and was also predicted to reduce the score of the splicing consensus site (Human Splicing Finder v. 2.4; http://www.umd.be/SSF). This mutation could thus result in an aberrant mRNA sequence. To confirm this, we isolated the mRNA from the patient's blood leukocytes and amplified the SPAST cDNA with primers for exons 2 and 7. The last 4 nucleotides of exon 3 were missed in the transcript, that would thus be translated into an aberrant protein (Figure 2A).</p>
              <figure data-origid="F2">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2964648/#F2">Figure 2</a>
                <figcaption>
                  <p id="s5s4f1p1">A) The cDNA from patient 138 (carrier of c.583C&gt;G in exon 3 of SPAST) was synthesised from leukocytes mRNA. The sequencing of the fragment amplified with primers that matched exons 2 and 7 showed the deletion of the last 4 nucleotides of exon 3. B) The cDNA from patient 212 (X617Glu) was amplified with primers for exons 11 and 17 of SPAST, and the sequence showed similar electropherogram peaks for the two alleles. C) The cDNA of patient 322 was amplified with primers for exons 7 and 17 of SPAST, and the sequence showed the effect on splicing of the c.1414-1G&gt;C mutation. D) MLPA of the SPAST gene in patient 199, showing the exons 10-16 deletion. The cDNA was amplified with primers that matched exons 9 and 17, and two PCR fragments were obtained. Sequencing of the smaller fragment confirmed the absence of exons 10 to 16. P = patient (cDNA); M = Patient's mother's (cDNA); C = Control (cDNA); G = Genomic DNA</p>
                </figcaption>
              </figure>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s5h1">SPAST sense mutation</h3>
            <div class="content">
              <p id="s5s5p1">We found a novel mutation in the SPAST stop codon, c.1849T&gt;G (p.X617Glu) in two non related patients without known affected relatives. The mutated transcript was predicted to be translated into a 46 amino acids longer protein. To determine the stability of the mRNA containing this mutation, we synthesized the cDNA from RNA obtained from leukocytes of one of the patients and amplified and sequenced a PCR-fragment generated with primers that matched exons 11 and the 3' untranslated region (UTR). The two alleles gave equally intense sequence electropherogram peaks, suggesting that c.1849G did not increase the mRNA decay and was likely translated into a mutated protein (Figure 2B).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s6h1">SPAST mutations affecting splicing</h3>
            <div class="content">
              <p id="s5s6p1">In addition to the above described c.583C&gt;G in exon 3 of SPAST, we found five changes in intron splicing consensus nucleotides. The in silico analysis (Splice Site Prediction by Neural Network; http://www.fruitfly.org/seq_tools/splice.html) indicated that the five changes would affect the pre-mRNA splicing. We could synthesize the cDNA from four of the patients, that was amplified with primers that matched exons 7 and 17, and exons 11 and 17. In all these cases two PCR-fragments were amplified, and the longer fragment corresponded to the wild type transcript while the shorter resulted from an exon skipping. In this way, we confirmed the existence of defective transcripts (Figure 2C).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s7h1">Large SPAST deletions</h3>
            <div class="content">
              <p id="s5s7p1">We performed MLPA analysis in the 77 ADHSP cases who were negative for ATL1 and SPAST sequencing mutations. Two patients (2.5%) showed a significantly reduced amplification signal for exons 10-16, or exons 6-7 of SPAST. In the patient with a putative deletion of exons 10-16 we amplified the cDNA with primers that matched exons 9 and 17, and confirmed the skipping of exons 10-16 (Figure 2D). In this pedigree, the age at onset (third decade) was similar in the three affected members who were studied, and the oldest patient (an 81-year-old man) had been confined to a wheelchair for the previous five years.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s8h1">ATL1 mutations</h3>
            <div class="content">
              <p id="s5s8p1">The ATL1 exons were sequenced in a total 88 patients with ADHSP and in 99 patients without a family history of HSP and an onset age ≤20 years. We found seven mutations and a variant with uncertain pathogenic effect in 11 patients (Table 2). Three were novel missense changes (p.His256Asp, p.Gln154Glu, and p.Phe413Val), while 5 had been reported. The p.Arg239Cys was a recurrent mutation found in 3 non related patients.</p>
              <figure class="table" data-origid="T2">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2964648/#T2">Table 2</a>
                <figcaption>
                  <p id="s5s8f1p1">Mutations in the ATL1 gene.</p>
                </figcaption>
              </figure>
              <p id="s5s8p2">Mutations p.Gln154Glu, (c.460 C&gt;G), p. His256Asp (c.766C&gt;A), and p.Phe413Val (c.1237T&gt;G) are located in the guanilate binding protein domains. The p.Gln154Glu mutation was found in a patient with a pure phenotype and onset of symptoms in childhood. His son showed gait problems at the age of two years. The p.His256Asp patient was a 45 years old woman with the onset of symptoms at the age of 3 years and a progressive spastic paraplegia (Additional file 2, Figure S1). In her family, we identified mutation carriers who were asymptomatic at ages of 60 and 63 years, and this could thus be classified as a variant with uncertain pathogenic effect. Mutation p.Phe413Val was found in a patient with an onset age of 6 years and a complicated phenotype (neuropathy). Her mother was affected, but she was death and we could not confirm whether carried the mutation. None of the 77 ADHSP patients studied through MLPA showed evidence for large ATL1 deletions.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s9h1">SPAST and ATL1 polymorphisms</h3>
            <div class="content">
              <p id="s5s9p1">We found several polymorphisms in the two genes (Additional file 1, Table S3). All the new polymorphisms were intronic, or exonic silent amino acid changes with one exception: c.844 T&gt;A in exon 5 of SPAST (p.Ser282Thr). This missense change affected an amino conserved between species and was not found in any of the 400 controls. Although it could be considered a putative mutation, was in a patient with the p.Ser261X mutation (located also in exon 5). Segregation analysis to determine whether the two nucleotide changes were in different chromosomes was not possible. However, the electrophoretic pattern of bands after double restriction enzyme digestion (MnlI + MboI) of the exon 5 fragment indicated that the mutation was in cis with the c.844 A allele (data not shown). This suggested that this was likely a rare variant, rather than an HSP mutation. Two missense polymorphisms that have been proposed as modifiers of the clinical phenotype (p.Ser44Leu and p.Pro45Leu) were not found in our patients [29].</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s10h1">Genotype-phenotype correlation</h3>
            <div class="content">
              <p id="s5s10p1">The phenotype associated with SPAST and ATL1 mutations was pure in 48 (87%) and 10 (90%) of the index patients, respectively. We analysed all the available relatives of the 63 index cases with mutations, and identified a total of 54 SPAST and 15 ATL1 mutation carriers. A total of 13 mutation carriers (10 for SPAST and 3 for ATL1 gene) were asymptomatic at the time of our analysis. The mean onset age of the disease (index cases and relatives) was 34.5 (± 17.72) years for SPAST mutation carriers, and 7.67 (± 5.9 years) for ATL1 mutation carriers (p &lt; 0.001). A difference was also observed between ATL1 mutation carriers and patients without SPAST/ATL1 mutations (7.67 ± 5.9 vs. 27.78 ± 20.238; p &lt; 0.001). The mean duration of symptoms at the time of examination did not differ between patients with SPAST (14.5 ± 11.1) and ATL1 (18.78 ± 15.057) mutations. No difference in disease duration was observed between SPAST or ATL1 mutation carriers and patients without mutations.</p>
            </div>
          </section>
        </div>
      </section>
      <section data-type="">
        <h2 id="s6h1">Discussion</h2>
        <div class="content">
          <p id="s6p1">SPAST mutations (including large gene deletions) were found in 15% (54/370) of the patients, but this frequency increased to 31% (44/141) among patients with ADHSP. Most of the SPAST mutations were novel, and this was in agreement with previous reports that described a high rate of private mutations in this gene [30,10,15]. We found an ATL1 mutation in 6% of the patients studied for this gene, but this frequency could be underestimated because we did not include ADHSP patients with a SPAST mutation or patients with sporadic/uncertain HSP and an onset age &gt;20 years. However, we think that the ATL1 mutation rate should be right because double mutated patients (SPAST + ATL1) have not been reported, and ATL1 mutations were rare among non-ADHSP patients with an onset age &gt;20 years [22,25,26].</p>
          <p id="s6p2">Considering the ADHSP cases, the mutational screening of SPAST and ATL1 identified a total of 54 mutation carriers, a frequency (38%) within the range reported in other populations [10,12,31,32]. Also in agreement with previous reports, the frequency of SPAST and ATL1 mutation carriers was much lower among patients with sporadic/uncertain HSP [12]. Four of the sporadic cases had a de novo mutation. A high rate of de novo mutational events for SPAST and ATL1 has been previously described, indicating that sporadic cases should also be screened for mutations in these genes after exclusion of other major neurological causes of spasticity [15,23,33-35].</p>
          <p id="s6p3">SPAST mutations are mostly restricted to the AAA protein domain. It is thus remarkable that three of the missense mutations in our patients were out of this domain (p.Leu195Val, p.Pro293Leu, and p.Asp613Ala). In the case of ATL1 all the mutations were missense changes in the GTPase and transmembrane domains of atlastin, and would disrupt the normal protein folding and oligomerization [20,21]. The SPAST c.C583G variant was previously reported as a missense mutation (p.Leu195Val) [10]. However, this change was within the consensus splicing sequence of exon 3 and we confirmed its effect on pre-RNA splicing. Nucleotide changes in the last nucleotides of exons could result in both, normal and splicing defective transcripts [36-38]. Although we found an aberrant transcript we could not define whether the c.583 G resulted also in normal transcripts.</p>
          <p id="s6p4">In the SPAST gene we also found one mutation in the stop codon (c.1849T&gt;G; X617Glu), that would result in the translation of 46 amino acid beyond the stop codon. To our knowledge, this is the first sense mutation reported in this gene. The abolishment of a stop codon and the appearance of a longer ORF has been found in other hereditary diseases, such as the British Familial Dementia caused by a sense mutation in the ITM2B gene [39].</p>
          <p id="s6p5">We performed MLPA analysis in 78 patients with ADHSP and we found two large deletions in the SPAST gene, which accounts for 2.5% of patients. In a recent work Shoukier et al. reported a similar frequency for this type of mutation [12]. However, previous studies reported a much higher frequency (18-23%) [13,15]. Additional studies are thus necessary to define the frequency of large deletions/duplications in the aetiology of HSP.</p>
          <p id="s6p6">Finaly, SPAST and ATL1 mutations have been associated with variable penetrance, leading to heterogeneous HSP phenotypes in terms of onset age and clinical symptoms (pure vs. complicated) [40,15]. This heterogeneity could be partly explained by different mutated genes. As discussed above, we observed a significantly higher frequency of familial dominant HSP among ATL1 patients, compared to SPAST mutation carriers, and pure HSP was also more frequent among the patients with ATL1 mutations. However, phenotypic variability was also common among mutation carriers from the same family.</p>
        </div>
      </section>
      <section data-type="">
        <h2 id="s7h1">Conclusions</h2>
        <div class="content">
          <p id="s7p1">In conclusion, we reported the mutational spectrum of the SPAST and ATL1 genes in a large cohort of Spanish patients with spastic paraplegia. We found a mutation in 15% of the cases, and a frequency of mutation carriers significantly higher among ADHSP compared to sporadic cases. Thus, the genetic screening should be more relevant in patients (pure and complicated phenotypes) with a family history of the disease. However, the fact that a significant number of apparently sporadic cases had a mutation suggested that these patients should not be excluded from the genetic study. The mutational report could be of limited value to predict the phenotype associated to these mutations, as demonstrated by the heterogeneous behavior of most of the mutations.</p>
        </div>
      </section>
      <section data-type="">
        <h2 id="s8h1">Competing interests</h2>
        <div class="content">
          <p id="s8p1">The authors declare that they have no competing interests.</p>
        </div>
      </section>
      <section data-type="">
        <h2 id="s9h1">Authors' contributions</h2>
        <div class="content">
          <p id="s9p1">All the authors contributed to this work by recruiting the patients, obtaining the clinical and analytical information, or performing the laboratory work. VA designated the work and analyzed the results. VA, ES-F, CB, M., BA, and AIC performed all the genetic analysis. VA and EC wrote the manuscript. All authors have read and approved the submission of this manuscript.</p>
        </div>
      </section>
      <section data-type="">
        <h2 id="s10h1">Note</h2>
        <div class="content">
          <p id="s10p1">* Manuel Amorín, Eugenia Marzo-Sola, Carlos H. Lahoz, Pia Gallano, Concepción Alonso-Cerezo, Rafael Palencia-Luaces, Eduardo Gutiérrez- Rivas, Rogelio Simón, Loreto Martorell, Eduardo López-Laso, José M. Asensi, Luis Hernández- Echebarria, Yolanda Morgado, Alonso González-Masegosa, Juan J. Garcia- Peñas, Irene Catalina-Alvarez, José L. Muñoz-Blanco, Miguel Fernández-Burriel, Juan B. Espinal, Mariano Aparicio- Blanco, Jon Infante, María Vázquez-Espinar, Elena Maside</p>
        </div>
      </section>
      <section data-type="">
        <h2 id="s11h1">Pre-publication history</h2>
        <div class="content">
          <p id="s11p1">The pre-publication history for this paper can be accessed here:</p>
          <p id="s11p2">http://www.biomedcentral.com/1471-2377/10/89/prepub</p>
        </div>
      </section>
      <section data-type="supplementary-material">
        <h2 id="s12h1">Supplementary Material</h2>
      </section>
    </article>
  </body>
</html>
