<!DOCTYPE html >
<html id="a2tj1z6UROq65P8u_kSQ3cSejz2W-nature09639" data-origid="nature09639" class="anndoc" data-anndoc-version="2.0" lang="" xml:lang="" xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="UTF-8"/>
    <meta name="generator" content="com.jmcejuela.anndoc.v2.bio.NcbiJournalArticleParser_v0_2$"/>
    <meta name="dcterms.source" content="http://doi.org/10.1038/nature09639"/>
    <title>a2tj1z6UROq65P8u_kSQ3cSejz2W-nature09639</title>
  </head>
  <body>
    <article>
      <section data-type="title">
        <h2 id="s1h1">Exome sequencing identifies frequent mutation of the SWI/SNF complex gene PBRM1 in renal carcinoma</h2>
      </section>
      <section data-type="abstract">
        <h2 id="s2h1">Summary</h2>
        <div class="content">
          <p id="s2p1">The genetics of renal cancer is dominated by inactivation of the VHL tumour suppressor gene in clear cell carcinoma (ccRCC), the commonest histological subtype. A recent large-scale screen of ~3500 genes by PCR-based exon re-sequencing identified several new cancer genes in ccRCC including UTX (KDM6A)1, JARID1C (KDM5C) and SETD22. These genes encode enzymes that demethylate (UTX, JARID1C) or methylate (SETD2) key lysine residues of histone H3. Modification of the methylation state of these lysine residues of histone H3 regulates chromatin structure and is implicated in transcriptional control3. However, together these mutations are present in fewer than 15% of ccRCC, suggesting the existence of additional, currently unidentified cancer genes. Here, we have sequenced the protein coding exome in a series of primary ccRCC and report the identification of the SWI/SNF chromatin remodeling complex gene PBRM14 as a second major ccRCC cancer gene, with truncating mutations in 41% (92/227) of cases. These data further elucidate the somatic genetic architecture of ccRCC and emphasize the marked contribution of aberrant chromatin biology.</p>
        </div>
      </section>
      <section data-origid="S1" data-type="methods">
        <h2 id="s3h1">Methods Summary</h2>
        <div class="content">
          <p id="s3p1">DNA samples from ccRCC patients tumour and matching normal were all obtained under local IRB and LREC approvals for this study and processed as previously described2. DNA fragmentation, library preparation and solution phase hybrid capture were according to manufacturer instructions (Agilent Technologies, US) and modified from previously published protocols5. Capillary-based Sanger sequencing for confirmations and PBRM1 followup were done as previously described2 with manual inspection of all sequencing traces. mRNA was extracted from snap-frozen mouse pancreatic lesions and subjected to RT-PCR using a nested PCR approach utilising primers of mouse Pbrm1 exon 23/24 and the Carp-Î²-Actin Splice acceptor sequence of the T2Onc transposon cassette. Resulting bands were gel-purified and subjected to capillary-based Sanger sequencing. PBRM1 or scrambled control siRNAs (Santa Cruz, CA) were transfected into ccRCC cell lines using Lipofectamine 2000 (Invitrogen, CA) according to the manufacturer's conditions. Real-time PCR and western blotting were all done utilising standard protocols essentially as described1. Expression analyses were carried out as previously described2.</p>
        </div>
      </section>
      <section data-origid="SM" data-type="supplementary-material">
        <h2 id="s4h1">Supplementary Material</h2>
      </section>
    </article>
  </body>
</html>
