<!DOCTYPE html >
<html id="a2YX8KTeVzFWztDbsTxLAaPXQ0WK-PMC4104028" data-origid="PMC4104028" class="anndoc" data-anndoc-version="2.0" lang="" xml:lang="" xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="UTF-8"/>
    <meta name="generator" content="com.jmcejuela.anndoc.v2.bio.NcbiJournalArticleParser_v0_3$"/>
    <meta name="dcterms.source" content="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4104028/"/>
    <title>a2YX8KTeVzFWztDbsTxLAaPXQ0WK-PMC4104028</title>
  </head>
  <body>
    <article>
      <section data-type="title">
        <h2 id="s1h1">WIPI2 Links LC3 Conjugation with PI3P, Autophagosome Formation, and Pathogen Clearance by Recruiting Atg12–5-16L1</h2>
      </section>
      <section data-type="abstract">
        <h2 id="s2h1">Summary</h2>
        <div class="content">
          <p id="s2p1">Mammalian cell homeostasis during starvation depends on initiation of autophagy by endoplasmic reticulum-localized phosphatidylinositol 3-phosphate (PtdIns(3)P) synthesis. Formation of double-membrane autophagosomes that engulf cytosolic components requires the LC3-conjugating Atg12–5-16L1 complex. The molecular mechanisms of Atg12–5-16L1 recruitment and significance of PtdIns(3)P synthesis at autophagosome formation sites are unknown. By identifying interacting partners of WIPIs, WD-repeat PtdIns(3)P effector proteins, we found that Atg16L1 directly binds WIPI2b. Mutation experiments and ectopic localization of WIPI2b to plasma membrane show that WIPI2b is a PtdIns(3)P effector upstream of Atg16L1 and is required for LC3 conjugation and starvation-induced autophagy through recruitment of the Atg12–5-16L1 complex. Atg16L1 mutants, which do not bind WIPI2b but bind FIP200, cannot rescue starvation-induced autophagy in Atg16L1-deficient MEFs. WIPI2b is also required for autophagic clearance of pathogenic bacteria. WIPI2b binds the membrane surrounding Salmonella and recruits the Atg12–5-16L1 complex, initiating LC3 conjugation, autophagosomal membrane formation, and engulfment of Salmonella.</p>
        </div>
      </section>
      <section data-type="abstract-graphical">
        <h2 id="s3h1">Graphical Abstract</h2>
        <div class="content">
          <figure data-origid="undfig1">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4104028/#undfig1">Figure</a>
            <figcaption></figcaption>
          </figure>
        </div>
      </section>
      <section data-type="abstract-author-highlights">
        <h2 id="s4h1">Highlights</h2>
        <div class="content">
          <p id="s4p1">• WIPI2 binds Atg16L1 directly and recruits the Atg12–5-16L1 complex • WIPI2 binding to Atg16L1 is required for LC3 lipidation and autophagosome formation • WIPI2-Atg16L1 function requires PI3P binding by WIPI2 and is independent of FIP200 • Autophagosomal engulfment of Salmonella requires the WIPI2-Atg16L1 complex</p>
        </div>
      </section>
      <section data-type="abstract-teaser">
        <h2 id="s5h1">Abstract (teaser)</h2>
        <div class="content">
          <p id="s5p1">Starvation-induced autophagy requires PtdIns(3)P locally produced on ER-derived membranes. Dooley et al. demonstrate that the PtdIns(3)P effector WIPI2b binds Atg16L1 to recruit Atg12–5-16L1 to PtdIns(3)P-positive omegasomes, resulting in LC3 lipidation and starvation-induced autophagy. These findings suggest that WIPI2b senses increases in PtdIns(3)P and directs LC3 conjugation to developing autophagosomes.</p>
        </div>
      </section>
      <section data-origid="sec1" data-type="intro">
        <h2 id="s6h1">Introduction</h2>
        <div class="content">
          <p id="s6p1">Autophagy is a conserved degradation pathway, present in all eukaryotes, required for cell survival during starvation and cell homeostasis. In higher eukaryotes, autophagy is also required for development, immunity, and combating infection (Choi et al., 2013). During macroautophagy, here called autophagy, double-membrane phagophores (isolation membranes) expand and sequester cytosolic components, forming autophagosomes that fuse with endosomes and lysosomes. Phagophores arise from endoplasmic reticulum (ER)-derived omegasomes (Axe et al., 2008), and during autophagosome expansion, membrane from other subcellular compartments (Golgi, mitochondria, and plasma membrane) is probably incorporated (Lamb et al., 2013).</p>
          <p id="s6p2">The formation of autophagosomes requires autophagy-related (Atg) proteins, first identified in yeast (Mizushima et al., 2011), which are recruited in a sequential manner (Itakura and Mizushima, 2010). The first Atg protein complex to respond to upstream signals, in particular inactivation of mammalian target of rapamycin complex 1 (mTORC1), is the Unc-51-like kinase (ULK) complex. Activation of ULK1/ULK2 and subsequent phosphorylation of ULK complex members Atg13 and FIP200 initiates autophagy. The class III Vps34 lipid kinase, part of the Beclin1 complex, is then activated (Wirth et al., 2013), translocates to ER sites, and produces a pool of phosphatidylinositol 3-phosphate (PtdIns(3)P). Live-cell imaging has shown that the ULK complex translocates first to the ER formation site, but PtdIns(3)P produced by the Beclin1 complex is required for stabilization of the ULK complex at the omegasome (Karanasios et al., 2013, Koyama-Honda et al., 2013).</p>
          <p id="s6p3">PtdIns(3)P at the omegasome facilitates recruitment of PtdIns(3)P effectors such as DCFP1 and the WIPI (WD-repeat PtdIns(3)P effector protein) family of proteins (Karanasios et al., 2013, Koyama-Honda et al., 2013, Polson et al., 2010), the function of which is unknown. The final Atg proteins recruited prior to LC3 lipidation are the Atg12–5-16L1 complex and the Atg3-LC3 conjugate. In the Atg12–5-16L1 complex, Atg16L1 binds to Atg5, which is conjugated to Atg12. This complex acts as an E3-like enzyme to recruit the E2-like protein Atg3, conjugated to LC3-I, to the omegasome. Atg3 binds to and is activated by Atg12 (Sakoh-Nakatogawa et al., 2013), facilitating conjugation of the LC3 to phosphatidylethanolamine (PE), thus converting LC3-I to LC3-II. Therefore, the site of Atg12–5-16L1 complex recruitment determines the site of LC3-II formation (Fujita et al., 2008). The LC3 family (Atg8 in yeast) is required for phagophore expansion, closure, and cargo recruitment (Slobodkin and Elazar, 2013). The mechanism by which the Atg12–5-16L1 complex is recruited to membranes is unknown. While Atg16L1 binds FIP200 (Gammoh et al., 2013, Nishimura et al., 2013, Fujita et al., 2013), recruitment of Atg12–5-16L1 depends on PtdIns(3)P formation (Itakura and Mizushima, 2010). However, no PtdIns(3)P binding protein has been implicated in recruiting the Atg12–5-16L1 complex, and the molecular mechanism underlying the requirement for PtdIns(3)P production during phagophore formation in mammalian autophagy remains unclear.</p>
          <p id="s6p4">We have addressed the function of the PtdIns(3)P effector WIPI proteins by identifying WIPI1 and WIPI2 interacting proteins. Our comparative analysis revealed that Atg16L1 directly interacts with WIPI2b, and we identified the residues required for interaction between WIPI2b and Atg16L1. We demonstrate that WIPI2b acts immediately upstream of Atg16L1 and is responsible for Atg12–5-16L1 recruitment to PtdIns(3)P-positive omegasomes, resulting in LC3 lipidation and starvation-induced autophagy. We also show that Atg16L1 mutants able to bind FIP200, but not WIPI2b, cannot rescue starvation-induced autophagy in Atg16L1-deficient mouse embryonic fibroblasts (MEFs). Finally, we show that WIPI2b function is also required for the innate immune response to Salmonella Typhimurium infection. WIPI2b binds the phagophore membrane surrounding Salmonella, recruits Atg16L1, initiating LC3 lipid conjugation, autophagosomal membrane formation, and engulfment of Salmonella, and restricts bacterial proliferation.</p>
        </div>
      </section>
      <section data-origid="sec2" data-type="results">
        <h2 id="s7h1">Results</h2>
        <div class="subsections">
          <section data-origid="sec2.1" data-type="">
            <h3 id="s7s1h1">WIPI2b Binds Atg16L1</h3>
            <div class="content">
              <p id="s7s1p1">We have shown that WIPI2 is required for starvation-induced autophagy (Polson et al., 2010). WIPI1, a closely related family member, also regulates autophagy (Proikas-Cezanne et al., 2004, Polson et al., 2010). In cells expressing WIPI1 and WIPI2, loss of WIPI1 increases autophagy, suggesting that WIPI1 may have an inhibitory role (Polson et al., 2010). WIPI2 has five isoforms (a, b, c, d, and e) that are differentially recruited to autophagosomes upon amino acid starvation (Mauthe et al., 2011). WIPI2a and WIPI2c are not recruited to the membrane upon starvation, and WIPI2a does not colocalize with DFCP1 (see below). To identify proteins that interact with WIPI2 and/or WIPI1, we used human embryonic kidney 293 (HEK293) cell lines stably expressing GFP, GFP-WIPI1a, and GFP-WIPI2b, which responded to amino acid starvation and formed wortmannin-sensitive (PtdIns(3)P-dependent) WIPI puncta, for a mass spectroscopy screen for interactors (Figures S1A–S1D, available online).</p>
              <p id="s7s1p2">A distinct set of proteins was identified in the GFP-WIPI1a and GFP-WIPI2b pull-downs (Figure S1E). The most abundant interacting proteins for GFP-WIPI1a were coatomer subunits, validated by immunoblot (Figure 1A). GFP-WIPI2b bound Atg5 and Atg16L1, and the interaction between WIPI2b, but not WIPI1a, and the Atg16L1 complex was validated by immunoblot (Figure 1A). The interaction of endogenous WIPI2 with Atg16L1 (Figure 1B) was confirmed using a crosslinker to stabilize the interaction. This low-affinity or transient interaction between WIPI2 and Atg16L1 is reminiscent of that between Atg16L1 and FIP200, detected after crosslinking (Nishimura et al., 2013) or in Atg3−/− MEFS (Gammoh et al., 2013).</p>
              <p id="s7s1p3">We chose to focus on the interaction of the Atg16L1 complex with WIPI2b, as the PtdIns(3)P-dependent recruitment of Atg12–5-16L1 to the omegasome and phagophore could be the function of WIPI2 in autophagy. In support of our hypothesis that WIPI2 acts upstream of Atg16L1, WIPI2 puncta are seen in Atg16L1Δ/Δ MEFs in both fed and starved conditions (Figures 1C and 1D). These puncta are phagophores and open autophagosomes (Figures 1E–1H).</p>
              <p id="s7s1p4">Endogenous WIPI2 colocalized with Atg16L1 (Figure 1I) and Atg12 (Figure 1J) on both GFP-LC3-negative and -positive autophagosomes. Note that myc-WIPI2b colocalization with DFCP1 on omegasomes is starvation and PtdIns(3)P-binding dependent, while WIPI2a is not detected on omegasomes (Figure S1E) (Mauthe et al., 2011). To probe the relationship between WIPI1a and WIPI2b, we tested if GFP-WIPI1a and GFP-WIPI2b were present on Atg16L1-positive autophagosomes. In cells expressing low levels of GFP-WIPI1a, where no displacement of endogenous WIPI2 occurred (Polson et al., 2010), the GFP-WIPI1a-positive puncta did not contain Atg16L1 in contrast to GFP-WIPI2b (Figures S1E and S1H). GFP-stable cell lines have GFP-negative, WIPI2-Atg16-positive autophagosomes (Figure S1I). Under starvation, live-cell imaging of GFP-WIPI2b and mCherry-Atg16L1 revealed that Atg16L1 translocated to GFP-WIPI2b-positive puncta (Figure 1K and Movie S1), supporting our proposed sequential recruitment of WIPI2b to an omegasome-derived pool of PtdIns(3)P followed by the Atg16L1 complex.</p>
            </div>
          </section>
          <section data-origid="sec2.2" data-type="">
            <h3 id="s7s2h1">A Region in Atg16L1 Not Conserved in Atg16L2 Binds WIPI2b</h3>
            <div class="content">
              <p id="s7s2p1">Atg16L1 is the protein through which the Atg12–5-16L1 complex is recruited to the forming autophagosome (Fujita et al., 2008), and we propose that by binding Atg16L1, WIPI2b is recruiting the Atg12–5-16L1 complex. To map the binding site, we used deletion constructs of Atg16L1 (Figure 2A). Previous work has identified the domains of Atg16L1, which bind Atg5 (Mizushima et al., 2003) and Rab33b (Itoh et al., 2008) and mediate dimerization (Itoh et al., 2008) and LC3-spot formation (Ishibashi et al., 2011). Deletion of the N-terminal Atg5 binding domain (1–78 aa) or C-terminal WD domain (266–623 aa) had no effect on the Atg16L1-WIPI2b interaction in GFP pull-down experiments (Figure 2B). We concluded that WIPI2b was interacting with Atg16L1 between 79 and 265. A similar result was obtained using human Atg16L1 constructs (Figures S2A and S2B). We refined the interaction site using truncation mutants of FLAG-Atg16L1 constructs, with GFP and CFP-Atg5 as controls, and GFP-WIPI1a or GFP-WIPI2b (Figures 2C and S2C). GFP-WIPI2b bound most strongly to Atg16L1 1–230, while GFP-WIPI1a showed an increased binding to Atg16L1 1–230; neither bound to 1–207. These data suggest that the WIPI2b-binding site is located in Atg16L1 between 207 and 230.</p>
              <p id="s7s2p2">Atg16L2, an isoform of Atg16L1, cannot function in autophagy, possibly due to its inability to localize to phagophores (Ishibashi et al., 2011). A region in Atg16L1 (229–242) was identified that is poorly conserved in Atg16L2 and may be important in membrane localization (Ishibashi et al., 2011). As this region overlaps with the WIPI2 binding region, we tested if WIPI2b binds Atg16L2. As shown in Figure 2D, GFP-WIPI2b does not bind to Atg16L2.</p>
            </div>
          </section>
          <section data-origid="sec2.3" data-type="">
            <h3 id="s7s3h1">WIPI2 and FIP200 Bind to Independent Sites on Atg16L1</h3>
            <div class="content">
              <p id="s7s3p1">To identify residues required for WIPI2b binding, we aligned residues 207–242 of Atg16L1 and Atg16L2 (Figure 3A). The main difference between the two proteins in this region is the presence of acidic residues in Atg16L1 that are not conserved in Atg16L2. To test if these acidic residues in Atg16L1 mediated binding to WIPI2b, we separately mutated all nine residues to arginine (to mimic the charge in Atg16L2) and tested the binding of these FLAG-tagged charge-change mutants to GFP-WIPI2b (Figures 3B and 3C). Mutation of either residue E226 or E230 to arginine (E226R and E230R) abolished binding of Atg16L1 to WIPI2b, and E208R or D212R mutants reproducibly showed reduced binding.</p>
              <p id="s7s3p2">FIP200 binds Atg16L1 in the region between 230 and 242 (Gammoh et al., 2013, Nishimura et al., 2013). This region encompasses part of the WIP2b binding site, so we asked if the WIPI2 and FIP200 binding sites overlapped by testing if our FLAG-Atg16L1 constructs could bind FIP200 (Figures 3D and 3E). Endogenous FIP200 bound to wild-type Atg16L1, but not to Atg16L2. FIP200 also bound to the point mutants that reduce or abolish WIPI2 binding (E208R, D212R, E226R, and E230R). However, the Atg16L1 point mutations from E235 to E241 bound very weakly or not at all to FIP200. These results show distinct, but adjacent, binding sites for WIPI2b and FIP200 within 207–242 of Atg16L1. Recent data suggest that the FIP200 binding site on Atg16L1 encompasses residue 246 (Fujita et al., 2013), so we extended residue mapping to E249 and saw a recovery of FIP200 binding to wild-type (WT) Atg16L1 (Figure S3A).</p>
              <p id="s7s3p3">We next asked if the binding of WIPI2 to Atg16L1 in MEFs lacking FIP200 was altered compared to wild-type MEFs. Endogenous WIPI2b and Atg16L1 coimmunoprecipitated in both WT and FIP200−/− MEF cells, with no significant difference, demonstrating that FIP200 is not required for the WIPI2b-Atg16L1 interaction (Figure 3F). In support of this finding, in approximately 10% of FIP200−/− MEFs, WIPI2 can be found on LC3-positive tubular structures (Figure S3B). As expected, WIPI2 puncta are observed in autophagy-deficient MEFs including Atg5 and Atg3, which are LC3 negative (Figure S3B). Additionally, double charge-change mutants of Atg16L1 that cannot bind WIPI2b have unaltered binding to FIP200, and vice versa (Figure 3G). A model of Atg16L1 207–246 (Figure 3H) shows that the WIPI2b and FIP200 binding sites on Atg16L1 lie on either side of a proline-induced loop, suggesting that one surface of Atg16L1 would bind to WIPI2, while a distinct surface opposite would bind FIP200. Note that a trimeric complex between GFP-WIPI2b, FLAG-Atg16L1, and FIP200 can exist as shown in Figure 4G.</p>
            </div>
          </section>
          <section data-origid="sec2.4" data-type="">
            <h3 id="s7s4h1">Mutational Analysis of WIPI2b Reveals the Binding Mechanism for Atg16L1</h3>
            <div class="content">
              <p id="s7s4p1">WIPI2b is a member of the WD-repeat SVP1-like family of seven-bladed β-propeller proteins that bind phosphatidylinositols (PROPPINs) (Dove et al., 2004). All known members have two PtdIns(3)P binding sites formed around a conserved FRRG motif and a hydrophobic loop that is predicted to insert into the membrane (Baskaran et al., 2012, Krick et al., 2012, Watanabe et al., 2012) (Figures S4A and S4B). Recruitment of the PtdIns(3)P effector Atg18 and its homologs (Krick et al., 2008, Obara et al., 2008, Nair et al., 2010, Polson et al., 2010) to PtdIns(3)P occurs via a conserved FRRG motif. We began mapping the Atg16L1 interaction site on WIPI2b using GFP-tagged WIPI proteins and blotting for coimmunoprecipitated Atg16L1. Inhibition of PtdIns(3)P binding through mutation of the PtdIns(3)P binding motif FRRG to FTTG (GFP-WIPI2b FTTG) had no significant effect on the ability of WIPI2b to bind Atg16L1 (Figures 4A and 4B). Atg16L1 bound to GFP-WIPI2b, but not detectably to GFP-WIPI2a (Figures 4A and 4E) or GFP-WIPI4 (Figure S4C). WIPI2a is identical to WIPI2b, except for an 18 amino acid insertion in between β1 and β2 of blade 1 of the β-propeller (Figures 4C, S4A, and S4D). Using our data and a model of WIPI2 based on the structure of a yeast ortholog, Hsv2 (Baskaran et al., 2012, Krick et al., 2012, Watanabe et al., 2012) (Figure 4C), we predicted possible sites on WIPI2 that might bind Atg16L1. As the site on Atg16L1 is acidic, we looked for conserved basically charged residues proximal to the site of the 18 amino acid insert in the WIPI2a isoform (which does not bind Atg16L1). We identified two solvent-exposed arginine residues, R108 and R125, in the cleft between blades 2 and 3 of WIPI2b (Figure 4D). Mutation of R108 to glutamate (WIPI2b R108E) reduced Atg16L1 binding dramatically, while R125E showed decreased binding (Figure 4D).</p>
              <p id="s7s4p2">These arginine residues (R108 and R125) are well conserved in WIPI1a, but we detected only a weak binding of WIPI1a to Atg16L1 (Figures 2C and 2D). As the C-terminal domains of the PROPPINs are less well conserved (Baskaran et al., 2012, Krick et al., 2012), we tested the role of the C terminus in Atg16L1 binding. Deletion of the C terminus of WIPI2b significantly increased binding to Atg16L1 (Figures 4E and 4F), and this was mirrored by a small, but significant, increase in WIPI1a ΔCT binding to Atg16L1. These results suggest that the conserved Atg16L1-binding site is potentially masked by the insertion in WIPI2a and that the C-terminal domains may modulate binding.</p>
              <p id="s7s4p3">To test if the WIPI2b-Atg16L1 interaction is direct, we used charge-change binding experiments with the WIPI2b and Atg16L1 mutants. We mixed cell lysates expressing WT, single, or double point mutants of GFP-WIPI2b and FLAG-Atg16L1 in all possible permutations (Figures 4G and S5A). The WIPI2b-Atg16L1 interaction is significantly restored when the GFP-WIPI2b R108E mutant is mixed with FLAG-Atg16L1 E230R, compared to FLAG-Atg16L1 binding to GFP-WIPI2b R108E (Figure 4H). This implies that residues R108 in WIPI2b and E230 in Atg16L1 interact directly. A proposed model of the predicted binding site on WIPI2b with Atg16L1 (207–246) interaction is shown in Figure S5B.</p>
            </div>
          </section>
          <section data-origid="sec2.5" data-type="">
            <h3 id="s7s5h1">WIPI2b Binding to Atg16L1 Is Required for LC3 Localization and Autophagy</h3>
            <div class="content">
              <p id="s7s5p1">We used the double charge-change mutant GFP-WIPI2b R108E R125E (RERE) that completely abolishes Atg16L1 binding to probe the function of the WIPI2b-Atg16L1 interaction in starvation-induced autophagy. In WIPI2-depleted cells, WT GFP-WIPI2b increased starvation-induced LC3-II levels; however, expression of GFP-WIPI2b RERE inhibited autophagy (Figures 5A and 5B). This inhibition in WIPI2-depleted cells may be due to GFP-WIPI2b RERE outcompeting any residual endogenous WIPI2b for PtdIns(3)P binding and therefore stalling autophagosome formation by blocking Atg12–5-16L1 recruitment. Consistently, the inhibition of LC3 lipidation by GFP-WIPI2b RERE was dependent on the PtdIns(3)P binding ability of WIPI2b: expression of GFP-WIPI2b RERE FTTG (which is unable to bind PtdIns(3)P) did not inhibit LC3 lipidation in WIPI2-depleted cells (Figures 5C and 5D). Levels of SQSTM-1/p62, an autophagy cargo receptor, are used to monitor autophagic flux (Klionsky et al., 2012). p62 accumulated after expression of GFP-WIPI2b RERE in WIPI2-depleted cells, and this was abolished in the WIPI2b RERE FTTG mutant (Figures 5C and 5D). GFP-WIPI2b FTTG had no inhibitory effect on LC3 lipidation or p62 levels.</p>
              <p id="s7s5p2">Immunofluorescence analysis of WIPI2-depleted HEK cells shows that WT WIPI2b and WIPI2b RERE form puncta upon autophagy initiation (Figures 5E and 5F). GFP-WIPI2b RERE puncta are open autophagosomes (Figures 4I and 4J). The PtdIns(3)P-binding deficient WIPI2b FTTG and WIPI2b RERE FTTG do not form puncta (Figures 5E and 5F). Consistent with LC3 lipidation data, WT WIPI2b puncta are LC3 positive, while the WIPI2b RERE puncta are LC3 negative, and the numbers of LC3-postive autophagosomes are reduced by expression of WIPI2b RERE, but not after expression of WIPI2b RERE FTTG (Figures 5E and 5G). These results show that WIPI2b binding to both Atg16L1 and PtdIns(3)P is necessary for LC3 lipidation and autophagosome formation.</p>
            </div>
          </section>
          <section data-origid="sec2.6" data-type="">
            <h3 id="s7s6h1">Ectopic Localization of WIPI2b to the Plasma Membrane Is Sufficient to Drive LC3 Lipidation through Recruitment of the Atg16L1 Complex</h3>
            <div class="content">
              <p id="s7s6p1">Our data suggest that WIPI2b functions in autophagy by recruiting the Atg12–5-16L1 complex to PtdIns(3)P-positive membranes by binding Atg16L1. To confirm this, we tested if ectopic localization of WIPI2b to the plasma membrane was sufficient to drive LC3 lipidation. We used mCherry-WIPI2b-CAAX constructs based on the KRas CAAX sequence (Ahearn et al., 2012) using WIPI2b-FTTG constructs. Ectopically localized Atg16L1 can recruit LC3 to the plasma membrane (Fujita et al., 2008). Expression of mCherry-WIPI2b-CAAX significantly increased LC3-II in fed and wortmannin-treated starved cells (Figures 6A and 6B), and this increase was lost in mCherry-WIPI2b-RERE-CAAX mutants unable to bind Atg16L1 (Figures 6A–6C). Both mCherry-WIPI2b-CAAX and mCherry-WIPI2b-RERE-CAAX were plasma membrane localized, but GFP-LC3 was only recruited by mCherry-WIPI2b-CAAX, and this occurred in fed, starved, or wortmannin-treated 2GL9 cells (Figure 6D) or MCF-7 cells (Figure S6). We conclude that WIPI2b membrane localization is sufficient to recruit the Atg12–5-16L1 complex and subsequently drive LC3 lipidation independently of upstream signals, including PI3-kinases and mTORC1 inactivation.</p>
              <p id="s7s6p2">As WIPI2b-CAAX can drive LC3 lipidation and recruitment to the plasma membrane, we asked if double-membrane phagophores could be detected on or emerging from the plasma membrane. In the regions on the plasma membrane that contain mCherry-WIPI2b-CAAX and GFP-LC3, no typical double membranes were detected in the cells examined, but in all doubly transfected cells, we saw a cluster of small vesicular structures under the plasma membrane (Figures 6E and 6F). The nature of these vesicles requires characterization.</p>
              <p id="s7s6p3">To determine whether WIPI2b-CAAX-driven lipidation requires FIP200, we expressed hemagglutinin (HA)-WIPI2b-CAAX (FTTG) in RISC-free control, ATG16L1, or FIP200 siRNA-treated cells. CAAX-driven LC3 lipidation was lost in Atg16L1-depleted cells, but not in FIP200-depleted cells (Figure 6G), demonstrating that LC3 recruitment to the plasma membrane via WIPI2b-CAAX requires Atg16L1, but not FIP200.</p>
            </div>
          </section>
          <section data-origid="sec2.7" data-type="">
            <h3 id="s7s7h1">WIPI2b, but Not FIP200, Binding Is Required for Atg16L1 to Rescue Autophagy in Atg16L1Δ/Δ MEFs</h3>
            <div class="content">
              <p id="s7s7p1">Atg16L1Δ/Δ MEFs do not support starvation-induced LC3 lipidation or autophagy, as they contain Atg16L1 with a deletion (Δ69–213) (Saitoh et al., 2008). To test the requirement for WIPI2b and FIP200 binding to Atg16L1 for autophagy, we expressed Atg16L1 in Atg16L1Δ/Δ MEFs. We used WT or Atg16L1 mutants of either the WIPI2b-binding site (E226R E239R/ERER) or the FIP200-binding site (D237R D239R/DRDR) to abolish WIPI2b or FIP200 binding, respectively (see Figure 3G). The number of LC3-positive autophagosomes formed per cell was significantly less with FLAG-Atg16L1 ERER mutant, which cannot bind WIPI2b, compared to WT rescue; in addition, LC3 lipidation was not rescued by this mutant (Figures 7A–7D). In contrast, there was no significant difference between the number of autophagosomes with WT FLAG-Atg16L1 and FLAG-Atg16L1 DRDR, and there was a significant restoration of LC3 lipidation (Figures 7A–7D). Clearly, Atg16L1 must bind WIPI2b to function in autophagy. Additionally, Atg16L1 mutants that cannot bind FIP200, but can bind WIPI2, rescue autophagy to an extent similar to that of WT Atg16L1.</p>
            </div>
          </section>
          <section data-origid="sec2.8" data-type="">
            <h3 id="s7s8h1">WIPI2b Is Required for LC3 Recruitment during Salmonella Infection</h3>
            <div class="content">
              <p id="s7s8p1">Intracellular bacterial pathogens, such as Salmonella, are targeted by autophagy as part of a host cell bacterial clearance response (Birmingham et al., 2006). After invasion, Salmonella resides in a specialized organelle, the Salmonella-containing vacuole (SCV) (Gorvel and Méresse, 2001). Phagophore membranes form around the bacteria following Salmonella-induced damage to the SCV membrane (Fujita et al., 2013, Birmingham et al., 2006, Kageyama et al., 2011). As in canonical autophagy, LC3 is conjugated to these phagophore membranes using Atg12–5-16L1 (Kageyama et al., 2011); however, recruitment of the Atg12–5-16L1 complex involves a number of redundant mechanisms (Fujita et al., 2013). Both FIP200 and ubiquitin binding by Atg16L1 are important for LC3 recruitment (Fujita et al., 2013). After infection with Salmonella, WIPI2 on p62-positive SCVs colocalized with LC3 (Figure 7E), LAMP1, and ubiquitin (Figures S7A and S7B). These WIPI2-positive Salmonella were detected on ER structures (Figure S7C) as expected (Huang et al., 2011). We asked if WIPI2 interaction with ATG16L1 was also required for the recruitment of the LC3 to the phagophore during bacterial infection. WIPI2 depletion significantly reduced LC3 on the SCV (Figures 7E and 7F). Autophagy restricts bacterial proliferation; we therefore assayed for replication in HeLa cells treated with either control or WIPI2 siRNA (Figure S7D). Colony formation was significantly increased in cells lacking WIPI2. To differentiate between the FIP200 and WIPI2 binding requirements for Atg12–5-16L1 recruitment to the phagophore during Salmonella infection, we transfected Atg16L1Δ/Δ MEFs with WT FLAG-Atg16L1, ERER (WIPI2-binding mutant), or DRDR (FIP200-binding mutant) (Figure 7G). Significantly less LC3 was recruited to p62-positive Salmonella in cells expressing the WIPI2-binding mutant compared to WT Atg16L1 (Figure 7H). There was no significant difference between rescue with the FIP200-binding mutant and WT Atg16L1. We conclude that WIPI2 is required for Atg12–5-16L1 complex recruitment and subsequent LC3 lipidation during autophagic targeting of Salmonella.</p>
            </div>
          </section>
        </div>
      </section>
      <section data-origid="sec3" data-type="discussion">
        <h2 id="s8h1">Discussion</h2>
        <div class="content">
          <p id="s8p1">Here, we show that the PtdIns(3)P binding protein WIPI2b directly binds Atg16L1 and recruits the Atg12–5-16L1 complex during starvation-induced autophagy and during clearance of pathogenic intracellular bacteria. The Atg16L1 complex acts as an E3-like enzyme for LC3 lipidation, an essential step in autophagosome formation, and the mechanism of its recruitment to membranes has been an unresolved question.</p>
          <p id="s8p2">We propose that WIPI proteins recruit upstream effectors to the site of phagophore formation, the omegasome, in a PtdIns(3)P-dependent manner. There are four members of the WIPI family of proteins, WIPI1–WIPI4. These proteins are PROPPINs, β-propeller-containing proteins that bind PtdIns (Michell et al., 2006). WIPI homologs, including the Atg18 proteins in S. cerevisiae (Barth et al., 2001, Guan et al., 2001), S. pombe (Sun et al., 2013), and the C. elegans WIPI4 homolog Epg-6, which is downstream of the C. elegans Atg18 (WIPI1/2 homolog) (Lu et al., 2011), have the ability to bind PtdIns(3)P but have different interactors. S. cerevisiae Atg18 (Rieter et al., 2013) and Epg-6 (Lu et al., 2011) interact with Atg2, as does WIPI4 (Velikkakath et al., 2012, Lu et al., 2011). Yeast Atg16 proteins do not have the C terminus containing the WIPI2b binding sites. However, S. pombe Atg18 a binds Atg5, and this binding is required for the targeting of the Atg12–5-16 complex to the PAS (Sun et al., 2013), implying that the overall function of Atg18/WIPI2 in Atg16 complex recruitment may be conserved between S. pombe and human.</p>
          <p id="s8p3">In mammals, WIPI2 and WIPI4 have different functions (Lu et al., 2011, Polson et al., 2010, Velikkakath et al., 2012). We show that WIPI2b binds and recruits Atg12–5-16L1, and we speculate, based on the data from C. elegans, that WIPI4 may act after this step and recruit Atg2 similarly to S. cerevisiae Atg18 (Rieter et al., 2013), but not Atg12–5-16. Atg2 has been shown to bind WIPI4 and is essential for autophagy and lipid droplet formation (Velikkakath et al., 2012).</p>
          <p id="s8p4">Whether the WIPI2-Atg16L1 interaction is regulated is not clear. Although the interaction is starvation independent (see Figure 1A), the formation of WIPI2-Atg16L1-positive puncta requires starvation. Intriguingly, WIPI1a, which contains the two conserved Atg16L1-binding residues, shows a very weak binding to Atg16L1 unless a truncated form of Atg16L1 (1–230) or a C-terminal deletion of WIPI1a is used. This suggests that the binding between the WIPI2a and Atg16L1 may be limited by its C terminus and Atg16L1 residues between 230 and 242.</p>
          <p id="s8p5">Recruitment of LC3 to Salmonella requires the Atg12–5-16L1 complex (Birmingham et al., 2006, Kageyama et al., 2011), and the requirement for PtdIns(3)P-binding proteins in this process is unclear (Huang et al., 2011, Kageyama et al., 2011). However, the targeting of Atg12–5-16L1, and subsequently LC3, to bacteria appears to be complex, with redundancy present in the system (Fujita et al., 2013). Therefore, subtle effects of removing one of the Atg12–5-16L1 recruitment mechanisms may be difficult to observe, as in the recent identification of ubiquitin binding by Atg16L1 and FIP200 (Fujita et al., 2013). Our results show that WIPI2b is also required for targeting of the Salmonella SCV by the autophagic machinery.</p>
          <p id="s8p6">Autophagy is an ancient cellular process found from yeast to humans. For example, the primitive metazoan Hydra undergoes starvation-induced autophagy, and omegasomes are seen in C. elegans. Although the core components in metazoans are evolutionarily conserved, they are distinct from yeast. The WIPI2b and Atg16L1 interaction sites are conserved in starlet sea anemone, and the WIPI2b site is conserved in the ancient marine sponge Amphimedon, indicating that this mechanism of recruitment of the LC3 conjugation machinery via WIPI2b emerged early in metazoan evolution. It is likely that the WIPI2b-Atg16L1 interaction occurs in most metazoan groups. Interestingly, the FIP200-binding site is missing in metazoans predating emergence of zebrafish and may only be present in vertebrates. The Atg16L1-binding site of WIPI2b is conserved in the S. cerevisiae homolog Atg18, although the WIPI2b-binding site in Atg16L1 is not present in yeast Atg16.</p>
          <p id="s8p7">The binding sites for WIPI2b and FIP200 are adjacent on Atg16L1, and we show that WIPI2 and FIP200 can bind Atg16L1 both individually and simultaneously. However, we did not observe interaction between WIPI2 and FIP200 after overexpression of ATG16L1 (data not shown). FIP200, a component of the ULK complex that regulates autophagy activation, was shown to recruit Atg16L1 (Gammoh et al., 2013, Nishimura et al., 2013). However, our mapping experiments show that the mutations used previously will have abolished WIPI2b binding, and therefore the inability of these FIP200 constructs to rescue autophagy may be attributable to the loss of the WIPI2-Atg16L1 interaction. An Atg16L1 mutant that cannot bind WIPI2, but can bind FIP2000, is unable to rescue autophagy, suggesting that WIPI2b rather than FIP200 is responsible for Atg16L1 recruitment. The close proximity of the WIPI2b and FIP200 binding sites on Atg16L1 is intriguing, and there is potential for regulation or cooperation between WIPI2b and FIP200 at this site.</p>
          <p id="s8p8">FIP200 can bind to Atg16L1 Δ78–230 (Gammoh et al., 2013, Nishimura et al., 2013) that lacks the coiled-coil domain required for dimerization, suggesting that the affinity of monomeric Atg16L1 for FIP200 is strong enough to make a stable interaction. However, the human Atg16L1 ΔCCD (70–212) construct, also lacking the coiled-coil domain, did not bind WIPI2 (Figure S3B). This suggests that Atg16L1 monomers have a weak affinity for WIPI2 and that dimerization is required for Atg16L1 to be recruited to membranes by WIPI2b. We modeled the interaction between WIPI2b and Atg16L1 based on the crystal structures of Hsv2 and a model of Atg16L1 (207–265), taking into account the residues required for complex formation (Figure S5B). This model suggests that an Atg16L1 dimer would be perpendicular to the membrane. We propose that WIPI2 binds PtdIns(3)P on the omegasome, and that the recruitment and positioning of the Atg12–5-16L1 complex on the membrane is critical for progression of the phagophore formation at the omegasome. Phagophores that form in close proximity to the ER are sandwiched between two cisternae of ER (Hayashi-Nishino et al., 2009, Ylä -Anttila et al., 2009), but LC3 lipidation is limited to the phagophore and does not extend onto the ER (Karanasios et al., 2013). We postulate that WIPI2 on the ER at the omegasome would position the LC3 lipidation machinery (Atg12–5-16L1 and Atg3-LC3) pointing out from the ER membrane toward the phagophore, allowing for LC3 lipidation on the phagophore membrane only. Finally, given the apparent utility of this mechanism for localizing the LC3 conjugation machinery via the WIPI2-Atg16L1 interaction, the question follows as to how positioning of this complex on a membrane gives rise in a double-membrane phagophore.</p>
        </div>
      </section>
      <section data-origid="sec4" data-type="methods">
        <h2 id="s9h1">Experimental Procedures</h2>
        <div class="subsections">
          <section data-origid="sec4.1" data-type="">
            <h3 id="s9s1h1">Cell Culture and Reagents</h3>
            <div class="content">
              <p id="s9s1p1">Cell lines, transfection protocols, siRNA, DNA constructs, primers, antibodies, and siRNA are detailed in the Supplemental Experimental Procedures.</p>
            </div>
          </section>
          <section data-origid="sec4.2" data-type="">
            <h3 id="s9s2h1">Protein Complex Purification and Mass Spectroscopy</h3>
            <div class="content">
              <p id="s9s2p1">Protein complexes were pulled down from either fed cells or cells starved for 1 hr in Earle’s balanced salt solution (EBSS) via a GFP tag using GFP-Trap beads (ChromoTek). Proteins were resolved by SDS-PAGE, fixed, and stained with GelCode, and full lanes were cut into 48 mm × 1 mm slices for tryptic digestion and mass spectrometry analysis. For more details, please see Supplemental Experimental Procedures.</p>
            </div>
          </section>
          <section data-origid="sec4.3" data-type="">
            <h3 id="s9s3h1">Immunoprecipitation</h3>
            <div class="content">
              <p id="s9s3p1">Endogenous immunoprecipitation was performed using DSP (Lomant’s Reagent) (Thermo Scientific) on intact cells at the indicated concentrations for 30 min on ice. Tagged proteins were immunoprecipitated with either GFP-Trap beads or anti-FLAG M2 affinity gel (Sigma). See Supplemental Experimental Procedures for more information.</p>
            </div>
          </section>
          <section data-origid="sec4.4" data-type="">
            <h3 id="s9s4h1">In Vitro Translation</h3>
            <div class="content">
              <p id="s9s4p1">In vitro translated protein was produced using TNT Quick Coupled Transcription/Translation System (Promega) and EasyTag [35S]-Methonine (PerkinElmer).</p>
            </div>
          </section>
          <section data-origid="sec4.5" data-type="">
            <h3 id="s9s5h1">Bacterial Infection</h3>
            <div class="content">
              <p id="s9s5p1">Salmonella Typhimurium SL 1344, a kind gift from David Holden, was grown in Luria broth (LB) overnight. Overnight cultures were diluted 100× in LB and grown to mid-to-late log phase at optical density 600 (OD600) = 2.0–2.5. Mammalian cell cultures in antibiotic-free Dulbecco’s modified Eagle’s medium (DMEM) were infected at a multiplicity of infection (moi) of 100 (HeLa) or 25 (MEF cell lines), centrifuged (2000 × g for 10 min at room temperature), and incubated at 37°C for 20 min. After 20 min, the medium was replaced with fresh medium containing gentamicin (50 μg/ml) and incubated at 37°C for 1 hr. Cells were then washed with PBS and fixed for immunofluorescence. Colony-forming unit assays were performed as described (Tattoli et al., 2012).</p>
            </div>
          </section>
        </div>
      </section>
    </article>
  </body>
</html>
