<!DOCTYPE html >
<html id="a7zefU6irg815RNNu7e__WF8vUHO-PMC165443" data-origid="PMC165443" class="anndoc" data-anndoc-version="2.0" lang="" xml:lang="" xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="UTF-8"/>
    <meta name="generator" content="com.jmcejuela.anndoc.v2.bio.NcbiJournalArticleParser_v0_3$"/>
    <meta name="dcterms.source" content="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC165443/"/>
    <title>a7zefU6irg815RNNu7e__WF8vUHO-PMC165443</title>
  </head>
  <body>
    <article>
      <section data-type="title">
        <h2 id="s1h1">The rat STSL locus: characterization, chromosomal assignment, and genetic variations in sitosterolemic hypertensive rats</h2>
      </section>
      <section data-type="abstract">
        <h2 id="s2h1">Abstract</h2>
        <div class="subsections">
          <section data-type="">
            <h3 id="s2s1h1">Background</h3>
            <div class="content">
              <p id="s2s1p1">Elevated plant sterol accumulation has been reported in the spontaneously hypertensive rat (SHR), the stroke-prone spontaneously hypertensive rat (SHRSP) and the Wistar-Kyoto (WKY) rat. Additionally, a blood pressure quantitative trait locus (QTL) has been mapped to rat chromosome 6 in a New Zealand genetically hypertensive rat strain (GH rat). ABCG5 and ABCG8 (encoding sterolin-1 and sterolin-2 respectively) have been shown to be responsible for causing sitosterolemia in humans. These genes are organized in a head-to-head configuration at the STSL locus on human chromosome 2p21.</p>
            </div>
          </section>
          <section data-type="methods">
            <h3 id="s2s2h1">Methods</h3>
            <div class="content">
              <p id="s2s2p1">To investigate whether mutations in Abcg5 or Abcg8 exist in SHR, SHRSP, WKY and GH rats, we initiated a systematic search for the genetic variation in coding and non-coding region of Abcg5 and Abcg8 genes in these strains. We isolated the rat cDNAs for these genes and characterized the genomic structure and tissue expression patterns, using standard molecular biology techniques and FISH for chromosomal assignments.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s2s3h1">Results</h3>
            <div class="content">
              <p id="s2s3p1">Both rat Abcg5 and Abcg8 genes map to chromosome band 6q12. These genes span ~40 kb and contain 13 exons and 12 introns each, in a pattern identical to that of the STSL loci in mouse and man. Both Abcg5 and Abcg8 were expressed only in liver and intestine. Analyses of DNA from SHR, SHRSP, GH, WKY, Wistar, Wistar King A (WKA) and Brown Norway (BN) rat strains revealed a homozygous G to T substitution at nucleotide 1754, resulting in the coding change Gly583Cys in sterolin-1 only in rats that are both sitosterolemic and hypertensive (SHR, SHRSP and WKY).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s2s4h1">Conclusions</h3>
            <div class="content">
              <p id="s2s4p1">The rat STSL locus maps to chromosome 6q12. A non-synonymous mutation in Abcg5, Gly583Cys, results in sitosterolemia in rat strains that are also hypertensive (WKY, SHR and SHRSP). Those rat strains that are hypertensive, but not sitosterolemic (e.g. GH rat) do not have mutations in Abcg5 or Abcg8. This mutation allows for expression and apparent apical targeting of Abcg5 protein in the intestine. These rat strains may therefore allow us to study the pathophysiological mechanisms involved in the human disease of sitosterolemia.</p>
            </div>
          </section>
        </div>
      </section>
      <section data-type="">
        <h2 id="s3h1">Background</h2>
        <div class="content">
          <p id="s3p1">The human disorder of sitosterolemia, also known as phytosterolemia (MIM 210250), is an autosomal recessive disorder characterized by the marked elevation of plasma phytosterols e.g. β-sitosterol, campesterol and stigmasterol [1,2]. Hypercholesterolemia primarily during childhood, accelerated atherosclerosis in some adult patients leading to premature death, as well a hemolytic anaemia, arthralgias and tendon and tuberous xanthoma formations are other clinical features. Studies have shown that the sitosterolemic patients hyperabsorb all sterols (cholesterol and phytosterols) and fail to excrete phytosterols into bile [3-7]. Positional mapping of the STSL locus to human chromosome 2p21 led to the identification of two genes, ATP binding cassette (ABC) half-transporters G5 and G8 that were mutated in affected individuals [8-12]. The protein products of these two genes (sterolin-1 and sterolin-2 respectively) are thought to form heterodimers to constitute a functional transporter, based primarily on the genetic evidence that affected individuals are either completely mutated in sterolin-1 or sterolin-2, but not both [12]. More recently, evidence from in vitro expression of these proteins has been presented that these two proteins co-immunoprecipitate, supporting this conclusion [13]. ABCG5 and ABCG8 appear to be present in all mammalian genomes examined to date and are highly conserved between species, indicating that these genes may have a similar function in other species. In order to identify naturally occurring animal models of sitosterolemia, we previously reported a screen of 20 in-bred mouse strains [14]. This screen showed considerable genetic variation at these loci in the mouse strains, but none of these strains exhibited sitosterolemia. However, a literature search revealed that a few strains of rats, studied primarily for investigation of hypertension and stroke, have been reported to be sitosterolemic as well [15,16]. We hypothesised that these rat strains may be mutated in either Abcg5 or Abcg8. Tissue (and plasma) plant sterol levels in the Wistar Kyoto (WKY) rat, the spontaneously hypertensive rat (SHR) and the stroke-prone spontaneously hypertensive rat (SHRSP) were significantly higher than in Wistar and WKA rats fed either a regular chow diet or diets that have higher plant sterols and these were correlated with increased death, or red cell hemolysis etc. [15-22]. Dietary sterol absorption rates were increased, with a decrease in sterol excretion [16], characteristics similar to that observed in sitosterolemic patients.</p>
          <p id="s3p2">In this study, we report the gene structure organisation, cDNA and tissue expression patterns of rat Abcg5 and Abcg8. Using fluorescence in situ hybridization, the rat STSL locus was assigned to the chromosomal 6q12. Interestingly, a cluster of 2 blood pressure (BP) quantitative trait loci (QTL) in (GH × BN) F2 with LOD scores 4.63 (tail BP) and 6.82 (intra-arterial BP) has also been reported to map to rat chromosome 6 between D6Mit3 and D6Mit12, an area containing the STSL locus. We interrogated this region for genetic variations and found a correlation between sitosterolemic strains and a missense mutation in Abcg5.</p>
        </div>
      </section>
      <section data-type="methods">
        <h2 id="s4h1">Methods</h2>
        <div class="subsections">
          <section data-type="">
            <h3 id="s4s1h1">DNA Samples of rat strains</h3>
            <div class="content">
              <p id="s4s1p1">The DNA samples from the following rat strains were used: Sprague Dawley (SD, Charles River Laboratories, Wilmington, MA), Wistar (Sea: Wistar, Outbred, SPF, Seac Yoshitomi) and WKA (WKA/Sea, inbred, SPF, Seac Yoshitomi), WKY (WKY/NCrj, inbred, SPF, Charles River Japan, Kanagawa, Japan or Charles River, Wilmington, MA), SHR/Crl (Charles River, Wilmington, MA), SHRSP (SHRSP/Sea, inbred, SPF, Seac Yoshitomi, Fukuoka, Japan), GH and BN. The genomic DNAs from GH and BN rats were a kind gift from Dr. E. L Harris (New Zealand). The liver tissue genomic DNAs from SD, WKY and SHR rats were extracted by digestion overnight at 55°C in lysis buffer (10 mM Tris pH8.0, 100 mM NaCl, 50 mM EDTA, 0.5% SDS and 0.2 mg/ml proteinase K) followed by phenol-chloroform extraction and ethanol precipitation.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s2h1">Genomic analysis and determination of exon-intron boundaries</h3>
            <div class="content">
              <p id="s4s2p1">Rat cDNAs for Abcg5 and Abcg8 were identified previously by amplification of these sequences from a Sprague Dawley rat enterocyte cDNA library, using primers located in areas conserved between mouse and human (Genbank AF312714 and AF351785 respectively). For Abcg5, the first PCR fragment was obtained by PCR using primers, ABCG5 F4, 5'-ATCCTCAAAGATGTCTCCTTGTAC-3', and ABCG5 R3, 5'-TCCTGACTCTCCTGGTCGCT-3'. For Abcg8 cDNA sequences, the primers, F11 5'-AGATGCTGGCCATCATAGGG-3' and R13 5'-GATGGAGAAGGTGAAGTTGCC-3', were used to amplify rat Abcg8 cDNA fragments. These products were sequenced and used to design primers specific for rat cDNA and a rat intestinal cDNA library was screened by anchor PCR.</p>
              <p id="s4s2p2">Direct sequencing of the PCR products led to the identification of rat cDNA sequence and allowed for design of rat specific primers (Table 1, see Additional file 1). Based upon the structure of mouse genes, putative rat-specific exonic primers were designed and used to amplify rat genomic regions as previously described. All PCR products were directly sequenced and both strands were sequenced as a routine. The complete rat genomic sequences have been deposited with GenBank (AY145899).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s3h1">Chromosomal localisation by fluorescence in situ hybridization (FISH)</h3>
            <div class="content">
              <p id="s4s3p1">For FISH analysis, metaphase chromosomes were prepared from rat embryo fibroblasts. The cells were cultured in DMEM medium at 37°C for 2 days and treated with 0.05 μg/ml colcemid for 45 min. The cells were harvested by mitotic shake-off and resuspended in 0.075 M KCl at room temperature for 10 min. Fixation was carried out as described previously [23].</p>
              <p id="s4s3p2">Fluorescence in situ hybridization (FISH) was performed essentially according to Pinkel et al [24] with minor modifications [25]. The rat sequences used for FISH labelling were amplified genomic DNA fragments (Abcg5 exons 6–10, primers RG5-F6 and RG5-R10, and Abcg8 exons 4–13, primers RG8-F4 and RG8-R13, Table 1, see Additional file 1). The two gene fragments were labelled separately with digoxigenin-11-dUTP (Boehringer Mannheim, Germany) by nick-translation (Life Technologies Inc., Gaithersburg, MD). Appropriate length (50 to 500 bp) of the nick translated fragments was monitored by checking the nick-translation products on a 1 % agarose gel. The two gene fragments were precipitated separately using 800 ng of labelled DNA with 20 μg unlabeled rat Cot1 DNA, which was included in order to suppress probe hybridization to repetitive sequences. The probe DNA mixtures were ethanol precipitated and dissolved in hybridization buffer (50 % formamide, 2 × SSC, 10 % dextran sulphate). After denaturation, the highly repetitive sequences in the probe mixture DNA were allowed to bind to the Cot1 DNA (10 min at 37°C). The probe mixtures were then applied to rat fibroblast metaphase chromosome slides (previously denatured at 73°C for 2 min in 70 % formamide, 2 × SSC). Hybridization was allowed to proceed in a moist chamber for 70 h at 37°C. Subsequently, the slides were washed for 15 min at 45°C in 55 % formamide, 2 × SSC, and then in 2 × SSC (pH 7.0) at 43°C for 9 min. The labelled probe molecules were detected with rhodamine anti-digoxigenin (Oncor, Inc., Gaithersburg, MD) and the slides were washed 3 times in PBS and 0.1 % Nonidet NP-40. Finally, chromosome spreads were counterstained with 4,6-diamino-2-phenylindole (DAPI, 0.5 μg/ml) in an antifade solution (Vectashield®, Vector Laboratories, Inc., Burlingame, CA). The images were captured using the LEICA DM RXA in combination with the Q-FISH software for microphotography.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s4h1">Tissue expression of Abcg5 and Abcg8</h3>
            <div class="content">
              <p id="s4s4p1">A rat multiple tissue Northern blot (Origene Technologies, Rockville, MD) containing immobilised poly A+ mRNA from Sprague Dawley rat was used. Rat sterolin cDNA probes were labelled and hybridization and washing steps were performed as described previously [26]. Total tissue RNAs were extracted from multiple tissues from a Sprague Dawley rat using Trizol reagent as described previously. Random hexamers were used as primers for reverse transcription of 2 μg of rat RNA from the multiple tissues. The primers were annealed to template at 25°C for 10 min, and the reaction were incubated at 53°C for 1 h in the presence of Superscript Reverse transcriptase, 5 mM dNTP mix and RN'ase Out inhibitor. The rat Abcg5-specific primers (RG5c-F1 and RG5Fc-R1), Abcg8-specific primers (RG8c-F1 and RG8c-R1) and β-actin primers (sense: 5'-CTACGTCGCCCTGGACTTCGAGC-3', antisense: 5'-GATGGAGCCGCCGATCCACACGG-3') were used for further PCR amplification (an initial denaturation at 94°C for 3 min, 35 cycles of PCR amplification with each cycle consisting of 30 min at 94°C, 45 s at 60°C, and 2 min at 72°C, followed by an extension step of 72°C for 5 min). The resultant PCR products were gel-purified and sequenced.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s5h1">Real-Time PCR</h3>
            <div class="content">
              <p id="s4s5p1">For quantitation, real-time PCR was employed. Primers used were based upon previously published mouse primer sets [27,28], or designed using MacVector from cDNA sequences between exons, avoiding the amplification of potentially contaminating genomic DNA in the total RNA sample (Table 3). Samples of total RNA (2 μg) from rat livers were reverse transcribed according to the SuperScript' First-Strand Synthesis System (Invitrogen, Carlsbad, CA) using random hexamers with a final volume of 20 μl. Real-time PCR was performed on a PE Biosystems GeneAmp 5700 sequence Detection System (Forest City, CA). Standard reaction volume was 10 μl and contained 1X QuantiTect SYBR Green PCR master mix (Qiagen, Valencia, CA), 0.002 U AmpErase UNG enzyme (PE Biosystems, Forest City, CA), 0.7 μl of cDNA template and 100–500 nM of oligonucleotide primers. Initial steps of RT-PCR were 2 minutes at 50°C for UNG erase activation, followed by a 15 minute hold at 95°C. Cycles (n = 40) consisted of a 15 second melt at 95°C, followed by a 1 minute annealing/extension at 60°C. The final step was a 60°C incubation for 1 minute. All reactions were performed in triplicate. Threshold cycle (CT) analysis of all samples was set at 0.5 relative fluorescence units.</p>
              <figure class="table" data-origid="T1">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC165443/#T1">Table 1</a>
                <figcaption>
                  <p id="s4s5f1p1">Sequence of the rat Abcg5 and rat Abcg8 intron-exon boundaries</p>
                </figcaption>
              </figure>
              <p id="s4s5p2">The relative amounts of Abcg5, Abcg8, Cyp7a1, HMG-CoA reductase, and Srebp-2 message from rat liver samples used in real-time RT-PCR were normalised to GAPDH expression levels to compensate for variations in input RNA amounts. The data were analysed using the comparative threshold cycle method (CT) method. Briefly, the CT values were averaged, from triplicate determinations, defining a Δ-CT value calculated by taking the average CT of the gene of interest and subtracting it from the average CT of GAPDH. The ΔΔ-CT was calculated by subtracting the average Δ-CT(calibrator) values from the Δ-CT(sample). The relative quantification was then calculated by the expression 2(AverageΔΔCT). The mRNA quantity for the calibrator (SD samples) is expressed as 1X sample and all other quantities are expressed as a fold difference relative to the calibrator (SD samples).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s6h1">Total membrane protein preparations</h3>
            <div class="content">
              <p id="s4s6p1">Approximately 500 mg of liver tissue cut into small pieces was homogenised in ice cold lysis buffer (5 mM Tris-pH7.5, 250 mM sucrose with protease inhibitors PMSF, Leupeptin, Aprotinin) by a dounce homogenizer times10 strokes. Samples were then subjected to centrifugation 1000 g for 10 minutes at 4°C. The supernatant was saved. The pellet was resuspended and subjected to dounce homogenisation by10 strokes, centrifuged at 1000 g for 10 minutes at 4°C and this step was repeated once more. The supernatants from all three steps were pooled and subjected to ultracentrifugation at 100,000 g for 40 minutes at 4°C, and the pellet was resuspended in 25 mM Tris-pH7.5, 150 mM NaCl, 0.1% TritonX-100 and 0.1% SDS, by passaging through a 25-gauge needle. Membrane protein concentrations were determined by a colorimetric assay according to manufacturer's protocol (BioRad, Hercules, CA) and 20 μg of total membrane proteins resolved by 7.5% SDS acrylamide gel.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s7h1">Immunoblotting and Immunolocalization</h3>
            <div class="content">
              <p id="s4s7p1">Proteins resolved by SDS-PAGE were transferred to nitrocellulose membranes and the membranes blocked for 2 hours in 5% dry milk in PBS-T (Phosphate Buffered saline and 0.1% Tween 20). Blots were then probed with a primary rabbit anti-peptide (116-VNGCELRRDQFQDCCVSYLLQS-136) antibody against rat sterolin-1 (1:1000 dilution) overnight at 4°C. The blot was washed once for 5 minutes in PBS-T, incubated with goat-anti-rabbit conjugated HRP (1:10000 dilution) for 1 hour, washed for 5 minutes in PBS-T and developed with Western Lightning Chemiluminescence Reagent Plus (PerkinElmer Life Sciences, Inc. Boston, MA).</p>
              <p id="s4s7p2">Snap-frozen rat liver and intestine tissues were sectioned to 8 μm thickness using a cryotome (Bright), air-dried for 30 min and keep at -80°C until used. The slides were stained with hematoxylin. Fixation was performed by incubating samples for 10 min with cooled methanol at -20°C and washed with PBS three times. The slides were treated with blocking solution (10% goat serum in 0.1 M glycine/PBS) for 30 min., primary antibody applied and incubated overnight at 4°C. The slides were washed with PBS and secondary antibody (goat-anti-rabbit FITC conjugated) was applied for 20–30 min at room temperature, washed and the signals captured and analyzed using an Olympus BX-50 microscope with Fluoview software. The fluorescent stained images were observed under blue filter excitation and capture excitation images at 488 nm wavelength.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s8h1">Nucleotide polymorphisms in Abcg5 and Abcg8 genes</h3>
            <div class="content">
              <p id="s4s8p1">Direct sequencing of PCR amplified genomic fragments was performed as described previously [11]. The 5' flanking region, all the exons and the 3'UTR region, as well as the introns of Abcg5 and Abcg8 with size less than 1 Kb, were individually amplified using primers, as described in Table 2. The amplified PCR products were sequenced by the use of an automated capillary sequencer. The Sprague Dawley nucleotide sequence of Abcg5 and Abcg8 was regarded as wild-type. Comparisons to mammalian sterolins (GenBank accession Nos. AF312713, AF312714, AF312715, AF324494 AF324495, AF351785) as well as Fugu (GenBank No. CAAB01001529), was made using ClustalW [29].</p>
              <figure class="table" data-origid="T2">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC165443/#T2">Table 2</a>
                <figcaption>
                  <p id="s4s8f1p1">Oligonucleotide primers used for real time PCR or rat sequences</p>
                </figcaption>
              </figure>
            </div>
          </section>
        </div>
      </section>
      <section data-type="">
        <h2 id="s5h1">Results</h2>
        <div class="subsections">
          <section data-type="">
            <h3 id="s5s1h1">Genomic organisation and chromosome mapping of the rat STSL locus</h3>
            <div class="content">
              <p id="s5s1p1">Rat cDNA sequences for Abcg5 and Abcg8 were isolated and characterized as indicated in Methods. Based upon the known conservation between the murine and human STSL loci [14], we designed oligonucleotides in putative exonic regions and used PCR amplification of rat genomic region to characterise the genome organisation. Figure 1 shows a summary composite of the genomic organisation. The intron-exon boundary sequences are indicated in Table 3. In keeping with the human and mouse STSL loci, the rat Abcg5 and Abcg8 genes are arranged in a head-to-head configuration, with a distance of 379 bp separating their respective initiation codons. The exact start transcriptional sites for either gene has not been definitively established. Comparison of the DNA sequence of isolated cDNAs, as well as the polypeptide sequence with human and mouse sequences revealed a very high degree of homology (&gt;80%). To assign the chromosomal location, we performed FISH analyses, and we were able to unambiguously map the rat STSL locus to chromosome 6q12 (Fig. 2). There is specific labelling of both chromatids of both homologues of rat chromosome (RNO) 6 (Fig. 2A and 2B). Based on the DAPI-banding pattern the Abcg5 and Abcg8 genes were sub-localized to RNO6q12. The STSL locus was mapped to the same position as shown by the diagrammatic representation of the rat Chr. 6 as shown in Fig. 2. The human ABCG5 and ABCG8 genes are present in a group of 18 genes ABCG5, ABCG8, SLC8A1, PRKR, GCKR, KHK, PPP1CB, APOB, POMC1, SDC1, MYCN, DDX1, VSNL1, RRM2, ODC1, ID2, TPO and ACP1 in the region HSA2p21-p25, and this group is also present on RNO6. Thus, the ABCG5 and ABCG8 genes fit into a group of genes that are syntenic both in HSA2 and in RNO6. A conserved segment between HSA2, RNO6, and MMU17 on RNO6q11-q16 has been demonstrated by Zoo-FISH analysis and gene mapping data, confirming this synteny [30-32].</p>
              <figure data-origid="F1">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC165443/#F1">Figure 1</a>
                <figcaption>
                  <p id="s5s1f1p1">Organisation of the rat Abcg5 and Abcg8 genes The top line indicates the restriction endonuclease map of the STSL locus; B, Bam HI; E, EcoRI; S, Sal I, X, XbaI. The second line indicates Abcg5 and Abcg8 gene organisation, with the vertical lines representing exons, numbered as shown. The 379 bp separating 5'ends of Abcg5 and Abcg8 is indicated. The direction of transcription of Abcg5 and Abcg8 is as indicated by the arrows, resulting in mRNAs indicated on the line below. The gray portions indicate the UTRs for Abcg5 and Abcg8. The polypeptides are depicted with the predicted transmembrane spanning regions, as well as the A, B and C Walker motifs, characteristic for the ABC family of protein.</p>
                </figcaption>
              </figure>
              <figure data-origid="F2">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC165443/#F2">Figure 2</a>
                <figcaption>
                  <p id="s5s1f2p1">Chromosomal mapping by FISH A representative photograph of the FISH analyses, shows the localisation of rat Abcg5 (panel A) and Abcg8 (panel B) to both pairs of chromosome 6 in a metaphase spread (arrows). An ideogram of rat chromosome 6 showing the mapped position of both Abcg5 and Abcg8 genes (vertical bar) is shown at the right hand side.</p>
                </figcaption>
              </figure>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s2h1">Expression patterns of rat Abcg5 and Abcg8 mRNAs</h3>
            <div class="content">
              <p id="s5s2p1">Rat Abcg5 and Abcg8 mRNA expression patterns were determined, using a commercial multiple tissue poly A+ Northern filter. Expression appeared mainly to be confined to the liver (Fig. 3A), with dominant transcripts sizes of ~2.5 kb for Abcg5 and of ~4.5 kb for Abcg8. However, the poly A+ RNA loaded onto this Northern blot has been adjusted to obtain an identical β-actin signal, and the intestinal RNA loaded may thus have less enterocyte RNA. Using a more sensitive RT-PCR analyses of RNA obtained from different tissues confirmed that rat Abcg5 and Abcg8 genes were expressed in both in the liver and small intestine, with a low level of expression in the colon (Fig. 3B).</p>
              <figure data-origid="F3">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC165443/#F3">Figure 3</a>
                <figcaption>
                  <p id="s5s2f1p1">Expression analyses of rat Abcg5 and Abcg8 Panel A, Northern blot analyses of poly (A)+ RNA from a variety of rat tissues (see Materials and Methods) showed that expression of Abcg5 (top panel) and Abcg8 (middle panel) was confined mainly to liver. Panel B, RT-PCR analyses confirmed the liver expression pattern, but further showed that both Abcg5 and Abcg8 were also expressed in the small intestine and colon, but not in any other tissues screened. As a control, β-actin expression was used in both Northern and RT-PCR analyses.</p>
                </figcaption>
              </figure>
              <p id="s5s2p2">Additionally, while screening an enterocyte cDNA library for full length cDNAs using PCR, we consistently amplified two DNA products when using Abcg8 primers flanking exon 2; one of the expected size, and another very low copy product ~60 bases larger. Direct sequencing of these products confirmed both of these contained Abcg8 rat cDNA sequences, but the larger product contained a 66 bp insertion, resulting in a predicted in-frame addition of 22 amino acids, between exons 2 and 3. To confirm that this insertion resulted from an alternatively spliced mRNA, we amplified and sequenced the intragenic region between exons 2 and 3 and identified a 66 bp exon (exon 2b), flanked by canonical splice-sites (see Table 2 and Fig. 4). We next screened total liver and intestinal RNA, using RT-PCR, from SD, SHR and WKY rats (Fig. 4). In all samples a larger DNA fragment of low abundance was identified (Fig. 4) and direct sequencing showed that these fragments contained exon 2b sequences (data not shown). It is not established whether these transcripts are a result of aberrant splicing, or are translated and code for a slightly larger sterolin-2. Careful interrogation of the mouse and human genomic sequences did not identify any homologous sequences, suggesting this is unique to the rat. Additionally, both the mouse and human Abcg8 genes have alternatively spliced mRNA, involving a CAG repeat at exon 2 and exon 8 splice acceptor sites respectively [12,14]. However, despite the presence of such a repeat at the rat exon 2 splice acceptor site for Abcg8 (Table 2), none of the rat mRNAs showed any heterogeneity of splice-site selection at this site.</p>
              <figure data-origid="F4">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC165443/#F4">Figure 4</a>
                <figcaption>
                  <p id="s5s2f2p1">Alternative splicing of Abcg8 mRNA expression in SD, SHR and WKY rats by RT-PCR Panel A shows the gene organisation of Abcg8, and below it the organisation of an alternatively spliced form detected by sequencing clones from a rat intestinal cDNA library (see Text). Panel B shows that the effect of the inclusion of exon 2b in the mRNA results in an in-frame addition of 22 amino acids. Total RNAs were prepared from liver and small intestine from SD, SHR and WKY rats maintained on a standard rodent chow diet and analysed by RT-PCR with primers (see Methods) based outside the alternatively spliced exon 2b. Direct amplification of a pooled cDNA library sample indicated a major DNA product of 396 bp, with a minor product of 462 bp, containing exon 2b. DNA products, using cloned wild type Abcg8 cDNA (wt) and alternatively spliced cDNA (AS) are shown in tracks 3 and 4 respectively. As can be seen, liver and intestine RNA from all three strains show a minor DNA product of 462 bp (confirmed by sequencing) contained the insertion of exon2b sequences.</p>
                </figcaption>
              </figure>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s3h1">Hypertensive rats have sitosterolemia and a mis-sense change in Abcg5</h3>
            <div class="content">
              <p id="s5s3p1">A number of different groups have reported that genetically hypertensive strains of rats (SHR, SHRSP, WKY) have elevated levels of plasma sitosterol [15-22]. Although the WKY strain is not considered hypertensive, dietary alterations involving plant sterols appear to increase blood pressure in this strain [21]. We screened two commercially available hypertensive rats, as well as a 'genetically hypertensive' strain, GH, for plasma plant sterol levels and compared these to two control strains, Sprague-Dawley and Wistar (Table 4). Our analyses confirmed that WKY, and SHR are sitosterolemic on a chow diet, but neither the control strains, nor the GH rat showed any significant differences.</p>
              <figure class="table" data-origid="T3">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC165443/#T3">Table 3</a>
                <figcaption>
                  <p id="s5s3f1p1">Sterol compositions in SD, Wistar, GH, WKY and SHR rats a</p>
                </figcaption>
              </figure>
              <p id="s5s3p2">To identify if alterations at the STSL locus could account for these observations, we amplified and directly sequenced Abcg5 and Abcg8 from Wistar, Sprague-Dawley, WKY, WKA, GH, SHR, and SHRSP rats. Table 5 shows a compilation of both coding and non-coding alterations detected. There were 24 polymorphic changes detected in Abcg5 and 14 polymorphic changes in Abcg8. However, only one non-synonymous nucleotide change, G1754T, resulting in a conservative cysteine substitution for glycine at amino acid 583 (Gly583Cys) in sterolin-1 was identified (Fig. 5A). This 'mutation' was only detected in SHR, SHRSP and WKY rats and was present in all of these strains as a homozygous change (Table 5). These hypertensive strains also shared uniquely 4 other nucleotide changes not present in the other strains (Table 5). These latter changes were confined to intronic areas and the 3'UTR region. Although the intronic changes are unlikely to alter the biology of these genes, the 3' UTR insertion in Abcg8 mRNA could be important, if message stability were affected. This possibility was not formally investigated.</p>
              <figure data-origid="F5">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC165443/#F5">Figure 5</a>
                <figcaption>
                  <p id="s5s3f2p1">Demonstration of G1754T (Gly583Cys) mutation in SHR, SHRSP and WKY rats. Panel A shows a representative sequence electropherogram from SD, Wistar, WKA, GH and BN rats (top electropherogram) and from SHR, SHRSP and WKY rats (bottom electropherogram) for exon 12 of Abcg5. A homozygous change (G1754T, Gly583Cys) is present in the sitosterolemic rats and this mutation abolishes the recognition site for restriction enzyme Hae III. Panel B shows the restriction fragment analyses of DNA amplified from these different rat strains, further confirming the presence of the mutational change. In the absence of digestion a 252 bp PCR product is present, which, after Hae III digestion (wt) results in two products of 180 bp and 72 bp (indicated by the arrows). The sitosterolemic strains WKY, SHR and SHRSP (lanes 6–8) are all resistant to digestion by Hae III, but SD and Wistar remain sensitive (lanes 3 and 4). Lanes 1 and 10 shows molecular weight markers, and lanes 2 and 9 show undigested PCR products (252 bp) from SD rat and SHRSP respectively. Panel C shows the alignment of amino acids 565 to 592 of sterolin-1 from human, mouse, rat and Fugu. An arrowhead indicates the position of Gly583Cys mutation.</p>
                </figcaption>
              </figure>
              <figure class="table" data-origid="T4">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC165443/#T4">Table 4</a>
                <figcaption>
                  <p id="s5s3f3p1">Polymorphic changes at the STSL locus identified in different rat strains</p>
                </figcaption>
              </figure>
              <p id="s5s3p3">The identified G1754T (Gly583Cys) missense mutation of the Abcg5 generates loss of a restriction site for Hae III after PCR amplification of exon 12, and this allowed a PCR-RFLP assay to further confirm the detected change (Figure 5B). Note that the DNA chosen from a single Wistar rat was heterozygous for almost all of these changes. The amino acid G583 is conserved between all known mammalian sterolin-1 proteins (Fig. 5C) and its alteration may have functional consequences, as it is also conserved in evolution, being conserved in the Fugu fish (Fig. 5C). Fugu sterolin-1 shows a 55% homology to rat sterolin-1 at the amino acid level, and its gene structure is also organized in a similar fashion (data not shown). Currently, a gap in the Fugu genomic sequence has not been completed that may encode for exon 2. The complete Fugu STSL locus is available on-line (GenBank No. CAAB01001529).</p>
              <p id="s5s3p4">In Abcg8, a large insertional polymorphism was identified in exon 13, that resulted in the insertion of 143 nucleotides in the 3'UTR (*2099_2100ins143nt). WKY, SHR, SHRSP and GH rats are homozygous, Sprague Dawley and WKA strains wild-type and the Wistar sample heterozygous for this polymorphism (Fig. 6).</p>
              <figure data-origid="F6">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC165443/#F6">Figure 6</a>
                <figcaption>
                  <p id="s5s3f4p1">Insertional polymorphisms in rat Abcg8 exon 13. Panel A shows the sizes of PCR products amplified using primers that amplify the 3'-UTR of Abcg8. Two distinct products of 1.11 Kb and 1.252 Kb bands were noted. SD and WKA were homozygous for the 1.11 kb size, while WKY, SHR and SHRSP were homozygous for the 1.25 kb size. Interestingly, the sample from a Wistar rat was heterozygous for this change. Direct sequencing of the larger product showed a 143 bp insertion at position 2099 in the 3' UTR of Abcg8 (panel B).</p>
                </figcaption>
              </figure>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s4h1">Comparison of expression of Abcg5 and Abcg8 in hypertensive rat strains</h3>
            <div class="content">
              <p id="s5s4p1">Expression levels of Abcg5 and Abcg8 in these rat strains were examined in two ways. For mRNA expression, quantitative real time PCR was utilised, and for protein, western blotting analyses, as well as immunohistochemistry was employed. Fig. 7 shows the results of the real time PCR assays of liver RNAs from fasted animals. Using GAPDH mRNA expression as a control, expression of Abcg5 and Abcg8 mRNAs appeared to be increased in liver from SHR compared to Sprague Dawley (Fig. 7), the message for Cyp7 a1 (except for Wistar and WKY) HMG CoA reductase, SR-B1, and SREBP-2 (except for SHR) were relatively unchanged. Interestingly, the message for LDL-R was reduced in both WKY and SHR livers.</p>
              <figure data-origid="F7">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC165443/#F7">Figure 7</a>
                <figcaption>
                  <p id="s5s4f1p1">Real time PCR quantification of selected rat mRNAs. Real time PCR and quantitation of message abundance was performed as described in Methods. Relative to the levels in livers from SD rats, expression of mRNAs for both sterolins was increased ~3–5 fold in the SHR livers, as was the message for Srebp-2 (~2 fold). No significant change in SR-B1 or Cyp7a1 for SHR compared to SD liver RNAs was noted. However, the expression of LDL-R was reduced in both WKY and SHR rats.</p>
                </figcaption>
              </figure>
              <p id="s5s4p2">To compare protein expression levels of the sterolins, we performed western blotting. A reliable antibody to sterolin-1 was available only and was used to probe liver microsomal lysates from Sprague Dawley, SHR and Wistar rats (Fig. 8). Although equal amounts of microsomal protein lysates were loaded from the livers of these different strains, we consistently observed that the protein expression patterns were always different between SHR and the control strains (Fig. 8, panel C, cf. tracks 5, 6 and 8). Thus, all of the data were normalised in one of two ways, either by using total microsomal protein, or by using the intensity of transferrin signal, after western blotting. Both of these methods gave consistent results and indicated that expression of Abcg5 was increased approximately 2-fold in the SHR but not the Wistar or SD (Fig. 8). To further assess if the mutant protein is mis-targeted, we performed immunolocalization, using an anti-peptide rabbit antibody, on various rat liver and intestinal sections (Fig. 9). We did not observe any differences between the staining patterns observed for the control rat strains (Wistar or SD) and the mutant hypertensive strains (SHR, SHRSP). Interestingly, we observed a clear apical staining pattern for Abcg5 in the intestine and this was not altered in the mutant strains (Fig. 9, panel B).</p>
              <figure data-origid="F8">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC165443/#F8">Figure 8</a>
                <figcaption>
                  <p id="s5s4f2p1">Protein Expression of Abcg5 in liver total membrane preparations. Membrane preparations (20 μg protein) from rat tissues were isolated and analysed by SDS-PAGE, blotted and probed with anti-Abcg5 antiserum (R116, panel A), probed with anti-transferrin antiserum (panel B) or the gel was fixed and stained with Coomassie Brilliant Blue R-250 (panel C). Lane 1 contains a recombinant E. coli expressed N-terminal fragment of sterolin-1 as a control for antibody binding, lanes 2 and 3 contain Sprague-Dawley (SD) spleen membrane extracts. Liver from Wistar (lanes 4 and 5), SHR (lanes 6 and 7) and Sprague-Dawley (SD, lanes 8 and 9) Densitometry measurements when normalised to transferrin show a 2-fold increase in sterolin-1 protein in SHR rat liver samples, compared to SD and Wistar rat liver. No antibody for sterolin-2 is currently available.</p>
                </figcaption>
              </figure>
            </div>
          </section>
        </div>
      </section>
      <section data-type="">
        <h2 id="s6h1">Discussion</h2>
        <div class="content">
          <p id="s6p1">Recent studies on ABC transporters have significantly advanced our understanding of intestinal sterol absorption and biliary excretion [33]. Sterols are essential constituents of cellular membranes in human, animals and plants. In humans and animals, cholesterol is the sterol that is synthesised, absorbed and utilised, whereas plants do not synthesise cholesterol, but use molecules that are similar, such as sitosterol, campesterol and stigmasterol. Although similar amounts of cholesterol and plant sterols are present in the typical human diet, the intestine of normal subjects provides an efficient barrier against the absorption of the majority of plant sterols [3]. On average, &lt; 1–5% of dietary plant sterols are absorbed and retained compared with ~55% of dietary cholesterol [3,34]. Thus, in healthy individuals, there exists an exquisite mechanism that allows the body to distinguish between cholesterol and non-cholesterol sterols. ABCG5 and ABCG8 are two genes located adjacently in a head-to-head configuration on human chromosome 2p21, encoding for two half-transporters, sterolin 1 and sterolin 2 respectively [9,10,12]. Mutations in either of these genes causes sitosterolemia, characterized by hyperabsorption of phytosterols, reduced excretion of sterols into the bile and very elevated levels of plasma plant sterol levels. In sitosterolemic patients, the absorption of phytosterols in intestine is dramatically increased, with absorption of plant sterols to about the same extent (~60%) as cholesterol.</p>
          <p id="s6p2">Interestingly, a similar whole-body accumulation of plant sterols, less severe than in human sitosterolemia, has been reported recently in SHR, SHRSP and WKY rats, but not in the other genetically related rat strains, such as SD, Wistar and WKA rats [16]. These rats exhibit defects at two levels: increased absorption of sterols (plant and animal) at the intestinal level and decreased excretion of sterols into bile [16]. The higher sterol absorption in the intestine and the lower biliary excretion of plant sterols in these rat strains led us to hypothesise that Abcg5 or Abcg8 gene mutations may be responsible for the sitosterolemia observed in these hypertensive rats.</p>
          <p id="s6p3">Characterization of the rat STSL locus showed that it is highly conserved in organization, and maps to rat chromosome 6q12, a region also syntenic with the mouse and human STSL loci. Rat Abcg5 and Abcg8 are expressed in a tissue-specific manner, with expression confined to the liver and the intestine. We sequenced the STSL loci in sitosterolemic and control rat strains and identified a number of nucleotide changes. Most of these nucleotide changes were in intronic regions and, based upon their locations, are likely to be 'silent'. However, one change, G1754T in exon 12 was identified that led to a coding change of Gly583Cys in sterolin-1 (Abcg5). This change was found only in rat strains that were confirmed to be sitosterolemic (WKY, SHR and SHRSP), but not in any of the other strains screened (SD, Wistar, WKA, GH or BN).</p>
          <p id="s6p4">The significance of other genetic variances in Abcg5 and Abcg8 in different rat strains documented in this study is presently unknown. Plant sterol metabolism has been extensively evaluated in different rat strains, but the accumulation of plant sterols occurs only in SHR, SHRSP and WKY rats. The predicted 6 trans-membrane domain model of sterolin-1 indicates that the Gly583Cys missense mutation is located within the third extracellular loop (residues 554–620), in a conserved region (Fig 5C) and is adjacent to two potential N-glycosylation sites (residues 585 and 594). Thus a mutation at residue 583 could disrupt the first site. Finally, both the sterolin-1 and sterolin-2 are cysteine-rich proteins (16 cysteines in sterolin 1 and 14 in sterolin 2), five of these cyteines in sterolin-1 are present in the extracellular loop containing the mutation. It is tempting to speculate that the gain of an additional cysteine in sterolin-1 by the Gly583Cys missense mutation might affect intracellular trafficking or the heterodimerization with sterolin-2. This possibility is diminished but not eliminated by the demonstration that in both the mutant and control rats, Abcg5 is located apically in the intestine (Fig. 9). Evidence that sterolin-1 and sterolin-2 may heterodimerize has now been presented, based upon in vitro analyses [13]. However, our preliminary analyses of immunohistochemistry, using an antibody to sterolin-1 suggests that the pattern of expression in the liver and intestine is similar for both SD as well as WKY or SHR rats (Fig. 9). These studies are limited in scope, as antibody reagents to detect rat sterolin-2 are not currently available.</p>
          <figure data-origid="F9">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC165443/#F9">Figure 9</a>
            <figcaption>
              <p id="s6f1p1">Immunohistochemical detection of Abcg5 in liver and intestine from different rat strains. The panels on the left hand show liver sections and those on the right hand show intestinal sections. The right hand images of each panel show the hematoxylin stained images, viewed with transmitted light and the left hand images show the fluorescence images captured at the same laser wavelength and intensity. Panel A shows results obtained with the pre-immune serum, panel B, C and D the results for Wistar, WKY and SHR rat tissues. All sections were captured at 40 × magnification. Scale bars (50 μm) are shown in the bottom left hand corner of each image. The data for SD rat tissues were similar to those obtained with Wistar and are not shown.</p>
            </figcaption>
          </figure>
          <p id="s6p5">Another formal possibility not excluded is that the Gly583Cys missense mutation may have no significance in the accumulation of phytosterols in WKY, SHR and SHRSP, and that the alteration in the 3' UTR or even the intronic changes reported above may be the causative 'mutations'. Our analyses of the mRNA and protein profiles would not strongly support these possibilities. However, further investigation will be required to determine if the 3-base insertion in the predicted 3'UTR of Abcg8 has any functional consequence. In the absence of a direct assay for sterolin-1 and -2 function, the genetic changes presented, together with the correlation of the phenotypes, remains the strongest evidence that these genetic alterations are functional, and that the Gly583Cys is a mutational change.</p>
          <p id="s6p6">A more relevant question raised in this study is whether the plant sterol accumulation in hypertensive rat strains has significance in the pathophysiology of hypertension. Recent linkage mapping studies mapped hypertension QTLs to a number of chromosomal regions in SHR and SHRSP rats [35]. Our FISH study assigned the STSL locus rat chromosome 6q12, a region where a cluster of blood pressure QTL with Lod score 4~6 has been reported to be present (M6MIT12-M6MIT3, 39 cM distance) in a New Zealand genetically hypertensive rat (GH rat) [35]. With respect to hypertension, although the SHR and SHRSP strains carry the Gly583Cys mutation, in extensive QTL mapping studies using these strains, no loci mapping to the rat STSL region have been reported [35]. In our study, the GH rat is neither sitosterolemic, nor has any nucleotide alteration at the STSL locus that would implicate this region as mutated. Thus we do not believe the STSL locus plays a role in determining blood pressure. However, it may play a significant role in modifying the pathophysiology.</p>
          <p id="s6p7">Studies reported by different investigators have shown that a dietary intake of 10 % canola oil (as part of semi-purified diets) significantly shortened the life-span of SHRSP rats, with accumulation of phytosterols in plasma, red blood cells and tissues [15,17-19,21,22]. Lower red blood cell (RBC) membrane deformability, lower density of platelets, as well as a higher blood pressure were also observed, when compared to a SHRSP group fed with vegetable oils lower in phytosterols (10 % soybean oil vs. 10% canola oil). Canola oil has a higher sitosterol content than soybean oil. In comparison, extensive studies on evaluation of high phytosterol diets in SD rats (non-sitosterolemic) have consistently shown to be safe without any adverse consequences [36-42]. These intriguing differences between SHRSP and SD rats may now be explained, at least in part, by presence of the Gly583Cys missense mutation in sterolin-1. The WKY rat also carries this mutation and a high dietary intake of phytosterols will also increase the blood pressure and decrease the density of platelets, though to a lower extent than that observed in SHRSP rats [20]. These data support a pathological role of elevated plant sterols in these rats and support the notion that the STSL loci evolved to not only allow for the excretion of cholesterol into bile, but also to keep 'xeno-sterols' from accumulating in the body. With respect to sitosterolemia in humans, although atherosclerotic complications have been well-described, hypertension has not been mentioned as feature. However, it is not known whether the atherosclerosis in sitosterolemic patients with hypertension is accelerated, relative to those affected individuals with lower blood pressures.</p>
          <p id="s6p8">The Gly583Cys mutation in Abcg5 is likely to have originated in the original outbred Wistar colony bred in Kyoto (from which WKY was subsequently derived), which has been widely employed as a normotensive control in experimental studies of hypertension. It gave rise to the SHR and SHRSP strains. Thus, this mutation in WKY, SHR and SHRSP lines may not contribute to the hypertension phenotype per se, but may only be responsible for sitosterolemia. The original Wistar colony was established at the Wistar Institute in 1906, and the Sprague Dawley colony was derived by Dawley in approximately 1925 from a single male of unknown origin and a Wistar female. The SHR strain was derived by selective inbreeding from the Kyoto outbred Wistar colony, and both SHR and SHRSP, as well as WKY strains have been maintained over many generations through brother-sister matings [43,44]. Healthy WKA and GH rat strains were also generated from outbred Wistar rat strains, but these strains were isolated independently in Japan and New Zealand, respectively (and maintained at other sites as well). The GH rat was selected for systolic hypertension that increased with successive generations and maintained through filial matings [45]. In view of this, the Gly583Cys missense mutation of Abcg5 might result from occurrence of a de novo mutation during breeding at the stage of developing inbred WKY strains and propagated into the SHR and SHRSP lines. In contrast, the 143 nucleotide fragment insertion polymorphism in the 3'-UTR of Abcg8 may have resulted from acquisition of a foreign chromosomal fragment during the Wistar rat strain breeding and established as homozygous change in the WKY, SHR and SHRSP, as well as GH rat strains. In this context, it is important to note that 3 other mutant loci have been reported in the SHR rat; missense mutations in the KAT-1 (kynurenine aminotransferase-1), Srebf-1 (sterol regulatory element binding factor-1) and CD36 (fatty acid translocase) are all present as homozygous changes [46-48]. The generation of congenic lines may assist with addressing many of these issues. Such lines will be useful in demonstrating that the variant sterolin-1 protein is responsible for causing sitosterolemia, but will allow comparisons between animals that are hypertensive and those that are hypertensive and sitosterolemic. Currently, there are no appropriate genetic controls available, as SHR, SHRSP and WKY are all maintained as colonies by brother-sister matings. While this manuscript was in review, Scoggan et al also reported identifying Gly583Cys alteration in Abcg5 of SHR, SHRP and WKY rats [49].</p>
        </div>
      </section>
      <section data-type="">
        <h2 id="s7h1">Conclusion</h2>
        <div class="content">
          <p id="s7p1">Three rat strain, WKY, SHR and SHRSP, that are hypertensive are also sitosterolemic. All of these strains are homozygous mutant for Gly583Cys in sterolin-1, encoded by Abcg5. This mutation is not present in rats that were not sitosterolemic, even though one of these was hypertensive (the GH rat). The rat STSL locus, comprising of Abcg5 and Abcg8, maps to rat chromosome 6q12, a region syntenic with both mouse and human STSL loci. Preliminary antibody studies show that this mutant protein is expressed and appears to be apically targeted in the intestinal cells.</p>
        </div>
      </section>
      <section data-type="">
        <h2 id="s8h1">Competing Interests</h2>
        <div class="content">
          <p id="s8p1">None declared.</p>
        </div>
      </section>
      <section data-type="">
        <h2 id="s9h1">Authors' contributions</h2>
        <div class="content">
          <p id="s9p1">HY conceived this study, HY, BP, and KL performed the molecular genetic studies, EK performed the real time PCR and the western blotting assays, M-HL performed the immunostaining, KH performed the FISH assay, RK and AB performed the sterol analyses, MS and NE performed the DNA isolations from the rats. HY, GS, IK, TR and SBP were responsible for supervision, and data analyses of the experiments, as well as for providing the funding. HY and SBP wrote the paper. All authors read and approved the final manuscript.</p>
        </div>
      </section>
      <section data-type="">
        <h2 id="s10h1">Pre-publication history</h2>
        <div class="content">
          <p id="s10p1">The pre-publication history for this paper can be accessed here:</p>
        </div>
      </section>
      <section data-type="supplementary-material">
        <h2 id="s11h1">Supplementary Material</h2>
      </section>
    </article>
  </body>
</html>
