<!DOCTYPE html >
<html id="a81fF0zx_ph9pWC5lIVDUu15FDqW-PMC3731650" data-origid="PMC3731650" class="anndoc" data-anndoc-version="2.0" lang="" xml:lang="" xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="UTF-8"/>
    <meta name="generator" content="com.jmcejuela.anndoc.v2.bio.NcbiJournalArticleParser_v0_4$"/>
    <meta name="dcterms.source" content="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3731650/"/>
    <title>a81fF0zx_ph9pWC5lIVDUu15FDqW-PMC3731650</title>
  </head>
  <body>
    <article>
      <section data-type="title">
        <h2 id="s1h1">Lys63-linked polyubiquitination of BRAF at lysine 578 is required for BRAF-mediated signaling</h2>
      </section>
      <section data-type="abstract">
        <h2 id="s2h1">Abstract</h2>
        <div class="content">
          <p id="s2p1">The RAF kinase family is essential in mediating signal transduction from RAS to ERK. BRAF constitutively active mutations correlate with human cancer development. However, the precise molecular regulation of BRAF activation is not fully understood. Here we report that BRAF is modified by Lys63-linked polyubiquitination at lysine 578 within its kinase domain once it is activated by gain of constitutively active mutation or epidermal growth factor (EGF) stimulation. Substitution of BRAF lysine 578 with arginine (K578R) inhibited BRAF-mediated ERK activation. Furthermore, ectopic expression of BRAF K578R mutant inhibited anchorage-independent colony formation of MCF7 breast cancer cell line. Our studies have identified a previously unrecognized regulatory role of Lys63-linked polyubiquitination in BRAF-mediated normal and oncogenic signalings.</p>
        </div>
      </section>
      <section data-type="results">
        <h2 id="s3h1">Results</h2>
        <div class="subsections">
          <section data-type="">
            <h3 id="s3s1h1">Activated BRAF is modified by Lys63-linked polyubiquitination in human cancer cell lines</h3>
            <div class="content">
              <p id="s3s1p1">To test whether Lys63-linked polyubiquitination plays a role in BRAF activation and BRAF-mediated activation of downstream signaling pathway, we examined the Lys63-linked polyubiquitination status of BRAF in several human cell lines with or without BRAF or RAS oncogenic mutations, including MCF10A (BRAF wild-type), MDA-MB-231 (KRAS G13D, BRAF G464V)22, HT-29 (BRAF V600E)23, A549 (KRAS G12S)24, and SW480 (KRAS G12V)25. As shown in Fig. 1A and 1B, using anti-Ub-K63-specific antibodies, we found that the endogenous BRAF proteins were modified with Lys63-linked polyubiquitin chain in these cancer cells harboring BRAF or RAS oncogenic mutations, in comparison with a non-tumorigenic breast epithelial cell line, MCF10A, which possesses wild-type RAS and BRAF.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s3s2h1">BRAF with constitutively active mutation is linked with Lys63-linked polyubiquitin chains</h3>
            <div class="content">
              <p id="s3s2p1">To further determine whether BRAF with constitutively active mutation is modified by Lys63-linked polyubiquitination, expression vectors encoding FLAG-BRAF wild-type or constitutively active V600E or K601E mutants were co-transfected into HEK-293T cells with HA-Ub containing one lysine only at position 63 (HA-Ub-K63-only). As shown in Fig. 2A, we found that BRAF V600E and K601E proteins were strongly polyubiquitinated with Lys63-ubiquitin chain compared with BRAF wild-type. Consistent with this result, we found that FLAG-BRAF V600E proteins were strongly modified by Ub K48R but not Ub K63R mutant proteins (Fig. 2B). To further confirm this result, expression vectors encoding FLAG-BRAF wild-type or constitutively active mutants were transfected into HEK-293T cells. Then FLAG-BRAF proteins in the cell lysate were immunoprecipitated with anti-FLAG antibodies and immunoblotted with anti-Ub-K63-specific antibodies. Unlike BRAF wild-type, BRAF V600E and K601E proteins were modified by endogenous Lys63-linked ubiquitin chains (Fig. 2C). BRAF phosphorylation at Thr 599 and Ser 602 is essential for its activation in physiological condition. BRAF T599A-S602A double mutant (BRAF-AA) abolishes activation of BRAF whereas BRAF T599E-S602D double mutant (BRAF-ED) renders BRAF constitutively active4. We found that BRAF-ED, but not BRAF-AA mutant, was modified with Lys63-linked polyubiquitination and was able to activate endogenous ERK (Fig. 2D).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s3s3h1">BRAF with constitutively active mutation is modified by Lys63-linked polyubiquitination at Lys 578 within its kinase domain</h3>
            <div class="content">
              <p id="s3s3p1">To determine the role of Lys63-linked polyubiquitination in BRAF activation and BRAF-mediated ERK activation, it is essential to map BRAF Ub lysine acceptor site(s). To determine the location of Ub lysine acceptor site on BRAF, we generated the N-terminal deletion constructs of BRAF wild-type, V600E and K601E mutants in which the BRAF regulatory domain to amino acid 446 was deleted (BRAF CT) (Fig. 3A). Then these BRAF CT mutants were co-transfected with HA-Ub-K63-only expression vectors into HEK-293T cells. As shown in Fig. 3B, BRAF CT V600E and K601E proteins were strongly modified by Lys63-linked polyubiquitination compared with the BRAF CT wild-type. Consistent with that result, BRAF CT V600E and K601E induced a stronger ERK activation compared with BRAF CT wild-type. Therefore, we reasoned that one or more Lys residues within BRAF C-terminal kinase domain (446–766 amino acids) may serve as the Ub Lysine acceptor site(s).</p>
              <p id="s3s3p2">BRAF contains 21 lysine residues within its kinase domain (Fig. 3C). To identify the possible Ub lysine acceptor site(s) in BRAF with constitutively active mutation, each of 14 evolutionary conserved Lys residues in FLAG-BRAF-K601E full length (or CT) were systematically replaced with an Arg residue, which maintains the positive charge but is not able to serve as an acceptor site for Ub modification. The expression vectors encoding FLAG-BRAF-K601E full length (or CT) and mutants with different Lys to Arg mutations were co-transfected into HEK-293T cells with HA-Ub-K63-only expression vectors. Among the 14 Lys to Arg mutants examined, only K483R and K578R mutants failed to induce BRAF polyubiquitination as well as ERK activation compared with all the other mutants (Fig. 3D, Fig. S1A). Consistent with these results, luciferase reporter analysis showed that BRAF-K601E CT with K483R or K578R mutation failed to induce AP-1-dependent luciferase reporter gene expression whereas BRAF-K601E CT and other mutants induced a higher level of AP-1 reporter activity in HEK-293T cells (Fig. S1B). Since the BRAF Lys 483 residue is a conserved lysine residue in the ATP binding pocket essential for coordinating kinase ATP-binding and activation26, it can be inferred that Lys 578 is the predominant Lys63-linked Ub acceptor site for BRAF polyubiquitination. In addition, we tested the impact of K578R mutation in BRAF with constitutively active V600E mutation. As shown in Fig. 3E, we found that BRAF with V600E-K578R double mutations failed to be modified by Lys63-linked polyubiquitination and resulted in significantly decreased ERK phosphorylation compared with BRAF with V600E mutation. Conforming to these results, we found that BRAF-ED with K578R mutation failed to be modified by Lys63-linked polyubiquitination and resulted in significantly decreased ERK phosphorylation compared to BRAF-ED mutant (Fig. S2). As indicated by previous studies indicate that BRAF Lys 578 residue is on the surface of the protein structure and is exposed to solvent27 (Fig. S3). Therefore, BRAF K578R mutation should not induce any significant structural perturbation. These results suggest that Lys 578 serves as an acceptor site for Lys63-linked polyubiquitination and activation of BRAF with constitutive active mutation.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s3s4h1">EGF induces Lys63-linked BRAF polyubiquitination at Lys 578</h3>
            <div class="content">
              <p id="s3s4p1">Since EGF induces BRAF activation via EGFR/RAS-GTP, we hypothesized that EGF induces BRAF Lys63-linked polyubiquitination in physiological condition. Indeed, Lys63-linked BRAF polyubiquitination was detected in serum starved HEK-293T cells transfected with different HA-tagged ubiquitin constructs after EGF treatment (Fig. 4A). To further test this hypothesis, HEK-293T cells were serum starved and treated with human EGF. As shown in Fig. 4B, EGF rapidly induced Lys63-linked endogenous BRAF polyubiquitination and ERK activation. Furthermore, we found that BRAF K578R mutant inhibited EGF-induced BRAF Lys63-linked polyubiquitination and delayed ERK phosphorylation (Fig. 4C).</p>
              <p id="s3s4p2">To further examine the role of Lys63-linked polyubiquitination in BRAF-mediated function in the cancer cells, MCF7, a breast cancer cell line with wild-type BRAF, was used to generate stable cell line that express empty vector or FLAG-BRAF wild-type or K578R mutant. Colony formation assay was performed to examine the effect of BRAF K578R on cell behavior. As shown in Fig. 4D, MCF7 cells with FLAG-BRAF K578R stable expression displayed significantly decreased colony number compared with the MCF7 cells with stable expression of empty vector or FLAG-BRAF wild-type. Taken together, these results indicate that Lys 578 residue is an acceptor site for Lys63-linked polyubiquitination of BRAF in response to EGF stimulation. In addition, Lys63-linked BRAF polyubiquitination is required for EGF-induced ERK activation as well as anchorage-independent colony formation potential of MCF7 breast cancer cell line.</p>
            </div>
          </section>
        </div>
      </section>
      <section data-type="discussion">
        <h2 id="s4h1">Discussion</h2>
        <div class="content">
          <p id="s4p1">It is worth noting that KRAS is subject to modification by monoubiquitination, which increases KRAS GTP loading and the affinity to its specific downstream activity28. Whereas the ubiquitination of HRAS stabilizes the association of HRAS with endosomes and upregulates its activity to induce the activation of downstream signaling29. Here, we demonstrate that BRAF, a downstream effector of RAS, is also modulated by ubiquitination during its activation.</p>
          <p id="s4p2">Previous studies suggested that the kinase domain of BRAF wild type and constitutively active V600E mutant differ in their conformation and the latter has increased dimerization potential630. With increased complexity for BRAF kinase regulation, our results demonstrate that BRAF with constitutive mutation is subject to modification by Lys63-linked polyubiquitination. Since substitution of constitutively active BRAF V600E or K601E mutant Lys 578 with Arg abolishes its Lys63-linked polyubiquitination and BRAF V600E or K601E-mediated ERK activation, we conclude that the Lys63-linked polyubiquitination is essential in BRAF V600E or K601E mutant -mediated oncogenic signaling.</p>
          <p id="s4p3">Our finding shows that EGF induces Lys63-linked polyubiquitination of BRAF, which may serve as a platform to assemble BRAF-containing signaling complex to mediate the activation of downstream signaling. Interestingly, a lysine to arginine mutation at Lys 578 of BRAF blocks EGF induced BRAF Lys63-linked polyubiquitination and ERK activation (Fig. 4C). These results suggest that Lys 578 is the predominant Ub acceptor site of BRAF Lys63-linked polyubiquitination induced by EGF and BRAF Lys63-linked polyubiquitination plays an essential role in EGF-induced ERK activation.</p>
          <p id="s4p4">Collectively, our study reveals that BRAF Lys63-linked polyubiquitination at Lys 578 residue within its kinase domain upon its activation is required for BRAF signaling in normal or oncogenic settings. In the light of the data presented here and in previous reports, we proposed two working models to present an integral modulation of BRAF activation by Lys63-linked polyubiquitination (Fig. 5). Obviously, more studies are needed to identify the specific E3 ligase(s) and deubiquitinating enzyme(s) that are responsible for the positive and negative regulation of BRAF Lys63-linked polyubiquitination. Further characterization of ubiquitination control in BRAF oncogenic signaling will help us to identify novel molecular therapeutic targets for curing the cancers harboring RAS and BRAF oncogenic mutations.</p>
        </div>
      </section>
      <section data-type="methods">
        <h2 id="s5h1">Methods</h2>
        <div class="subsections">
          <section data-type="">
            <h3 id="s5s1h1">Cell culture and transfection</h3>
            <div class="content">
              <p id="s5s1p1">HEK-293T, colon cancer cell line HT-29, SW480, breast cancer cell line MCF7, MDA-MB-231 and alveolar adenocarcinoma cell line A549 were maintained in Dulbecco's modified Eagle's medium (high glucose) supplemented with 10% fetal bovine serum, 100 units/ml penicillin, and 100 μg/ml streptomycin at 37°C in 5% CO2. MCF10A cells were cultured in the standard, the Dulbecco's Modified Eagle Medium: Nutrient Mixture F-12 (DMEM/F12, 1:1) containing 17 mM glucose, and supplemented with 5% horse serum, 10 mg/L insulin, 20 μg/L epidermal growth factor, 50 μg/L cholera toxin, 50 mg/L hydrocortisone, 100 units/ml penicillin, and 100 μg/ml streptomycin in a humidified environment at 37°C with 5% CO2. HEK-293T and MCF7 were transfected with mammalian expression plasmids using Fugene XtremeGENE 9 (Roche) or Lipofectamine 2000 (Invitrogen), respectively.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s2h1">Expression vectors</h3>
            <div class="content">
              <p id="s5s2p1">A cDNA construct containing the full length open reading frame of the wild-type human BRAF was subcloned into the Flag-tagged mammalian expression vector pcDNA3.1. Point mutations were made by site-directed PCR mutagenesis and verified by DNA sequencing analysis. Ubiquitin K63R, K48R and ubiquitin with all lysine to arginine mutation except Lys 63 or Lys 48 were sub-cloned into mammalian expression vector pcDNA3.1 with HA-tag at their N-terminal and verified by DNA sequencing analysis. Their expression in HEK-293T cells after transfection were confirmed by immunoblotting analysis with anti-HA antibodies (Fig. S4).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s3h1">Establishment of stable cell lines</h3>
            <div class="content">
              <p id="s5s3p1">Human breast cancer cell line MCF7 was transfected with FLAG-BRAF wild type, FLAG-BRAF-K578R, and control vector using Lipofectamine 2000 according to the manufacturer's instructions. Then G418 (1.3 mg/ml) was added to these transfected cells. After 15 days selection, stable cell lines were established. Western blotting was used to validate the expression of different BRAF constructs.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s4h1">Antibodies and reagents</h3>
            <div class="content">
              <p id="s5s4p1">Antibodies against ERK1/2, phospho-ERK1/2 and secondary antibodies conjugated to horseradish peroxidase were purchased from Cell Signaling Technology (Beverly, MA). The anti-FLAG antibody (M2) and anti-β-actin antibodies as well as protease and phosphatase inhibitors were obtained from Sigma (St. Louis, MO). Antibodies for HA epitope, BRAF (F-7) and Protein A-agarose beads were purchased from Santa Cruz Biotechnology, Inc. (Santa Cruz, CA). Anti-Ub-K63-specific antibody was from EMD Millipore (Temecula, CA). Rabbit anti-Flag antibodies were generated and purified by Genemed Synthesis Inc. (San Antonio, TX). Human recombinant epidermal growth factor (EGF) was purchased from R&amp;D systems (Minneapolis, MN).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s5h1">Immunoprecipitation and immunoblotting</h3>
            <div class="content">
              <p id="s5s5p1">To prepare the total cell lysates, cells were washed with ice-cold PBS on ice, then were lysed by adding lysis buffer (50 mM Tris-HCl, pH 7.4, 135 mM NaCl, 1% Triton X-100, 25 mM β-glycerophosphate, 0.1 mM sodium orthovanadate, 0.5 mM phenylmethylsulfonyl fluoride, 1 mM dithiothreitol, 10 μg/ml aprotinin,10 μg/ml leupeptin, 1 mM Benzamidine, 20 mM disodium p-nitrophenylphosphate, 1 mM phenylmethylsulfonyl fluoride, 3 mM EDTA, and phosphatase inhibitor cocktail A and B). Then the cell lysate were boiled in the presence of 1% SDS and then diluted with lysis buffer to 0.1% SDS in order to disrupt noncovalent protein-protein interactions. After collecting the cell lysate by 15000×g centrifugation for 15 min at 4°C, the supernatant were incubated with the indicated antibodies for 3 h at 4°C. Followed by adding Protein A-agarose beads and rotated for 3 h at 4°C. The precipitates were washed four times in cold washing buffer (20 mM HEPES,pH 7.7, 50 mM NaCl, 2.5 mM MgCl2, and 0.05% Triton X-100), then the beads were re-suspended in gel loading buffer and boiled for 5 min. The immunoprecipitates or the whole cell lysates were resolved by SDS-PAGE and transferred to nitrocellulose or PVDF membranes (Bio-Rad). The membranes were probed with the appropriate antibodies. The IgG horseradish peroxidase-conjugated antibodies were used as the secondary antibodies. The proteins were detected by using the ECL-Plus Western blotting detection system (GE Healthcare).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s6h1">Luciferase reporter assay</h3>
            <div class="content">
              <p id="s5s6p1">HEK-293T cells were seeded at a concentration of 1 × 105 cells per well and cultured overnight in 12-well plates. AP-1-dependent firefly luciferase reporter and effector plasmids were co-transfected along with the Renilla luciferase plasmid into HEK-293T cells. Forty eight hours after transfection, cells were harvested in lysis buffer (Promega), and luciferase assays were performed using the Dual-Luciferase Reporter Assay System (Promega). The relative luciferase activity was calculated by dividing the firefly luciferase activity by the Renilla luciferase activity. Data represents three independent experiments performed in triplicates.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s7h1">Colony formation assays</h3>
            <div class="content">
              <p id="s5s7p1">The 6-well plate was filled with 2 ml of 0.5% agar gel in DMEM with 10% FBS as lower gel. Cell numbers of MCF7 stable cell lines were counted (1 × 104) and mixed with 1.5 ml of 0.3% agar gel in DMEM with 10% FBS. The mixture was added on the concretionary lower gel and incubated at 37°C in 5% CO2 for 15 days. The cells were stained with 5 mg/ml MTT to detect the existence of colony formation of each cell lines in the plate. The colony number was counted by 1-D gel image colony counting software from Bio-Rad.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s8h1">Statistical analysis</h3>
            <div class="content">
              <p id="s5s8p1">All values are presented as mean ± S.D. P-values were calculated using unpaired two-tailed student's t-test. A p-value &lt; 0.05 is considered statistically significant.</p>
            </div>
          </section>
        </div>
      </section>
      <section data-type="">
        <h2 id="s6h1">Author Contributions</h2>
        <div class="content">
          <p id="s6p1">L.A., W.J. and Y.Y. designed and performed experiments, analyzed the data and edited the manuscript. L.A. wrote the draft of the manuscript. H.Z. and J.H.Y. obtained funding, supervised the project and edited the manuscript. N.Z., Y.H.F., Y.Y. and Z.C.S. analyzed the results and commented on the manuscript. G.F.X. analyzed the 3-D structure of BRAF. G.L. built some initial constructs. L.L., Y.L.Z. and J.C. revised the manuscript.</p>
        </div>
      </section>
      <section data-origid="s1" data-type="supplementary-material">
        <h2 id="s7h1">Supplementary Material</h2>
      </section>
      <section data-type="resources">
        <h2 id="s8h1">Resources</h2>
        <div class="content">
          <figure data-origid="f1">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3731650/#f1">Figure 1</a>
            <figcaption>
              <h5 id="s8f1h1">Activated BRAF is modified by Lys63-linked polyubiquitination in human cancer cell lines.</h5>
              <p id="s8f1p1">(A) Endogenous BRAF with constitutive active mutation is modified by Lys63-linked polyubiquitin chain. MCF10A cells (wild-type BRAF) and MDA-MB-231 cells (with KRAS G13D, BRAF G464V mutation) were lysed and immunoprecipitated with anti-normal IgG or BRAF antibodies followed by immunoblotting with anti-Ub-K63-specific antibodies to detect the presence of BRAF with Lys 63-linked polyubiquitination. (B) Hyperactivated RAS induces BRAF modification by Lys 63-linked polyubiquitination. HT29 (BRAF V600E mutation), A549 (with KRAS G12S mutation), and SW480 cell lines (with KRAS G12V mutation) were lysed and immunoprecipitated with anti-normal IgG or BRAF antibodies followed by immunoblotting with anti-Ub-K63-specific antibodies to detect the presence of Lys63-ubiquitinated endogenous BRAF. Cropped blots were used under the same experimental conditions.</p>
            </figcaption>
          </figure>
          <figure data-origid="f2">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3731650/#f2">Figure 2</a>
            <figcaption>
              <h5 id="s8f2h1">BRAF with constitutively active mutation is linked with Lys63-linked polyubiquitin chains.</h5>
              <p id="s8f2p1">(A) BRAF with constitutively active mutation was linked with HA-Ub-K63-only. The expression vector encoding full length FLAG-BRAF WT or constitutively active V600E or K601E mutant was co-transfected into HEK-293T cells with HA-Ub-K63-only expression vector. Lys 63-polyubiquitinated FLAG-BRAF was detected by immunoprecipitation with anti-HA antibodies and immunoblotting with anti-FLAG antibodies. (B) BRAF with V600E constitutive active mutation was modified by Lys63- but not Lys48-linked polyubiquitination. Empty vector or expression vector encoding HA-Ub-K63R or -K48R was co-transfected into HEK-293T cells with FLAG-BRAF. Ubiquitinated FLAG-BRAF proteins were detected by immunoprecipitation with anti-HA antibodies and immunoblotting with anti-FLAG antibodies. (C) BRAF with constitutively active mutation was endogenously modified by Lys63-linked polyubiquitination. The expression vector encoding full length FLAG-BRAF WT, constitutively active V600E or K601E mutant was transfected into HEK-293T cells. Lys63-linked polyubiquitinated FLAG-BRAF was detected by immunoprecipitation with anti-FLAG antibodies and immunoblotting with anti-Ub-K63-specific antibodies. (D) BRAF ED mutant was modified by Lys63-linked polyubiquitin chains compared with BRAF WT or BRAF AA mutant. Expression vector encoding FLAG-BRAF WT, AA and ED were co-transfected into HEK-293T cells with HA-Ub-K63-only vector, respectively. Lys63-linked polyubiquitinated FLAG-BRAF was detected by immunoprecipitation with anti-HA antibodies and immunoblotting with anti-FLAG antibodies. Cropped blots were used under the same experimental conditions.</p>
            </figcaption>
          </figure>
          <figure data-origid="f3">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3731650/#f3">Figure 3</a>
            <figcaption>
              <h5 id="s8f3h1">BRAF with constitutively active mutation is modified by Lys63-linked polyubiquitination at Lys 578 within its kinase domain.</h5>
              <p id="s8f3p1">(A) Schematic view of BRAF full length (FL) and BRAF C-terminal (CT). (B) BRAF C-terminal (CT) with constitutively active mutation was modified with Lys63-linked polyubiquitination. The expression vector encoding FLAG-BRAF CT wild-type, constitutively active V600E or K601E mutant was co-transfected into HEK-293T cells with HA-Ub-K63-only expression vector. Lys63-linked polyubiquitinated FLAG-BRAF was detected by immunoprecipitation with anti-HA antibodies and immunoblotting with anti-FLAG antibodies. (C) BRAF sequence with the evolutionary conserved lysine residues within its C-terminal kinase domain amino acids indicated. (D) BRAF with K601E mutation was modified by Lys63-linked polyubiquitination at Lys 578. Expression vector encoding HA-Ub-K63-only was co-transfected into HEK-293T cells with control vector or FLAG-BRAF K601E or K601E plus other lysine to arginine mutants. Lys63-linked polyubiquitinated FLAG-BRAF was detected by immunoprecipitation with anti-FLAG antibodies and immunoblotting with anti-HA antibodies. (E) BRAF with V600E mutation was modified by Lys63-linked polyubiquitination at Lys 578. Expression vector encoding FLAG-BRAF WT, V600E or V600E plus K578R double mutant was co-transfected into HEK-293T cell with HA-Ub-K63-only vector. Lys63-linked polyubiquitinated FLAG-BRAF was detected by immunoprecipitation with anti-HA antibodies and immunoblotting with anti-FLAG antibodies. Cropped blots were used under the same experimental conditions.</p>
            </figcaption>
          </figure>
          <figure data-origid="f4">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3731650/#f4">Figure 4</a>
            <figcaption>
              <h5 id="s8f4h1">EGF induces Lys63-linked BRAF polyubiquitination at Lys 578.</h5>
              <p id="s8f4p1">(A) BRAF was modified with Lys63-linked polyubiquitination after EGF treatment. Expression vector encoding FLAG-BRAF wild-type was co-transfected into HEK-293T cells with HA-Ub with Lys63- or Lys48-only expression vector. After 24 hours, cells were starved for 16 hours and treated with EGF (50 ng/ml) at time points indicated. Polyubiquitinated FLAG-BRAF was detected by immunoprecipitation with anti-FLAG antibodies and immunoblotting with anti-HA antibodies. (B) Endogenous BRAF was modified by Lys63-linked polyubiquitination after EGF treatment. HEK-293T cells were starved for 16 hours and treated with EGF (50 ng/ml) at time points indicated. Lys63-linked polyubiquitinated FLAG-BRAF was detected by immunoprecipitation with anti-BRAF antibodies and immunoblotting with anti-Ub-K63-specific antibodies. (C) EGF induced BRAF Lys63-linked polyubiquitination at Lys 578, which was required for EGF-induced ERK activation. Expression vector encoding empty vector, FLAG-BRAF wild-type or K578R mutant was co-transfected into HEK-293T cell with HA-Ub-K63-only expression vector. After 24 hours, cells were starved for 16 hours and treated with EGF (50 ng/ml) at time points indicated. Lys63-linked polyubiquitinated FLAG-BRAF was detected by immunoprecipitation with anti-FLAG antibodies and immunoblotting with anti-HA antibodies. (D) BRAF with K578R mutation blocked anchorage-independent growth of MCF7 breast cancer cells. MCF7 cells with stable expression of empty vector, FLAG-BRAF wild-type or K578 mutant was used for colony formation assay in triplicates. **p &lt; 0.01. Cropped blots were used under the same experimental conditions.</p>
            </figcaption>
          </figure>
          <figure data-origid="f5">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3731650/#f5">Figure 5</a>
            <figcaption>
              <h5 id="s8f5h1">Working models for the role of Lys63-linked polyubiquitination in BRAF activation.</h5>
              <p id="s8f5p1">(A) Normal BRAF activation. Upon EGF binding to its receptor, the Ras-GTP recruits BRAF and induces its phosphorylation at Thr 599-Ser 602 residues and Lys63-linked polyubiquitination at Lys 578 within its kinase domain, which in turn mediates the activation of MEK-ERK and regulates downstream gene expression. (B) Oncogenic BRAF activation. BRAF with constitutively active mutation gains its Lys63-linked polyubiquitination at Lys 578 to activate downstream effector gene independent of RAS activity.</p>
            </figcaption>
          </figure>
        </div>
      </section>
    </article>
  </body>
</html>
