<!DOCTYPE html >
<html id="a9ZqkY4nYtV9MlqPSLGIR.90tOyy-PMC2185579" data-origid="PMC2185579" class="anndoc" data-anndoc-version="2.0" lang="" xml:lang="" xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="UTF-8"/>
    <meta name="generator" content="com.jmcejuela.anndoc.v2.bio.NcbiJournalArticleParser_v0_3$"/>
    <meta name="dcterms.source" content="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2185579/"/>
    <title>a9ZqkY4nYtV9MlqPSLGIR.90tOyy-PMC2185579</title>
  </head>
  <body>
    <article>
      <section data-type="title">
        <h2 id="s1h1">Acaps Are Arf6 Gtpase-Activating Proteins That Function in the Cell Periphery</h2>
      </section>
      <section data-type="abstract">
        <h2 id="s2h1">Abstract</h2>
        <div class="content">
          <p id="s2p1">The GTP-binding protein ADP-ribosylation factor 6 (Arf6) regulates endosomal membrane trafficking and the actin cytoskeleton in the cell periphery. GTPase-activating proteins (GAPs) are critical regulators of Arf function, controlling the return of Arf to the inactive GDP-bound state. Here, we report the identification and characterization of two Arf6 GAPs, ACAP1 and ACAP2. Together with two previously described Arf GAPs, ASAP1 and PAP, they can be grouped into a protein family defined by several common structural motifs including coiled coil, pleckstrin homology, Arf GAP, and three complete ankyrin-repeat domains. All contain phosphoinositide-dependent GAP activity. ACAP1 and ACAP2 are widely expressed and occur together in the various cultured cell lines we examined. Similar to ASAP1, ACAP1 and ACAP2 were recruited to and, when overexpressed, inhibited the formation of platelet-derived growth factor (PDGF)–induced dorsal membrane ruffles in NIH 3T3 fibroblasts. However, in contrast with ASAP1, ACAP1 and ACAP2 functioned as Arf6 GAPs. In vitro, ACAP1 and ACAP2 preferred Arf6 as a substrate, rather than Arf1 and Arf5, more so than did ASAP1. In HeLa cells, overexpression of either ACAP blocked the formation of Arf6-dependent protrusions. In addition, ACAP1 and ACAP2 were recruited to peripheral, tubular membranes, where activation of Arf6 occurs to allow membrane recycling back to the plasma membrane. ASAP1 did not inhibit Arf6-dependent protrusions and was not recruited by Arf6 to tubular membranes. The additional effects of ASAP1 on PDGF-induced ruffling in fibroblasts suggest that multiple Arf GAPs function coordinately in the cell periphery.</p>
        </div>
      </section>
      <section data-type="">
        <h2 id="s3h1">Introduction</h2>
        <div class="content">
          <p id="s3p1">The ADP-ribosylation factors (Arfs) are a family of ubiquitous and highly conserved GTP-binding proteins in eukaryotes (for reviews see Donaldson and Klausner 1994; Donaldson et al. 1995; Moss and Vaughan 1995, Moss and Vaughan 1998). Based on biochemical activities and sequence similarity, proteins within the Arf family can be divided into Arf and Arf-like proteins. The Arf proteins have been further subdivided into class I (Arf1, Arf2, and Arf3), class II (Arf4 and Arf5), and class III (Arf6) Arfs. Similar to other GTP-binding proteins, Arfs function as molecular switches with multiple targets affecting a variety of cellular events. Arfs regulate membrane traffic (Donaldson and Klausner 1994; Donaldson et al. 1995; Moss and Vaughan 1995, Moss and Vaughan 1998) and the actin cytoskeleton (Radhakrishna et al. 1996, Radhakrishna et al. 1999; D'Souza-Schorey et al. 1997; Frank et al. 1998; Song et al. 1998; Franco et al. 1999; Zhang et al. 1999), and directly stimulate phospholipase D and phosphatidylinositol 4-phosphate 5-kinase (Brown et al. 1993; Cockcroft et al. 1994; Honda et al. 1999). Multiple Arf family members may function in the same membrane trafficking system, and Arf1 and 6 impinge on the actin cytoskeleton in the cell periphery. Arf6 affects membrane ruffling, filopodia, and actin-rich protrusions; Arf1 affects focal adhesions (Radhakrishna et al. 1996; Frank et al. 1998; Norman et al. 1998; Franco et al. 1999). In addition, a single Arf family member may fulfill the same or similar functions at multiple sites in the same pathway. For instance, Arf1 regulates membrane traffic from the ER through the Golgi complex and from the trans-Golgi network (Balch et al. 1992; Donaldson et al. 1992; Palmer et al. 1993; Stamnes and Rothman 1993; Traub et al. 1993; Ooi et al. 1998). Similarly, Arf6 affects the formation of actin-rich membrane protrusions (Radhakrishna et al. 1996; Frank et al. 1998), rac-induced ruffles (Radhakrishna et al. 1999), and phagosomes (Zhang et al. 1998, Zhang et al. 1999). The presence of multiple Arfs at a given site and single Arf isoforms at multiple sites suggest that the regulation of Arf activity is specific both for the given Arf isoform and for the compartment in which the Arf is located.</p>
          <p id="s3p2">Regulation that is specific for both Arf isoform and intracellular site is most likely achieved through guanine nucleotide exchange factors (GEFs) and GTPase-activating proteins (GAPs). Four families of GEFs have been identified: GBF, BIG1/2, ARNO, and EFA6 (Morinaga et al. 1997; Frank et al. 1998; Klarlund et al. 1998; Claude et al. 1999; Franco et al. 1999; Mansour et al. 1999; Togawa et al. 1999; Jackson and Casanova 2000). Five mammalian proteins have been reported to have Arf GAP activity: Arf GAP1, GIT1/CAT1, GIT2/CAT2, PAP, and ASAP1 (Cukierman et al. 1995; Brown et al. 1998; Premont et al. 1998; Andreev et al. 1999; Vitale et al. 2000). Compartment and substrate specificity have been established for some of these proteins. Two, GBF1 and BIG1/2, are Golgi-localized GEFs specific for class I and II Arfs (Claude et al. 1999; Mansour et al. 1999; Morinaga et al. 1997; Togawa et al. 1999). ARNO and EFA6 family GEFs function in the cell periphery, affecting membrane traffic and the actin cytoskeleton by regulating Arf6 activity (Frank et al. 1998; Klarlund et al. 1998; Franco et al. 1999; Jackson and Casanova 2000). Of the Arf GAPs, in vitro analysis has revealed that Arf GAP1, ASAP1, and PAP use Arf1 and Arf5 more efficiently than Arf6 (Randazzo 1997b; Brown et al. 1998; Andreev et al. 1999), whereas GIT1 has no preference among Arf1, 5, and 6 (Vitale et al. 2000). The in vivo Arf specificities of these GAPs have not yet been determined. The GAPs, like the GEFs, have distinct sites of action. Arf GAP1 specifically affects Golgi function (Aoe et al. 1997). ASAP1 functions in the periphery, regulating focal adhesion turnover and reorganization of the actin cytoskeleton (Randazzo et al. 2000a). PAP (PAG3) binds to paxillin, impacting cytoskeleton organization, and colocalizes with Arf6 in the cell periphery, though it does not affect Arf6-dependent cytoskeletal changes (Kondo et al. 2000). These data support the hypothesis that Arf GEFs and Arf GAPs act at specific cellular locations. However, the ability of GEFs and/or GAPs, at a particular cellular location, to discriminate between Arfs and/or distinct Arf-dependent events has not been examined through direct comparison of the proteins.</p>
          <p id="s3p3">Here, we have identified two Arf GAPs, which we call ACAP1 and ACAP2, that can be grouped with PAP and ASAP1 as a family based on structural and functional homologies. We examined the in vitro GAP activities, in vivo activities, and the localization of ACAP1, ACAP2, and ASAP1. Although all functioned in the cell periphery, affecting the actin cytoskeleton, and colocalized with Arf6, ACAP1 and ACAP2, but not ASAP1, specifically affected Arf6-dependent events. We propose a model in which multiple Arf GAPs coordinate distinct events occurring within a biological pathway. With the number of potential Arf GAPs, these results could extrapolate to Arf function in other compartments.</p>
        </div>
      </section>
      <section data-type="materials|methods">
        <h2 id="s4h1">Materials and Methods</h2>
        <div class="subsections">
          <section data-type="">
            <h3 id="s4s1h1">Plasmids</h3>
            <div class="content">
              <p id="s4s1p1">ACAP1/KIAA0050 cDNA was obtained from Dr. Takahiro Nagese at the Kazusa DNA Research Institute (Chiba, Japan). ACAP2 was cloned from a SuperScript human leukocyte cDNA library (Life Technologies) using a 2-kb KpnI fragment from KIAA0041 (Dr. Takahiro Nagese) as a probe. Six positive clones were isolated, sequenced, and found to overlap over a region of 3,227 bp. An open reading frame extends from nucleotide 61–2,397 and encoded a predicted protein of 778 amino acids. The sequence was entered into Genbank/EMBL/DDBJ under accession no. AJ238248, under the name centaurin β2 (available at http://www.ncbi.nim.nih.gov/BLAST/). A mammalian expression vector for epitope-tagged ACAP1 was generated by PCR with oligonucleotide primers incorporating BstZI and SalI sites. After digestion, the PCR product was ligated into the NotI and SalI sites of pFLAG-CMV2 to give an in frame fusion between ACAP1 and the NH2-terminal FLAG tag. For ACAP2, two overlapping clones encompassing the open reading frame were used to amplify overlapping 5′ and 3′ fragments, the former contained a NotI site, the latter contained a KpnI site. The products were purified, mixed in equimolar amounts, and used as template for combinatorial PCR. The resulting product was purified and digested with NotI and KpnI and subcloned into pFLAG-CMV2 (Randazzo et al. 2000b). A FLAG-tagged ASAP1b has been described (Brown et al. 1998). ASAP1b was also cloned into pFLAG-CMV (Sigma-Aldrich) using the EcoRI site. The results using this construct were identical to those using the previously described construct. Glutamine mutations in arginine 448 of ACAP1 and arginine 442 of ACAP2 were introduced using a kit (Quikchange; Stratagene). The expressed proteins with these point mutations lacked GAP activity (see Results), which is consistent with the lack of activity of mutants of ASAP1 (Randazzo et al. 2000a) and PAP (Mandiyan et al. 1999) that had the equivalent arginine mutated.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s2h1">Determination of Tissue Distribution of mRNAs</h3>
            <div class="content">
              <p id="s4s2p1">Tissue cDNA panels were obtained from CLONTECH Laboratories Inc. Primers that would yield ∼500-bp PCR products were designed. For ACAP1, the forward primer was CCTTCAgTCAggCTCgCCTT gATgAC, the reverse primer was CCTCgAATCT CAggCAgCTT ggTCAg, and the product was predicted to be 498 bp. For ACAP2, the forward primer was CATCTCCATC CACAggAAgC CTAgATTCTg, the reverse primer was TgAACTTTgg gCAgATgCTC TgACTTggTC, and the product was predicted to be 537 bp. Conditions for thermal cycling were from the manual from CLONTECH Laboratories Inc. that accompanied the cDNA panels. PCR products were fractionated on a 1% agarose gel and stained with ethidium bromide. The data shown are for 38 cycles for ACAP1 and 34 cycles for ACAP2. Glyceraldehyde 3-phosphate dehydrogenase was used as a control target using primers provided by CLONTECH Laboratories Inc. The product obtained with 30 cycles is shown.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s3h1">Proteins</h3>
            <div class="content">
              <p id="s4s3p1">FLAG-tagged ACAP1, ACAP2, and ASAP1 were expressed in COS7 or HEK 293 cells by transient transfection using Fugene 6 (Roche). The cells were lysed in 20 mM Tris-HCl, pH 8.0, 25 mM NaCl, containing 1% Triton X-100, complete protease inhibitor cocktail (Roche), and 10% glycerol. The proteins were sequentially fractionated by anion exchange on a HiTrap Q column (Amersham Pharmacia Biotech) in a NaCl gradient and by chromatography on an Econo-Pac CHT-II column (Bio-Rad Laboratories) in a KPi gradient (Randazzo et al. 2000b). Less than 1% of the activity in these preparations was due to endogenous GAPs that copurified with the ectopically expressed protein (based on GAP activity in similar fractions obtained from cells transfected with an empty vector). Native ASAP1 was purified from bovine brain through the hydrophobic interaction column step (Randazzo 1997b; Brown et al. 1998).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s4h1">Assays of Cellular Activities</h3>
            <div class="content">
              <p id="s4s4p1">Cells were maintained in DME with 10% FBS. The effects of the ACAPs on platelet-derived growth factor (PDGF)–induced ruffling were measured using NIH 3T3 fibroblasts. The cells were transiently transfected with the plasmids directing expression of FLAG-tagged ACAPs using Fugene 6 (Roche). 24 h after transfection, the cells were removed from the culture plates using trypsin/EDTA and replated on fibronectin-coated coverslips (Randazzo et al. 2000a) in serum free OptiMEM (GIBCO BRL). Cells were incubated 5–7 h in serum-free medium followed by a 4–5-min treatment with 10 ng/ml PDGF (recombinant BB; GIBCO BRL) before fixation. Transfected cells were identified by immunostaining for the FLAG tag. Actin was visualized using rhodamine–conjugated phalloidin (Molecular Probes). Arf6-dependent changes in the actin cytoskeleton were measured in HeLa cells as described previously (Radhakrishna et al. 1996).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s5h1">Immunofluorescence</h3>
            <div class="content">
              <p id="s4s5p1">Cells were prepared for immunofluorescence as described previously (Radhakrishna et al. 1996). PDGF-induced ruffling experiments were analyzed by epifluorescence microscopy. Arf6-dependent effects were visualized by confocal microscopy using a ZEISS LSM 510 confocal microscope with a Planapo 63×/1.3 NA objective. Adobe Photoshop® software was used to prepare figures from the digitized images obtained with the confocal microscope.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s6h1">Sequence Analysis</h3>
            <div class="content">
              <p id="s4s6p1">Pleckstrin homology (PH)–, Arf GAP-, and ankyrin (ANK)-repeat domains were identified with the program Pfam run at the Sanger Center and with ProfileScan run on the ISREC server. As described previously, the predicted ANK repeats agree, to a large extent, with the published structure for the Arf GAP- and ANK-repeat domains of PAP (Mandiyan et al. 1999). Coiled coils were identified with the program COILS (Lupas et al. 1991), which was accessed through the ExPASy Molecular Biology Server of the Swiss Institute for Bioinformatics and run with the MTIDK matrix. A 28-residue window was used to detect potential coiled-coil domains and a 21-residue window was used to identify the ends. A probability of 0.6 with less than a 25% change with weighting was used as a cutoff. ACAP1 and ACAP2 sequences were aligned using the GAP module within a GCG run on the National Center for Biotechnology Information (NCBI) server. Multiple alignments were performed by ClustalW on the NCBI server.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s7h1">Antibodies</h3>
            <div class="content">
              <p id="s4s7p1">mAb M5 to the FLAG epitope was from Sigma-Aldrich. Fluorescein isothiocyanate (FITC)-donkey anti–mouse, FITC-conjugated donkey anti–rabbit, and rhodamine–conjugated goat anti–mouse antibodies were from Jackson ImmunoResearch. Alkaline phosphatase–conjugated donkey anti–rabbit and alkaline phosphatase–conjugated donkey anti–mouse antibodies were from Bio-Rad Laboratories. Rabbit antiserum to the COOH-terminal peptide of Arf6 is described in Song et al. 1998. Rabbit antiserum 642 against ASAP1 is described in Randazzo et al. 2000a. Rabbit antisera J150 and J151 against residues 722–740 and 1–14 of ACAP1 conjugated via an additional NH2-terminal cysteine, or a COOH-terminal tyrosine residue to PPDT (purified protein derivative of tuberculin) and rabbit antiserum J149 against residues 761–778 of ACAP2 conjugated to an additional NH2-terminal cysteine residue to PPDT were raised according to a standard booster immunization protocol (Hammonds-Odie et al. 1996).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s8h1">Miscellaneous</h3>
            <div class="content">
              <p id="s4s8p1">GAP assays were performed as described previously (Randazzo 1997a,Randazzo 1997b). Lipids were from Sigma-Aldrich and were used as mixed micelles with Triton X-100 in GAP assays. Western blotting was performed using enhanced chemiluminescence for visualization.</p>
            </div>
          </section>
        </div>
      </section>
      <section data-type="">
        <h2 id="s5h1">Results</h2>
        <div class="subsections">
          <section data-type="">
            <h3 id="s5s1h1">The ACAPs Are Members of a Highly Conserved Protein Family in Multicellular Organisms</h3>
            <div class="content">
              <p id="s5s1p1">To identify additional phosphoinositide-dependent Arf GAPs, we searched existing cDNA databases for predicted proteins with a similarity to ASAP1 and PAP. Two cDNAs were identified: KIAA0050 (sequence data available from EMBL/GenBank/DDBJ accession no. D30758), which was a full-length clone, and KIAA0041 (accession no. D26069), which lacked an initiating methionine. Further screening of a lymphocyte library led to the identification of the full-length cDNA (accession no. CAB41450) for KIAA0041. The predicted proteins are similar to ASAP1 and PAP in the amino acid sequence of the Arf GAP domain (Brown et al. 1998) and in the relative position of four domains, containing, in order from the NH2 terminus: coiled coil, PH, Arf GAP, and ANK-repeat domains. The expressed proteins contained phosphoinositide-dependent GAP activity (see below). We have called KIAA0050, ACAP1, and CAB41450, ACAP2, for Arf GAP with coiled coil, ANK repeat and PH domains. Based on structural and biochemical considerations, we propose that ASAP1, PAP, and the ACAPs comprise a family. (These proteins have previously been called centaurins [Randazzo et al. 2000b] because of sequence similarity to centaurin α. However, whereas centaurin α binds PIP3 and lacks GAP activity, ACAP1 and ACAP2 are preferentially activated by phosphatidylinositol 4,5-bisphosphate [PIP2; not shown] and do contain GAP activity. Therefore, to avoid confusion, we have chosen “ACAP” as a name that reflects both the enzymatic activity and structure of the proteins.)</p>
              <p id="s5s1p2">The ACAPs have various features that distinguish them from ASAP1 and PAP. The amino termini of ACAP1 and ACAP2 are similar to that of oligophrenin-1 (23% identity for both, 37% similarity with ACAP2) (Fig. 1) and contain coiled-coil domains within the 45 residues in the NH2-terminal. In contrast, although the first 330 amino acids of PAP and ASAP1 are 70% identical to each other (Brown et al. 1998), they are &lt;25% identical to ACAP1 and ACAP2 and the predicted coiled-coil domains are on the NH2 terminus of the PH domains. Similarly, the PH domains of these four proteins fall into two distinct groups, with &gt;70% identity between ACAP1 and ACAP2 and between PAP and ASAP1, but &lt;20% identity when comparing ACAP1 or 2 with either PAP or ASAP1 (Fig. 1). Furthermore, as noted previously (Brown et al. 1998; Andreev et al. 1999), PAP and ASAP1 contain proline-rich domains that interact with SH3 domains of Src and Crk, and SH3 domains that interact with FAK family members. ACAP1 and ACAP2 lack these domains. The ACAPs each have an additional recognizable sequence motif. ACAP1 has homology to the PI-PLC X-box at residues 76–142. ACAP2 contains a second coiled-coil domain that correlates to the position within the protein to the coiled-coil domains found in PAP and ASAP1.</p>
              <p id="s5s1p3">ASAP/ACAP family proteins occur in multicellular organisms. ACAP orthologs are found in Caenorhabditis elegans (sequence data available from EMBL/GenBank/DDBJ accession no. AJ132698 and AJ132699), Drosophila melanogaster (accession no. AA803046 and AE003742), and Arabidopsis thaliana (GCP1, accession no. AJ130878). A PAP/ASAP1 ortholog is found in D. melanogaster (accession no. AC008370). No orthologs of ASAP/ACAP family proteins were found in Saccharomyces cerevisiae. Among mammals, the proteins are highly conserved. For instance, mouse expressed sequence tags (ESTs) have been identified that are 95% identical over the translated region to human ACAP1 and ACAP2. Human and mouse PAP are 95% identical (Andreev et al. 1999), as are mouse (Brown et al. 1998; accession no. AF075461), human (accession no. AB033075 and KIAA1249), and bovine (King et al. 1999; accession no. AF112886) ASAP1.</p>
              <p id="s5s1p4">ACAPs are widely expressed. PAP and ASAP1 have been shown previously to be ubiquitously expressed (Brown et al. 1998; Andreev et al. 1999). Message for ACAP1 and ACAP2 was detected by PCR using cDNAs prepared from different tissues as templates, and primers specific for ACAP1 and ACAP2 were designed to yield 500-bp products (Fig. 2 A). The relative abundance of message for ACAP1 varied. Of the tissues examined, the highest levels were found in spleen and lung. Message was also detected in heart, kidney, liver, and pancreas. Little or no signal was detected in testis or brain. Two additional PCR products ∼50 and 100 bp larger than the major product were observed in spleen, lung, and pancreas. A search of the EST data base also revealed that ACAP1 was expressed in multiple tissues (Fig. 2 B). The ACAP2 message was at similar levels in all tissues examined and was found in ESTs from multiple tissues (Fig. 2). Consistent with the distribution of message, ACAP2 and ASAP1 protein, detected by Western blotting, were found in all cell lines examined, and ACAP1 was detected in most cells examined (Fig. 2 C). The three proteins occurred together in CHO (not shown), HeLa, Jurkat, C1R, JY, Daudi, B, HEK 293, and NIH 3T3 cells. Monocytes contained ACAP2 and ASAP1, with no detectable ACAP1.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s2h1">ACAPs Are Arf6 GAPs Dependent on PIP2 and Phosphotidic Acid for Activity</h3>
            <div class="content">
              <p id="s5s2p1">GAP activity of epitope-tagged ACAP1, ACAP2, and ASAP1 was determined using Arf1, 5, and 6, representatives from each class of Arf, as substrate (Fig. 3). ACAP1 used Arf6 about twice as efficiently as Arf1 and Arf5. ACAP2 used Arf6 about fivefold better than Arf1 and Arf5. In contrast, ASAP1 used Arf1 and Arf5 10–100-fold more efficiently than Arf6 (Fig. 3) (Brown et al. 1998). The GAP activity reported in Fig. 3 is normalized to the amount of Arf GAP protein, quantified through Western blotting, to the epitope tag each of these purified proteins contained. ASAP1 had a much higher specific activity than either ACAP1 or 2 in these in vitro assays (note the scale on GAP activity).</p>
              <p id="s5s2p2">ACAP GAP activity was dependent on phospholipids (Fig. 4). In previous studies, the GAP activity of ASAP1 was found to be stimulated by the coordinate binding of PIP2 and phosphatidic acid (PA) (Randazzo 1997b). ACAP1 and ACAP2 showed similar phospholipid dependencies when Arf6 was used as a substrate. In both cases, half-maximal activity was achieved with 10–20 μM PIP2 in the presence of 380 μM PA. PAP is also comodulated by PIP2 and PA (not shown). PA/PIP2-dependent GAP activity appears to be a defining feature of the ASAP/ACAP family of proteins.</p>
              <p id="s5s2p3">A highly conserved arginine is critical for Arf GAP activity. All proteins known to have Arf GAP activity contain an arginine five residues on the COOH-terminal from the fourth cysteine constituting the zinc finger. ASAP1, PAP, and Arf GAP1, containing a mutation in this residue, have little or no detectable GAP activity (Mandiyan et al. 1999; Randazzo et al. 2000a; Szafer et al. 2000). ACAPs were similar. [R448Q]ACAP1 and [R442Q]ACAP2 had no detectable activity using Arf6 as a substrate (Fig. 5).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s3h1">ACAPs Affect the Actin Cytoskeleton</h3>
            <div class="content">
              <p id="s5s3p1">NIH 3T3 fibroblasts treated with PDGF form actin-rich dorsal ruffles. Examples of these unique structures are indicated with arrowheads in Fig. 6. ASAP1 localizes to actin-rich dorsal ruffles and overexpression of ASAP1 inhibits the formation of these ruffles (Randazzo et al. 2000a). ACAPs were similar to ASAP1. Overexpression of either ACAP inhibited the formation of PDGF-induced dorsal ruffles (Fig. 6 and Fig. 7, first and third rows), and the ACAPs were recruited to those ruffles that did form (not shown). The effects of ACAPs on ruffling were partly dependent on GAP activity. GAP-deficient ACAP1 and ACAP2 had less of an effect than wild-type protein, but still decreased the rate of ruffling compared with the empty vector control (Fig. 7). In contrast, expression of a GAP-deficient ASAP1 enhanced ruffle formation (Fig. 7). The mutant ACAPs, like mutant ASAP1, were efficiently recruited to the actin-rich ruffles (Fig. 6, rows 2 and 4, arrowheads). Like ASAP1, ACAPs specifically affected Arf function in the cell periphery. Neither ACAP1, ACAP2, nor ASAP1 (Brown et al. 1998) caused changes in the morphology of the Golgi apparatus (not shown).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s4h1">ACAPs Function as Arf6 GAPs In Vivo</h3>
            <div class="content">
              <p id="s5s4p1">Given the localization and effects of the ACAPs, Arf6 seemed a likely substrate for these Arf GAPs. Arf6 functions in the cell periphery and has well-documented effects on the actin cytoskeleton. Because HeLa cells have been used as a model system for a large part of this work on Arf6 (Radhakrishna et al. 1996; Frank et al. 1998; Song et al. 1998), we used HeLa cells to test for an effect of the ACAPs on Arf6 function in vivo (Fig. 8 and Fig. 9). First, we examined whether overexpression of an ACAP would affect the formation of actin-rich membrane protrusions. These structures are observed acutely in cells overexpressing Arf6 upon treatment with tetrafluoroaluminate (AlF4), as can be seen in Fig. 8 (top right, arrowhead). Coexpression of Arf6 with either ACAP inhibited the formation of protrusions in response to AlF4 treatment (Fig. 8, second and third rows, right). We quantified the proportion of cells making protrusions in each case compared with those formed in cells expressing Arf6 alone. As shown in Fig. 9, ACAP1 was most effective at inhibiting protrusions. ACAP2 reduced the number of cells generating protrusions by 50%. The effects of ACAP1 and ACAP2 were dependent on GAP activity. Point mutants of the ACAPs that lacked GAP activity had much less of an effect than did the wild-type ACAPs (Fig. 9, and see Fig. 11, third row). Although ASAP1 was recruited to the cell edge and membrane protrusions, colocalizing with Arf6, it had little or no effect on protrusions formed in the presence of Arf6 and AlF4 (Fig. 8, bottom row, and Fig. 9).</p>
              <p id="s5s4p2">We also assessed the localization of the ACAPs with respect to Arf6 in cotransfected cells. In untreated cells, although ACAP1 and ACAP2 were primarily cytosolic, they could be observed on tubular portions of the Arf6 endosomal compartment (Fig. 8, arrows), and also at the plasma membrane on small protrusive buds where they colocalized with Arf6 (Fig. 8). With AlF4 treatment, recruitment of ACAPs to this tubular endosome was greatly enhanced, and Arf6 could also be observed on these tubes that persisted during AlF4 treatment (Fig. 8, arrows). These endosomal tubes were often oriented towards regions along the edges of the cells where protrusions would ordinarily form (Fig. 8, third row, two right images, arrowheads). Recruitment to this portion of the Arf6 endosome was not dependent upon ACAP overexpression, since endogenous ACAP2 was recruited into protrusions and small tubular structures feeding into protrusions induced by AlF4 in cells overexpressing Arf6-HA (Fig. 10, arrowhead and arrow). The recruitment of ACAP1 and ACAP2 was dependent on the presence of Arf6-GTP. ACAP1 remained cytosolic and was not recruited to the endosome or the plasma membrane in cells expressing the dominant-negative Arf6, T27N, even in the presence of AlF4 (Fig. 11, top row). In cells expressing the constitutively active mutant of Arf6, Q67L, the ACAP1 (Fig. 11, middle row) could be observed localized to the plasma in the absence of AlF4 treatment. A point mutant of ACAP1 that lacked GAP activity was not observed on the tubular endosomes, but rather was associated with the plasma membrane protrusions (Fig. 11, bottom row). Similar results were observed with ACAP2 (not shown). Taken together, these data are consistent with ACAP1 and ACAP2 functioning as Arf6 GAPs.</p>
            </div>
          </section>
        </div>
      </section>
      <section data-type="">
        <h2 id="s6h1">Discussion</h2>
        <div class="content">
          <p id="s6p1">We have identified two Arf GAPs, ACAP1 and ACAP2, that are structurally and biochemically similar to two previously described Arf GAPs, ASAP1 and PAP, and propose that the four proteins constitute the ASAP/ACAP family of Arf GAPs. Like ASAP1, the ACAPs functioned in the cell periphery affecting the actin cytoskeleton. In contrast with ASAP1, ACAPs specifically affected Arf6-dependent events. These results support the hypothesis that Arf GAPs can discriminate between Arfs at a given location. Given that ACAP1 and ACAP2 occur together in a single cell, we propose that the Arf6-specific ACAPs may discriminate between particular Arf6-dependent activities. Site- and Arf specificity would enable Arf GAPs to function together in coordinating a set of cellular events leading to a biological response.</p>
          <p id="s6p2">Data base searches for proteins homologous to ASAP1 led to the identification of the widely expressed ACAPs. The proteins were similar to ASAP1 and PAP in having a coiled-coil domain, followed by PH, Arf GAP, and ANK-repeat domains and had GAP activity that was stimulated by the coordinate action of PA and PIP2. Based on these criteria, we propose that ACAP1, ACAP2, PAP, and ASAP1 constitute a family. The ASAP/ACAP family proteins are found primarily in multicellular organisms and are involved in reorganization of the actin cytoskeleton. As a class of proteins, these Arf GAPs could regulate processes, such as cell migration, adhesion, and cell–cell contact, that are important for development and wound healing.</p>
          <p id="s6p3">The data provided here support the idea that GAPs functioning at the plasma membrane can distinguish between class I/II Arfs and the class III Arf6. During PDGF stimulation of NIH 3T3 cells, ACAP1, ACAP2, and ASAP1 colocalized in dorsal ruffles and at the edge of resting and protrusive HeLa cells. However, these effects were mediated by different Arfs. In vitro ACAP1 and ACAP2 used Arf6, relative to Arf1 and Arf5, better than did ASAP1. In vivo, both ACAPs affected Arf6-dependent cytoskeleton events in HeLa cells, whereas ASAP1 had no effect even though it colocalized with Arf6 at the plasma membrane and in protrusions. Thus, in cells, ACAPs are Arf6-specific GAPs, whereas ASAP1 affects either a class I or II Arf. Arf1 could be the class I Arf that ASAP is affecting, since Arf1 has been implicated in focal adhesion formation. Taken together, these results suggest that Arf6 may cooperate with a second Arf to regulate actin cytoskeleton reorganization.</p>
          <p id="s6p4">ACAP1 and ACAP2 both affected Arf6-dependent processes, yet were present together in a single cell. Both ACAPs were recruited to and stabilized membrane tubules. These peripheral tubules in HeLa cells represent the site where Arf6 is activated to Arf6-GTP, a prerequisite for membrane recycling back to the plasma membrane (Radhakrishna and Donaldson 1997). Once at the plasma membrane, accumulation of Arf6-GTP induces the formation of protrusions (Radhakrishna et al. 1996). Here, ACAP1 was more efficient at blocking protrusions than was ACAP2. Conversely, ACAP2 was more efficient at inhibiting cell spreading in NIH 3T3 fibroblasts (Nie, Z., and Randazzo, P.A., unpublished). ACAP1 and ACAP2 may function as a complementary pair, similar to GCS1 and GLO3 in yeast (Poon et al. 1999), that are able to discriminate between particular Arf6-dependent events. The reason for having two Arf6 GAPs present in a single cell may be that their functions are not strictly equivalent. With the proper assays, we may be able to identify which GAPs are specific for distinct Arf6-mediated events, such as adipsin secretion (Millar et al. 1999), GLUT4 trafficking (Yang and Mueckler 1999), phagocytosis (Zhang et al. 1998), rac trafficking (Radhakrishna et al. 1999), and protrusion formation (Radhakrishna et al. 1996).</p>
          <p id="s6p5">Regulation of the ASAP/ACAP family of Arf GAPs is likely to be multifaceted to allow discrimination and coordination of events within a pathway. Preliminary analysis of phospholipid activation indicates that all ASAP/ACAP family Arf GAPs are dependent on regulation by PIP2 and PA. However, the PH domains of the ACAPs do have significant similarity to the phosphatidylinositol 3,4,5-triphosphate (PI3,4,5P3)–binding PH domain of centaurin α (Blader et al. 1999) (21% identity), with conservation of residues in the NH2-terminal, a potential lipid-binding site (32% identity over residues 269–305 of ACAP1). Therefore, a more thorough comparison of the effects of inositide isomers on ACAPs is needed, particularly for PI3,4P2, PI3,5P2, and PI3P, and is the subject of additional studies (Jackson, T.R., manuscript in preparation). Other signaling pathways could also impinge on the ASAP/ACAP family of Arf GAPs. The possible role of phosphatidic acid has been previously discussed (Liscovitch and Cantley 1995; Randazzo 1997a,Randazzo 1997b; Toker 1998; Kam et al. 2000). Yet to be examined are protein interactions mediated by the proline-rich and SH3-domains of PAP and ASAP1 and the ANK repeats of all members of this family. The crystal structure of a fragment of PAP has revealed that the ANK repeats physically associate with the Arf GAP domain (Mandiyan et al. 1999), and, therefore, could influence the activity of this domain. Since ANK repeats occur in all the ASAP/ACAP family Arf GAPs, as well as in the GIT/CAT family of Arf GAPs, this domain could provide a common mechanism to regulate GAP activity. A combination of specificity of protein interactions and differences in phosphoinositide specificity in activating the GAP could provide independent regulation of each Arf GAP.</p>
          <p id="s6p6">Several molecular mechanisms by which Arf GAPs may affect the actin cytoskeleton have been or are being considered. The ASAP/ACAP Arf GAPs are multidomain proteins that could have one or more roles. One possibility is that the GAPs are negative regulators through which multiple signals converge to control Arf-GTP levels. In this way, the effects of Arf could be coordinated with the effects of other proteins that mediate changes in the actin cytoskeleton, such as the Rho family proteins. An alternative model is that, in addition to the role as a negative regulator of Arf, the Arf GAPs are scaffolds, bringing proteins with direct effects on the cytoskeleton together. If any of the protein interactions were affected by Arf-GTP binding, the GAP could have an active role, functioning as an Arf effector. There is also evidence that Arf GAPs can directly interact with actin, affecting depolymerization (Blader et al. 1999). The similarity of the ACAPs to oligophrenin-1, a Rho GAP identified by genetic analysis of X-linked mental retardation (Billuart et al. 1998), raises the additional possibility that Arf and Rho family proteins can independently affect a common pathway.</p>
          <p id="s6p7">Our data suggest that activated Arf6 can recruit ACAP1 and ACAP2 to membranes. We found ACAPs colocalized with the constitutively active form [Q67L]Arf6, but not with a dominant-negative [T27N]Arf6. The effects of AlF4 on the localization of the ACAPs to tubular endosomes are also consistent with recruitment by Arf-GTP, since previous studies have demonstrated that Arf6 must be converted to Arf6-GTP on the endosome to allow membrane recycling back to the plasma membrane (PM) (Radhakrishna and Donaldson 1997).</p>
          <p id="s6p8">The striking recruitment of ACAPS to the Arf6 tubular endosome upon AlF4 treatment raises the possibility that AlF4 may be stabilizing an Arf6–AlF4–GAP complex, analogous to those found for Ras and its GAPs in vitro (Mittal et al. 1996). In such a complex, AlF4 would occupy the position normally taken by the γ-phosphoryl group of GTP in the transition state of the GTPase reaction. In other words, the complex of the GDP bound form of Arf, AlF4 and GAP, mimics the complex of Arf-GTP with GAP. If we assume that AlF4 can form such a complex in cells, then, in AlF4- treated cells overexpressing Arf6, endogenous GAP would be locked in an unproductive complex and the excess Arf6-GTP would be able to interact with effectors, generating protrusions. Conversely, coexpression of GAP would sequester this excess Arf6 in a transition state complex, blocking protrusion formation. GAP mutants lacking the catalytic arginine are unable to form this transition state, and, therefore, AlF4 treatment would be predicted to fail to form such a complex. Consistent with this, AlF4 treatment of cells overexpressing Arf6 and mutant ACAPs did not result in recruitment of GAPs to tubules. Whether or not this AlF4 complex formation is the mechanism for “activation” of Arf6 observed in our cells will need to be tested. Interestingly, for Arf1, we have detected an Arf1–GDP–AlF4–ASAP1 complex in vitro (Randazzo, P.A., Miura, K., and Nie, Z., manuscript in preparation)</p>
          <p id="s6p9">Although we have examined the role of ACAPs in the cell periphery, other Arf GAPs function here as well. GIT1 was recently shown to use Arf6 as a substrate (Vitale et al. 2000); Premont et al. 1998 showed that overexpression of GIT1 inhibited trafficking of the ligand activated β-adrenergic receptor into a clathrin-dependent internalization pathway. Additionally, both ASAP1 and the putative Arf GAP p95Pkl have been found associated with focal adhesions (Turner et al. 1999; Randazzo et al. 2000a). ASAP1, Pkl, and PAP have been found to bind paxillin (Turner et al. 1999; Kondo et al. 2000). ASAP1, PAP, and GIT1 have been implicated in the regulation of cell movement (Bagrodia et al. 1999; Kondo et al. 2000; Randazzo et al. 2000a). Understanding how these different Arf GAPs could function together to coordinate complex cellular activities will be a key area of future research.</p>
        </div>
      </section>
    </article>
  </body>
</html>
