<!DOCTYPE html >
<html id="aFkRM47cVingPhonG59o8iIPhNQa-PMC3416829" data-origid="PMC3416829" class="anndoc" data-anndoc-version="2.0" lang="" xml:lang="" xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="UTF-8"/>
    <meta name="generator" content="com.jmcejuela.anndoc.v2.bio.NcbiJournalArticleParser_v0_4$"/>
    <meta name="dcterms.source" content="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3416829/"/>
    <title>aFkRM47cVingPhonG59o8iIPhNQa-PMC3416829</title>
  </head>
  <body>
    <article>
      <section data-type="title">
        <h2 id="s1h1">Prevalence and Clinical Features of Hearing Loss Patients with CDH23 Mutations: A Large Cohort Study</h2>
      </section>
      <section data-type="abstract">
        <h2 id="s2h1">Abstract</h2>
        <div class="content">
          <p id="s2p1">Screening for gene mutations in CDH23, which has many exons, has lagged even though it is likely to be an important cause for hearing loss patients. To assess the importance of CDH23 mutations in non-syndromic hearing loss, two-step screening was applied and clinical characteristics of the patients with CDH23 mutations were examined in this study. As a first screening, we performed Sanger sequencing using 304 probands compatible with recessive inheritance to find the pathologic mutations. Twenty-six possible mutations were detected to be pathologic in the first screening. For the second screening, using the probes for these 26 mutations, a large cohort of probands (n = 1396) was screened using Taqman amplification-based mutation analysis followed by Sanger sequencing. The hearing loss in a total of 52 families (10 homozygous, 13 compound heterogygous, and 29 heterozygous) was found to be caused by the CDH23 mutations. The majority of the patients showed congenital, high frequency involved, progressive hearing loss. Interestingly, some particular mutations cause late onset moderate hearing loss. The present study is the first to demonstrate the prevalence of CDH23 mutations among non-syndromic hearing loss patients and indicated that mutations of the CDH23 gene are an important cause of non-syndromic hearing loss.</p>
        </div>
      </section>
      <section data-origid="s1" data-type="">
        <h2 id="s3h1">Introduction</h2>
        <div class="content">
          <p id="s3p1">Mutations in the CDH23 (NM_22124) gene are known to be responsible for both Usher syndrome type ID (USH1D) and non-syndromic hearing loss (DFNB12) [1], [2]. Molecular confirmation of CDH23 mutations has become important in the diagnosis of these conditions.</p>
          <p id="s3p2">This gene encodes cadherin 23, a protein of 3354 amino acids with 27 extracellular (EC) domains, a single transmembrane domain and a short cytoplasmic domain. Cadherin-specific amino acid motifs such as DRE, DXNDN, and DXD, that are highly conserved in sequence and spacing and required for cadherin dimerization and calcium binding were found in each extracelluar domain [3].</p>
          <p id="s3p3">The cadherin 23 protein is known to be an important composition of the tip link that maintains the arrangement of streocilia [4].</p>
          <p id="s3p4">More than 50 mutations have been reported for the Usher phenotype (USH1D) and 24 mutations reported for the non-syndromic hearing loss phenotype (DFNB12) [1], [2], [5]–[7]. As suggested by genotype–phenotype correlation study, Usher 1D, which has congenital profound hearing impairment, vestibular dysfunction, and retinitis pigmentosa, is usually associated with nonsense mutations, whereas DFNB12, which has a milder phenotype, is associated with missense mutations [1], [2], [5]–[8].</p>
          <p id="s3p5">We previously reported that four pathologic mutations were identified in 5 out of 64 Japanese families compatible with autosomal recessive inheritance, suggesting that CDH23-caused deafness may be commonly found among non-syndromic hearing loss patients [6]. GJB2 has been shown to be a common gene involved in congenital hearing impairment. SLC26A4 is also frequently involved among those patients. GJB2 and SLC26A4 are comparatively small making Sanger sequencing relatively easy. The latter is also associated with the typical inner ear anomaly, enlarged vestibular aqueduct. Therefore, screening is relatively easy and many studies have focused on just these two genes. Clinical molecular diagnosis has been dramatically improved for these genes. However, screening strategy of other hearing loss genes is difficult and Sanger sequencing of the candidate genes, such as CDH23, with many exons is time consuming. Consequently, only a few reports are available for the mutation spectrum of CDH23.</p>
          <p id="s3p6">In the present study, we performed Sanger sequencing using 304 patients whose pedigrees are compatible with recessive inheritance to find additional pathologic mutations. Also, to find the novel pathologic mutations and to clarify the frequency and clinical characteristics of patients with CDH23 mutations, a large cohort of probands from unrelated families (n = 1396) was screened using TaqMan amplification-based mutation analysis of the variants observed in the initial 304 patients.</p>
        </div>
      </section>
      <section data-origid="s2" data-type="">
        <h2 id="s4h1">Results</h2>
        <div class="content">
          <p id="s4p1">The first screening using 304 Japanese probands compatible with autosomal recessive inheritance identified 26 candidates for disease causing mutations. These include four previously reported pathologic mutations: p.P240L, p.R301Q, p.Q1716P, and p.R2029W, as well as 6 possible pathologic variants in the coding region of CDH23. All of the mutations were missense mutations. The following second screening based on TaqMan assay followed by Sanger sequencing confirmed 10 “possibly pathologic” mutations (Table 1) and 17 variants with uncertain pathogenicity (Table 2) in a large cohort of the patients. “Possible pathologic” mutations were defined as 1) mutations found to be homozygotes or compound heterozygotes (and determined by segregation study), 2) variants which were not found or were very few in 192 control subjects, 3) amino acids that were well-conserved among various species, 4) compatible with next generation sequencing database, and 5) compatible with the predicted effect of missense mutations on CDH23 protein function. Results of the compatibility of the next generation sequence database, the SIFT and PolyPhen2 score for prediction are shown in Tables 1 and 2.</p>
          <figure class="table" data-origid="pone-0040366-t001">
            <a href="http://doi.org/10.1371/journal.pone.0040366.t001">Table 1</a>
            <figcaption>
              <h5 id="s4f1h1">Possible pathologic variants found in this study.</h5>
            </figcaption>
          </figure>
          <figure class="table" data-origid="pone-0040366-t002">
            <a href="http://doi.org/10.1371/journal.pone.0040366.t002">Table 2</a>
            <figcaption>
              <h5 id="s4f2h1">Variants with uncertain pathogenicity found in this study.</h5>
            </figcaption>
          </figure>
          <p id="s4p2">The 17 variants found as heterozygous and therefore with uncertain pathogenicity did not fulfill all the above criteria. For example, p.A1443G was uncertain because DNA samples from family members were not available and we could not confirm its pathogenicity by segregation study. p.R1588W was found to be homozygous in 4 patients and heterozygous in 16 patients, but only 1 was found in 384 control alleles. However, a member of the patient's family (#2841) showed normal hearing instead of being homozygous. Also p.V803I, p.V1807M and p.I2669V are obscure from the functional prediction analysis.</p>
          <p id="s4p3">In one family (#4685), three mutation were found in proband and two of them were found in same allele p.[D16126A;V1807M] confirmed by segregation analysis.</p>
          <p id="s4p4">As p.V1807M predicted to have no effect on CDH23 structure, p.D1626A might be a pathogenic mutation.</p>
          <p id="s4p5">For 10 possible pathologic mutations, amino acids were well-conserved among various species, including Homo sapiens, P. troglodytes, B. traurus, M. musculus, R. norvegicus, G. gallus, and D. rario. Many mutations (5 out of 10 possible pathologic mutations, 2 out of 17 uncertain variants) were found in DRE, DXNDN, and DXD motif (Table 1 and 2). Ten possible pathologic mutations were found to be either homozygotes (n = 11, Table 3, Fig. 1) or compound heterozygotes (n = 15) (Table 4, Fig. 2). Twenty-nine patients were found to be heterozygous without a second mutation (Table 5).</p>
          <figure data-origid="pone-0040366-g001">
            <a href="http://doi.org/10.1371/journal.pone.0040366.g001">Figure 1</a>
            <figcaption>
              <h5 id="s4f3h1">Pedigrees, mutations, and audiograms of the patients with homozygous CDH23 mutations.</h5>
            </figcaption>
          </figure>
          <figure data-origid="pone-0040366-g002">
            <a href="http://doi.org/10.1371/journal.pone.0040366.g002">Figure 2</a>
            <figcaption>
              <h5 id="s4f4h1">Pedigrees, mutations, and audiograms of the patients with compound heterozygous CDH23 mutations.</h5>
            </figcaption>
          </figure>
          <figure class="table" data-origid="pone-0040366-t003">
            <a href="http://doi.org/10.1371/journal.pone.0040366.t003">Table 3</a>
            <figcaption>
              <h5 id="s4f5h1">Details of phenotype and genotype of 11 patients in 10 families with homozygous CDH23 mutation.</h5>
            </figcaption>
          </figure>
          <figure class="table" data-origid="pone-0040366-t004">
            <a href="http://doi.org/10.1371/journal.pone.0040366.t004">Table 4</a>
            <figcaption>
              <h5 id="s4f6h1">Details of phenotype and genotype of 15 patients in 13 families with compound heterozygous CDH23 mutation.</h5>
            </figcaption>
          </figure>
          <figure class="table" data-origid="pone-0040366-t005">
            <a href="http://doi.org/10.1371/journal.pone.0040366.t005">Table 5</a>
            <figcaption>
              <h5 id="s4f7h1">Details of phenotype and genotype of 29 patients with heterozygous CDH23 mutation.</h5>
            </figcaption>
          </figure>
          <p id="s4p6">Tables 3 and 4 summarize 23 families with hearing loss caused by the CDH23 mutations (homozygous or compound heterozygous cases) and Table 5 summarizes 29 families with hearing loss potentially caused by the CDH23 mutations (heterozygous cases). The frequency was 1.6% (23/1396) or 2.1% (29/1396) of the overall hearing loss population. When restricted to patients compatible with recessive inheritance, the frequency was increased to 2.5% (23/919) or 3.2% (29/919). Table 3, 4 and 5 also summarize clinical characteristics including hereditary form, hearing threshold, severity, residual hearing in the lower frequencies, hearing in the higher frequencies, onset age (age of awareness), progressiveness of hearing loss, use of hearing aid/cochlear implantation, visual impairment, and vestibular symptoms. The ages of these patients were from 1 to 71 years. Age of onset (awareness of hearing loss) ranged from congenital to 60 years old, though the majority was congenital or early onset. There were some correlations between genotype and phenotype (onset age). The patients associated with p.P240L showed congenital and severe hearing loss regardless of whether associated with one more mutation, whereas the patients with p.R2029W or p.T1368M showed late-onset moderate hearing loss (Tables 3 and 4). Concerning type of hearing loss, the majority of the patients had some residual hearing in the lower frequencies, and overlapping audiograms showed characteristic high frequency involved hearing loss (Fig. 3). The majority of the patients showed progressive nature of hearing loss evaluated by serial audiogram (Fig. 4). No patients had associated visual impairment or vestibular symptoms (Tables 3, 4 and 5). Seven patients received cochlear implantation due to the insufficient amplification of hearing aids (Tables 3, 4 and 5).</p>
          <figure data-origid="pone-0040366-g003">
            <a href="http://doi.org/10.1371/journal.pone.0040366.g003">Figure 3</a>
            <figcaption>
              <h5 id="s4f8h1">Overlapping audiograms of the patients with CDH 23 mutations.</h5>
              <p id="s4f8p1">A: patients with hearing loss caused by the CDH23 mutations (homozygous or compound heterozygous cases), B: patients potentially caused by the CDH23 mutations (heterozygous cases).</p>
            </figcaption>
          </figure>
          <figure data-origid="pone-0040366-g004">
            <a href="http://doi.org/10.1371/journal.pone.0040366.g004">Figure 4</a>
            <figcaption>
              <h5 id="s4f9h1">Hearing progression of the patients with CDH23 mutations.</h5>
              <p id="s4f9p1">Note that the high frequency portion was already worsened, and the low frequency portion was deteriorated by ages.</p>
            </figcaption>
          </figure>
        </div>
      </section>
      <section data-origid="s3" data-type="">
        <h2 id="s5h1">Discussion</h2>
        <div class="content">
          <p id="s5p1">Mutations in the CDH23 gene are known to be responsible for both Usher syndrome type ID (USH1D) as well as non-syndromic hearing loss (DFNB12), and molecular confirmation of CDH23 mutations is clinically important for diagnosis of these conditions. However, clinical application of the detection of CDH23 mutations has lagged because of the size of the gene. Especially for DFNB12, which is not associated with visual impairment, screening is comparatively difficult, and therefore, little is known about frequencies among the hearing loss population as well as clinical characteristics.</p>
          <p id="s5p2">In this study, we have applied two-step screening and identified a significant number of novel pathologic mutations of CDH23 responsible for non-syndromic hearing loss in a large cohort of patients. All of the possible pathologic mutations identified in this study (Table 1) were missense mutations, being consistent with previous reports that DFNB12 patients associated with missense mutations have milder hearing impairment than in USH1D, which is associated with nonsense, splice-site, or frameshift mutations [2], [5]–[7]. None had visual impairment, also supporting this rule. That the majority was found in the EC domain with only one exception found in the cytoplasmic domain, was also in line with the previous reports on DFNB12 [2], [5]–[7]. Of these 26 mutations, five out of 10 possible pathologic mutations were found in DRE, DXNDN, and DXD motifs, which are thought to be important for calcium binding property. These highly conserved EC calcium binding motifs are thought to be essential for linearization, rigidification, and dimerization of the cadherin molecules [9], [10]. And the results of computer analysis to predict the impact of amino acid change, all of 10 possible pathologic mutations predicted to cause a severe damage for protein function of CDH23.</p>
          <p id="s5p3">As a result, 26 patients (from 23 families) had two mutations (in a homozygous or compound heterozygous state), and met criteria for recessive inheritance. A hallmark of recessive mutations is the detection of two mutations in the paternal and maternal alleles and the parents having normal hearing. As seen in previous mutation screening reports, including those for CDH23 [6], [7] as well as GJB2 and SLC26A4 [11], [12], we encountered a significant number of heterozygous cases without a second mutation even after direct sequencing of the coding region of the gene. Possible explanations are: 1) the existence of a second mutation in the intron or regulatory region of CDH23, which has not been explored, 2) the observed mutations are rare polymorphisms, 3) the screening method fails to detect the second mutation, and 4) an additional modulatory gene may contribute to hearing loss (for example, PCDH15). Although we have not reached the final conclusion, it is most likely that these heterozygous cases are also related to CDH23 mutations because: 1) allele frequencies are found to be higher in the hearing loss group (Table 2), and 2) the phenotype is similar to that of the patients with two mutations. As shown in Fig. 3, overlapping audiograms of the patients with only one mutation was similar to that with the patients with two mutations (high frequency involved sensorineural hearing loss with residual hearing at the lower frequencies).</p>
          <p id="s5p4">Based on the frequencies of 3.7% (including heterozygous cases) of the hearing loss population and 5.7% (including heterozygous cases) of the recessive inherited cases in this study, we confirmed that mutations of CDH23 are an important cause for non-syndromic hearing loss and should be borne in mind next to GJB2 or SLC26A4 screening. This study revealed that p.P240L account for nearly 43.3%(45/104) of all CDH23 mutated families in Japan. Common mutations, such as c.35delG or c.235delC in GJB2 or p.H723R in the SLC26A4 gene, have been reported in many recessive deafness genes, and usually they are population-specific [12]–[14]. It is an interesting question whether p.P240L is frequent because of a founder effect or mutational hot spot, but the existence of such a common mutation makes mutation screening easier. Additional frequent mutations found in this study together with TaqMan procedures will facilitate genetic testing for deafness patients.</p>
          <p id="s5p5">Concerning mutation spectrum, as in our previous report [6], the CDH23 mutation spectrum in Japanese is very different from that found in Caucasians and may be representative of those in Eastern Asian populations. Its elucidation is expected to facilitate the molecular diagnosis of DFNB12 and USH1D. It has also been known that prevalent GJB2 mutations are highly ethnic-specific (see The connexin-deafness homepage; http://davinci.crg.es/deafness/): c.35delG is common in the Caucasoid population, c.167delT was reported as prevalent in Ashkenazi Jews, p.R143W in a restricted village in Africa, and c.235delC in East Asian populations. A series of studies proved a founder effect for these frequent mutations [11], [15].</p>
          <p id="s5p6">In the present study, using a large cohort of patients, clinical characteristics (onset age, progression, audiograms) of patients with CDH23 mutations were clarified.</p>
          <p id="s5p7">Concerning genotype/phenotype correlations, hearing of the patients with p.[P240L];[P240L] is worse than in those with the other mutations, and tends to be congenital and severe. In contrast, the patients with p.[R2029W];[R2029W] showed a milder phenotype of middle age onset. Overlapping audiograms showed typical high frequency involved sensorineural hearing loss with residual hearing at the lower frequencies.</p>
          <p id="s5p8">Concerning age of onset (awareness of hearing loss), the majority was congenital or early onset. But rather later-onset was seen in three patients (#2806, 3255, 501), and they were associated with some particular mutations (p.R2029W and p.T1368M). Their phenotype was rather mild and gradually progressive. It is interesting to note that their phenotype was similar to presbycusis. Actually, CDH23 mutations have been reported as responsible for age-related hearing loss in mice [16], [17].</p>
          <p id="s5p9">Progressive nature of hearing loss and the presence of residual hearing are particular phenotypic features of the patients with CDH23 mutations. Our previous genetic analysis for the patients with high frequency involved hearing loss successfully identified CDH23 mutations [18]. Seven patients received cochlear implantation and showed good performance after implantation. For the patients with residual hearing, newly developed cochlear implantation; EAS (Electric Acoustic Stimulation) is a good therapeutic option and therefore much attention should be paid to the etiology when considering individual intervention, i.e., regular cochlear implantation or EAS. Genetic testing will be very important prognostic information together with various hearing tests.</p>
          <p id="s5p10">In conclusion, a large cohort study using Taqman amplification-based mutation analysis indicated that mutations of the CDH23 gene are important causes of non-syndromic hearing loss. A mutation screening strategy using TaqMan assay based on the ethnic-specific frequent mutations is a powerful and effective method for such a large gene. Clinical characteristics of patients with CDH23 mutations is that hearing loss is progressive, high frequency involved sensorineural hearing loss with residual hearing in the lower frequencies. Most cases are congenital but care is needed because some patients show presbycusis-like hearing loss. Cochlear implantation (including EAS) is a good therapeutic intervention for the patients with CDH23 mutations.</p>
        </div>
      </section>
      <section data-origid="s4" data-type="materials|methods">
        <h2 id="s6h1">Materials and Methods</h2>
        <div class="content">
          <p id="s6p1">To identify additional pathologic CDH23 mutations, two-step screening was applied in this study. Subjects from independent families were collected from 33 ENT departments nationwide in Japan. All subjects gave prior informed consent for participation in the project, which was approved by the ethical committee of each hospital. Genomic DNA was isolated from peripheral blood by DNeasy Blood and Tissue Kit (QIAGEN, Düsseldorf, Germany) according to the manufacturer's procedure.</p>
        </div>
        <div class="subsections">
          <section data-origid="s4a" data-type="">
            <h3 id="s6s1h1">First screening (Direct sequencing)</h3>
            <div class="content">
              <p id="s6s1p1">First, we sequenced the CDH23 gene in 304 Japanese non-syndromic sensorineural hearing loss probands (including our previously reported 64 samples [6]) compatible with autosomal recessive inheritance or sporadic cases. None of the subjects had any other associated neurological signs, vestibular or visual dysfunction. Sanger sequencing was applied to these samples to find mutations responsible for deafness. Detailed procedures were described in our previous report [6]. 26 candidates for disease causing mutations were collected according to the following criteria; 1) non-synonymous variants, and 2) allele carrier rates were less than 2% in control subjects.</p>
            </div>
          </section>
          <section data-origid="s4b" data-type="">
            <h3 id="s6s2h1">Second screening (TaqMan genotyping assay based screening and Direct sequencing)</h3>
            <div class="content">
              <p id="s6s2p1">For the second screening, probes of these 26 mutations selected in the first screening was applied for a custom TaqMan® SNP Genotyping Assays (Applied Biosystems, Foster City, CA) [19]. 1396 probands of sensorineural hearing loss patients including 304 probands used in the first screening were used for the second assay. Of them, 1347 had bilateral sensorineural hearing loss and 49 had unilateral sensorineural hearing loss. The inheritance composition of the subjects was as follows: 298 subjects from autosomal dominant or maternally inherited families (two or more generations affected); 919 subjects from autosomal recessive families (parents with normal hearing and two or more affected siblings) or subjects with sporadic deafness (compatible with recessive inheritance or non-genetic hearing loss); the rest had unknown inheritance mode. After TaqMan assay, Sanger sequencing was performed: 1) to confirm these mutations found in TaqMan genotyping assays, 2) to confirm whether mutations were homozygotes or heterozygote, and 3) in cases found in heterozygous state, direct sequencing of the coding region of the CDH23 was performed.</p>
            </div>
          </section>
          <section data-origid="s4c" data-type="">
            <h3 id="s6s3h1">Controls</h3>
            <div class="content">
              <p id="s6s3p1">The control group consisted of 192 unrelated Japanese individuals without any noticeable hearing loss evaluated by auditory testing.</p>
            </div>
          </section>
          <section data-origid="s4d" data-type="">
            <h3 id="s6s4h1">Next generation sequencing and computer analysis</h3>
            <div class="content">
              <p id="s6s4p1">To elucidate the allele frequency of 26 mutations, comparison was made between allele frequency found in 216 deafness patients and 72 controls based on a next generation sequencing database that is currently being established at Shinshu University (unpublished). In brief, exome sequencing was performed with SureSelect target DNA enrichment (Agilent Technologies, Santa Clara, CA) and Illumina GAIIx sequencing (Illumina, San Diego, CA) according to the manufacturers' procedures. In the SureSelect library, 76 already reported genes responsible for sensorineural hearing loss and syndromic hearing loss were contained. After base calling, sequence results were aligned with a bowtie program [20] and allele frequencies of each CDH23 mutation in patients and the control population were calculated. Computer analysis to predict the effect of missense variants on CDH23 protein function was performed with Sorting Intolerant from Tolerant (SIFT; http://sift.jcvi.org/), and Polymorphism Phenotyping (PolyPhen2; http://genetics.bwh.harvard.edu/pph2/) [21], [22].</p>
            </div>
          </section>
        </div>
      </section>
    </article>
  </body>
</html>
