<!DOCTYPE html >
<html id="aFvyr1EZ_1e9h7pM6ZrCH2Bg6Hiu-PMC1459411" data-origid="PMC1459411" class="anndoc" data-anndoc-version="2.0" lang="" xml:lang="" xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="UTF-8"/>
    <meta name="generator" content="com.jmcejuela.anndoc.v2.bio.NcbiJournalArticleParser_v0_4$"/>
    <meta name="dcterms.source" content="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1459411/"/>
    <title>aFvyr1EZ_1e9h7pM6ZrCH2Bg6Hiu-PMC1459411</title>
  </head>
  <body>
    <article>
      <section data-type="title">
        <h2 id="s1h1">Human Ape2 protein has a 3′–5′ exonuclease activity that acts preferentially on mismatched base pairs</h2>
      </section>
      <section data-type="abstract">
        <h2 id="s2h1">Abstract</h2>
        <div class="content">
          <p id="s2p1">DNA damage, such as abasic sites and DNA strand breaks with 3′-phosphate and 3′-phosphoglycolate termini present cytotoxic and mutagenic threats to the cell. Class II AP endonucleases play a major role in the repair of abasic sites as well as of 3′-modified termini. Human cells contain two class II AP endonucleases, the Ape1 and Ape2 proteins. Ape1 possesses a strong AP-endonuclease activity and weak 3′-phosphodiesterase and 3′–5′ exonuclease activities, and it is considered to be the major AP endonuclease in human cells. Much less is known about Ape2, but its importance is emphasized by the growth retardation and dyshematopoiesis accompanied by G2/M arrest phenotype of the APE2-null mice. Here, we describe the biochemical characteristics of human Ape2. We find that Ape2 exhibits strong 3′–5′ exonuclease and 3′-phosphodiesterase activities and has only a very weak AP-endonuclease activity. Mutation of the active-site residue Asp 277 to Ala in Ape2 inactivates all these activities. We also demonstrate that Ape2 preferentially acts at mismatched deoxyribonucleotides at the recessed 3′-termini of a partial DNA duplex. Based on these results we suggest a novel role for human Ape2 as a 3′–5′ exonuclease.</p>
        </div>
      </section>
      <section data-type="">
        <h2 id="s3h1">INTRODUCTION</h2>
        <div class="content">
          <p id="s3p1">Cellular DNA is subject to attack by a variety of agents of exogenous and endogenous origins. Oxidative attack on DNA by free radical species happens continuously during normal cellular metabolism, generating abasic sites and DNA strand breaks with modified 3′-termini, such as 3′-phosphate or 3′-phosphoglycolate (3′-PG) (1,2). Abasic (AP) site, which is one of the most common lesions that arise in cellular DNA, can also be formed due to the action of DNA glycosylases on modified bases or by spontaneous base hydrolysis (3). It has been estimated that as many as 10 000 bases are lost spontaneously in a mammalian cell per day (4). DNA strand breaks with blocking 3′-termini can also arise due to the β-lyase activity of some DNA glycosylases (5). The non-coding AP-sites as well as modified 3′-DNA ends are inhibitory to synthesis by DNA polymerases (Pols), or if bypassed, are highly mutagenic; consequently, their repair is essential for retaining the stability of the genome (1,2).</p>
          <p id="s3p2">AP sites and single strand DNA breaks with 3′ modified termini are repaired mainly by the base excision repair (BER) system (6,7). In BER, class II AP endonucleases play a central role by incising the DNA 5′ to AP sites to generate accessible 3′-OH termini prior to repair synthesis. In addition, several class II AP endonucleases are able to function in the removal of 3′-blocking termini thereby generating accessible 3′ DNA ends for repair synthesis by DNA polymerases (8).</p>
          <p id="s3p3">Class II AP endonucleases have been classified into two families, the exonuclease III (ExoIII) and endonuclease IV (EndoIV) families, based on their homology to the two Escherichia coli enzymes. In E.coli the main AP endonuclease is ExoIII representing 90% of the cellular AP-endonuclease activity, while Endo IV accounts for 10% of the total activity (5). In Saccharomyces cerevisiae, the Apn1 and Apn2 proteins represent the EndoIV and the ExoIII family, respectively (9,10). Apn1 exhibits a strong AP-endonuclease activity and accounts for &gt;90% of the total AP-endonuclease activity in yeast cells (5,11). In contrast, the Apn2 protein has only a weak AP-endonuclease activity but exhibits a strong 3′-&amp;exonuclease and 3′-phosphodiesterase activities (10,12,13). The functional consequences of the enzymatic activities of Apn1 and Apn2 is indicated by that the apn1Δ strain displays a much higher level of sensitivity to the alkylating agent methyl-methanesulfonate (MMS) than the apn2Δ strain; in contrast, neither of the single mutant strains, only the apn1Δapn2Δ double mutant showed sensitivity to the oxidative agent H2O2 (12). These observations indicate that in S.cerevisiae Apn1 has a more prominent role in the repair of AP sites than the Apn2, whereas for the processing of 3′ oxidatively damaged DNA termini Apn1 and Apn2 contribute equally. Contrary to these findings, in Schizosaccharomyces pombe, which is generally believed to be phylogenetically closer to mammalian cells than S.cerevisiae, Apn2 provides the major AP-endonuclease activity while the Apn1 serves only as a back-up activity (14).</p>
          <p id="s3p4">In humans, EndoIV homologs are not known, but two ExoIII family proteins, Ape1 and Ape2, have been identified. Ape1 exhibits robust AP-endonuclease activity, which accounts for &gt;95% of the total cellular activity, and Ape1 is considered to be the major AP endonuclease in human cells. It also possesses 3′-phosphodiesterase and 3′ exonucleaseactivities that are about two orders of magnitude less efficient than its AP-endonuclease activity (8,15).</p>
          <p id="s3p5">The role and the biochemical properties of Ape2 have not been ascertained yet. This is largely due to the difficulty in the purification of human Ape2 as its overexpression causes extreme cytotoxicity in E.coli (16). A partially purified sample of human Ape2 has been shown to exhibit a weak AP-endonuclease activity (16). Its limited AP-endonuclease activity taken together with the fact that in human cells Ape1 accounts for &gt;95% of the total AP-endonuclease activity indicates that human Ape2, similarly to its S.cerevisiae homolog may play only a limited role as an AP endonuclease. Also, a novel role of Ape2 is suggested by recent studies that showed that APE2-null mice have a growth retardation phenotype which is accompanied by a G2/M arrest of proliferating lymphocyes, indicating that Ape2 is required for proper cell cycle progression (17,18). In addition, Ape2 has been shown to localize not only to the nucleus but also to some extent to the mitochondria, in which it may help to maintain the function and integrity of mitochondrial DNA, even if threatened by the attack of reactive oxygen species produced during oxidative phosphorylation (19). These observations strongly suggest that the function of Ape1 and Ape2 proteins differs considerably in human cells.</p>
          <p id="s3p6">To shed more light on the role of human Ape2, we have purified recombinant Ape2 from yeast, and characterized it biochemically. Here, we report novel enzymatic activities for human Ape2. We find that recombinant Ape2 exhibits only a weak AP-endonuclease activity; however, it shows strong 3′–5′ exonuclease and 3′-phosphodiesterase activities. Also, we show that human Ape2 is able to remove preferentially mismatched 3′-nucleotides from DNA. Based on these findings, we discuss a possible role for Ape2 in processing 3′-damaged termini or 3′-mismatched nucleotides.</p>
        </div>
      </section>
      <section data-type="materials|methods">
        <h2 id="s4h1">MATERIALS AND METHODS</h2>
        <div class="subsections">
          <section data-type="">
            <h3 id="s4s1h1">Cloning, expression and purification of the wild type and mutant Ape 2 proteins</h3>
            <div class="content">
              <p id="s4s1p1">Human Ape2 cDNA was amplified by PCR from a human cDNA library. The full-length, sequence verified PCR product was cloned into a Gateway entry vector, pENTR3C (Invitrogen Corporation, Carlsbad, CA, USA) resulting in pIL1045 plasmid. The point mutation D277A in the wild-type Ape2 was generated by the QuickChange site-directed mutagenesis kit (Stratagene Corporation, La Jolla, CA, USA) and the oligonucleotides O1026 (5′-CTC CCG GCT TGC CTA TGT GCT GGG-3′) and O1027 (5′-CCC AGC ACA TAG GCA AGC CGG GAG-3′). For expressing Ape2 proteins we cloned the wild type and mutant Ape2 cDNA-s in fusion with glutathione S-transferase (GST) under the control of the S.cerevisiae galactose-inducible phosphoglycerate promoter using the plasmid pBJ842 (20), which resulted in plasmids pIL1047 and pIL1247, respectively.</p>
              <p id="s4s1p2">The wild type and mutant GST-Ape2 proteins were overexpressed in parallel in the protease-deficient BJ5464 yeast strain. Cells were grown to stationer phase in synthetic medium lacking leucine to select for the plasmids. The cultures were diluted 10-fold in fresh medium lacking dextrose but containing 2% lactic-acid and 3% glycerol followed by incubation overnight before galactose was added to 0.2% final concentration. Four hours after induction cells were harvested and disrupted by bead beater in buffer A (20 mM Tris–HCl, pH 7.5, 1 mM DTT, 0.01% Nonidet P-40 and 10% glycerol) supplemented with 1000 mM NaCl, 5 mM EDTA and protease inhibitor mixture (Mini-Complete; Roche, Indianapolis, IN). After clarification of the crude extract by centrifugation it was loaded onto a glutathione–Sepharose column (Amersham Pharmacia Biotech, USA). First, the column was washed with buffer A+1000 mM NaCl followed by washing with buffer A+100 mM NaCl. The GST-hApe2 proteins were eluted with 20 mM reduced glutathione in buffer A+100 mM NaCl. The Ape2-containing fractions were collected, frozen in aliquots under liquid nitrogen, and stored at −80°C.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s2h1">DNA substrates</h3>
            <div class="content">
              <p id="s4s2p1">DNA substrate (S1) used for the AP-endonuclease assay was generated by annealing a 75 nt template O82 (5′-TCG ATG GTA CGG ACG GAG TTC TTA AGC ATT ATA CGG ATG TGA CCT CAT GGC CTC GTA GCA GCA CTG ACC CTT TTG-3′) to the 75 nt 5′-32P-labeled oligomer O14 (5′-CAA AAG GGT CAG TGC TGC TAC GAG GCC ATG AGG TCA CAT CCG TAX AAT GCT TAA GAA CTC CGT CCG TAC CAT CGA-3′). In this oligomer the X represents the place of a single tetrahydrofuran, an abasic site analog. DNA substrate (S2) used for the 3′-phosphodiesterase assay were generated by annealing a 70 nt template O105 (5′-AGC TGG ATT CGT GTC AGC CGC TAG CCA TTA GCT GGC GCA TTC CTC GAG AGA GTC CCG AGC ATC GTG ACT G-3′) to the 35 nt 5′-32P-labeled primer O102 (5′-CAG TCA CGA TGC TCG GGA CTC TCT CGA GGA ATG CG-PG-3′) containing a 3′ PG terminus. DNA substrate (S3) used for the 3′–5′ exonuclease assay was generated by annealing a 52 nt oligomer template O56 (5′-TTC GTA TTA TGC CTA CAC TTG AGT ACC GGA GCA TCG TCG TGA CTG GGA AAA C-3′) to the 33 nt 5′-32P-labeled primer, O67 (5′-GTT TTC CCA GTC ACG ACG ATG CTC CGG TAC TCA-3′). DNA substrates containing all the possible 16 bp at the primer–template junction used for the proofreading assay in Figure 6 were generated by annealing the 52 nt oligomer template, O56, containing a G, A, T or C nucleotides at position 33 counting from the 3′ end to the 5′-32P-labeled primer, O67, containing a G, A, T or C at 3′ end. For generating a nick, 1, 2, 3 and 4 nt long gap containing double-stranded DNA substrates, we annealed three oligonucleotides, a 66 nt template oligomer O1049 (5′-AGC AAG TCA CCA ATG TCT AAG AGT TCG TAT TAT GCC TAC ACT GGA GTA CCG GAG CAT CGT CGT GAC-3′), a 32 nt 5′-32P-labeled primer O76 (5′-CGA CGA TGC TCC GGT ACT CCA GTG TAG GCA TA-3′), and a complementary 30, 29, 28, 27 or a 26 nt oligomer, respectively.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s3h1">3′-phosphodiesterase, 3′–5′ exonuclease and AP-endonuclease assays</h3>
            <div class="content">
              <p id="s4s3p1">A standard reaction mixture (10 μl) contained 40 mM Tris–HCl (pH 7.5), 0.5 mM MnCl2, 1 mM DTT, 100 μg/ml BSA, 10 nM DNA substrate and 5 nM Ape 2 protein. In Figure 2B, 1 U Ape1 (Trevigene, Gaithersburg, MD) was used. Where indicated the above buffer was supplemented with 150 mM NaCl (Figure 2A), MnCl2 was substituted with 8 mM MgCl2 (Figure 4A), and for experiments shown in Figure 4E HEPES was used for pH 5.0, 5.5 and 6.0, and Tris–HCl was used for pH 7.0, 7.5, 8.0, 8.8 and 10.0. Reactions were assembled on ice, incubated at 37°C for 5 min, and stopped by the addition of loading buffer (20 μl) containing 20 mM EDTA, 95% formamide, 0.25% xylene cyanol and 0.25% bromophenol blue. The reaction products were resolved on 10% polyacrylamide gel containing 8 M urea. Quantification of the results was done using Molecular Dynamics STORM phosphorimager and ImageQuant 5.0 software. To eliminate error derived from unequal loading, the concentration of the reaction products were calculated from the ratio of the intensity of bands of products to the intensity of the total radioactivity in each lane.</p>
            </div>
          </section>
        </div>
      </section>
      <section data-type="">
        <h2 id="s5h1">RESULTS</h2>
        <div class="subsections">
          <section data-type="">
            <h3 id="s5s1h1">Purification of wild type and active site mutant Ape2 proteins</h3>
            <div class="content">
              <p id="s5s1p1">The family of ExoIII-type endonucleases contains two members in humans, the Ape1 and Ape2 proteins. The conserved domains involved in catalytic activity are located at the N-terminal part of both proteins. In addition, the Ape2 protein has a C-terminal extension, which is not present in Ape1, but can also be found in homologs of human Ape2 such as Apn2 proteins of S.cerevisiae and S.pombe (Figure 1A) (9,10). While Ape1 has been characterized extensively, very little is known about the function of Ape2. Here, our goal was to characterize the enzymatic properties of purified human Ape2 protein.</p>
              <p id="s5s1p2">To facilitate the purification of the Ape2 protein, the Ape2 cDNA was PCR amplified from a human cDNA library followed by cloning it in frame downstream of the GST gene in a yeast overexpression vector. We also made a similar construct expressing the Ape2D277A protein containing a point mutation at the D277 residue, which corresponds to a conserved metal-binding site in the active center of class II AP endonucleases (21) (Figure 1A). We expected this mutation to render Ape2 catalytically inactive. The resulting GST–Ape2 fusion proteins were purified from a protease-deficient S.cerevisiae strain on Glutathione–Sepharose beads. During purification the wild type and mutant proteins behaved similarly and displayed identical electrophoretic mobility on a SDS–polyacrylamide gel (Figure 1B). Western blot analysis by anti-GST antibody confirmed the dominant protein band as GST–Ape2 and also revealed the presence of a ∼68 kDa proteolytic product of Ape2 protein (data not shown).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s2h1">Ape2 exhibits very weak AP-endonuclease activity</h3>
            <div class="content">
              <p id="s5s2p1">The presence of the ExoIII domain in Ape2 as well as its high degree of homology to Ape1 had suggested that Ape2 is an AP-endonuclease. Supporting this, a partially purified Ape2 that was expressed in E.coli has been reported to show a weak AP-endonuclease activity (16). To examine whether the purified recombinant Ape2 has an intrinsic AP-endonuclease activity, we assayed Ape2 using a 75 nt duplex DNA substrate, in which the 5′-32P end-labeled strand contained a single abasic site at position 31 from the 5′ end. Incubation of Ape2 with the 75 nt DNA substrate generated a 30 nt labeled oligomer indicating that Ape2 like other class II AP-endonucleases incised the DNA 5′ to the AP site (Figure 2A, lanes 2 and 3). Using the Ape2 D277A mutant protein in the same assay we could not detect any incision of the DNA substrate confirming that the D277A mutation indeed rendered Ape2 catalytically inactive and, that the observed AP-endonuclease activity is intrinsic to Ape2 (Figure 2A, lanes 4 and 5). AP-endonuclease reaction with Ape1 was carried out as control for determing the incision specificity of Ape2. Incision by Ape1 resulted in the same 30 nt reaction product generated by Ape2 (Figure 2B), which further confirms that Ape2, similarly to Ape1, incised the DNA 5′ to the AP-site. Interestingly, however, the endonuclease activity of Ape1 and Ape2 showed different salt sensitivity; the activity of Ape1 was reduced at 150 mM NaCl concentration, but Ape2 was more active at 150 mM than at 0 mM NaCl concentration (Figure 2B). Measuring the salt dependence between 0 and 300 mM NaCl concentration we found that 150 mM NaCl concentration is optimal for the AP-endonuclease activity of Ape2 (Figure 2C). However, Ape2 is a weak AP-endonuclease indicated by that even at optimal NaCl concentration Ape2 incised only 7% of the DNA substrate despite its two and half molar excess over DNA substrate.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s3h1">Ape2 has a robust 3′–5′ exonuclease activity</h3>
            <div class="content">
              <p id="s5s3p1">When we examined the AP-endonuclease activity of Ape2 in a buffer containing no NaCl, we noticed the gradual shortening of the 75mer template that indicated an exonucleolytic digestion of the DNA from the 3′ end. To determine whether the observed 3′–5′ exonuclease activity was intrinsic to Ape2, or due to an exonuclease contamination of our protein sample, we assayed the wild type and active center mutant Ape2 D277A proteins on a double-stranded DNA substrate containing a 5′-32P-labeled strand with recessed 3′ terminus. Incubation of increasing amount of Ape2 with this DNA substrate generated a series of smaller products, indicating that Ape2 exonucleotically digested the DNA from the 3′ terminus (Figure 3A, lanes 2 and 3). In contrast, the D277A active center mutant Ape2 did not show any exonuclease activity demonstrating that the observed robust 3′–5′ exonuclease activity was intrinsic to the Ape2 protein (Figure 3A, lanes 4 and 5). Importantly, when the DNA substrate was in 2-fold molar excess over Ape2, under optimal reaction conditions Ape2 hydrolyzed nearly 100% of the exonuclease substrate and only 7% of the AP-endonuclease substrate. Therefore Ape2 is dominantly a 3′–5′ exonuclease.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s4h1">3′-Phosphodiesterase activity of Ape2</h3>
            <div class="content">
              <p id="s5s4p1">Next we tested whether Ape2 is also able to remove 3′-blocking termini. For this we constructed a DNA substrate by annealing a 70 nt long oligonucleotide to a 35 nt oligomer containing a 3′-PG terminus. Enzymatic removal of the 3′-PG results in a 3′-hydroxyl terminus (3′-OH), and 3′-PG is released in the form of phosphoglycolic acid. The 3′-OH group confers lower electrophoretic mobility to the 35 nt oligomer on a denaturing polyacrylamide gel. Incubation of Ape2 with the PG-containing DNA substrate resulted in the appearance of a lower mobility band on the gel, which indicated that Ape2 had a 3′-phosphodiesterase activity (Figure 3B, lanes 2 and 3). Unlike the wild-type protein, the Ape2 D277A mutant did not exhibit such an activity (Figure 3B, lanes 4 and 5). These results indicate that the Ape2 protein has a 3′-phosphodiesterase activity.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s5h1">DNA substrate specificity and enzymatic characteristics of Ape2 exonuclease</h3>
            <div class="content">
              <p id="s5s5p1">To characterize further Ape2 exonuclease, first we examined for its optimal reaction conditions. The exonuclease activity of Ape2 was strongly dependent upon metal ion. Ape2 was ∼5-fold more active in the presence of manganese than of magnesium ion (Figure 4A and B). Addition of salt inhibited the exonuclease activity, and at 100 mM NaCl concentration Ape2 exhibited only 20% of its exonuclease activity measured in buffer containing no salt (Figure 4C and D), and in the presence of 200 mM salt almost no exonuclease activity could be detected. The Ape2 showed optimal activity in the pH range of 6.0–8.0 (Figure 4E and F).</p>
              <p id="s5s5p2">To determine the DNA substrate requirement for the exonuclease activity, we compared the exonuclease activity of Ape2 for its ability to hydrolyze single-stranded DNA, blunt-ended duplex DNA, partial DNA duplexes with either a protruding or a recessed 3′-terminus or a single nucleotide gap containing heteroduplex DNA. As shown on Figure 5A, Ape2 exhibited the strongest exonuclease activity on a recessed 3′ end containing partial DNA duplex (Figure 5A, lane 8) and fairly strong activity on a single nucleotide gap containing heteroduplex DNA (Figure 5A, lane 10). In addition, Ape2 also exhibited exonuclease activity on blunt-ended substrate (Figure 5A, lane 6). However, Ape2 showed only residual activity on single-stranded DNA and protruding 3′-terminus.</p>
              <p id="s5s5p3">Short-patch BER proceeds via a nick and a single nucleotide gap containing heteoduplex DNA. To examine whether Ape2 has any preferences for these DNA substrates, we compared the exonuclease activity of Ape2 on a nick and 1, 2, 3 or 4 nt long gap containing DNA duplexes. Our results revealed that Ape2 exonuclease had 2-fold stronger activity on a single nucleotide gap containing substrate than on a nicked DNA, and increasing the gap size from 1 to 4 nt did not make significant difference (Figure 5B and C).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s6h1">3′-Mispair specific exonuclease activity of Ape2</h3>
            <div class="content">
              <p id="s5s6p1">To examine the effect of base–base hydrogen bounding at the primer–template junction on the exonuclease activity of Ape2, we assayed Ape2 on various combinations of recessed DNA with 3′-mispairs as substrates. Figure 6 shows the exonuclease activity of Ape2 on DNA substrates containing all the 16 possible base pairing at the 3′ end. Quantification revealed that Ape2 removed 3′ mismatched nucleotides 2- to 40-fold more efficiently than those matched correctly (Figure 6). For example, Ape2 had a very limited activity on a 3′ recessed DNA duplex containing a matched 3′A opposite from a template T (Figure 6, lane 3); however, it exhibited a strong nuclease activity on a DNA containing unmatched 3′ A opposite from G, A or C (Figure 6, lanes 2, 4 and 5). From these experiments we conclude that Ape2 exonuclease has a significant preference for mispaired nucleotides at the recessed 3′ end of DNA.</p>
            </div>
          </section>
        </div>
      </section>
      <section data-type="">
        <h2 id="s6h1">DISCUSSION</h2>
        <div class="content">
          <p id="s6p1">In this study, we have clarified the enzymatic activities of human Ape2 and show that it is a multifunctional enzyme. Ape2 has only a weak AP endonuclease, a fairly strong 3′-phosphodiesterase, but a robust 3′–5′exonuclease activity. Importantly, we reveal that the Ape2 exonuclease is most active on 3′-recessed heteroduplex DNA and is able to remove mismatched nucleotides preferentially.</p>
          <p id="s6p2">All the proteins of the ExoIII family share the highly conserved ExoIII-domain in the N-terminus but proteins in the new Apn2 subfamily within the ExoIII family contain an additional C-terminal extension. While the classical ExoIII-type AP-endonucleases have been studied extensively, very little is known about the proteins of the Apn2 subfamily. Previously, we have studied the Apn2 protein of S.cerevisiae and shown that it provides only a back-up AP-endonuclease activity but plays an important role in the repair of DNA strand breaks arising from the reaction of DNA with reactive oxygen species (10,12,13). In contrast, in S.pombe cells Apn2 appears to be the major AP endonuclease (14). Our current study indicates that the enzymatic properties of Ape2 are more similar to those of Apn2 of S.cerevisiae than to those of Apn2 of S.pombe.</p>
          <p id="s6p3">Previously, Ape2 has been purified only partially due to its insolubility and its cytotoxicity on the E.coli and it has been shown to possess a very weak AP-endonuclease activity (16). In agreement with that study, our attempt to purify active recombinant Ape2 from E.coli was not successful. However, we managed to overcome the problem by overexpressing Ape2 in fusion with GST in the yeast S.cerevisiae. To confirm that the observed 3′-exonuclease, 3′-phosphodiesterase and the extremely weak AP-endonuclease activities are intrinsic to Ape2, we also purified a mutant Ape2 protein, in which we altered the conserved Asp277 residue to Ala. This amino acid in Ape2 is equivalent to residues Asp328 in S.cerevisiae Apn2, and Asp283 in human Ape1, which have been shown to be involved in metal binding (21). As expected, the Ape2 D277A protein did not show activity in any of the enzymatic assays thereby confirming that the detected AP-endonuclease, 3′-phosphodiesterase and 3′-exonuclease activities are intrinsic to Ape2.</p>
          <p id="s6p4">It has been published recently that APE2-null mice show abnormalities in proliferating haemopoietic organs, such as dyshaematopoiesis, defect in lymphopoiesis, and delayed S-phase and G2/M-phase arrest. It has been suggested that the increased accumulation of AP sites in the absence of the AP-endonuclease activity of Ape2 is the underlying cause of these phenotypes (17,18). However, our biochemical data show a very inefficient AP-endonuclease activity of Ape2 arguing that this activity cannot play a major role in vivo. In contrast, owing to its significant 3′-phosphodiesterase activity, Ape2 might have a major function in the removal of 3′-PG termini formed in DNA by oxidative agents. This theory is also supported by our previous study, in which we provided genetic evidence for the involvement of S.cerevisiae Apn2 in the repair of 3′-PG residues (12).</p>
          <p id="s6p5">Another clue for the main function of Ape2 can be found in its strong 3′–5′ exonuclease activity and in its preference for removing mismatched nucleotides from the 3′ primer end. Ape2 might have a role as a proofreader in processes where new DNA synthesis occurs such as DNA repair synthesis. During short patch BER, Polβ carries out the DNA synthetic step. However, Polβ lacks proofreading exonuclease activity, and it is quite error prone making an average one mistake per 4000 base insertions. Without any proofreading mechanism, Polβ would introduce several mutations into the genome of each cell per day. Human Ape1 protein has been shown already to be able to proofread the errors of Polβ (22). It is possible that Ape2 could also function as an alternative proofreader for Polβ and thereby improve the fidelity of BER.</p>
          <p id="s6p6">Recently, several class II AP endonucleases such as yeast Apn1 and human Ape1 have been demonstrated to be able to remove 3′ incorporated 8-oxodG damaged nucleotide (23,24). Further studies should determine, whether Ape2 can also function in the cleansing of 3′ end of newly synthesized DNA from 8-oxodG, or other damaged nucleotides.</p>
          <p id="s6p7">One might question the need for an exonuclease for the removal of mismatches or 8-oxodG from the 3′-termini of newly synthesized DNA when cells contain MSH2 complexes to recognize these errors and to eliminate them by mismatch repair mechanisms. However, class II AP exonucleases can provide a backup mechanism to the mismatch repair system. Yeast genetic experiments have already indicated that beside mismatch repair system, the 3′ exonuclease activity of Apn1 functions in an alternative way for prevention the mutagenic threat of 8-oxodG (23).</p>
          <p id="s6p8">In summary, our biochemical characterization of Ape2 support a main function for Ape2 as a 3′-exonuclease and predicts its involvement in the processing of 3′-blocking termini or 3′ mismatched nucleotides of newly synthesized DNA.</p>
        </div>
      </section>
      <section data-type="resources">
        <h2 id="s7h1">Resources</h2>
        <div class="content">
          <figure data-origid="fig1">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1459411/#fig1">Figure 1</a>
            <figcaption>
              <p id="s7f1p1">Purification of the wild type and mutant human Ape2. (A) Schematic representation of human Ape2. The N-terminus of Ape2 contains the conserved ExoIII domain characteristic of class II AP endonucleases, while the C-terminus is unique to Ape2-subfamily. The arrow indicates the amino acid D227 residue that was changed to alanine in the active site of Ape2. (B) Purity of wild type and mutant Ape2 proteins. Purified wild-type GST–Ape2 and GST–Ape2D277A fusion proteins were analyzed on a 10% denaturating polyacrylamide gel and stained with Coomassie blue. Lane 1, molecular weight standard; lane 2, 0.5 μg of purified GST–Ape2; lane 3, 0.5 μg of purified GST–Ape2D277A. The arrow identifies full-length GST–Ape2, while asterisk indicate the co-purifying proteolytic product of Ape2.</p>
            </figcaption>
          </figure>
          <figure data-origid="fig2">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1459411/#fig2">Figure 2</a>
            <figcaption>
              <p id="s7f2p1">AP-endonuclease activity of Ape2. (A) The AP-endonuclease activity of Ape2 was assayed using a 75 nt double-stranded DNA substrate (S1) containing a single AP-site at position 31 on the 5′-labeled strand. The DNA substrate (10 nM) was incubated without (lane 1) or with wild type (lanes 2–3) or D277A mutant Ape2 (lanes 4–5) in standard reaction buffer but containing 150 mM NaCl for 5 min at 37°C. The reaction products were analyzed on 10% polyacrylamide gels containing 8 M urea, and the DNA bands were visualized by autoradiography. The position of the incision product at 30 nt is indicated. (B) Comparison the specificity of Ape2 and Ape1 endonuclease activities. The DNA substrate (S1) was incubated without (lane 1) or with Ape1 (lanes 2 and 4) or Ape2 (lanes 3 and 5) in the presence or absence of 150 mM NaCl in standard reaction buffer for 5 min at 37°C. (C) The NaCl concentration dependence of the endonuclease activity of Ape2. S1 DNA substrate (10 nM) was incubated with Ape2 (25 nM) in standard reaction buffer supplemented with 0–300 mM NaCl as indicated.</p>
            </figcaption>
          </figure>
          <figure data-origid="fig3">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1459411/#fig3">Figure 3</a>
            <figcaption>
              <p id="s7f3p1">3′-Exonuclease and 3′-phosphodiesterase activities of Ape2. (A) The 3′–5′ exonuclease activity of wild-type (lanes 2 and 3) and mutant (lanes 4 and 5) Ape2 was assayed on partial DNA duplex (S3) (10 nM) in standard reaction buffer for 5 min at 37°C. (B) The phosphodiesterase activity of Ape2 was tested on a DNA substrate (S2) containing a 5′-labeled 35 nt oligomer with a 3′-PG terminus annealed to a 70 nt oligomer. Wild type (lanes 2 and 3) and mutant (lanes 4 and 5) Ape2 proteins were incubated with 10 nM DNA substrates in standard reaction buffer for 5 min at 37°C. The positions of the reaction product and the substrate are indicated by 3′-OH and 3′-PG, respectively.</p>
            </figcaption>
          </figure>
          <figure data-origid="fig4">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1459411/#fig4">Figure 4</a>
            <figcaption>
              <p id="s7f4p1">Optimal reaction conditions for the 3′–5′ exonuclease activity of the Ape2. The 3′–5′ exonuclease activity of GST–Ape2 was assayed under various reaction conditions using a partial DNA duplex (S3) (10 nM) in which the 5′-labeled oligonucleotide contained a 3′-recessed terminus. (A) Metal ion dependence. The first lane contains reaction mixture without any metal ions in the reaction buffer. Reactions were carried in the presence of 8 mM MgCl2 (lanes 2–4) or 0.5 mM MnCl2 (lanes 5–7) and increasing concentrations of Ape2 as indicated. (B) Graphical representation of results in (A); (C) NaCl concentration dependence. The first lane contains reaction without any Ape2 protein. (D) Graphical representation of results in (C); (E) pH dependence. (F) Graphical representation of results in (E).</p>
            </figcaption>
          </figure>
          <figure data-origid="fig5">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1459411/#fig5">Figure 5</a>
            <figcaption>
              <p id="s7f5p1">Exonuclease activity of Ape2 protein on different DNA substrates. (A) In each reaction single-stranded (lanes 1 and 2), 3′ overhanging partial heteroduplex (lanes 3 and 4), blunt-ended (lanes 5 and 6), 3′ recessed partial heteroduplex (lanes 7 and 8), or single nucleotide gap containing heteroduplex DNA substrates (lanes 9 and 10) (10 nM) were incubated with Ape2 (5 nM) for 10 min at 37°C. (B) Exonuclease activity of Ape2 on nick or gap containing DNA duplexes. Parallel reactions were carried out using 5 nM of Ape2 and 10 nM DNA duplex containing a nick, or 1, 2, 3 or 4 nt gap, or a 3′ recessed partial heteroduplex DNA as indicated. Asterisks indicate the radioactively labeled terminus. (C) Graphical representation of results in (B).</p>
            </figcaption>
          </figure>
          <figure data-origid="fig6">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1459411/#fig6">Figure 6</a>
            <figcaption>
              <p id="s7f6p1">Comparing the 3′ exonuclease activity of Ape2 on paired and mispaired DNA substrates. The 3′–5′ exonuclease activity of GST–Ape2 (5 nM) was assayed on partial DNA duplexes (10 nM) containing all the possible 16 matched and mismatched nucleotide pairs at the primer–template junctions; control reactions without Ape2 (lanes 1, 6, 11 and 16); exonuclease reactions on the correctly paired 3′ end substrates (lanes 3, 7, 15 and 19) and on mispaired DNA (lanes 2, 4, 5, 8–10, 12–14, 17, 18 and 20). The products of three independent experiments, of which one is shown on top, were quantified by PhosphorImager, and the cleavage efficiency of the Ape2 exonuclease on these various DNA substrates are shown under each lane by diagrams.</p>
            </figcaption>
          </figure>
        </div>
      </section>
    </article>
  </body>
</html>
