<!DOCTYPE html >
<html id="aGceapPDe3PhApPZ5JJCTi1e6wgG-PMC35353" data-origid="PMC35353" class="anndoc" data-anndoc-version="2.0" lang="" xml:lang="" xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="UTF-8"/>
    <meta name="generator" content="com.jmcejuela.anndoc.v2.bio.NcbiJournalArticleParser_v0_3$"/>
    <meta name="dcterms.source" content="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC35353/"/>
    <title>aGceapPDe3PhApPZ5JJCTi1e6wgG-PMC35353</title>
  </head>
  <body>
    <article>
      <section data-type="title">
        <h2 id="s1h1">A missense mutation (Q279R) in the Fumarylacetoacetate Hydrolase gene, responsible for hereditary tyrosinemia, acts as a splicing mutation</h2>
      </section>
      <section data-type="abstract">
        <h2 id="s2h1">Abstract</h2>
        <div class="subsections">
          <section data-type="">
            <h3 id="s2s1h1">Background</h3>
            <div class="content">
              <p id="s2s1p1">Tyrosinemia type I, the most severe disease of the tyrosine catabolic pathway is caused by a deficiency in fumarylacetoacetate hydrolase (FAH). A patient showing few of the symptoms associated with the disease, was found to be a compound heterozygote for a splice mutation, IVS6-1g-&gt;t, and a putative missense mutation, Q279R. Analysis of FAH expression in liver sections obtained after resection for hepatocellular carcinoma revealed a mosaic pattern of expression. No FAH was found in tumor regions while a healthy region contained enzyme-expressing nodules.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s2s2h1">Results</h3>
            <div class="content">
              <p id="s2s2p1">Analysis of DNA from a FAH expressing region showed that the expression of the protein was due to correction of the Q279R mutation. RT-PCR was used to assess if Q279R RNA was produced in the liver cells and in fibroblasts from the patient. Normal mRNA was found in the liver region where the mutation had reverted while splicing intermediates were found in non-expressing regions suggesting that the Q279R mutation acted as a splicing mutation in vivo. Sequence of transcripts showed skipping of exon 8 alone or together with exon 9. Using minigenes in transfection assays, the Q279R mutation was shown to induce skipping of exon 9 when placed in a constitutive splicing environment.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s2s3h1">Conclusion</h3>
            <div class="content">
              <p id="s2s3p1">These data suggest that the putative missense mutation Q279R in the FAH gene acts as a splicing mutation in vivo. Moreover FAH expression can be partially restored in certain liver cells as a result of a reversion of the Q279R mutation and expansion of the corrected cells.</p>
            </div>
          </section>
        </div>
      </section>
      <section data-type="">
        <h2 id="s3h1">Background</h2>
        <div class="content">
          <p id="s3p1">Tyrosinemia type I (HTI, McKusick 276700) is an autosomal recessive disease caused by a deficiency in fumarylacetoacetate hydrolase (FAH, EC 3. 7. 1. 2), the last enzyme of the catabolic pathway of tyrosine [1,2,3,4]. In the absence of FAH, metabolites such as maleylacetoacetate (MAA), fumarylacetoacetate (FAA), and succinylacetone (SAc) accumulate during tyrosine degradation. FAA has been shown to have a mutagenic activity in a mammalian cell assay [5], to induce cell cycle arrest at G2/M, and apoptosis [6]. Tyrosinemia is characterized by hepatic failure, cirrhosis, renal dysfunction, hepatocarcinoma, and neurologic crisis. Liver is the most severely affected organ. Two distinct forms of the disease have been described according to the symptoms and age of onset [7, 8]. The acute form of tyrosinemia is diagnosed in the first months of life and results in rapid deterioration of hepatic and renal functions leading to early death due to hepatic failure. In the chronic form, symptoms appear later in childhood and often culminate with development of hepatocarcinomas. Molecular studies on some patients presenting either the acute or the chronic form revealed identical genotypes [9] suggesting that genotypic variability does no account alone for the different clinical forms observed in tyrosinemia.</p>
          <p id="s3p2">The gene encoding the FAH protein has been cloned and mapped to chromosome 15, region q23-q25 [10]. It contains 14 exons, spanning over 35 kb of DNA [11]. At this time, 34 different mutations of the FAH gene have been identified in HTI patients: 18 missense mutations, 5 nonsense mutations, and 10 splicing mutations [4, 12]. One mutation (R341W) causes a pseudodeficient phenotype with a reduced amount of FAH immunoreactive material [13]. FAH is a cytoplasmic homodimeric enzyme. Its crystal structure shows that each subunit of 46 kDa consists of a 120 residue N-terminal domain and a 300 residue C-terminal domain. These subunits interact through their C-terminal domains and the active site is located in close proximity to the dimer interface [14]. It has been shown that the FAH active site, which is highly conserved from fungi to human, comprises two metal ions that participate in substrate binding and catalysis.</p>
          <p id="s3p3">Recently, Kim et al. [15] described the case of a HTI patient who showed few of the symptoms associated with HTI until the age of 37 when hepatocellular carcinoma was diagnosed. This patient is one of the few reported cases of HTI who lived over 30 years and has been genotyped as a compound heterozygote for a frequent splice mutation, IVS6-1g-&gt;t, and a new mutation, Q279R (836A-&gt;G). Since this patient showed an almost normal phenotype during the first 36 years of her life, a molecular analysis of both mutations was carried out in vivo and in vitro to determine if this particular phenotype was caused by a neutral missense mutation (Q279R) like in the pseudodeficiency phenotype or by other mechanisms such as mutation reversion, as described in a number of HTI patients [16]. It was shown that FAH was expressed in a mosaic pattern in the patient's liver, with non-tumoral regions expressing FAH [15]. Here we report that the Q279R mutation acts as a splicing mutation in vivo and that correction of this mutation in some cells leads to restored FAH function and partial liver repopulation by corrected cells.</p>
        </div>
      </section>
      <section data-type="">
        <h2 id="s4h1">Results</h2>
        <div class="subsections">
          <section data-type="">
            <h3 id="s4s1h1">Expression of FAH in a non-tumoral liver region results from reversion of the Q279R mutation</h3>
            <div class="content">
              <p id="s4s1p1">Immunostaining of sections from the resected liver of the HTI patient with an anti-FAH antibody showed a mosaic pattern of FAH reactivity [15]. The non-tumoral region was FAH immunopositive and expressed full-length FAH as demonstrated by western blot analysis, in contrast to tumoral regions where no FAH was detected (no truncated protein form was detected either [15]). Spectrophotometric measurement of FAH hydrolytic activity against FAA in microdissected regions of frozen liver sections confirmed that the enzyme expressed in the non-tumoral region was active (data not shown). The DNA in microdissected regions of liver sections was next examined in order to determine whether one of the two mutations had reverted (Figure 1). Restriction enzyme analysis revealed that DNA extracted from tumor regions presented both the IVS6-1g-&gt;t and Q279R mutations. As for the DNA extracted from a FAH positive nodule (NT), it showed the pattern expected for IVS6-1g-&gt;t heterozygocity (three bands of 156-, 104- and 75-bp, Figure 1). In the Q279R test, a weak mutated band (58-bp) was detected along with a strong band of normal length (78-bp) indicating the presence of a normal allele likely resulting from a reversion of the mutation (Figure 1, lane NT). Reversion of the Q279R mutation to Q279Q on one FAH allele was confirmed by direct sequencing (see below).</p>
              <figure data-origid="F1">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC35353/#F1">Figure 1</a>
                <figcaption>
                  <p id="s4s1f1p1">Mutation analysis in different liver regions. DNA was extracted from different liver regions and amplified by PCR. PCR products were digested with either Alu I to detect IVS6-1g-&gt;t or with Msp I to detect Q279R. For IVS6-1 g-&gt;t, the same heterozygous pattern is seen in both the reverted nodule (NT), tumor section (T) and fibroblast DNA (F), showing 3 bands, one at 156-, another at 104- and the last at 75-bp. The control (wt/wt) shows two bands, one at 156- and the other at 75-bp, indicating the absence of IVS6-1g-&gt;t (M: molecular weight marker, 100- and 200-bp). For Q279R both the 78- and 58-bp bands are seen in the tumor section (T) and fibroblast DNA (F) indicating an heterozygous genotype while only the 78-bp wild-type band is seen in the control (wt/wt). In the region suspected of reversion (NT), a strong 78-bp wild-type band is seen with a weak 58-bp mutated band (M: molecular weight marker, 100-bp).</p>
                </figcaption>
              </figure>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s2h1">The Q279R mutation is associated with altered mRNA splicing in vivo</h3>
            <div class="content">
              <p id="s4s2p1">In order to determine if Q279R-containing mRNA was present in liver cells, we used RT-PCR to examine the transcripts in various liver specimens and in fibroblasts of the patient (Figure 2). Interestingly, RT-PCR amplification of transcripts showed an unexpected alternative splicing pattern in different liver regions. Thus in a FAH expressing nodule (NT) the main amplified band was of a length expected for a normal mRNA (537-bp, Figure 2A). Indeed the sequence of this major product was identical to wild-type FAH mRNA, without neither the Q279R nor the IVS6-1g-&gt;t mutation (data not shown).</p>
              <figure data-origid="F2">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC35353/#F2">Figure 2</a>
                <figcaption>
                  <p id="s4s2f1p1">RT-PCR on RNA from different liver sections and fibroblasts. Total RNA was extracted from various samples, reverse transcribed and amplified by PCR. The 537-bp fragment starts in the middle of exon 6 and ends in the middle of exon 12. A- RT-PCR products amplified from total RNA extracted from different liver regions i.e. non-tumoral (lane 2) and tumoral (lane 3) and from fibroblasts of the patient (lane 4). (lane 1: Control RNA from T19-EBV normal cells). B- RT-PCR products amplified from total RNA extracted from transformed lymphocytes of the parents (father; lane 1, mother; lane 2) and cultured fibroblasts of another patient homozygous for IVS6-1g-&gt;t (lane 3). A normal length cDNA is seen in the reverted nodule (537-bp, A, lane 2) and in both parents. Three of the products seen in the tumor liver section (A' lane 3) are also observed in a patient homozygous for the IVS6-1g-&gt;t mutation (B, lane 3).</p>
                </figcaption>
              </figure>
              <p id="s4s2p2">The amplified products of transcripts from the FAH negative region of a tumor section presented three bands of slightly lower molecular weight (Figure 2A, lane 3), as observed in the patient's fibroblasts (lane 4). Cloning and sequencing of the fragments amplified from the tumor region revealed 7 mRNA species (summarized in Figure 3). The largest band in the tumoral region actually comprises two transcripts that comigrate, as determined by cloning and sequencing. One includes a 5-bp deletion (del5, 532-bp) at the beginning of exon 7, and the other, the same deletion together with another 13-bp deletion at the beginning of exon 8 (del5+del13, 519-bp). The second band (about 400-bp) observed in the tumoral region also consists of two transcripts, the first one with the 5-bp deletion in exon 7 and a skipping of exon 8 (del5+del100, 432-bp), and the second one (del100), which is a product of the Q279R mutation (see below). These first three transcripts were already reported for the IVS6-1g-&gt;t mutation [17], (see, Figure 2B, lane 3). Another rare transcript was also identified by cloning of the RT-PCR product from the tumoral region and consists in retention of intron 6 (InsI6, see lane 3, Figure 2B).</p>
              <figure data-origid="F3">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC35353/#F3">Figure 3</a>
                <figcaption>
                  <p id="s4s2f2p1">Diagram of RNA alternative transcripts found in the IVS6-1g-&gt;t and Q279R patient.The splicing pattern is based on sequence analysis of the RT-PCR transcription products amplified with the primers Tan5 and Tan 51. The alternative transcripts identified are pictured from exons 6 to 10 (represented by boxes). The introns are represented by lines. Black boxes indicate that a cryptic splice site is activated with a subsequent skipping of exonic sequences. Premature stop codons that appear due to frameshift are represented by a stop sign.</p>
                </figcaption>
              </figure>
              <p id="s4s2p3">The other three products identified by cloning and sequencing of the PCR products from the tumoral region seem to be a consequence of the Q279R allele, with two of them resulting from alternative splicing (see Figure 2 and the diagram of Figure 3). The first one is a transcript with a skipping of exon 8 (del100, 437-bp), which comigrates with the del5+del100 transcript. The second one results from the skipping of both exons 8 and 9 (del231, 306-bp). The last transcript identified (537-bp, see 1 lane, Figure 2B) and due to the Q279R paternal allele is of normal length and contains the Q279R mutation in exon 9.</p>
              <p id="s4s2p4">Four of the six alternative transcripts identified in the tumoral region of the patient are caused by the IVS6-1g-&gt;t mutation (InsI6, del5, del5+del13, del5+del100). The other two alternative transcripts (del100, and del231) seem to be due to the Q279R allele. This is summarized in the diagram of Figure 3. Del231, which lacks both exons 8 and 9, is a major product. These data suggest that the Q279R mutation disturbs the splicing pattern in vivo.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s3h1">Q279R acts as a splicing mutation in a minigene assay</h3>
            <div class="content">
              <p id="s4s3p1">To test whether the Q279R mutation is itself a splicing mutation, we constructed two minigenes in which a wild-type exon 9 of the FAH gene or an exon 9 with the Q279R mutation was placed in a constitutive splicing context. Reed [18] has previously reported that exon recognition would require an mRNA region of about 300-bp including the exon itself and the adjacent intronic sequences. A 387-bp fragment comprising the wild-type or the mutated exon 9 and their surrounding intronic sequences was amplified and inserted in a globin splicing cassette (see Methods, Figure 4A). Both minigenes were sequenced prior to transient transfection in HeLa cells. Total RNA was then amplified with the HG1S and HG3AS primers, which are located at each end of the splicing cassette (see Figure 4A).</p>
              <figure data-origid="F4">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC35353/#F4">Figure 4</a>
                <figcaption>
                  <p id="s4s3f1p1">Analysis of the splicing pattern obtained with the minigenes. A- The splicing K7 consists of exon 1 of β-globin and its downstream intronic sequences joined to β-globin exon 3 and its upstream intronic sequences. Exon 9, with or without the Q279R mutation was inserted in K7 at the intronic junction. HeLa cells were transiently transfected with both constructs, the wild-type Q279Q-K7 and Q279R-containing Q279R-K7. After 24 hours, cells were harvested and the splicing pattern of each minigene was examined by RT-PCR analysis of the transcripts. Exons are represented by boxes and introns by lines. The primers used for RT-PCR are indicated at each end of the splicing K7. B- Total RNA extracted from transfected HeLa cells was amplified with HG1S and HG3AS. Plasmidic DNA Q279Q-K7 and Q279R-K7 (pDNA) were also amplified as a control. The band obtained for Q279Q-K7 transfected cells (RT+) is of expected size, in contrast to the band obtained in the case of Q279R-K7 (RT+) transfected cells, which is of lower molecular weight. RT- serves as a negative control: the reverse transcription reaction was performed without any enzyme. In the two RT- fractions, the amplification of about 900-bp is due to plasmidic DNA contamination.</p>
                </figcaption>
              </figure>
              <p id="s4s3p2">As shown in Figure 4B, the major product of Q279Q-K7 is of the expected size and the two globin exons and the fah exon 9 are correctly spliced together. This was confirmed by sequencing. In contrast, the major product of Q279R-K7 is of lower size. In this case, the β-globin exons are joined together and exon 9 is skipped from the mature transcript. These results strongly argue that the Q279R mutation acts as a splicing mutation. Of importance is the presence of a small but significant amount of spliced forms containing the mutated exon 9 (Figure 4B). This observation is consistent with the presence of a low fraction of normally spliced mRNA species in the patient specimens (see above).</p>
            </div>
          </section>
        </div>
      </section>
      <section data-type="">
        <h2 id="s5h1">Discussion</h2>
        <div class="content">
          <p id="s5p1">The observation of FAH mosaicism in the liver of a HTI patient led us to suspect that reversion of one of the mutations might have occurred, as reported in patients with other mutations [9, 16]. This proved to be the case as confirmed by the mutation detection test in different regions of the liver. The heterozygocity pattern found in the tumoral region was no longer present in the non-tumoral normal looking section of the patient liver. The site-specific reversion of the Q279R mutation in this region was also demonstrated by direct sequencing. This event leads to both the production of normal sized FAH mRNA and expression of functional FAH, as shown by western blot and enzymatic activity measurements. FAH positive cells have been reported to show a selective growth advantage in vivo. Overturf et al. [19] showed that the introduction of as few as 1000 normal or corrected hepatocytes in a FAH knock-out mice model led to an extensive repopulation of the liver. Therefore, in man, a reversion event might occur in a cell, which expands and forms FAH expressing nodule. The amount of FAH being expressed by those reverted cells appears to be sufficient to reduce the symptoms of HTI. Kelsey et al. [20] showed that the expression of levels as low as 2% of the FAH level in normal liver was sufficient to rescue the lethal phenotype in the albino mouse model of HTI.</p>
          <p id="s5p2">In the present patient, cancer is only present in non-FAH expressing cells. In a mouse model of HTI where the mouse lethal phenotype was rescued by gene therapy with either retroviruses or adenoviruses harboring FAH, tumors and dysplasia were always observed in FAH negative areas [19, 21]. This is consistent with studies showing that the toxic metabolite in HTI, FAA, is indeed mutagenic [6] and can induce mitotic abnormalities and genome instability [22]. FAH-negative cells may be more prone to cancer development since FAA can accumulate in these cells while restoration of FAH activity by mutation reversion should remove this toxic metabolite in the reverted cells thereby reducing the risk of carcinogenesis.</p>
          <p id="s5p3">The present data show that a dysfunctional allele would not be transcriptionally silenced in favor of a functional one since in the reverted region of the liver, the IVS6-1g-&gt;t transcripts are still detectable. As shown here, the normal size transcript produced due to reversion of the Q279R mutation seems to be sufficient to allow FAH expression in the non-tumoral liver nodules.</p>
          <p id="s5p4">The Q279R mutation has recently been identified in the patient and her father [15], as a missense mutation. However, the complex pattern of transcripts obtained by RT-PCR in different regions of the liver reported here suggests that the Q279R mutation could act as a splicing mutation in vivo. In fact, we identified at least six alternative transcripts in the tumoral region and in the patient's fibroblasts (Figures 2 and 3). Four of these are produced as a result of the IVS6-1g→t mutation; among these, three are identical to those previously reported in other patients harboring this mutation [17]. We also identified an unreported high molecular weight transcript, with a retention of intron 6, as a minor product. The Q279R mutation seems to be responsible for the other two transcripts. mRNA analysis in the compound heterozygous patient also revealed that this exon 9 mutation partially affects the splicing at the 5' splice site both in vivo and in transfected HeLa cells, whereas the IVS6-1g-&gt;t mutation seems to completely abolish the normal splicing. Among the abnormal splicing isoforms produced from either the Q279R or the IVS6-1g-&gt;t alleles, three contain premature termination codons (PTCs) and are expressed at low levels when compared to the other major in-frame mRNA transcripts (Figure 3). These PTC-containing transcripts are likely to be targeted for nonsense-mediated mRNA decay (NMD; [23,24,25]).</p>
          <p id="s5p5">The nucleotide change in the Q279R mutation (836A-&gt;G) is located at position -2 of the 5' (donor) splice site. The wild-type adenine at this position is present in 64 % of all 5' splice sites [26]. The A-&gt;G mutation most likely weakens the splice site, but does not completely abolish its utilization as demonstrated in the transfection experiments (Figure 4).</p>
          <p id="s5p6">Structural analysis of FAH also suggested that the Gln-&gt;Arg amino acid replacement might be structurally tolerated and would not interfere with the structure as predicted by circular dichroism [27]. Indeed functional analysis of a Q279R-containing FAH variant showed that the mutated protein was enzymatically active [27]. Altogether, these data lead to the conclusion that while Q279R introduces a missense mutation at the cDNA level, it likely acts as a splicing mutation in vivo thereby inhibiting the production of normal FAH mRNA and the corresponding protein. This is supported by the analysis of transcripts in the liver and by the minigene assay, but it cannot be excluded that the mutation is leaky as a low amount of normal transcripts is found in both cases. Whether this low amount of FAH mRNA is translated in sufficient functional protein to alleviate the clinical phenotype remains uncertain. While FAH could not be detected in the tumor tissue by western blot analysis, the presence of low amounts of FAH undetectable by the blot assay cannot be excluded.</p>
        </div>
      </section>
      <section data-type="">
        <h2 id="s6h1">Conclusion</h2>
        <div class="content">
          <p id="s6p1">This clinical case of HTI is particularly interesting since the patient ceased dietary restriction at age 14 and showed no evidence of sickness until 37 of age. One explanation for this particularly mild phenotype was that the missense Q279R allele was enzymatically functional like in the case of the pseudodeficient allele (R341W), which produces low levels of catalytically active FAH [13, 27, 28]. Our data rather suggest that the Q279R mutation disturbs mRNA splicing in vivo with the result that very little if any functional mRNA is made. Finally, analysis of transcripts in a FAH expressing region suggests that mutation reversion in the Q279R allele leads to the production of functional FAH and clonal expansion of the corrected cells and that this restoration may account for the mild phenotype observed.</p>
        </div>
      </section>
      <section data-type="materials|methods">
        <h2 id="s7h1">Materials and Methods</h2>
        <div class="subsections">
          <section data-type="">
            <h3 id="s7s1h1">Case History</h3>
            <div class="content">
              <p id="s7s1p1">The full clinical history of this patient is reported elsewhere [15]. Briefly, the patient presented abnormal liver functions at 9 months of age. She was diagnosed for chronic tyrosinemia at age 6 with elevated levels of tyrosine metabolites. She followed a restrictive diet low in phenylalanine, tyrosine, and methionine until she was 14 years of age. She was regularly monitored and, except for high levels of tyrosine, was considered healthy. At age 37, the patient was diagnosed with hepatocarcinoma and underwent partial liver resection. She subsequently had liver transplantation. Frozen and paraffin-embedded liver specimens from a non-tumor and two tumor regions (T1 and T2) were obtained and analysed. The patient is heterozygous for mutations IVS6-1g-&gt;t (maternal) and Q279R (paternal) [15].</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s7s2h1">Detection of the Q279R and IVS6-1g-&gt;t mutations in liver samples</h3>
            <div class="content">
              <p id="s7s2p1">The presence of the Q279R and IVS6-1g-&gt;t mutations in various regions of the liver (non-tumoral and tumoral) was verified by PCR analysis. DNA was extracted from microdissected liver sections as previously described [9]. The IVS6-1g-&gt;t mutation was detected with Alu I (New England Biolabs) using a protocol described in Poudrier et al., 1999 [29]. For the Q279R mutation, a modified (bold) forward primer RT75 (5' TGC TGT GCC CAA CCC GAA CC 3') was designed such as to create, together with the mutation itself (836A-&gt;G), a Msp I (C/CGG) restriction site. Hence the 78-bp PCR product obtained using RT75 and the reverse primer Tan77 (5' CTG GGT TTG GGA GCT TCC TGC AGA 3') generates a 58-bp and a 20-bp fragment specifically from the mutated allele upon Msp I digestion, whereas the normal allele product remains uncleaved. PCR amplification started with DNA denaturation at 95°C for 5 min, followed by 30 cycles of: 95°C for 30 s, 63°C for 30 s and 72°C for 30 s. The last step consisted of an elongation period of 2 min at 72°C. The PCR product was then digested with Msp I (New England Biolabs), loaded on a 12 % polyacrylamide gel and stained with ethidium bromide. The 20-bp Msp I fragment is not detectable with this method. Mutation detection tests were performed in triplicates on each region of the liver with identical results.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s7s3h1">RT-PCR</h3>
            <div class="content">
              <p id="s7s3p1">RT-PCR was performed on RNA extracted from dissected liver sections, EBV transformed lymphocytes [30], T19 EBV control cells and skin fibroblasts of a different patient homozygous for IVS6-1g-&gt;t. RNA was extracted using Trizol Reagent (GibcoBRL) as described by the manufacturer. Two micrograms (2 μg) of total RNA was incubated at 70°C for 10 min with 500 ng of oligo d(T)12-18 and then chilled on ice. Reverse transcription was performed at 42°C for 50 min by adding 50 mM Tris-HCl (pH 8.3), 75 mM KCl, 3 mM MgCl2, 10 mM DTT, 10 mM of each dNTP and 200 U SuperScript II reverse transcriptase (GibcoBRL). Raising the temperature to 70°C for 15 min stopped the reaction. Two microliters (2 μl) of this cDNA product was used to amplify FAH from exon 2 (forward primer Tan 5, 5'CTG GAC CTC AGC ATC ATC 3') to exon 12 (reverse primer Tan 51, 5' CAG GTT GCA GCC GTT GAC 3'). The conditions used were 95°C for 5 min, 55°C for 5 min and 72°C for 40 min; 35 cycles of the following program: 95°C for 30 s, 55°C for 30 s and 72°C for 1 min 30 s; and a final extension of 15 min at 72°C. The amplification product was next digested with Sph I (New England Biolabs) and Pst I (New England Biolabs) and cloned in pET30FAH [27]. Different clones were selected and sequenced on a 373A DNA sequencer (Applied Biosystems). A shorter fragment was also amplified using the forward primer RT76 (5 'CGT GCC TCC TCT GTC GTG 3 ') located in exon 6 and the reverse primer Tan51. The amplification was performed as follows: 35 cycles at 95°C for 30 s, 61°C for 30 s and 72°C for 1 min; and a final extension of 15 min at 72°C.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s7s4h1">Minigene constructs and analysis</h3>
            <div class="content">
              <p id="s7s4p1">A fragment surrounding exon 9 was amplified with the forward primer Tan76 (5'GCA GAT CAG CTC CAG ATT CTA ATG 3') and the reverse primer RT80 (5'GCT AGC TGG CAT GTC CAC 3') which contains a Nhe I restriction site at its 5' end. This amplification was performed on a normal individual and on the patient's genomic DNA using the Expand High Fidelity PCR system (Roche Molecular Diagnostics). The conditions used were of 95°C for 5 min, followed by 35 cycles of: 95°C for 30 s, 57°C for 30 s and 68°C for 1 min 30 s. The last step consisted of an elongation period of 10 min at 68°C. The fragment obtained was extracted from a 1% agarose gel using the UltraClean TM DNA purification kit (Mo Bio Laboratories, Inc.). One microliter (1 μl) of purified fragment was amplified with RT79 (5' GGT AAC CAT TGA ATG CTC TTG C 3'), which was modified at its 5' end to create a Bst EII restriction site and RT80 (see above). The PCR amplification started with a denaturation step of 5 min at 95°C followed by 30 cycles of: 95°C for 30 s, 56°C for 30 s and 68°C for 1 min. The final elongation step was of 5 min at 68°C. The 387-bp expected fragment was purified from a 2% agarose gel and digested with Bst EII (New England Biolabs) and Nhe I (New England Biolabs) and cloned in K7, a minigene cassette containing human β-globin exons 1 and 3 (Deguillien, M., Huang, S. C., Morinière, M., Dreumont, N., Benz, E. J. Jr and Baklouti, F., unpublished). The two minigenes obtained (Q279Q-K7 and Q279R-K7) were sequenced on a 373A DNA sequencer. One point six micrograms (1.6 μg) of each construction were cotransfected with 0.4 μg of pEGFP-NI (Clontech) using Lipofectamine (Gibco BRL). The efficiency of cotransfection was respectively of 20% and 30% for Q279Q-K7 and Q279R-K7. After 24 hours, cells were harvested and lysed in Trizol Reagent. Total RNA was digested with 1 μl of RNase-free DNase I (Boehringer-Manheim) for 30 min at 37°C. RNA was extracted for a second time with 1 ml Trizol Reagent and resuspended in 50 μl of RNA storage solution (Ambion, Inc.). Reverse transcription was performed on 2 μg of total RNA in a final volume of 40 μl with 2 μl of the M-MLV reverse transcriptase (Gibco-BRL) and 1 μg of oligo (dT) 12-18, according to the manufacturer's recommendations. Two microliters (2 μl) of reverse transcription product were amplified using Taq polymerase (Pharmacia Biotech.). The forward primer HG1S (5' TAG CGG CCG CAT ATA AGT CAG GGA AGA GCC A 3') and the reverse primer HG3AS (5' TAT CTA GAG TTG GAC TTA GGG AAC AAA 3') are located at each extremity of the splicing cassette. The PCR amplification was performed as follows: a denaturation step of 5 min at 95°C, followed by 35 cycles of: 95°C for 30 s, 61°C for 30 s and 72°C for 1 min. The final elongation step was 10 min a 72°C. The splicing products were extracted from the agarose gel and sequenced.</p>
            </div>
          </section>
        </div>
      </section>
      <section data-type="">
        <h2 id="s8h1">List of abbreviations used</h2>
        <div class="content">
          <p id="s8p1">FAA, fumarylacetoacetate; FAH, fumarylacetoacetate hydrolase; HTI, hereditary tyrosinemia type I; MAA, maleylacetoacetate; NMD, nonsense-mediated mRNA decay; PTC, premature termination codon; RT, reverse transcription; Sac, succinylacetone; wt, wild-type.</p>
        </div>
      </section>
    </article>
  </body>
</html>
