<!DOCTYPE html >
<html id="ahItoJKHuYJIWRYrjdPchjjiQHei-PMC2324116" data-origid="PMC2324116" class="anndoc" data-anndoc-version="2.0" lang="" xml:lang="" xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="UTF-8"/>
    <meta name="generator" content="com.jmcejuela.anndoc.v2.bio.NcbiJournalArticleParser_v0_4$"/>
    <meta name="dcterms.source" content="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2324116/"/>
    <title>ahItoJKHuYJIWRYrjdPchjjiQHei-PMC2324116</title>
  </head>
  <body>
    <article>
      <section data-type="title">
        <h2 id="s1h1">Five novel mutations of the FRMD7 gene in Chinese families with X-linked infantile nystagmus</h2>
      </section>
      <section data-type="abstract">
        <h2 id="s2h1">Abstract</h2>
        <div class="subsections">
          <section data-type="">
            <h3 id="s2s1h1">Purpose</h3>
            <div class="content">
              <p id="s2s1p1">Infantile nystagmus (IN) is an inherited disorder characterized by bilateral ocular oscillatory movements. Recently, mutations in FRMD7 were found to be responsible for X-linked idiopathic infantile nystagmus . We investigated the role of the FRMD7 gene mutations in seven Chinese families with infantile nystagmus.</p>
            </div>
          </section>
          <section data-type="methods">
            <h3 id="s2s2h1">Methods</h3>
            <div class="content">
              <p id="s2s2p1">Linkage analysis was performed with fluorescently labeled microsatellite markers, DXS1001 and DXS1047. Analysis of FRMD7 gene mutations was performed by direct sequence to the whole coding regions and exon-intron boundaries of FRMD7 gene in all affected members in seven families with IN.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s2s3h1">Results</h3>
            <div class="content">
              <p id="s2s3p1">Five novel FRMD7 gene mutations, 70 G&gt;T(p.G24W) in exon 2, c.689–690delAG (p.Ser232del) in exon8, c. 782G&gt;A (p.R260Q) and c. 812G&gt;T (p. C271F) in exon 9, and c. 910C&gt;T (R303X) in exon 10, were identified in five of seven Chinese families with X-linked infantile nystagmus. But we didn’t detect the FRMD7 gene mutation in one of seven families, although a positive LOD score of 2.42 (θmax=0.1) was obtained at DXS1047 . We also found the same mutation, which is c. 782G&gt;A (p.R260Q), occurred in two different families.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s2s4h1">Conclusions</h3>
            <div class="content">
              <p id="s2s4p1">This is first report that five kinds of FRMD7 gene mutation types occurred in Chinese families with IN, which further support that FRMD7 gene mutations are the underlying pathogenesis of the molecular mechanism for infantile nystagmus.</p>
            </div>
          </section>
        </div>
      </section>
      <section data-type="intro">
        <h2 id="s3h1">Introduction</h2>
        <div class="content">
          <p id="s3p1">Infantile nystagmus (IN) is a relatively common ocular motor disorder characterized by rapid to and fro oscillations of the eyes. It usually presents itself at birth or develops within the first few months of life. The etiology of infantile nystagmus is not clear and usually occurs without sensory defect. This is different from “sensory defect nystagmus,” which is caused by inherited ocular diseases including albinism, achromatopsia, Leber congenital amaurosis, congenital cataract, and anterior segment dysgenesis.</p>
          <p id="s3p2">Between 7% and 30% of IN patients have a positive family history [1] in which inheritance may be autosomal dominant, autosomal recessive, or X-linked. Five congenital nystagmus loci have been mapped including autosomal dominant NYS2 on chromosome 6p12 [2] (OMIM 164100), autosomal dominant NYS3 on chromosome 7p11.2 (OMIM 608345) [3], autosomal dominant NYS4 on chromosome 13q31-q22 (OMIM 193003) [4], and X-linked NYS1 on chromosome Xp11.4-p11.3 [5] (OMIM 310700) and Xq26-q27 [6]. X-linked inheritance is suggested to be the most common mode with incomplete penetrance in females [7].</p>
          <p id="s3p3">Recently, mutations in FRMD7 were found to be responsible for X-linked infantile nystagmus [8-10, 11,]. In this study, we report five novel mutations in FRMD7 in Chinese families with infantile idiopathic nystagmus.</p>
          <figure data-origid="f1">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2324116/#f1">Figure 1</a>
            <figcaption>
              <p id="s3f1p1">Pedigrees of seven families with X-linked congenital nystagmus. Squares indicate males, circles indicate females, slashed symbols indicate deceased, black symbols indicate affected individuals, unfilled symbols indicate unaffected individuals, and unaffected, obligate carriers are represented by a dotted circle.</p>
            </figcaption>
          </figure>
        </div>
      </section>
      <section data-type="methods">
        <h2 id="s4h1">Methods</h2>
        <div class="content">
          <p id="s4p1">This study was approved by the Ethics Committee of the Tianjin Eye Hospital, Tianjin, China and was conducted according to the Tenets of the Declaration of Helsinki. Seven families with X-linked infantile idiopathic nystagmus were investigated .To exclude ocular or neural visual pathway abnormalities, a detailed clinical examination including visual acuities, slit examination of the lens, examination of the vitreous, fundus, electroretinogram (ERG), and visual evoked potential (VEP)was performed for those patients who could accept clinical examination at Tianjin Eye Hospital.</p>
          <p id="s4p2">After informed consent was obtained, a 5 ml blood sample was collected from every participant and genomic DNA was isolated from peripheral blood lymphocytes using a DNA isolation kit (Roche Biochemical Inc., Basel, Switzerland.).</p>
          <p id="s4p3">Linkage analysis was performed on those pedigrees with a sufficient number of affected individuals. Fluorescently labeled microsatellite markers, DXS1001 and DXS1047, were analyzed by polymerase chain reaction (PCR) amplification in a 10 μl reaction mixture containing 50 ng of genomic DNA, 1X PCR buffer, 2.0 mM MgCl2, 0.2 mmol/l of each dNTP, 5 pmol/l of each forward and reverse primer, and 0.2 U of AmpliTaq Gold DNA polymerase. Markers were genotyped using an ABI 3130 Genetic Analyzer (Applied Biosystems, Foster City, CA). Genotypes were analyzed using the GeneMapper 3.7 Software (Applied Biosystems).</p>
          <p id="s4p4">Two-point LOD scores were calculated with easy Linkage plus v4.0 beta software, assuming an X-linked dominant trait with 100% penetrance on male and 50% in female, a disease allele of 0.001, and arbitrary marker allele frequencies of 1/n with n being the number of alleles observed for that specific marker.</p>
          <p id="s4p5">Mutation screening was performed by direct DNA sequence analysis. Whole coding regions and exon-intron boundaries of the FRMD7 gene were PCR-amplified in 50 μl of a standard PCR buffer containing 1.5 mmol/l MgCl2, 0.2 mmol/l of each dNTP, 0.5 μmol/l of each primer, 1 U of Taq polymerase, and 50 ng of DNA. The amplification program was an initial 2 min of denaturation at 98 °C followed by 30 cycles of 30 s at 94 °C, 30 s at 55 °C, 1 min at 72 °C, and a final 7 min extension step at 72 °C. The PCR products were extracted using the QIAquick Gel Extraction Kit (Qiagen, Valencia, CA). DNA sequencing analysis was performed using the BigDye Terminator Cycle Sequencing v3.1 kit on an ABI PRISM 3130 Genetic Analyzer (Applied Biosystems). Sequences were aligned using Chromas software (Conor McCarthy, Griffith University, Brisbane, Queensland,Australia). The reference cDNA sequence was obtained from the GenBank database (NM_194277.1), and +1 corresponds to the A of the ATG translation initiation codon.</p>
          <p id="s4p6">All four SNPs are in the region 60 kb upstream of the start of the FRMD7 gene transcript.</p>
          <figure class="table" data-origid="t1">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2324116/#t1">Table 1</a>
            <figcaption>
              <h5 id="s4f1h1">Sequence analysis of four SNPs in two affected males from two families.</h5>
            </figcaption>
          </figure>
        </div>
      </section>
      <section data-type="results">
        <h2 id="s5h1">Results</h2>
        <div class="content">
          <p id="s5p1">Four large families with infantile nystagmus were analyzed by linkage analysis. In family A (from the Sandong province), 16 of the 46 living family members are affected including 10 males and six females. In addition, three females are obligate unaffected carriers. No male-to-male transmission was observed. The proband is a nine-year-old girl (Figure 1A, individual IV12) who had nystagmus since three to four months after birth. She has a horizontal jerk ocular oscillation and a face turn to left with the null point to the right. The corrected visual acuity of both eyes at distance is 1.0. Her father is 32 years old with nystagmus since he was three to four months old. He has a horizontal ocular oscillation with a torsional component and with little head posture and head shaking. His corrected visual acuities for both eyes are 0.5. Other patients have various reduced visual acuity ranging from 0.2 to 1.0. Five of them have head posture, and two of them have head oscillations. The penetrance appears to be complete in males and about 60% in the females.</p>
          <figure data-origid="f2">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2324116/#f2">Figure 2</a>
            <figcaption>
              <p id="s5f1p1">Sequencing results of affected heterozygous females from families A and D and of affected hemizygous males from families B, E, F, and G . Five types of FRMD7 mutations were identified in six families, which were c. 812G&gt;T (p. C271F) in family A, c.689–690delAG (p.Ser232del) in family B, c.70 G&gt;T(p.G24W) in family D, c. 782G&gt;A (p.R260Q) in both families E and F, and c. 910C&gt;T (R303X) in family G.</p>
            </figcaption>
          </figure>
          <p id="s5p2">Significant LOD scores were obtained with polymorphic markers DXS1047 (Zmax=8.55, θmax=0) and DXS1001 (Z max=5.55, θ max=0). Direct DNA sequence analysis from all patients and obligate carriers reveals a c. 812G&gt;T missense mutation in exon 9, which would result in a substitution of the cysteine residue at codon 271 by a phenylalanine residue (C271F, Figure 2A).</p>
          <p id="s5p3">In family B (from the Henan province), nine patients (three males and six females) show horizontal jerk ocular oscillation. The affected males have more serious ocular oscillation than the affected females. Five females are obligate unaffected carriers. The proband is a nine-year-old boy (individual IV2 in Figure 1B) who developed nystagmus at three to four months of age with 0.7 corrected visual acuity in both eyes at distance. His mother is an asymptomatic obligate carrier. Other patients have various reduced visual acuity ranging from 0.4 to 1.0. A significant LOD Score of 3.61 (θmax=0) was obtained with polymorphic marker, DXS1047, and a LOD score of 2.03 (θmax=0.1) was obtained with DXS1001. Direct DNA sequence analysis of FRMD7 from all patients and obligate carriers revealed a 2 bp deletion (AG, 689–690) in exon 8 that was not present in unaffected family members (Figure 2B). The deletion is predicted to result in an aberrant truncated protein.</p>
          <p id="s5p4">Family C was from Liaoning province, and 12 patients (six males and six females) in this family were found with horizontal jerk ocular oscillations but without head posture. Three females are obligate unaffected carriers. The proband is an eight-year-old boy (individual V7 in Figure 1C) who developed nystagmus at three to four months of age with 0.3 corrected visual acuity of both eyes at distance. Other patients have various reduced visual acuity ranging from 0.2 to 1.0. Positive LOD scores of 2.42 (θmax=0.1) and 0.65 (θmax=0.25) were obtained with polymorphic markers, DXS1047 and DXS1001, respectively. No mutations were detected in the FRMD7 gene, although all coding regions and exon-intron boundaries were sequenced.</p>
          <p id="s5p5">In Family D (from Wuhan province), eight patients (two males and six females) show horizontal jerk ocular oscillation. The proband is a nine-year-old boy (individual III5 in Figure 1D), and his mother is the only asymptomatic obligate carrier in all the female members of this family. A positive LOD score of 1.57 (θmax=0) was obtained with polymorphic marker DXS1047, and a LOD score of 1.31 (θmax=0) was obtained with DXS1001. Sequence analysis revealed a c. 70G&gt;T missense mutation in exon 2 in all patients and obligate carriers, which would result in a substitution of the glycine residue at codon 24 by a tryptophan residue (Figure 2D).</p>
          <figure class="table" data-origid="t2">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2324116/#t2">Table 2</a>
            <figcaption>
              <h5 id="s5f2h1">Mutations in seven families and the significant LOD score in four families.</h5>
            </figcaption>
          </figure>
          <p id="s5p6">Families E and F are from Tianjin, and there is no indication by history that they are related. There is an insufficient number of available affected family members in these two families to provide a significant LOD score. A direct sequence of all affected patients in families E and F revealed the same sequence change in exon 9, a c. 782G&gt;A missense mutation, which would result in substitution of the arginine residue at codon 260 by a glutamine residue (R260Q, Figure 2E,F).</p>
          <p id="s5p7">To examine the possibility of a common origin between families E and F, four single nucleotide polymorphisms (SNPs; rs2180237, rs1569893, rs2748723, and rs2748724) in and around FRMD7 were genotyped in affected males, individual II1 of family E and individual IV1 of family F (Table 1). The haplotypes of these two affected males and by extension that in their respective families is quite distinct. This suggests that the same mutation, R260Q, occurred independently in these two families rather than descending from a common ancestor.</p>
          <p id="s5p8">Sequence analysis of all patients and obligate carriers in family G revealed a c. 910C&gt;T nonsense mutation in exon 10, which would result in substitution of the arginine residue at codon 303 by a stop codon (R303X, Figure 2G).</p>
          <p id="s5p9">The maximum LOD scores and mutations identified for each family are shown in Table 2. None of these five mutations were detected in 50 male and 50 female unaffected control individuals when tested by single strand conformation polymorphism (not shown). These results suggest that c.70 G&gt;T (p.G24W) in exon 2, c.689–690delAG (p.Ser232del) in exon 8, c. 782G&gt;A (p.R260Q) and c. 812G&gt;T (p. C271F) in exon 9, and c. 910C&gt;T (R303X) in exon 10 are novel mutations in the FRMD7 gene.</p>
        </div>
      </section>
      <section data-type="discussion">
        <h2 id="s6h1">Discussion</h2>
        <div class="content">
          <p id="s6p1">In this study, we identified five novel mutations in the FRMD7 gene. FRMD7, which has 12 exons and encodes 714 residues, is a previously unidentified member of the protein 4.1 superfamily. In situ hybridization experiments show that the FRMD7 gene is expressed in the ventricular layer of the forebrain, midbrain, cerebellar primordium, spinal cord, and the developing neural retina in embryos 56 days post-ovulation. In earlier embryos 37 days post-ovulation, the expression is restricted to the midbrain and hindbrain, which are known to influence the motor control of eye movement [8].</p>
          <figure data-origid="f3">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2324116/#f3">Figure 3</a>
            <figcaption>
              <p id="s6f1p1">Alignment of FRMD7 amino acids. The alignment of amino acids around p.G24, p. R260, and p. C271 of FRMD7 revealed evolutionary conservation of the residue from Canis familiaris, Rattus norvegicus, Mus musculus, and Gallus gallus to Homo sapiens.</p>
            </figcaption>
          </figure>
          <p id="s6p2">Previously, 32 mutations in the FRMD7 gene have been identified in different ethnic groups [8-12] including G24R in exon 2 and C271Y in exon 9, which is similar to G24W and C271F identified in families D and A, respectively. We identified five novel mutations in all but one family investigated here, and family C in which no mutation was identified shows linkage to the FRMD7 region. Missense, nonsense, and deletion mutations were observed. All changes we identified were novel mutations. The three missense mutations identified are G24W, C271F, and R260Q of which G24W and C271F occur at known mutation sites but substitute a different amino acid residue.</p>
          <p id="s6p3">The FRMD7 protein is most highly conserved near the NH2-terminus, which includes the B41 and FERM-C domains. The B41 domain is located at residues 1–192, and the FERM-C domain is located at residues 186–279 in FRMD7. Modeling of the core domain of the cytoskeletal protein 4.1R suggests that mutations at codon 24 and codon 271 are likely to destabilize the protein by the introduction of larger amino acids within restricted areas of the protein [8]. Multiple alignment of the region shows that codon 24, codon 260, and codon 271 are located within highly conserved regions that are invariant in Canis familiaris, Rattus norvegicus, Mus musculus, and Gallus gallus, suggesting that these amino acids are critical to the normal function of the protein (Figure 3).</p>
          <p id="s6p4">We also found a nonsense mutation, R303X, in exon 10 and a novel deletion mutation, p.S232Ffs2, in exon 8. Either of these two changes alters the length of the open reading frame, creating an aberrant truncated protein.</p>
          <p id="s6p5">In one of our familial cases, no mutation was found in the coding region and exon-intron boundaries of FRMD7, although FRMD7 was sequenced in all affected males and females. It is possible that congenital nystagmus in this family is caused by a mutation located in the promoter or other regulatory regions including those within introns or that there might be unknown splice variants that we didn’t detect with our experiment technique.</p>
          <p id="s6p6">As shown by the results above, we have further expanded the mutational spectrum of FRMD7 and confirmed that mutations in FRMD7 are a common cause for X-linked congenital idiopathic nystagmus, although no mutations have been detected in a small subset of families mapped on Xq26–27. Further biochemical studies of the mutations in FRMD7 will yield insight into its molecular mechanism underlying the pathogenesis of congenital idiopathic nystagmus.</p>
        </div>
      </section>
    </article>
  </body>
</html>
