<!DOCTYPE html >
<html id="aidy0SX6HhdUHgOhBUsSKLtNxlIC-PMC2836575" data-origid="PMC2836575" class="anndoc" data-anndoc-version="2.0" lang="" xml:lang="" xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="UTF-8"/>
    <meta name="generator" content="com.jmcejuela.anndoc.v2.bio.NcbiJournalArticleParser_v0_4$"/>
    <meta name="dcterms.source" content="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2836575/"/>
    <title>aidy0SX6HhdUHgOhBUsSKLtNxlIC-PMC2836575</title>
  </head>
  <body>
    <article>
      <section data-type="title">
        <h2 id="s1h1">CK2 phosphorylation-dependent interaction between aprataxin and MDC1 in the DNA damage response</h2>
      </section>
      <section data-type="abstract">
        <h2 id="s2h1">Abstract</h2>
        <div class="content">
          <p id="s2p1">Aprataxin, defective in the neurodegenerative disorder ataxia oculomotor apraxia type 1, resolves abortive DNA ligation intermediates during DNA repair. Here, we demonstrate that aprataxin localizes at sites of DNA damage induced by high LET radiation and binds to mediator of DNA-damage checkpoint protein 1 (MDC1/NFBD1) through a phosphorylation-dependent interaction. This interaction is mediated via the aprataxin FHA domain and multiple casein kinase 2 di-phosphorylated S-D-T-D motifs in MDC1. X-ray structural and mutagenic analysis of aprataxin FHA domain, combined with modelling of the pSDpTD peptide interaction suggest an unusual FHA binding mechanism mediated by a cluster of basic residues at and around the canonical pT-docking site. Mutation of aprataxin FHA Arg29 prevented its interaction with MDC1 and recruitment to sites of DNA damage. These results indicate that aprataxin is involved not only in single strand break repair but also in the processing of a subset of double strand breaks presumably through its interaction with MDC1.</p>
        </div>
      </section>
      <section data-type="">
        <h2 id="s3h1">INTRODUCTION</h2>
        <div class="content">
          <p id="s3p1">Aprataxin, defective in the human autosomal recessive disorder ataxia oculomotor apraxia type 1 (AOA1), is encoded by the APTX gene (1,2). AOA1 is a neurodegenerative disorder characterized by early onset cerebellar ataxia, oculomotor apraxia, hypoalbuminemia and late peripheral neuropathy (3). It resembles ataxia telangiectasia (A-T) in its neurodegenerative phenotype but lacks the extra-neurological features of A-T that include radiosensitivity, immunodeficiency and cancer predisposition (3,4). ATM, the protein defective in A-T, recognizes and is activated by DNA double strand breaks (DSB) to signal this damage to the DNA repair machinery and activate cell cycle checkpoints (5).</p>
          <p id="s3p2">Aprataxin contains three functional domains: an N-terminal FHA domain (6), a central histidine triad (HIT) domain (7) and a C-terminal zinc finger motif. Aprataxin interacts with XRCC1 and PARP-1, two key components of the DNA base excision repair machinery (8–10), suggesting a role for aprataxin in DNA single strand break (SSB) repair. Cells derived from AOA1 patients are sensitive to genotoxic agents that induce SSB in DNA (8–12). Consistent with the involvement of aprataxin in DNA repair, defective SSB repair has been reported in AOA1 cells in response to camptothecin, H2O2 and BSO (11–13). More recently aprataxin was shown to catalyze the nucleophilic release of adenylate groups covalently linked to 5′-termini of DNA molecules (14,15). Ligation of SSB or DSB involves the formation of an AMP-ligase complex, which subsequently transfers the AMP moiety onto the 5′ phosphate of the break site from which it is released after the formation of the phosphodiester bond (16). The AMP hydrolase activity of aprataxin appears to be important in resolving abortive DNA ligation intermediates that can form at ‘dirty’ or ‘complex’ SSB and potentially also at DSB formed by reactive oxygen species attack or clustered damage (15). The involvement of aprataxin in DNA repair is further supported by recent evidence of elevated levels of oxidative DNA damage in AOA1 cells coupled with a reduced expression of PARP-1, apurinic endonuclease 1 (APE1) and OGG1 (17). Furthermore, impaired base excision and gap filling repair efficiencies reveals a synergy between aprataxin, PARP-1, APE-1 and OGG1 in the DNA damage response and highlights both direct and indirect modulating functions for aprataxin on base excision repair (17). Although a role for aprataxin in the repair of abortive ligations at sites of DSB has been suggested by the interaction between aprataxin and XRCC4 (9) and the in vitro binding of aprataxin to DNA double strand ends (15,18), no direct evidence for its involvement in DNA double strand break repair has been reported (19).</p>
          <p id="s3p3">In response to DNA damage, many proteins involved in DNA damage signalling/repair, such as Mre11, NBS1 and 53BP1 quickly re-localize to nuclear foci (20,21). These foci co-localize with the phosphorylated form of histone H2AX (γH2AX), a well-established marker for DSB (22). While aprataxin interacts with XRCC1, XRCC4 and PARP-1, we have previously shown that in contrast to the other DNA repair proteins, it does not form detectable nuclear foci after DNA damage exposure induced by either low linear energy transfer (LET) ionizing radiation (IR) or H2O2 (8). The dynamics of recruitment of DNA repair proteins to sites of DNA damage has been investigated by laser irradiation combined with photosensitizers (23–25). Hirano et al. (2007) employed this approach to show that aprataxin was recruited in vivo to SSB. While laser-induced DNA damage produces SSB and DSB, the relative distribution and the density of the damage are greatly influenced by the laser type, energy output and the type of photosensitizer (26). Biological imaging of charged particle tracks represents an alternative approach for investigating the association of repair proteins with chromatin (27,28). This technology offers the advantage of defined intrinsic physical properties of the heavy ion beams and allows the density of lesions along the track to be varied with ion species with different LET values. Using a remote-controlled microscope coupled to the beamline that allows the acquisition of fluorescence images of living cells, in real-time during ion irradiation, we observed that GFP-aprataxin is rapidly localized to DNA damage induced by nickel-ion irradiation (8,28). To gain further insight how aprataxin is recruited to sites of DNA breaks, we investigated the in vivo recruitment of GFP-tagged aprataxin to localized DNA damage induced by high LET heavy ion irradiation. We demonstrate for the first time that aprataxin co-localizes and interacts with mediator of DNA damage checkpoint protein 1 (MDC1), a protein that amplifies ATM-dependent DNA damage signalling (29–35). Interaction between the two proteins is mediated through the FHA domain of aprataxin, which binds to a casein kinase II (CK2)-phosphorylated N-terminal region of MDC1. Mutation of aprataxin Arg29, a key residue conserved within FHA motifs (36), disrupted the interaction between aprataxin and MDC1, and abolished its recruitment to sites of DNA damage. These results together with those of Harris et al. (17) highlight the importance of the aprataxin FHA domain in regulating protein–protein interactions and targeting aprataxin to sites of DNA damage.</p>
        </div>
      </section>
      <section data-type="materials|methods">
        <h2 id="s4h1">MATERIALS AND METHODS</h2>
        <div class="subsections">
          <section data-type="">
            <h3 id="s4s1h1">Cell lines, inhibitors</h3>
            <div class="content">
              <p id="s4s1p1">Control lymphoblastoid cell line (LCL) (C3ABR) and cervical adenocarcinoma cells (HeLa) were cultured in RPMI 1640 (GIBCO BRL) and DMEM (GIBCO BRL), respectively containing 10% foetal calf serum (FCS) (Lonza), 2 mM l-glutamine (Life Technologies), 100 U/ml penicillin (GIBCO BRL), 100 U/ml streptomycin (GIBCO BRL) and maintained in a humidified incubator at 37°C/5% CO2. Wild type, Aptx−/− and Mdc1−/− mouse embryonic fibroblasts (MEF), and human AOA1 fibroblasts (FD105) hTERT-transformed (Gift from Prof. Keith Caldecott, University of Sussex, UK) were cultured in DMEM as described earlier. CK2 inhibitors 4,5,6,7-tetrabromotriazole (TBB), and 2-dimethylamino-4,5,6,7-tetrabromo-1H-benzimidazole (DMAT) were purchased from Calbiochem and dissolved in DMSO.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s2h1">Aprataxin-GST and MDC1-GST pull down assays</h3>
            <div class="content">
              <p id="s4s2p1">Aprataxin-GST proteins have been described earlier (8,37). Pull down assays, were performed as described in Supplementry Data. Proteins were separated on 5% and 10% SDS–PAGE, transferred onto nitrocellulose membranes (Hybond C, Amersham) and detected using the relevant antibody. Aprataxin antibodies have been described earlier (8,37). Sheep polyclonal MDC1 antibody was produced against GST-MDC1 fragments and purified by affinity column against GST only and the antigen using standard protocols. For immunoblotting, antibodies were used at dilutions of: MDC1 (1/2000), aprataxin (1/2000), anti-GST (1/2000) followed by the relevant species-specific HRP-conjugated secondary antibody from Chemicon or Sigma (1/10 000).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s3h1">In vivo CK2 inhibition and pull downs</h3>
            <div class="content">
              <p id="s4s3p1">HeLa cells were mock treated (DMSO) or treated with a range of concentrations of specific CK2 inhibitor TBB, or DMAT for 8 h in a six well-plate format and then lysed. Pull down was performed as described in Supplementary Data.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s4h1">In vitro CK2 phosphorylation and pull down</h3>
            <div class="content">
              <p id="s4s4p1">A region of MDC1 containing CK2 sites (amino acids 150–350) was cloned into the bacterial GST expression vector pGEX-6P-1, expressed in E. coli (BL21DE3) and purified as described by the manufacturer. Two micrograms of each GST fusion protein was in vitro phosphorylated by incubation with 200 ng of human recombinant CK2 (New England Biolabs) in buffer P (20 mM Tris–HCl pH 7.5, 50 mM KCl, 10 mM MgCl2, 200 µM ATP) for 1 h at 37°C. Once phosphorylated, GST fusion proteins were used in pull-down assays mixed with 2 mg of HeLa whole cell extracts (WCE) as described earlier.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s5h1">Immunoprecipitation and immunoblotting</h3>
            <div class="content">
              <p id="s4s5p1">Total cell extracts from control (C3ABR) cells were prepared as described in Supplementary Data and the equivalent of 1 mg of total protein was immunoprecipitated with non-specific serum (1 or 5 µg), aprataxin (1 µg) and MDC1 (5 µg) antibodies overnight at 4°C. 40 µl of protein G-Sepharose beads (Amersham) were added to the immunoprecipitation (IP) and incubated for 1 h on a rotating wheel at 4°C. IP were washed three times with lysis buffer and the beads were resuspended in 20 µl of sample loading buffer. Proteins were separated on 5% and 10% SDS−PAGE, transferred onto nitrocellulose membranes and detected using the relevant primary and secondary antibodies.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s6h1">Isothermal titration calorimetry</h3>
            <div class="content">
              <p id="s4s6p1">Phosphopeptides were synthesised by Dr W. Mawby (University of Bristol). Aprataxin FHA-phosphopeptide binding was quantified by isothermal titration calorimetry (ITC) using a Microcal Omega VP-ITC calorimeter (MicroCal Inc., Northampton, MA, USA). Protein was dialysed against ITC buffer (50 mM HEPES pH 7.5, 150 mM NaCl, 5 mM β-mercaptoethanol) and peptides were dissolved in the dialysis buffer. Experiments were carried out at 22°C and involved 30 successive 10 µl injections of 1 mM peptide solution into a sample cell containing 100 µM protein solution. Heats of dilution were subtracted and binding isotherms were plotted and analysed with MicroCal Origin version 7.0, assuming a single-site binding model.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s7h1">Crystallization and structure determination of the aprataxin FHA domain</h3>
            <div class="content">
              <p id="s4s7p1">Aprataxin FHA domain was crystallized by hanging-drop vapour diffusion. Protein at a concentration of 10 mg/ml in 50 mM HEPES pH 7.5, 150 mM NaCl, 5 mM β-mercaptoethanol was mixed with an equal volume of reservoir solution [0.2 M ammonium fluoride, 20% (w/v) PEG 3350] and incubated at 18°C. Thin plates grew within 1 week and were transferred into cryoprotectant (0.1M ammonium fluoride, 10 % PEG 3350, 25 mM HEPES pH 7.5, 75 mM NaCl, 25% PEG 400) and flash frozen in liquid nitrogen. Data were collected at 100 K on an Raxis-IV detector mounted on a Rigaku MicroMax-007HF rotating anode X-ray source, and integrated and scaled using DENZO and SCALEPACK (38). The structure was solved by molecular replacement using the coordinates of PDB entry 1YJM as a search model with AmoRe (39,40). Subsequent refinement was carried out at 1.65 Å using REFMAC5 (41) and manual model building in Coot (42). Data collection and refinement statistics are summarized in Supplementary Table S2. All structure figures were prepared with PyMol (http://www.pymol.org). Coordinates have been deposited in the Protein Databank: Accession code 3KT9.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s8h1">High LET heavy ion irradiation procedure</h3>
            <div class="content">
              <p id="s4s8p1">For heavy ion irradiation experiments, cells were seeded on sterilized glass coverslips and irradiated at the UNILAC beamline at GSI (Darmstadt, Germany) with high energy xenon ions (4.5 MeV/nucleon; LET 8815 keV/µm), krypton (5.4 MeV/nucleon; LET 5060 keV/µm), nickel (6.0 MeV/nucleon; LET 3430 keV/µm) or uranium ions (4.2 MeV/nucleon; LET 14925 keV/µm), respectively. Different heavy ion species were used for the recruitment studies since these experiments were carried out over several years and we have had to adhere to the schedule of radiation exposure at the Darmstadt GSI site in Germany. Directly before the irradiation, the coverslips were mounted into sample holders and placed in a medium filled tank. For the ion irradiation, the sample holders were automatically taken from the medium tank, exposed to the ion beam under a small angle of 15° between the axis of the ion beam and the plane of the cell monolayer and placed back again. For the applied fluence of 3 × 106 p/cm2, the whole procedure takes &lt;30 s.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s9h1">Immunofluorescence and microscopy</h3>
            <div class="content">
              <p id="s4s9p1">For immunostaining, cells grown on coverslips were washed with PBS, fixed in 2% formaladehyde/PBS for 20 min and permeabilized with PT-5 solution [PBS/0.5% Triton X-100] for 5 min. Non-specific binding was blocked by incubating with PTB-5 [PBS/0.5% BSA] for 20 min at room temperature. Coverslips were then incubated with mouse anti-γH2AX antibody (Upstate 1/500 in PTB-5) for 1 h at room temperature. Cells were stained with 5 µg/ml Alexa 568 goat anti-mouse F(ab)2 conjugate (Invitrogen) and the DNA counterstained with 1 µM ToPro3 (Invitrogen). Microscopy was performed on a Leica TCS NT confocal scanner equipped with an ArKr-Laser on the Leica DM IRBE inverted microscope (lens: HCX PlanApo 63x oil/NA1.32). Images were recorded as confocal stacks with optical sections separated by about 0.3 µm. Confocal images were displayed as maximum projections and assembled in Adobe Photoshop 7.0. Heavy ion irradiation procedure is described in Supplementary Data.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s10h1">In vivo real-time beamline microscopy and heavy ion irradiation</h3>
            <div class="content">
              <p id="s4s10p1">Cells were transfected with Amaxa Nucleofection technology (Amaxa, Koeln, Germany). Transfection of MEFs was performed with Amaxa Nucleofector Kit VDP-1004 and protocol A-023 according to the manufacturer instructions (Amaxa, Germany). Mdc1−/− and Mdc1+/+ cells transiently transfected with GFP-aprataxin were employed for real-time fluorescence intensity measurements. These cells were cultivated on polycarbonate foil (18-mm diameter; 40-µm thickness) and irradiated at the beamline microscope as described earlier (28).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s11h1">γH2AX foci repair kinetics</h3>
            <div class="content">
              <p id="s4s11p1">Analysis of γH2AX repair foci kinetics was performed as described earlier (43) and is described in Experimental Procedures in Supplementary Data.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s12h1">Analysis of SSB repair</h3>
            <div class="content">
              <p id="s4s12p1">SSB were induced by exposing confluent cells to low doses of H2O2 (50−100 µM) in RPMI 1640 medium without supplements for 15 min at 37°C. To remove H2O2, cells were washed two times with PBSCMF (140 mM NaCl, 3 mM KCl, 8 mM Na2HPO4 and 1 mM KH2PO4). The numbers of SSB were determined by an alkaline elution assay as described earlier (44). The numbers of SSB in untreated control cells were subtracted in all cases.</p>
            </div>
          </section>
        </div>
      </section>
      <section data-type="results">
        <h2 id="s5h1">RESULTS</h2>
        <div class="subsections">
          <section data-type="">
            <h3 id="s5s1h1">Recruitment of aprataxin to DNA damage and co-localization with MDC1 after high LET radiation exposure</h3>
            <div class="content">
              <p id="s5s1p1">A hallmark of DNA repair proteins is their capacity to re-localize in discrete aggregates or foci within the nucleus after DNA damage exposure. We have previously shown that GFP-aprataxin co-localized with XRCC1 along charged particle tracks (8). Irradiation of cells with heavy ions generates multiple damaged sites including DNA lesions like SSB, DSB and base modifications (45). Here, we have focused on sites of DSB that can be identified by the phosphorylation of histone H2AX (γH2AX), the earliest detectable marker for DSB (22) and by recruitment of MDC1 (30). To determine the requirements for aprataxin recruitment to complex DNA damage, cells were transfected with GFP-tagged aprataxin, subjected to heavy ion irradiation at a small angle (&lt;15°) from the horizontal and analysed by confocal fluorescence microscopy (Figure 1A). Expression and localization of GFP-aprataxin and GFP-MDC1 in unirradiated cells is shown in Supplementary Figure S1. High LET heavy ion irradiation at low specific energies produced discrete γH2AX aggregates/foci deposited along the particle trajectory (Figure 1B) as observed earlier (46). GFP-aprataxin was recruited to these sites of DSB as observed by the partial co-localization pattern between GFP-aprataxin and γH2AX (Figure 1B). To further confirm the recruitment of GFP-aprataxin to sites of DSB, we demonstrated co-immunostaining with MDC1 (Figure 1C), another DNA damage response factor that is recruited to DSB through binding to γH2AX (34). As was the case with γH2AX, co-localization was incomplete between aprataxin and MDC1 (Figure 1C). To confirm the recruitment of MDC1 to high LET heavy ion-induced DSB, we used a GFP-MDC1 fusion protein. As expected, GFP-MDC1 completely co-localized with γH2AX staining at sites of heavy ion-induced DSB (Supplementary Figure S2). To address possible artefacts due to either overexpression or the presence of a GFP tag attached to the N-terminus of aprataxin, immunostaining for endogenous aprataxin was also performed (Figure 1D). In this case also, endogenous aprataxin was recruited to sites of DSB as observed by partial co-localization with MDC1 along the particle tracks. To determine whether MDC1 mediates the recruitment of aprataxin to sites of DNA DSB, wild-type and Mdc1−/− MEFs were transiently transfected with GFP-aprataxin and analysed for protein track formation after heavy ion exposure. While aprataxin was recruited to DNA breaks in wild-type MEFs after heavy ion irradiation at 10 min post-irradiation (4.5 MeV/nucleon; LET 8815 keV/µm), no recruitment was observed in Mdc1−/− cells despite the formation of γH2AX tracks (Supplementary Figure S3A). Due to poor transfection efficiency of Mdc1 MEFs, the absence of aprataxin recruitment was only observed in a limited number of cells. Accordingly, to monitor the recruitment of aprataxin in MDC1-deficient cells over shorter time points (up to 4 min), we employed a remote-controlled system coupled to a beamline microscope that allows measurement of GFP fluorescence intensity at sites of DNA breaks in real-time during heavy ion irradiation (28). Both wild-type and Mdc1−/− MEFs were transiently transfected with GFP-aprataxin and subjected to high LET heavy ion irradiation as described earlier (28). Real-time imaging of GFP fluorescence revealed fast recruitment of aprataxin to sites of DNA damage within 1 min after irradiation in both Mdc1−/− and wild-type cells (Supplementary Figure S3B). Interestingly, fluorescence intensity at the sites of DNA damage drops out rapidly in Mdc1−/− compared with wild-type cells suggesting an increased rate of dissociation of GFP-aprataxin from sites of DNA DSB in these cells. This indicates that while MDC1 is not necessary for the initial recruitment of aprataxin to sites of DNA DSB, its absence affects the retention of aprataxin to these sites. On the other hand, loss of XRCC1 prevented any recruitment of aprataxin in cells exposed to high LET radiation (Supplementary Figure S4). Together, these results provide evidence that aprataxin localizes at least in part to sites of DNA DSB, and that MDC1 does not act to recruit aprataxin, but rather as part of a tethering platform for aprataxin and possibly other repair proteins, to ensure effective DNA DSB repair.</p>
              <figure data-origid="F1">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2836575/#F1">Figure 1.</a>
                <figcaption>
                  <p id="s5s1f1p1">Co-localization of aprataxin with γH2AX and MDC1 at sites of DSB. (A) Methodology employed to study the dynamic recruitment of GFP-tagged fusion proteins (aprataxin and MDC1) to localized DNA damage induced by heavy ion irradiation. Transfection of GFP constructs and irradiation procedures are described in Supplementary Data. (B) Recruitment of GFP-aprataxin to uranium (4.2 MeV/nucleon LET 14925 keV/µm) ions-induced DNA DSB in HeLa cells. DSB are visualized by γH2AX immunostaining and nuclei by ToPro3 staining. (C) Co-localization of GFP-aprataxin with MDC1 along particle tracks after uranium ions irradiation in HeLa cells. (D) Recruitment of endogenous aprataxin to DSB and co-localization with MDC1 along particle tracks after krypton ions irradiation (5.4 MeV/nucleon; LET 5060 keV/µm) revealed by immunostaining with anti-aprataxin and anti-MDC1 antibodies in human fibroblasts.</p>
                </figcaption>
              </figure>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s2h1">Interaction between aprataxin and MDC1</h3>
            <div class="content">
              <p id="s5s2p1">The co-localization of MDC1 and aprataxin to sites of DNA breaks prompted us to determine whether these two molecules associate. Using a combination of co-IP and GST pull-down assays we demonstrated an interaction between endogenous aprataxin and MDC1. MDC1 co-immunoprecipitated with aprataxin using a specific anti-aprataxin antibody while non-specific serum (Ig) failed to immunoprecipitate MDC1 (Figure 2A). Three discrete MDC1 bands were detected as has been reported earlier (30,34). This interaction was confirmed in a reverse co-IP with an anti-MDC1 antibody (Figure 2B). To further characterize the interaction between aprataxin and MDC1 and to map the interaction domain on aprataxin, we performed aprataxin GST pull-downs. Using a series of overlapping GST fusion proteins, encompassing aprataxin FHA, HIT and zinc finger domains (Figure 2C), we identified the N-terminal region (amino acid 1−110) of aprataxin as the MDC1 interaction domain (Figure 2D). Binding of MDC1 to the aprataxin FHA domain was retained in the presence of BenzonaseTM suggesting that this was not due to bridging with nucleic acid but rather a direct protein−protein interaction (Figure 2E). To confirm the direct interaction between aprataxin and MDC1, aprataxin GST pull downs were carried out in the presence of ethidium bromide (EtBr) (Supplementary Figure S5). EtBr did not disrupt the interaction between the two proteins. On the contrary, an increase in MDC1 binding was observed in the presence of EtBr which may reflect an increase in available soluble MDC1. Thus the co-localization of aprataxin and MDC1 to sites of DNA damage induced by high LET radiation is supported by evidence of a direct interaction between these two proteins.</p>
              <figure data-origid="F2">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2836575/#F2">Figure 2.</a>
                <figcaption>
                  <p id="s5s2f1p1">Direct interaction between aprataxin and MDC1. (A) Co-IP of MDC1 and aprataxin using an anti-aprataxin antibody followed by immunoblotting with the respective antibodies. Non-specific serum (Ig) is used as a negative control and the amounts of protein in WCE are shown. (B) Co-IP of aprataxin with an anti-MDC1 antibody. Non-specific serum (Ig) is included and WCE is also shown. (C) Diagram of aprataxin domains and GST-fragments used in the pull-down assays. (D) Protein extracts from control LCL (C3ABR) were either incubated with GST alone or GST-aprataxin fragments. Bound proteins were separated on SDS–PAGE and detected by anti-MDC1 antibodies. Coomassie staining shows equivalent loading of GSTs. (E) Direct binding of aprataxin to MDC1. Aprataxin FHA pull downs in the absence or presence of BenzonaseTM, a potent nuclease that degrades both RNA and DNA.</p>
                </figcaption>
              </figure>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s3h1">CK2 phosphorylation-dependent interaction of MDC1 with the aprataxin FHA domain</h3>
            <div class="content">
              <p id="s5s3p1">FHA domains are conserved sequences of 65–150 amino acid residues found principally within eukaryotic nuclear proteins that have been shown to mediate phospho-specific protein–protein interactions by binding to phosphopeptides (47,48). Previous reports have demonstrated that the interaction between aprataxin and XRCC1 was dependent on phosphorylation of XRCC1 by CK2 (9,10). Furthermore, XRCC4 which is involved in the repair of DNA DSB via the non-homologous end joining pathway was also shown to bind to the FHA domain of aprataxin (9). To determine whether XRCC4 could also target aprataxin to DNA breaks, we examined the recruitment of aprataxin in Xrcc4−/− cells. Normal recruitment of GFP-aprataxin was observed in Xrcc4−/− cells indicating that XRCC4, albeit binding to the FHA domain of aprataxin, is not involved in its targeting to heavy ion-induced DNA breaks (data not shown). Nevertheless, the interaction between aprataxin and XRCC4 may be important for the assembly of the rejoining complex and/or to ensure effective DNA end ‘clean-up’ during the repair process.</p>
              <p id="s5s3p2">To determine whether the interaction between MDC1 and aprataxin is also dependent on phosphorylation of MDC1, GST pull downs with the aprataxin FHA domain were performed in the presence of protein phosphatase (Figure 3A and B). Alkaline phosphatase treatment disrupted the binding of MDC1 to the aprataxin FHA domain demonstrating a phosphorylation-dependent interaction between the two proteins (Figure 3B). Since CK2 was found to phosphorylate XRCC1 to mediate the aprataxin–XRCC1 interaction (9,10), we therefore investigated whether CK2 could also phosphorylate MDC1 and thus mediate the interaction between the two proteins. As shown in Figure 3C, inhibition of CK2 activity in vivo with increasing concentration of the specific CK2 inhibitor, 4,5,6,7-Tetrabromotriazole (TBB), resulted in decreased binding of MDC1 to the aprataxin FHA domain. Use of another CK2-specific inhibitor, 2-dimethylamino-4,5,6,7-tetrabromo-1H-benzimidazole (DMAT), also showed a reduction in MDC1 binding to aprataxin FHA (Supplementary Figure S6). Analysis of MDC1 protein sequence using GPS2.0 TM, NetPhos2.0TM, ScansiteTM prediction programs and cross-reference with the Phospho-ELMTM database identified several putative CK2 phosphorylation sites within the N-terminal region of MDC1 (Supplementary Figure S7). To confirm the CK2 phosphorylation-dependent interaction between aprataxin and MDC1, we made a GST fusion protein corresponding to the N-terminal region of MDC1 residues (150–350) that contains nine potential sites for CK2, three of which are predicted to be pT and required for FHA binding. The GST-fusion protein (150–350) was in vitro phosphorylated with recombinant human CK2 and pull downs were performed. As shown in Figure 3D, MDC1 fragment 150–350 pulled down endogenous aprataxin from total cell extracts (Figure 3D, Lane 4) after in vitro CK2 phosphorylation, indicating that the CK2-phosphorylated 150–350 region is the aprataxin binding domain on MDC1.</p>
              <figure data-origid="F3">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2836575/#F3">Figure 3.</a>
                <figcaption>
                  <p id="s5s3f1p1">CK2-mediated phosphorylation-dependent binding of MDC1 to the aprataxin FHA domain. (A) Diagram of aprataxin GST fusion used in the pull-down assays. (B) Effect of protein phosphatase activity on MDC1 binding to the aprataxin FHA domain. Whole cell extracts from HeLa cells were mock treated or treated with alkaline phosphatase (Alk PPase: 2500 U) for 1 h at room temperature and subsequently used for pull-down assays using GST only, and GST-aprataxin FHA domain. (C) Reduced binding of MDC1 to the aprataxin FHA domain following CK2 inhibitor treatment. HeLa cells were treated for 8 h with increasing concentrations of CK2 inhibitor TBB, lysed and WCE were used for GST pull-down assays. (D) Schematic of MDC1 protein sequence showing the GST fusion protein (150–350) used in the pull-down assays. In vitro binding of CK2-phosphorylated (150–350) region of MDC1 with HeLa WCE. Binding endogenous aprataxin to CK2-phosphorylated MDC1 (150–350) fragment was revealed by immunoblotting.</p>
                </figcaption>
              </figure>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s4h1">Binding of aprataxin to CK2-phosphorylated MDC1 pSDpTD sites</h3>
            <div class="content">
              <p id="s5s4p1">To confirm and quantify the interaction between the aprataxin FHA domain and CK2-phosphorylated MDC1, binding experiments were performed by ITC using recombinant aprataxin FHA domain and phosphorylated MDC1 peptides. FHA domains typically target pT residues (47,48), and within the first 500 residues of MDC1 there are seven threonine residues found in putative CK2 phosphorylation sites. Six of these are present in highly conserved sequences with the consensus D/N-S-D-T-D-x-E/D-E/D-E/D with additional preferences for small hydrophobic residues (Ala/Val) in the pT +2 position, and medium hydrophobic residues (Leu/Ile) in the pT−4 site (Figure 4A and Supplementary Figure S7). Whilst the threonine sites are predicted to be the best CK2 substrates, the preceding serines (S) are also putative phosphorylation sites for CK2. Therefore, both mono- and di-phosphorylated peptides were tested in ITC experiments (Figure 4B and C and Supplementary Table S1). A 14-residue peptide encompassing S329 and T331 was chosen, as these residues have been detected in the phosphorylated form of MDC1 in vivo (49–52). As shown in Figure 4B, the peptide phosphorylated only on T331 bound the aprataxin FHA domain stoichiometrically, albeit with an affinity of 58 µM. However, introduction of a second phosphate on residue S329 increased binding affinity ∼10-fold to 5.6 µM (Figure 4C) which is well within the range of binding affinities previously observed for FHA domains in other signalling contexts and similar to the 4.1 µM affinity observed by ITC for binding of polynucleotide kinase (PNK) FHA to CK2-phosphorylated XRCC4 (53). Importantly, no detectable binding to peptides phosphorylated only on S329 was observed even at high concentration in the ITC experiment (Figure 4B), demonstrating an absolute dependence of binding on threonine phosphorylation. A shorter di-phosphorylated peptide, terminating after the pT+3 position, bound aprataxin with comparable affinity to the longer version, possibly echoing the unusual lack of sequence selectivity in residues C-terminal to the pT observed by in vitro library screening of the PNK FHA domain (53; Supplementary Table S1). Thus, these data clearly demonstrate that aprataxin binds to a confirmed CK2 phosphorylation site in MDC1 in a pT-dependent fashion, its affinity is significantly elevated in the di-phospho form and supports the MDC1/aprataxin interaction data described in Figure 3. To substantiate ITC binding data, aprataxin GST pull downs were performed using GFP-MDC1 mutant constructs (Figure 4E). Wild-type (SDTD) MDC1 sequence, a mutant where the SDTD region was deleted (ΔSDTD), and a mutant where the key Thr residue was changed to Ala (SDAD)2 were expressed in HeLa cells and GST pull downs using aprataxin FHA domain were carried out. As expected, wild-type MDC1 was efficiently pulled down, while both the (ΔSDTD) and (SDAD)2 were not (Figure 4F). Together, these data suggest that the aprataxin FHA is able to detect varying phosphorylation levels that directly modulate its affinity for MDC1. Given the similarity in sequences, it seems highly likely that the aprataxin FHA domain can bind to any of the six sequences in MDC1 that contain the core SDTD motif. Since both aprataxin and Nbs1 bind to the same CK2-phosphorylated region of MDC1 (51,52,54), we determined whether Nbs1 could be in a complex with aprataxin and MDC1, and whether it was required for the recruitment of aprataxin to sites of DNA damage. Using Nbs1 defective cells, we demonstrate that both aprataxin binding to MDC1 and its recruitment to DNA breaks was independent of Nbs1 (Supplementary Figure S8). Thus, this suggests that MDC1 interacts with multiple proteins through a common CK2-phosphorylated region, and that it may reside in several distinct protein complexes within the cell. The presence of multiple phosphorylated SDTD motifs would provide a high local concentration of binding sites for aprataxin, Nbs1 and, potentially, other pS/T-dependent binding proteins.</p>
              <figure data-origid="F4">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2836575/#F4">Figure 4.</a>
                <figcaption>
                  <p id="s5s4f1p1">Aprataxin binds to a conserved di-phosphorylated CK2 motif in MDC1. (A) MDC1 contains six conserved motifs with a core SDTD motif that is di-phosphorylated in vivo (48–50). (B) ITC binding isotherms for interaction of wild-type aprataxin FHA with pSDTD and SDpTD phosphopeptides (C, D) ITC binding isotherms for the titration of pSDpTD phosphopeptide into wild-type and K38A aprataxin FHA, respectively. (E) Diagram of GFP-MDC1 mutants used in the pull-down assays. (F) Aprataxin FHA GST pull downs from HeLa cells expressing wild-type GFP-MDC1, a mutant containing a deletion of the SDTD region (ΔSDTD), and a mutant where the Thr residues contained in the SDTD sequence were replaced by Ala (SDAD)2.</p>
                </figcaption>
              </figure>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s5h1">Structure of the aprataxin FHA domain and a model for pSDpTD interactions</h3>
            <div class="content">
              <p id="s5s5p1">To further investigate the molecular basis of the phosphopeptide aprataxin FHA interaction, we attempted to crystallize the aprataxin FHA domain both with and without bound phosphopeptide. The apo protein crystallized readily and the structure was determined by molecular replacement using PNK FHA domain as the search model (1YJM). Residues 1–102 of the aprataxin FHA domain were modelled into the density and the structure was refined to an Rfree value of 23.7% at 1.65Å, with excellent stereochemistry (Supplementary Table S2). As expected the aprataxin FHA domain is structurally similar to that of PNK, with which it shares 39% sequence identity, and both adopt the classical FHA β-sandwich topology. The two structures superimpose with an overall root mean-square deviation in alpha carbon atoms of 0.6 Å for a core of 82 matched Cα’s, with largest deviations occurring in the loops connecting β1–β2 and β6–β7 (Figure 5A), consistent with the pattern of sequence conservation (Figure 5B). Despite extensive screening, the aprataxin FHA domain has, thus far, failed to crystallize in complex with either the mono- or di-phosphorylated MDC1 peptide. However, although the PNK and aprataxin FHA domains have only 39% sequence identity, they do show a remarkably high structural conservation at and around the phosphopeptide binding site. We were, therefore, able to model the pSDpTD peptide into the apo structure, using the PNK FHA domain structure with an XRCC4 phosphopeptide bound as a guide (55; Figure 5C). As has been observed in all previously characterized FHA–phosphopeptide interactions, pT recognition is provided by Arg29 and Ser41 (equivalent to PNK Arg35 and Ser47).</p>
              <figure data-origid="F5">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2836575/#F5">Figure 5.</a>
                <figcaption>
                  <p id="s5s5f1p1">Structural comparison of the aprataxin and PNK FHA domains and modelling of the aprataxin–MDC1 interaction. (A) Superposition of Cα backbone structures of the aprataxin FHA domain (green) with that of the PNK FHA/XRCC4 phosphopeptide complex (grey). (B) Sequence alignment of the core FHA domains residues from aprataxin and PNK. Identical positions are shown in red with the highly conserved arginine and serine residues that make canonical pT contacts in all available FHA-phosphopeptide structures are boxed. Black dots highlight the five Arg/Lys residues conserved between the two proteins. (C) Modelling of the aprataxin FHA–pSDpTD MDC1 phospho-motif complex (left) based on the crystal structure of the aprataxin FHA domain and the PNK–XRCC4 complex structure (right).</p>
                </figcaption>
              </figure>
              <p id="s5s5p2">The weakly conserved isoleucine at the pT−4 position of the MDC1 peptide is in contact with Pro31, which is conserved in PNK FHA and which forms a platform for interaction with a pT−4 tyrosine in the PNK/XRCC4 complex (55). Arg42 (PNK Arg48) occupies a position that is weakly conserved in the FHA domain family, and is suitably positioned for pT interaction. Most importantly, the model suggests that selectivity for pS in the pT–2 position is provided by a basic surface created by Arg29 and Lys38. In support of this, mutation of Arg44 in PNK, which is structurally equivalent to aprataxin Lys38, abolishes PNK binding to XRCC4 phosphopeptides containing a glutamate rather than pS at the –2 position (55). To test this hypothesis ITC experiments were repeated with Arg29Ala (R29A) and Lys38Ala (K38A) mutant FHA domains. Consonant with its highly conserved role in pT interactions in all FHA domains, the R29A mutant reduced binding to the di-phosphorylated MDC1 peptide to undetectable levels. For K38A, both the monothreonine phosphorylated and di-phosphorylated peptides bind to the mutant with affinities comparable to that observed for wild-type FHA interaction with the monothreonine phosphorylated ligand (Figure 4D and Supplementary Table S1). Thus, substitution of Lys38 effectively eliminates the additional binding affinity provided by accessory serine phosphorylation, consistent with our structural model. Overall, the functional significance of the extended binding surface evident from these experiments is exemplified by the fact that Lys38, Lys39, Arg29 and Arg42 constitute four of only five basic residues conserved in PNK from a total of 15 present in the aprataxin FHA (Figure 5B).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s6h1">Aprataxin FHA Arg29Ala mutant ablates the phospho-dependent aprataxin–MDC1 interaction</h3>
            <div class="content">
              <p id="s5s6p1">ITC data using purified aprataxin FHA and synthetic peptides, together with the crystallographic structure of aprataxin FHA and in silico modelling of MDC1 peptide binding demonstrated the importance of basic residues which include Arg29 and Lys38 for the binding of di-phosphorylated MDC1 peptide. To confirm these data, aprataxin GST pull downs using both Arg29Ala (R29A) and Lys38Ala (K38A) mutants were performed with total cell extracts. While MDC1 bound to the wild-type FHA sequence, R29A FHA mutant ablated MDC1 binding (Figure 6A), in agreement with the ITC binding and modelling data. Similarly, substitution of Lys38 effectively eliminates the additional binding affinity provided by accessory serine phosphorylation (Figure 4D) and, consistent with the extremely low affinity (∼80 µM), GST pull downs with the K38A FHA mutant also did not show any significant interaction with MDC1 (Figure 6A).</p>
              <figure data-origid="F6">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2836575/#F6">Figure 6.</a>
                <figcaption>
                  <p id="s5s6f1p1">Key role of aprataxin FHA domain in targeting aprataxin to sites of DNA damage in vivo. (A) Substitution of Arg29 and Lys38 to Ala abrogated the interaction between aprataxin and MDC1 as determined by pull-down assays as described earlier. (B) Effect of Arg29Ala and Lys38Ala mutations on the recruitment of aprataxin to high LET heavy ion-induced DSB in HeLa cells. HeLa cells were transiently transfected with wild-type GFP-aprataxin, GFP-aprataxin R29A or GFP-aprataxin K38A mutants, and cells were subjected to nickel ions irradiation (6.0 MeV/nucleon; LET 3430 keV/µm). DSB are visualized by γH2AX immunostaining and nuclei by ToPro3 staining.</p>
                </figcaption>
              </figure>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s7h1">Recruitment of aprataxin to sites of DNA damage is abolished by Arg29Ala mutation</h3>
            <div class="content">
              <p id="s5s7p1">To investigate the functional importance of the aprataxin FHA domain as a protein–protein interaction platform and its potential role in targeting aprataxin to DNA damage, we examined the effect of Arg29Ala and Lys38Ala FHA mutations on the in vivo recruitment of aprataxin to sites of DNA breaks. Recruitment of aprataxin to heavy ion-induced DNA damage was visualized using GFP-aprataxin containing either the wild-type sequence, R29A or K38A substitutions. GFP-aprataxin carrying the wild-type sequence was effectively recruited to DNA breaks as demonstrated by the co-localization with γH2AX along the particle tracks (Figure 6B) and as previously shown in Figure 1. Consistent with the ITC experiments, no recruitment to sites of DNA damage was observed with the R29A FHA mutant, while recruitment was reduced in the K38A aprataxin mutant (Figure 6B). Similar results were obtained with laser irradiation (data not shown). These data demonstrate the importance of the FHA domain in mediating phosphorylation-dependent protein–protein interactions and in targeting aprataxin to sites of DNA breaks.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s8h1">Normal repair of IR-induced DSB in aprataxin-deficient cells</h3>
            <div class="content">
              <p id="s5s8p1">The interaction between aprataxin, XRCC4 and MDC1 and its recruitment to heavy ion irradiation-induced DNA damage suggest a role for aprataxin in the repair of DSB. To get further insight on the functional relevance of aprataxin recruitment to DNA breaks and its interaction with MDC1, we transfected both wild-type and FHA R29A mutant full length APTX cDNAs in AOA1 (FD105) fibroblasts and measured the repair kinetics of DSB in these cells. Repair of DSB was monitored by imaging the disappearance of γH2AX foci, a surrogate marker for DSB formation and rejoining, after low LET IR. Similar DSB repair kinetics were observed in AOA1 and AOA1 complemented with wild-type GFP-aprataxin, indicating that aprataxin is not necessary for the repair of DSB induced by low LET irradiation (Figure 7A). AOA1 cells transfected with GFP-aprataxin R29A exhibited a similar DSB repair kinetic (Figure 7A). While aprataxin has been implicated in DSB processing through its interaction with XRCC4, and the interaction with MDC1 described here, we failed to detect any significant difference in the repair kinetics of low LET-induced DSB in the presence or absence of aprataxin. Similar results were obtained for H2O2-induced DSB (Supplementary Figure S9). While surprising to some extent, these data are in agreement with previous reports which failed to detect a significant DSB repair defect in AOA1 after IR and H2O2 treatments (9,19). Not surprisingly, Mdc1−/− cells display some defect in the repair of DNA DSB induced by low LET IR (Figure 7B). These data are in agreement with the reported role of MDC1 in the signalling and repair of IR-induced DNA DSB (30–34,56). While high LET heavy ion radiation, IR and H2O2 are all capable of inducing DSB, they also induce a variety of DNA lesions that include base damage and SSB (45). Although MDC1 has been solely implicated in DSB recognition and repair, the induction of SSB by high LET heavy ion radiation prompted us to investigate whether MDC1 may also participate in SSB repair. In contrast to Aptx−/− cells where there is evidence for a defect in the repair of DNA SSB, normal SSB repair was observed in Mdc1−/− cells (Supplementary Figure S10), indicating that MDC1 is not required for the repair of SSB, and that the interaction described here between aprataxin and MDC1 may only be relevant to a subset of DSB, those resulting from abortive DNA ligation attempts.</p>
              <figure data-origid="F7">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2836575/#F7">Figure 7.</a>
                <figcaption>
                  <p id="s5s8f1p1">Normal repair of IR-induced DSB in AOA1 cells. (A) γH2AX foci repair kinetic after low LET ionizing radiation (IR; 2 Gy). AOA1 fibroblasts (FD105) were transfected with either GFP-aprataxin (wild-type) or GFP-aprataxin (R29A) mutant. Untransfected cells were used as control. Graph represents average numbers of foci per cell ± SD. Foci were counted in 50–60 individual cells per time point. DSB repair kinetics were similar regardless of the presence of absence of aprataxin indicating that aprataxin is not required for low LET IR-induced DSB repair. (B) Repair kinetics of DSB (γH2AX foci) after low LET irradiation (IR) in wild-type and Mdc1−/− cells. Graph represents average number of foci per cell ± SD. Foci were counted in 50–60 individual cells per time points a significant difference was observed at 6 and 8 h post-irradiation (*P &lt; 0.05, Student’s t-test).</p>
                </figcaption>
              </figure>
            </div>
          </section>
        </div>
      </section>
      <section data-type="discussion">
        <h2 id="s6h1">DISCUSSION</h2>
        <div class="content">
          <p id="s6p1">Previous reports have shown that aprataxin interacts with XRCC1, Ligase III and PARP-1, implicating it in the repair of SSB (8–10). Further evidence for its role in repair of single strand interruptions in DNA was provided by Ahel et al. (2006) who showed that it resolves abortive DNA ligation intermediates. We have provided additional evidence here for a defect in SSB repair in AOA1 cells after exposure to H2O2 which supports the specific role of aprataxin in repairing only a subgroup of relatively ‘slowly processed’ SSB carrying abortive ligation intermediates, since the reduced rate of repair in Aptx−/− cells is only significant at later time points (&gt;15 min). Importantly, a delay in the repair of DNA SSB was only detected in non-proliferating cells, consistent with the results of Reynolds et al. (2009) where a defect in SSB repair was observed in AOA1 cells after inhibiting proliferation with aphidicolin (57). These data are in agreement with the SSB repair delay observed in AOA1 fibroblasts caused by intracellular accumulation of reactive oxygen species, where this repair defect was also evident after 1 h post-treatment (12). Notwithstanding these observations there exists some disagreement in the literature on the capacity of AOA1 cells to repair SSB (9,19). This may be explained by the specific mutation in AOA1, the cell type used (proliferating versus non-proliferating) or the repair assay, alkaline comet versus the alkaline elution assay.</p>
          <p id="s6p2">More recently, Reynolds et al. (2009) demonstrated that short-patch SSB repair in AOA1 cell extracts bypasses the point of aprataxin action at oxidative breaks, resulting in the accumulation of adenylated DNA nicks. This defect results from insufficient levels of non-adenylated DNA ligase, and short-patch SSB repair could be restored in AOA1 extracts by the addition of recombinant DNA ligase (57). In addition, aphidicolin inhibition of long-patch repair uncovered a significant defect in SSB repair in Aptx−/− neural astrocytes while it did not affect SSB repair rates in wild-type mouse neural astrocytes (57). Most recently, Harris et al. (2009) provided evidence for reduced expression of key base excision repair proteins (PARP-1, APE1 and OGG1) in AOA1 cells, and revealed the presence of elevated levels of oxidative DNA damage in these cells coupled with reduced base excision and gap filling repair efficiencies (17). Thus, a synergy between aprataxin, PARP-1, APE-1 and OGG1 in the DNA damage response uncovered both direct and indirect modulating functions for aprataxin in base excision repair (17). In addition, we also have shown here that aprataxin was not recruited to DNA breaks in the absence of XRCC1. Together, these data demonstrate that aprataxin participates in chromosomal SSB repair in vivo, that it modulates base excision repair efficiency and suggest that short-patch SSB repair is arrested in AOA1 because of insufficient non-adenylated DNA ligase.</p>
          <p id="s6p3">Clements et al. (2004) identified XRCC4 as a binding partner for aprataxin using a yeast 2-hybrid screen and confirmed this by demonstrating co-IP with DNA ligase IV, a constitutive binding partner. Since XRCC4/Ligase IV is an essential component of the non-homologous end-joining (NHEJ) pathway these data suggested that aprataxin might also be involved in proof-reading or processing of DSB. We provide here compelling evidence that aprataxin is recruited to sites of DNA damage and interacts with MDC1, an important component of the DNA damage response. MDC1 is required for efficient phosphorylation of H2AX, for recruitment and retention of ATM and MRN adjacent to break sites (29,30,34,35) and for the recruitment of the ubiquitin ligase RNF8 to ubiquitinate H2AX at damaged sites (58–61). This protein appears to act as a scaffold to bring together several DNA damage response proteins to mediate and enhance the recognition and signalling of DSB (58–66). Our data add an additional DNA repair protein, aprataxin, to the list of proteins interacting with MDC1. MDC1 interacts with DNA damage response proteins in a number of different ways. Interaction with γH2AX and the anaphase-promoting complex occurs through its C-terminal, BRCT domain (33,34,65), while it binds DNA-PK through an internal set of repeat sequences (32). It also possesses an FHA domain (amino acids 55–124) that mediates interaction with Chk2 and Rad51 (33,66). It employs an S/TQ rich region for ATM-dependent interaction with the ubiquitin ligase RNF8 (58).</p>
          <p id="s6p4">We have described here a novel interaction between MDC1 and aprataxin, involving an extended cluster of CK2 phosphorylation sites (S-D-T-D) located downstream from the MDC1 FHA domain (amino acids 150–350). This was confirmed by the use of CK2 inhibitors, binding studies after in vitro CK2 phosphorylation, GST pull downs and modelling studies based on binding of mono- and di-phosphorylated peptides to the aprataxin FHA domain and the X-ray crystal structure of this domain. We provide here compelling evidence for a role for CK2 activity in the interaction of this region of MDC1 with aprataxin and show that a phosphopeptide containing a core pSDpTD motif corresponding to one of these sites binds to the FHA domain of aprataxin with high affinity. While a single threonine phosphorylation is both necessary and sufficient for binding, affinity is substantially increased by secondary serine phosphorylation. This, in turn, appears to provide for accessory interactions with an extended FHA phosphopeptide binding surface not previously documented. The data presented here uncover another important role for CK2 in mediating the interaction between aprataxin and the DSB repair protein MDC1.</p>
          <p id="s6p5">In three recent reports constitutive phosphorylation of the N-terminal SDT repeats of MDC1 by CK2 has been demonstrated in the recruitment of the MRN complex to sites of DSB through direct interactions with the FHA domain of Nbs1 (51,52,54). Spycher et al. (54) reported that doubly phosphorylated pSDpT motifs regulate the accumulation and retention of the MRN complex in the DSB-flanking chromatin, and depletion of CK2 disrupted the MDC1–MRN complex and DNA damage-induced Nbs1 foci formation. Moreover, disruption of the SDT phosphoacceptor sites on MDC1 prevented the recruitment of Nbs1 to damaged chromatin (51,52,54). We have demonstrated that Nbs1 is not required for the binding of aprataxin to MDC1 and its recruitment to sites of DNA damage. Therefore, MDC1 interacts with multiple proteins through a common CK2-phosphorylated region and may reside in several distinct protein complexes within the cell. It is likely that the interaction between MDC1/Nbs1 and MDC1/aprataxin is influenced or regulated by the relative abundance of these proteins in the cells. These data together with previous findings that identified a direct role for CK2 in the repair of chromosomal DNA strand breaks (67), the CK2-dependent aprataxin/XRCC1 (9,10) and aprataxin/XRCC4 (9) interactions provide growing evidence for a role for CK2 in maintaining genome integrity by contributing to protein–protein complexes formation and/or recruitment of DNA damage regulators/repair factors at sites of DNA lesions.</p>
          <p id="s6p6">As shown here, aprataxin re-localizes rapidly to discrete foci along charged particle tracks that overlap with γH2AX foci, a marker of DSB after exposure of cells to high LET radiation which gives rise to complex damage in DNA (22). The recruitment and co-localization of aprataxin to sites of γH2AX-labelled DNA breaks has previously been observed using different heavy ion species (8,17,28, this work). However, while MDC1 completely co-localized with γH2AX in the DSB-flanking microenvironment as reported earlier (68), aprataxin only partially co-localized with MDC1 and γH2AX in smaller foci (microfoci). This suggests that aprataxin may be recruited only to the DSB or a sub-group of DNA breaks directly within sites of complex or clustered damage or that its association with DNA damaged sites may be more transient. This transient behaviour has been described for Nbs1 in the absence of MDC1 (26,52). In addition, we provide evidence here that the lack of MDC1 doesn’t affect the rapid recruitment of aprataxin to sites of high LET-induced DNA DSB, but in contrast affects the retention of aprataxin at sites of DNA breaks, supporting a role for MDC1 as a scaffold during the DNA damage response. Proteins such as SMC1 and DNA-PKcs that are clearly involved in the DNA damage response cannot be localized to sites of DNA damage except after exposure to extremely high doses of radiation (68). Spatial-co-localization studies revealed that unlike γH2AX and MDC1, phosphorylated DNA-PKcs was localized only at very specific regions, presumably representing sites of DSB within the tracks (69). The presence of microfoci surrounded by regions of γH2AX-modified chromatin after laser-induced DNA damage has been reported (68). Several factors involved in DNA repair including Nbs1 and Mre11 are present in microfoci located in the centre of microirradiated tracks and surrounded by vast regions of γH2AX-modified chromatin (68,70). In the present context these microirradiated tracks coincide with MDC1-modified chromatin. The heterogeneity of staining may be explained by the presence of significant amounts of ssDNA adjacent to DSB and/or the presence of clustered or complex DNA damage (45,46). Exposure of cells to low LET radiation (IR) or H2O2 failed to localize aprataxin to sites of damage (8). This may be similar to that observed for SMC1 or DNA-PK where extremely high levels of damage are required to see such localization (68). Low LET radiation is more sparsely ionizing and thus less damaging to DNA than the complex forms of damage arising from high LET exposure which could explain the failure to see aprataxin foci after low LET radiation (45). The difference in capacity of aprataxin to localize to sites of DNA damage after the two types of IR correlates well with the extent of repair of DNA DSB. The rate of repair of DNA DSB in response to low LET radiation was unaffected by aprataxin status of the cells. Similar to previous reports (9,19), the failure to detect any significant DSB repair defect in the presence or absence of aprataxin after low LET irradiation or high doses of H2O2 suggests that aprataxin recruitment to DNA breaks may only be relevant for a sub-group of breaks, possibly more frequently/effectively induced by high energy and high LET heavy ions irradiation. This sub-group of breaks could represent those where abortive ligation was occurring, a property of AOA1 cells (14).</p>
          <p id="s6p7">The interaction of aprataxin with DNA damage recognition and repair proteins suggest that aprataxin participates in the processing of some SSB and DSB to resolve abortive DNA ligations. The model described in Figure 8 provides an integrated approach as to how aprataxin may function at sites of damage (DSB and SSB) in DNA. This involves recruitment to damaged sites by XRCC1, MDC1 and other unidentified factors, followed by interaction with either XRCC1/LigaseIII or XRCC4/LigaseIV to assist in the DNA end-processing and resealing of SSB and DSB, respectively. In summary, it is likely that aprataxin contributes to the repair of both DNA SSB and DSB in its capacity to resolve abortive ligation intermediates. This is most evident when cells are exposed to agents that form complex damage in DNA.</p>
          <figure data-origid="F8">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2836575/#F8">Figure 8.</a>
            <figcaption>
              <p id="s6f1p1">A model for the dual role of aprataxin in SSB and DSB repair. The FHA domain of aprataxin, through which it interacts with MDC1, XRCC1 and XRCC4, is shown three times to represent a series of dynamic interactions with these three proteins or to represent individual aprataxin molecules.</p>
            </figcaption>
          </figure>
        </div>
      </section>
      <section data-type="">
        <h2 id="s7h1">SUPPLEMENTARY DATA</h2>
        <div class="content">
          <p id="s7p1">Supplementary Data are available at NAR Online.</p>
        </div>
      </section>
      <section data-type="">
        <h2 id="s8h1">FUNDING</h2>
        <div class="content">
          <p id="s8p1">This work was supported by grants from the Australian National Health &amp; Medical Research Council (NHMRC) (M.F.L.); and by the Medical Research Council, UK (S.J.S.). Funding for open access charge: Australian National Health &amp; Medical Research Council (NHMRC).</p>
          <p id="s8p2">Conflict of interest statement. None declared.</p>
        </div>
      </section>
      <section data-type="supplementary-material">
        <h2 id="s9h1">Supplementary Material</h2>
      </section>
    </article>
  </body>
</html>
