<!DOCTYPE html >
<html id="akKYhif8WBbKGt7EysbPGXfcZYm4-PMC3049153" data-origid="PMC3049153" class="anndoc" data-anndoc-version="2.0" lang="" xml:lang="" xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="UTF-8"/>
    <meta name="generator" content="com.jmcejuela.anndoc.v2.bio.NcbiJournalArticleParser_v0_4$"/>
    <meta name="dcterms.source" content="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3049153/"/>
    <title>akKYhif8WBbKGt7EysbPGXfcZYm4-PMC3049153</title>
  </head>
  <body>
    <article>
      <section data-type="title">
        <h2 id="s1h1">Identification of FOXP1 Deletions in Three Unrelated Patients with Mental Retardation and Significant Speech and Language Deficits</h2>
      </section>
      <section data-type="abstract">
        <h2 id="s2h1">Abstract</h2>
        <div class="content">
          <p id="s2p1">Mental retardation affects 2-3% of the population and shows a high heritability. Neurodevelopmental disorders that include pronounced impairment in language and speech skills occur less frequently. For most cases, the molecular basis of mental retardation with or without speech and language disorder is unknown due to the heterogeneity of underlying genetic factors. We have used molecular karyotyping on 1523 patients with mental retardation to detect copy number variations (CNVs) including deletions or duplications. These studies revealed three heterozygous overlapping deletions solely affecting the forkhead box P1 (FOXP1) gene. All three patients had moderate mental retardation and significant language and speech deficits. Since our results are consistent with a de novo occurrence of these deletions, we considered them as causal although we detected a single large deletion including FOXP1 and additional genes in 4104 ancestrally matched controls. These findings are of interest with regard to the structural and functional relationship between FOXP1 and FOXP2. Mutations in FOXP2 have been previously related to monogenic cases of developmental verbal dyspraxia. Both FOXP1 and FOXP2 are expressed in songbird and human brain regions that are important for the developmental processes that culminate in speech and language. ©2010 Wiley-Liss, Inc.</p>
        </div>
      </section>
      <section data-type="">
        <h2 id="s3h1">INTRODUCTION</h2>
        <div class="content">
          <p id="s3p1">Mental retardation (MR) is defined as a significant impairment of cognitive and adaptive functions with an onset before the age of 18 years (Ropers, 2008). Based on the assessment of patients’ intelligence quotients (IQs), MR is usually classified into mild (IQ 50-70), moderate (IQ &gt; 35) and severe (IQ &gt; 20) forms. It has been shown that the causes of the disorder include environmental and genetic factors (Inlow and Restifo, 2004), still for most cases, the pathological basis remains unexplained. The high degree of heritability of MR is highlighted by the estimation that up to 50% of severe cases are caused by genetic abnormalities (Inlow and Restifo, 2004). Due to the heterogeneity of the underlying genetic factors, the identification of MR candidate genes still remains difficult to date. A promising approach to detect small chromosomal copy number variants (CNVs) in the genome is the use of molecular karyotyping techniques (Marshall, et al., 2008). CNVs have been shown as the underlying cause or susceptibility factors for a variety of autism spectrum disorders and conditions associated with MR (Berkel, et al., 2010; Marshall, et al., 2008; Sebat, et al., 2007; Zweier, et al., 2010). Array-based comparative genomic hybridization has also enabled the detection of interstitial submicroscopic copy number alterations in about 10 % of patients with MR (de Vries, et al., 2005). Here we present the identification of overlapping heterozygous deletions affecting the FOXP1 (MIM# 605515) gene in three unrelated patients with MR and significant speech and language disorder.</p>
        </div>
      </section>
      <section data-type="materials|methods">
        <h2 id="s4h1">MATERIALS AND METHODS</h2>
        <div class="subsections">
          <section data-type="">
            <h3 id="s4s1h1">Oligonucleotide arrays</h3>
            <div class="content">
              <p id="s4s1p1">CNV data were generated in different institutions within the MRNET consortium (Supp. Table S1). Patient 1 was part of a cohort of 387 patients investigated with Infinium Human550-Quad and Human610-Quad arrays (Illumina). Intensity data were normalized as described previously (Wagenstaller, et al., 2007). Segmentation was performed with circular binary segmentation as implemented in the R-package ‘DNAcopy'. Patient 2 was part of a cohort of 188 patients investigated using whole genome oligonucleotide 244K arrays (Agilent Technologies, Santa Clara, CA). Image data were analyzed using Feature Extraction 9.5.3.1 and CGH Analytics 3.4.40 software (Agilent Technologies, Santa Clara, CA) with the following analysis settings: aberration algorithm ADM-2; threshold: 6.0; window size: 0.2 Mb; filter: 5probes, log2ratio = 0.29. Patient 3 belonged to a cohort of 184 patients analyzed with genome-wide human SNP 6.0 arrays (Affymetrix, Santa Clara, CA). Analysis of data was performed using the Genotyping Console Software 3.0 (Affymetrix). For the detection of genomic deletions and duplications, automated analysis by Segment Reporting Tools was used. Regions showing at least 5-10 aberrant neighboring SNPs / markers and having a size of at least 100 kb were classified as being significant. Additional CNV data of 764 MR patients from other institutions not listed here were obtained through the database of the MRNET (www.german-mrnet.de).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s2h1">Control populations</h3>
            <div class="content">
              <p id="s4s2p1">Controls consisted of 1146 individuals from popgen, 813 individuals of a population-based cohort (KORA study), 972 patients with cardiac ischemia (AGNES study), 482 patients with early-onset lung cancer (LUCY study), and 691 long-lived individuals (LLI study).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s3h1">Breakpoint identification</h3>
            <div class="content">
              <p id="s4s3p1">PCR reactions on genomic DNA level were performed to amplify the junction fragments that spanning the telomeric and centromeric breakpoints. Fragments were directly sequenced with the respective forward and reverse primer: B35_FOXP1F/B8_FOXP1R (patient 1): 5′-atgctgaaggtggaatggg-3′, 5′-ggccacatacgtgttgtcag-3′; O06_for/Z02_rev (patient 2): 5′-cgttgccagctcaaggttat-3′, 5′-taagtgtgtgcgaagccaag-3′; bp_FOXP1_3for/rev (patient 3): 5′-gcacctgaccctctagctca-3′, 5′-ggttcagccactggtctttc-3′.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s4h1">Fluorescence in situ hybridization (FISH)</h3>
            <div class="content">
              <p id="s4s4p1">Preparation of chromosome metaphases of patient 3 and his parents and FISH were performed according to standard protocols using BAC DNA probes RP11-215K24, RP11-154H23, CTD-3121O8 (Invitrogen, Darmstadt, Germany) and RP11-788D09 (BACPAC Resource Center, Oakland, USA).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s5h1">Mutation screening</h3>
            <div class="content">
              <p id="s4s5p1">FOXP1 (NM_032682.4) exons were PCR amplified using intronic primers and investigated with 2 different methods: 197 DNAs were investigated by direct sequencing, 772 DNAs were analyzed using dye-binding/high-resolution DNA melting point analysis (LightScanner HR I 384, Idaho Technology). Genotyping of the controls were performed on a MALDI-TOF mass-spectrometer (Sequenom MassArray system) using the homogeneous mass-extension (hME) process for producing primer extension products. Primers were designed with ExonPrimer (http://ihg.helmholtz-muenchen.de/ihg/ExonPrimer.html) and are available on request. Nucleotide numbering reflects cDNA numbering with +1 corresponding to the A of the ATG translation initiation codon.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s6h1">SLIC study</h3>
            <div class="content">
              <p id="s4s6p1">The SLIC probands were selected from samples ascertained by the Newcomen Centre at Guys Hospital and by the Manchester Language Study (The SLI Consortium, 2002) and all had severe language impairments with language skills more than 1.5 SD below the normative mean for their chronological age in combination with full IQ scores at least 0.7 SD below that expected for their age. IQ scores are derived from the Wechsler Intelligence Scales for Children (WISC-III). The 46 (16 males, 30 females; average age: 11 years and 3 months) individuals would not meet strict SLI diagnostic criteria as individuals with IQ problems are usually excluded. Nonetheless all probands had severe language impairments which require special schooling arrangements and continued support.</p>
            </div>
          </section>
        </div>
      </section>
      <section data-type="">
        <h2 id="s5h1">RESULTS</h2>
        <div class="subsections">
          <section data-type="">
            <h3 id="s5s1h1">FOXP1 deletions</h3>
            <div class="content">
              <p id="s5s1p1">In a collaborative effort we performed a genome-wide microarray scan for CNVs in a German cohort of 1523 unrelated patients with unexplained mental retardation. Standard diagnostic tests like chromosomal karyotyping, fragile X testing and subtelomeric screenings were performed in most cases to rule out known causes of MR. The recruitment of patients was part of the German Mental Retardation Network (MRNET) study. Approval for the study was obtained by the ethical review boards of the participating institutions and informed written consent was obtained from all participants.</p>
              <p id="s5s1p2">Copy number analysis revealed overlapping deletions at chromosome 3p14.1 affecting solely the FOXP1 gene in three unrelated cases, two males and one female aged between 5.5 to 7 years. Deletion sizes of 498 kb, 659 kb and 1047 kb included all but the first of the coding exons in patient 1 (Wagenstaller, et al., 2007) and the entire coding region in the other two patients 2 and 3 (Figure 1A). The deletions were verified by fluorescence in situ hybridization (FISH; Figure 1B) or quantitative real-time PCR (data not shown). DNA analysis of the unaffected parents by molecular karyotyping and FISH indicated that patients 2 and 3 carried de novo deletions, while it could only be shown that the mother of patient 1 does not carry the deletion since the father was not available for investigation. We found that the deleted alleles were of paternal origin in all three patients. Sequencing of the coding exons of the remaining FOXP1 allele did not reveal any sequence variation compared to the annotated reference sequence (NM_032682). Characterization of the breakpoints was achieved by amplification and sequencing of the respective junction fragments (GenBank EF504249, HM124444, and GU980955; Figure 1C).</p>
              <figure data-origid="fig01">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3049153/#fig01">Figure 1</a>
                <figcaption>
                  <p id="s5s1f1p1">FOXP1 deletions in patients with MR.</p>
                  <p id="s5s1f1p2">(A) Schematic presentation of the position and size of the deleted regions (grey bars) at chromosome 3p14.1 in three patients with MR and in a control individual. Only the FOXP1 gene (NM_032682, minus strand) is affected. The coding exons are indicated by larger vertical lines compared to the non-coding exons.</p>
                  <p id="s5s1f1p3">(B) Results of fluorescence in situ hybridization (FISH) shown for patient 3 and his parents. The positions of the BAC clones RP11-215K24 and RP11-154H23 flanking the deletion, as well as CTD-3121O8 located within the deletion (green signals) are given in Figure 1A. BAC clone RP11-778D09 was used as a reference probe located on chromosome 3q27 (red signals). The white arrow indicates the missing signal for clone CTD-3121O8 on one of the patient's chromosomes 3 while the signal is present in both chromosomes of the parents.</p>
                  <p id="s5s1f1p4">(C) Breakpoint characterization by sequencing of junction fragments amplified by PCR on genomic DNA of the patients.</p>
                  <p id="s5s1f1p5">In patient 1, the distal breakpoint was located at position 70,807,767, the proximal breakpoint at position 71,305,965 with a deletion size of 498,198 bp. In patient 2, the two breakpoints were located within a microhomology of three nucleotides (AGA) at the telomeric side at position 70,778,067-70,778,070 and the centromeric side at 71,437,354-71,437,357 resulting in a deletion size of 659,287 bp. For patient 3, the telomeric breakpoint is located at position 70,341,246-70,341,247 and the centromeric breakpoint at position 71,388,173-71,388,174 with a deletion size of 1,046,927 bp. All positions are given according to hg18, UCSC Human Genome March 2006.</p>
                </figcaption>
              </figure>
              <p id="s5s1p3">For the deletion in patient 1, the distal breakpoint is located within a long interspersed nuclear element (LINE) repeat (L1/L1PA5) while the proximal breakpoint lies in a MER1-type repeat (MER58A). No repetitive sequences are present at the breakpoint loci in patient 2, however, microhomoloy of 3 bases (AGA) is observed at the breakpoints. The distal breakpoint of the deletion in patient 3 is located in a short interspersed nuclear element (SINE) while repetitive sequences are missing at the proximal breakpoint. The absence of homologous regions at the breakpoints suggests that the deletions were likely generated by double-strand breaks and non-homologous end-joining.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s2h1">Control populations</h3>
            <div class="content">
              <p id="s5s2p1">To assess the significance of these findings, we checked the FOXP1 region for CNVs in the Database of Genomic Variants (DGV) (Iafrate, et al., 2004) and in 4104 ancestrally matched controls which have been investigated with different types of high-density SNP arrays containing at least 500,000 SNPs (Supp. Table S1). No CNVs affecting the coding region were found in the DGV database. Copy number analysis in the 4104 control individuals revealed the presence of a single large 3p14.1p13 deletion of approximately 1.3 Mb affecting FOXP1, EIF4E3 (MIM# 609896), PROK2 (MIM# 607002) and GPR27 (MIM# 605187) in an individual of the LLI study, with no indications of a comparable MR phenotype. Detailed clinical data on intellectual abilities were lacking and further investigations not possible due to the design of the study. In particular, we could not investigate the presence of somatic mosaicism. Application of Fisher's exact test on these data, not considering the controls contained in the DGV database, resulted in a nominal p-value of 0.06. Thus, although the absence of deletions in the unaffected parents is consistent with a role of FOXP1 in the clinical findings of the patients incomplete penetrance of deletions in this chromosomal interval cannot be ruled out.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s3h1">Speech and language development</h3>
            <div class="content">
              <p id="s5s3p1">Detailed clinical investigation of the three patients revealed that they present moderate MR in combination with a general developmental delay (Table 1). In all patients, the non-verbal IQ score was assessed as &lt; 50 (3 SD below the mean). Speech and language development was estimated in the same range. All three patients started to speak at age of 3.5 years and used only combinations of two words at ages 5, 5.5 and 7 years, respectively. In all patients, the productive and receptive vocabulary came up to less than 100 words. Expressive language was more affected than receptive abilities. Dysgrammatism and very poor speech articulation with difficulties producing consonants at the beginning of words was present in all patients. Two showed oromotoric problems including difficulties with lip protrusion. In infancy, patients had a tendency to keep their mouth open and patient 2 suffered from swallowing difficulties. In addition, all showed considerably retarded gross-motor development with unsupported walking as late as 24 to 36 months. Brain magnetic resonance imaging and electroencephalography did not reveal any abnormalities. None of the patients showed sensorineural hearing loss. Ophthalmologic testing disclosed moderate myopia only in patient 1. Upon physical examinations, normal growth parameters regarding height and occipitofrontal head circumference were documented for two patients but presence of obesity was striking in both. Consistent craniofacial anomalies seen in patients 2 and 3 included broad and prominent forehead and frontal hair upsweep (Figure 2).</p>
              <figure data-origid="fig02">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3049153/#fig02">Figure 2</a>
                <figcaption>
                  <p id="s5s3f1p1">Facial Phenotype of patients 2 and 3. Consistent facial features of patients showing broad and prominent forehead, frontal hair upsweep, and a short nose. (A) Patient 2, at the age of 4 years and 3 months. (B) Patient 3, at the age of 6 years, had in addition sparse lateral eye brows and down-slanting palpebral fissures.</p>
                </figcaption>
              </figure>
              <figure class="table" data-origid="tbl1">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3049153/#tbl1">Table 1</a>
                <figcaption>
                  <p id="s5s3f2p1">Clinical description of FOXP1 deletion patients</p>
                </figcaption>
              </figure>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s4h1">FOXP1 point mutations</h3>
            <div class="content">
              <p id="s5s4p1">In order to detect possible FOXP1 point mutations, we sequenced the coding exons in 883 patients with MR from the MRNET database and discovered eight different non-synonymous, three synonymous and nine non-coding variants (Table 2). All variants were genotyped in up to 676 unrelated healthy controls. Three of the identified coding variants, p.Ser5Pro, p.Pro215Ala and p.Thr390Ser were also present at similar frequencies in healthy matched controls, but none of the coding variants was found in the HapMap or the NCBI dbSNP database. Five of the non-synonymous variants were transmitted through an apparently unaffected parent; for the other variants, heredity could not be established as either one or both parents were not available (Table 2). Variants transmitted by an unaffected parent are usually classified as non-pathogenic. Still, presuming a multigenic threshold model for MR, disease relevance cannot be completely ruled out. To uncover the neurobiological significance of these variants, investigation of the functional effects of the putative mutations is needed.</p>
              <figure class="table" data-origid="tbl2">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3049153/#tbl2">Table 2</a>
                <figcaption>
                  <p id="s5s4f1p1">Summary of FOXP1 variants identified in controls and in patients with mental retardation</p>
                </figcaption>
              </figure>
              <p id="s5s4p2">Since the core phenotype of the three patients carrying FOXP1 deletions consisted of MR with significant speech and language deficits, we additionally sequenced FOXP1 coding exons in DNA of 40 patients with a tentative diagnosis of Angelman syndrome (without microcephaly and negative for SNRPN imprinting). We also sequenced the DNA of 46 probands from the Specific Language Impairment Consortium (SLIC) collection (The SLI Consortium, 2002). For both groups, no sequence variants in the coding region of FOXP1 were detected.</p>
            </div>
          </section>
        </div>
      </section>
      <section data-type="">
        <h2 id="s6h1">DISCUSSION</h2>
        <div class="content">
          <p id="s6p1">This is the first report of 3p14.1 deletions that solely affect the FOXP1 gene in three unrelated patients with moderate mental retardation in combination with a significant impairment of speech and language abilities. We investigated a large control population and the apparent finding of a large 3p14.1p13 deletion in this cohort affecting FOXP1 and further genes illustrates the difficulties to assess the significance of rare mutational events in extremely heterogeneous diseases and may point to incomplete penetrance. The additional findings of obesity and mild craniofacial anomalies (prominent forehead and frontal hair upsweep) in two patients might be part of the clinical spectrum associated with FOXP1 deletions. A de novo deletion of this chromosomal interval on 3p14. 1p13 affecting part of FOXP1 and three additional genes (EIF4E3, PROK2 and GPR27) has been previously described in a boy with multiple abnormalities including speech and developmental delay (Pariani, et al., 2009). Compared to the clinical findings in our patients, there is a clear overlap with regard to the speech and developmental delay suggesting that haploinsufficiency of FOXP1 is indeed causative for this phenotype. As our patients did not show any of the other clinical signs like contractures, hypertonia and blepharophimosis, present in the boy carrying the larger 3p14.1p13 deletion (Pariani, et al., 2009), these might rather be caused by the haploinsufficiency of the additionally affected genes.</p>
          <p id="s6p2">FOXP1 belongs to a functionally diverse family of forkhead box (FOX) transcription factors that are all characterized by a highly conserved FOX domain. FOX genes have been shown to play important roles in diverse cellular functions including metabolic and developmental processes (Carlsson and Mahlapuu, 2002). While the function of FOXP1 during blood cell, lung and heart formation has already been addressed (Hu, et al., 2006; Shi, et al., 2004; Shu, et al., 2001; Wang, et al., 2004), the role of FOXP1 in neuronal processes, in particular brain development, is still unclear. In a mouse mutant model, Foxp1 has recently been shown to be an important accessory factor in Hox transcriptional output, thus regulating motor neuron diversification and connectivity to target muscles (Dasen, et al., 2008; Rousso, et al., 2008). These findings are of particular interest with regard to the gross-motor and oromotor deficits seen in our patients with FOXP1 deletions.</p>
          <p id="s6p3">Currently, four FOX genes (FOXC1 [MIM# 601090], FOXC2 [MIM# 602402], FOXP2 [MIM# 605317] and FOXP3 [MIM# 300292]) are listed in the Online Mendelian Inheritance in Man (OMIM) database that have been shown to be causative for human diseases (Hannenhalli and Kaestner, 2009). With regard to structure and expression profiles, the most interesting gene related to FOXP1 is FOXP2. Rare mutations of FOXP2 have been described in individuals having expressive and receptive language and speech deficits generally described as developmental verbal dyspraxia (Hurst, et al., 1990; Lai, et al., 2001; MacDermot, et al., 2005; Marshall, et al., 2008). It is striking, that patients with FOXP1 deletions show a comparable speech and language deficit. Most of the mutations affecting FOXP2 are deletions and maternal uniparental disomies. In almost all cases, the affected allele is of paternal origin suggesting differential parent-of-origin expression of FOXP2 in human speech development (Marshall, et al., 2008). Of note, there is some parallelism as the three FOXP1 deletions we describe are also of paternal origin.</p>
          <p id="s6p4">A functional relationship between Foxp1 and Foxp2 has been previously demonstrated in mouse, as they form homo- and heteromers necessary for efficient DNA binding (Marshall, et al., 2008). FoxP1 and FoxP2 also show overlapping expression patterns within brains of zebra finches and fetal human brains, particularly in subcortical regions that play important roles in sensorimotor integration and coordinated movements important for vocalization and speech respectively (Teramitsu, et al., 2004). From these findings it was speculated that FOXP1 might play an important role in the development of brain circuits that coordinate fine sequential motor control required for articulation.</p>
          <p id="s6p5">Although the mean non-verbal IQ of the members of the large family carrying FOXP2 mutation was lower than of the unaffected members, non-verbal intellectual impairment could not be considered characteristic of the phenotype associated with this FOXP2 mutation (Lai, et al., 2001). This is in contrast to the clinical findings in our patients with FOXP1 deletions who, in addition to their speech and language disabilities, exhibit significantly reduced non-verbal IQ scores. This may indicate that additional distinct functions of FOXP1 exist, presumably acting on a more global level in neuronal development compared to FOXP2. This would be in line with results from a previous study on 49 patients with developmental verbal dyspraxia and normal intelligence, where no causative FOXP1 mutations could be detected (Vernes, et al., 2009).</p>
          <p id="s6p6">In conclusion, we report a previously unknown cause of MR associated with significant speech and language disorder defined by FOXP1 deletions. We propose that haploinsufficiency of FOXP1 leads to abnormal development of neural structures that coordinate general cognitive and psychomotoric as well as verbal abilities.</p>
        </div>
      </section>
    </article>
  </body>
</html>
