<!DOCTYPE html >
<html id="am9WbqjUxwuRYTkyS3ruEf0sIjue-PMC3408813" data-origid="PMC3408813" class="anndoc" data-anndoc-version="2.0" lang="" xml:lang="" xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="UTF-8"/>
    <meta name="generator" content="com.jmcejuela.anndoc.v2.bio.NcbiJournalArticleParser_v0_4$"/>
    <meta name="dcterms.source" content="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3408813/"/>
    <title>am9WbqjUxwuRYTkyS3ruEf0sIjue-PMC3408813</title>
  </head>
  <body>
    <article>
      <section data-type="title">
        <h2 id="s1h1">De novo germline and postzygotic mutations in AKT3, PIK3R2 and PIK3CA cause a spectrum of related megalencephaly syndromes</h2>
      </section>
      <section data-type="abstract">
        <h2 id="s2h1">Abstract</h2>
        <div class="content">
          <p id="s2p1">Megalencephaly-capillary malformation (MCAP) and megalencephaly-polymicrogyria-polydactyly-hydrocephalus (MPPH) syndromes are sporadic overgrowth disorders associated with markedly enlarged brain size and other recognizable features1-5. We performed exome sequencing in three families with MCAP or MPPH and confirmed our initial observations in exomes from 7 MCAP and 174 control individuals, as well as in 40 additional megalencephaly subjects using a combination of Sanger sequencing, restriction-enzyme assays, and targeted deep sequencing. We identified de novo germline or postzygotic mutations in three core components of the phosphatidylinositol-3-kinase (PI3K)/AKT pathway. These include two mutations of AKT3, one recurrent mutation of PIK3R2 in 11 unrelated MPPH families, and 15 mostly postzygotic mutations of PIK3CA in 23 MCAP and one MPPH patients. Our data highlight the central role of PI3K/AKT signaling in vascular, limb and brain development, and emphasize the power of massively parallel sequencing in a challenging context of phenotypic and genetic heterogeneity combined with postzygotic mosaicism.</p>
        </div>
      </section>
      <section data-origid="S1" data-type="methods">
        <h2 id="s3h1">METHODS</h2>
        <div class="subsections">
          <section data-origid="S2" data-type="methods|subjects">
            <h3 id="s3s1h1">Study subjects</h3>
            <div class="content">
              <p id="s3s1p1">The study included 52 affected individuals from 50 unrelated families. Informed consent was obtained from all subjects (see Supplementary Note). Patients were diagnosed as having MCAP or MPPH using our published diagnostic criteria4. Genomic DNA was extracted from different tissues using standard procedures. When parental DNA samples were available, paternity-maternity testing was performed by genotyping six highly polymorphic short-tandem repeats. All exome capture and sequencing experiments were performed using whole-blood genomic DNA.</p>
            </div>
          </section>
          <section data-origid="S3" data-type="">
            <h3 id="s3s2h1">Exome sequencing of the three index families</h3>
            <div class="content">
              <p id="s3s2p1">Exome capture and sequencing of trio LR08-018 was performed at the PerkinElmer DNA Sequencing Service. Libraries were generated using the 50 megabases (Mb) SureSelect human exome kit (Agilent) and sequenced on a HiSeq 2000 (Illumina) according to the manufacturer’s recommendations for paired-end 100-bp reads. Exome sequencing of trio LR09-006 and subject LR00-016a1 were carried-out at the University of Washington Genome Sciences Genomic Resource Center using the SeqCap EZ Exome Library v2.0 capture kit (Roche) combined with sequencing of paired-end 76-bp reads on a Genome Analyzer IIx (Illumina). Reads were aligned to the human reference genome (hg19) as previously described44.</p>
              <p id="s3s2p2">Single-nucleotide variants and small indels were identified using the GATK Unified Genotyper45 and a variant quality score ≥ 10, and annotated using SeattleSeq SNP annotation (see URLs). Variants were then filtered using standard hard-filtering parameters45. Specifically, only variants with a quality score ≥ 30, sequencing depth ≥ 4, quality/depth ratio ≥ 5, length of homopolymer run ≤ 5.0, and allelic balance ≤ 0.80 were considered for downstream analysis. We also used exome sequencing data from 112 individuals sequenced at the University of Washington Genome Sciences or PerkinElmer DNA Sequencing Service. These included 95 unrelated control individuals from the University of Washington NIEHS Exome Project (see URLs) and 17 in-house exomes consisting of healthy individuals and patients with unrelated phenotypes.</p>
            </div>
          </section>
          <section data-origid="S4" data-type="">
            <h3 id="s3s3h1">Family-based analysis</h3>
            <div class="content">
              <p id="s3s3p1">For each index patient (LR08-018, LR09-006, and LR00-016a1), we generated a list of “rare variants” by focusing on protein-altering and splice-site DNA changes absent from dbSNP build 132, the 1000 Genomes project46, and 112 control exomes. As described previously44, we systematically identified potential de novo mutational events from the list of rare variants generated for subjects LR08-018 and LR09-006 by focusing on variants supported by ≥ 2 reads in the proband and absent in both parents (&lt; 2 variant reads) at base-pair positions covered by ≥ 4 reads in the entire trio. Candidate de novo mutations were manually inspected using the Integrative Genomics Viewer (IGV)15. This approach was unsuccessful for MCAP trio LR09-006, so we relaxed our criteria to include all variants that failed on the initial hard-filtering parameters used for variant calling, and generated a second list of rare variants and candidate de novo events using the aforementioned criteria.</p>
              <p id="s3s3p2">Genomic DNA samples from LR00-016a1 and both affected siblings were genotyped using the InfiniumII HumanHap610 Quad BeadChip array (Illumina) at the Center for Applied Genomics at Children’s Hospital of Philadelphia prior to the current study. We assumed autosomal recessive inheritance and pulled genes with either homozygous or two heterozygous rare variants. We then used the genome-wide single-nucleotide polymorphism (SNP) data obtained on all three affected siblings to further reduce the number of candidate genes. For a dominant mode of inheritance, we manually examined candidate heterozygous variants from the list of rare variants generated for LR00-016a1.</p>
            </div>
          </section>
          <section data-origid="S5" data-type="subjects">
            <h3 id="s3s4h1">Additional MCAP patients and control exomes</h3>
            <div class="content">
              <p id="s3s4p1">Exome capture and sequencing from seven unrelated MCAP patients was performed at the McGill University and Genome Quebec Innovation Centre (Montreal, Canada) using the 50 Mb SureSelect human exome kit (Agilent) and sequencing of 100-bp paired-end reads on Illumina Hiseq. We generated over 15 Gb of sequence per individual such that approximately 90% of the coding bases of the genome defined by the consensus coding sequence project were covered by at least 20 reads. Reads were aligned to hg19 with BWA47, and duplicate reads were marked using Picard (see URLs) and excluded from downstream analysis. For each sample, single-nucleotide variants and indels were called using SAMtools pileup and varFilter48 with the base alignment quality adjustment disabled, and were then quality filtered using a threshold of at least 20% of reads supporting the variant call. Due to low-level mosaicism in most MCAP patients, this initial analysis missed all but one mutation of PIK3CA, which were later found using other methods. The study also included exome sequencing data from 174 individuals sequenced at the University of Washington Genome Sciences using the SeqCap EZ Exome Library v2.0 (Roche) and Illumina sequencing platform. These 174 individuals consist of unrelated healthy individuals of European descent that are parents of children with sporadic autism.</p>
              <p id="s3s4p2">To detect low-level mosaicism, we systematically analyzed the 3,287 coding and splice-site bases of PIK3CA in exomes from the remaining six unsolved MCAP patients and 174 unrelated control individuals. Standard variant calling49 in the patient and control datasets identified 36 heterozygous variants supported by 41 to 66% of reads, and one homozygous change found in 99% of reads. These sites were considered as germline variants and excluded from downstream analysis. To detect candidate mosaic changes, we first pulled all sites with at least one read not matching the reference sequence using the SAMtools (v0.1.7) pileup48. Raw variant sites were then run through a custom pipeline to identify candidate mosaic changes supported by at least 1, 2, 3 or 4 variant reads using a base quality threshold of 20 and at least 1% of reads supporting the variant.</p>
            </div>
          </section>
          <section data-origid="S6" data-type="">
            <h3 id="s3s5h1">Sanger sequencing</h3>
            <div class="content">
              <p id="s3s5p1">We amplified the coding exons of AKT3 (NM_005465.4 and NM_181690.1), BDP1 (NM_018429.2), PIK3R1 (NM_181523.1, NM_181524.1 and NM_181504.2), PIK3R2 (NM_005027.2), PIK3R3 (AK302049.1 and EU832531.1), PIK3CA (NM_006218.2), PIK3CB (NM_006219.1), PIK3CD (NM_005026.3 and U57843.1), and PTEN (NM_000314.4) using custom intronic primers and standard polymerase-chain-reaction (PCR) protocols (primer sequences available on request) combined with Sanger sequencing. Amplicons were sequenced at the University of Washington High-Throughput Genomics Unit and Seattle BioMed Sequencing Core Facility. All mutations were tested in at least two independent amplification and sequencing reactions in the proband and available relatives. Sequence traces were analyzed using Mutation Surveyor v3.97 (SoftGenetics). For all mutations, nucleotide-level conservation and impact of amino-acid substitutions were assessed using GERP50 and Grantham matrix scores51, respectively.</p>
            </div>
          </section>
          <section data-origid="S7" data-type="">
            <h3 id="s3s6h1">Restriction-enzyme and genotyping assays</h3>
            <div class="content">
              <p id="s3s6p1">Six mutation sites (p.Glu453del, p.Glu726Lys, p.Gly914Arg, p.Ala1035Val, p.Met1043Ile, and p. His1047Tyr) were confirmed by PCR using HotStarTaq Plus DNA polymerase (Qiagen), and primer pairs including a carboxyfluorescein (FAM)-labeled primer and an unlabeled primer with a GTTTCTT sequence on the 5′ end52. Amplicons were digested using restriction enzymes (New England BioLabs) cutting either the wild-type or the mutant allele, depending on the mutation examined (Supplementary Table 11). Fragments were detected on the ABI 3730XL DNA Analyzer (Applied Biosystems) and results were analyzed using GenMarker v1.90 software (SoftGenetics). The fraction of mutant allele was calculated as follows: mutant peak area/(wild-type peak area + mutant peak area).</p>
              <p id="s3s6p2">The p.Glu726Lys PIK3CA mutation is located in a 7.2 kb segmental duplication (chr3:178932477-178939690) sharing 98% identity with a region on chromosome 22 (chr22:17049390-17056254). To rule out the possibility of non-specific amplification, we amplified a 12.2 kb fragment using the TaKaRa LA Taq and primers flanking the segmental duplication (primer sequences are listed in Supplementary Table 11). We then used the long-range PCR product as a template for our restriction-enzyme assay.</p>
              <p id="s3s6p3">The p.Glu453del PIK3CA mutation was confirmed by standard PCR using a forward FAM-labeled primer and an unlabeled reverse primer to amplify 238 and 235 bp fragments corresponding to the wild-type and mutant alleles, respectively.</p>
            </div>
          </section>
          <section data-origid="S8" data-type="">
            <h3 id="s3s7h1">Deep sequencing of mutation sites</h3>
            <div class="content">
              <p id="s3s7p1">We performed targeted ultra-deep sequencing of five mutation sites in PIK3CA in a series of MCAP and MPPH patients and unaffected parents using chimeric oligonucleotides containing Illumina adapter sequences combined with locus specific primers. Reverse oligonucleotides also contained 12 unique 8-mer barcodes for multiplexing of up to 12 samples per lane (Supplementary Table 12). Amplification was carried out in a MiniOpticon Real-time PCR system (Bio-Rad) using the iProof High-Fidelity Master Mix (Bio-Rad), 50 ng of genomic DNA, and SYBR Green. Samples were removed from the PCR machine before fluorescence began to plateau. Amplicons were purified using Ampure beads (Agencourt) and directly sequenced on an Illumina HiSeq 2000 using 100 bp paired-end reads. Sequencing reads were aligned to hg19 using BWA47. We used the SAMtools (v0.1.7) pileup48 combined with custom scripts to pull variant sites supported by at least 1% of reads with a base quality score threshold of 20.</p>
              <p id="s3s7p2">The two newly identified mutations (p.Glu81Lys and p.Arg88Gln in subjects LR06-342 and LR11-068, respectively) were validated by an independent ultra-deep sequencing experiment including two DNA sources per affected individual and DNA samples from parents of LR11-068. All six DNA samples were amplified and sequenced twice using 8-mer barcodes.</p>
            </div>
          </section>
          <section data-origid="S9" data-type="">
            <h3 id="s3s8h1">Lymphoblastoid cell lines</h3>
            <div class="content">
              <p id="s3s8p1">Epstein-Barr virus immortalized lymphoblastoid cell lines were established from peripheral blood samples of four megalencephaly patients (LR00-016a1, LR08-018, LR05-204, and LR09-006), one unaffected individual, and one patient with Cowden disease (GM10080). Lymphoblastoid cell lines were cultured in RPMI medium with 15% fetal calf serum (FCS), L-Gln and antibiotics (Pen-Strep) at 5% CO2.</p>
            </div>
          </section>
          <section data-origid="S10" data-type="">
            <h3 id="s3s9h1">Antibodies and PI3K inhibitor</h3>
            <div class="content">
              <p id="s3s9p1">Anti-phosphatidylinositol-3,4,5-trisphosphate (PIP3) monoclonal antibody was from Caltag-Medsystems Ltd (UK). The following antibodies were obtained from Cell Signalling Technology: PDK1-pSer241, S6-pSer240/244, 4E-BP-pThr37/46, and β-tubulin. Conjugated secondary antibodies, anti-mouse IgG-FITC (whole molecule), and anti-rabbit IgG-Cy3 F(ab′)2 fragment were from SIGMA-ALDRICH (UK). We used the PI3K inhibitor InSolution™ PI-103 (Calbiochem) from Merck-Millipore.</p>
            </div>
          </section>
          <section data-origid="S11" data-type="">
            <h3 id="s3s10h1">Indirect immunofluorescence</h3>
            <div class="content">
              <p id="s3s10p1">Exponentially growing lymphoblastoid cell lines were pelleted, swollen in 75mM KCL (10 minutes), immobilized onto poly-lysine coated slides by cytospinning (CytoSpin. Shandon), permeabilized (0.1% triton X-100 in 5% BSA/PBS for 2 minutes) and blocked in 5% BSA/PBS (10 minutes), prior to sequential incubation with primary and secondary antibodies. Slides were counterstained with DAPI and preserved in anti-fade mounting media (Vectashield). Slides were analyzed using the Zeiss AxioPlan platform. Images were captured using SimplePCI software and the same exposure time for each sample. Images (TIFF format) were then used to quantify signal (RGB) using ImageJ software.</p>
            </div>
          </section>
          <section data-origid="S12" data-type="">
            <h3 id="s3s11h1">SDS-page and Western Blot</h3>
            <div class="content">
              <p id="s3s11p1">Whole-cell extracts were prepared by lysing the cells in urea buffer (9M urea, 50mM Tris-HCl at pH 7.5 and 10mM 2-mercaptoethanol), followed by 15sec sonication, 30% amplitude using a micro-tip (SIGMA-Aldrich). The supernatant was quantified by Bradford Assay. Extracts were resolved by SDS-PAGE, semi-dry blotted onto polyvinylidene fluoride (PVDF) membranes for blocking and antibody incubation.</p>
            </div>
          </section>
          <section data-origid="S13" data-type="">
            <h3 id="s3s12h1">Statistical analysis</h3>
            <div class="content">
              <p id="s3s12p1">The likelihood of observing a second de novo mutation in AKT3 was calculated with the following: P = 1.28 × 10−8 × 1,540 × 98; with 1.28 × 10−8 as the estimated haploid substitution rate53, 1,540 as the number of coding and splice-site bases of AKT3 (NM_005465.4 and NM_181690.1), and 98 as the number of chromosomes tested (49 unrelated patients). We used two-tailed Fisher’s exact tests for genotype-phenotype correlations.</p>
            </div>
          </section>
        </div>
      </section>
      <section data-origid="S14" data-type="supplementary-material">
        <h2 id="s4h1">Supplementary Material</h2>
      </section>
      <section data-type="resources">
        <h2 id="s5h1">Resources</h2>
        <div class="content">
          <figure data-origid="F1">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3408813/#F1">Figure 1</a>
            <figcaption>
              <p id="s5f1p1">Craniofacial appearance and magnetic resonance imaging (MRI) of the three index patients. Photos and brain MRI of patients LR08-018 (a–c), LR00-016a1 (d–f), and LR09-006 (g–i). Photos of patients LR08-018 (a), LR00-016a1 (d) and LR09-006 (g) were taken at 11 months, 15 years, and 5 months, respectively. Note the prominent forehead and apparent macrocephaly in all three patients (a, d, g), and a midline facial capillary malformation (or nevus flammeus) in LR09-006 (g). Midline sagittal brain images (b, e, h) show prominent forehead, increased cranium-to-face ratio and cerebellar tonsillar ectopia (arrowheads), while axial or coronal images (c, f, i) show bilateral perisylvian polymicrogyria (arrows). Scale bars correspond to 1 cm. Additional photos of patient LR09-006 and a clinical description of the three index patients are provided in the Supplementary Note. We obtained written consent to publish photographs of the patients.</p>
            </figcaption>
          </figure>
          <figure data-origid="F2">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3408813/#F2">Figure 2</a>
            <figcaption>
              <p id="s5f2p1">PIP3 levels in lymphoblastoid cell lines derived from an unaffected control, a patient with Cowden disease (GM10080), and four megalencephaly patients. (a) Indirect immunofluorescence staining of PIP3 in exponentially growing lymphoblastoid cell lines using a mouse monoclonal anti-PIP3 antibody (Online Methods). Scale bar corresponds to 10μm. (b) Per-cell quantification of PIP3 levels based on anti-PIP3 signal intensity (a.u., arbitrary units). Levels of PIP3 signal in control cells (WT) are comparable to those of LR08-018 (AKT3 p.Arg465Trp). All other mutant cell lines show increased PIP3 signal compared to control cells. Elevated PIP3 signal is also evident in cells derived from the patient with Cowden disease (PTEN p.Glu261Ter), which served as a positive control. * Statistically significant difference compared to control cells (p&lt;0.05 two-tailed t-test assuming unequal variance, n=30 to 50 cells per cell line). Error bars indicate standard deviation. (c) Levels of PIP3 in cell lines from LR00-016a1 (PIK3R2 p.Gly373Arg) and LR05-204 (PIK3CA p.Glu453del) can be reduced following treatment with the PI3K-inhibitor PI-103 (5μM for 16 hours). Scale bar corresponds to 10μm.</p>
            </figcaption>
          </figure>
          <figure data-origid="F3">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3408813/#F3">Figure 3</a>
            <figcaption>
              <p id="s5f3p1">Distribution of mutations in AKT3, PIK3R2, and PIK3CA. The activating Akt3 mutation in mouse is indicated in grey (p.Asp219Val). For recurrent mutations, the number of occurrences is indicated in parentheses. PH: pleckstrin homology domain; C-terminal: carboxyl-terminal domain; SH2 and SH3: Src-homology-2 and -3 domains; Rho-GAP: Rho GTPase-activating protein domain; p85-BD and RAS-BD: p85- and RAS-binding domains; C2: protein-kinase-C-homology-2 domain. The PIK3CA mutations affect a total of 15 residues, mainly localized in the p85-binding, C2, and catalytic lipid kinase domains.</p>
            </figcaption>
          </figure>
          <figure class="table" data-origid="T1">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3408813/#T1">Table 1</a>
            <figcaption>
              <p id="s5f4p1">Summary of AKT3, PIK3R2 and PIK3CA mutations identified in 37 megalencephaly families</p>
            </figcaption>
          </figure>
          <figure class="table" data-origid="T2">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3408813/#T2">Table 2</a>
            <figcaption>
              <p id="s5f5p1">PIK3CA mutations identified or confirmed by deep sequencing of five mutation sites</p>
            </figcaption>
          </figure>
        </div>
      </section>
    </article>
  </body>
</html>
