<!DOCTYPE html >
<html id="aNDzh5Ar0TCoX1IIlBiy81PLoN3m-nar_gkp166" data-origid="nar/gkp166" class="anndoc" data-anndoc-version="2.0" lang="" xml:lang="" xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="UTF-8"/>
    <meta name="generator" content="com.jmcejuela.anndoc.v2.bio.NcbiJournalArticleParser_v0_2$"/>
    <meta name="dcterms.source" content="http://doi.org/10.1093/nar/gkp166"/>
    <title>aNDzh5Ar0TCoX1IIlBiy81PLoN3m-nar_gkp166</title>
  </head>
  <body>
    <article>
      <section data-type="title">
        <h2 id="s1h1">The solution structure of the first PHD finger of autoimmune regulator in complex with non-modified histone H3 tail reveals the antagonistic role of H3R2 methylation</h2>
      </section>
      <section data-type="abstract">
        <h2 id="s2h1">Abstract</h2>
        <div class="content">
          <p id="s2p1">Plant homeodomain (PHD) fingers are often present in chromatin-binding proteins and have been shown to bind histone H3 N-terminal tails. Mutations in the autoimmune regulator (AIRE) protein, which harbours two PHD fingers, cause a rare monogenic disease, autoimmune polyendocrinopathy-candidiasis-ectodermal dystrophy (APECED). AIRE activates the expression of tissue-specific antigens by directly binding through its first PHD finger (AIRE-PHD1) to histone H3 tails non-methylated at K4 (H3K4me0). Here, we present the solution structure of AIRE-PHD1 in complex with H3K4me0 peptide and show that AIRE-PHD1 is a highly specialized non-modified histone H3 tail reader, as post-translational modifications of the first 10 histone H3 residues reduce binding affinity. In particular, H3R2 dimethylation abrogates AIRE-PHD1 binding in vitro and reduces the in vivo activation of AIRE target genes in HEK293 cells. The observed antagonism by R2 methylation on AIRE-PHD1 binding is unique among the H3K4me0 histone readers and represents the first case of epigenetic negative cross-talk between non-methylated H3K4 and methylated H3R2. Collectively, our results point to a very specific histone code responsible for non-modified H3 tail recognition by AIRE-PHD1 and describe at atomic level one crucial step in the molecular mechanism responsible for antigen expression in the thymus.</p>
        </div>
      </section>
      <section data-type="">
        <h2 id="s3h1">INTRODUCTION</h2>
        <div class="content">
          <p id="s3p1">The plant homeodomain (PHD)-type zinc finger is a chromatin-binding domain of ∼60 amino acids found in ∼150 proteins in the human genome (1). Many PHD fingers act as nucleosome interaction determinants playing a fundamental role in histone recognition and epigenetic mechanisms (1–3). Recently, PHD fingers have emerged as motifs that specifically recognize the methylation status of lysine residues in histone H3 N-terminal tails (4–6). In particular, two distinct subclasses of PHD fingers have been identified, which can specifically bind with micromolar affinity either to methylated (H3K4me3, H3K9me3) (7–9) or non-methylated (H3K4me0) lysine residues on histone H3 N-terminal tail (10,11). Importantly, PHD fingers reside within multidomain proteins containing other histone or DNA-binding domains such as Bromo or SAND domains, respectively. Moreover, they take part in larger transcriptional coactivator or corepressor protein complexes, which operate on the chromatin template with multivalent interactions (12). The physiological relevance of PHD modules is highlighted by the occurrence of pathological PHD finger mutants in genes such as RAG2, ING2, NSD1 and ATRX, underlying different human diseases that include immunological disorders, cancer and neurological diseases (13). Two contiguous PHD fingers are present in autoimmune regulator (AIRE), a transcriptional activator that is defective in a monogenic autosomal recessive disease named autoimmune polyendocrinopathy-candidiasis-ectodermal dystrophy (APECED).</p>
          <p id="s3p2">Although rare, APECED is considered as an invaluable model to provide insights into mechanisms by which immune tolerance is maintained. APECED patients are characterized by multiple features of abnormal immunological tolerance leading to destructive autoimmune reactions in several organs (14). AIRE is predominantly expressed in thymic medullary epithelial cells where it controls the expression of many tissue-specific antigens (15). The absence of functional AIRE leads to the escape of self-reactive T-lymphocytes to the periphery and ultimately to pathogenic autoimmune reactions (16). The mechanism, by which AIRE promotes the expression of tissue-specific antigens in thymic medullary epithelial cells, is still under investigation but it has been shown that AIRE regulates the expression of its target genes via PHD domains (17–19). The presence of several APECED-causing missense and truncation mutations in PHD domains highlights their crucial role in AIRE function (20). Furthermore, AIRE PHD fingers have been implicated in nuclear dots formation and in transactivational capacity (18,21). Notably, we and others showed that AIRE-PHD1 binding to H3K4me0 is needed for activation of AIRE target genes (22,23). Using a combination of biochemical and biophysical methods, we showed that AIRE-PHD1 is a specialized histone tail reader module, which binds most tightly to H3K4me0 peptide (Kd ∼5 µM) (22). Interestingly, AIRE-PHD2 does not belong to the PHD class of histone readers and its functional role is still unknown (24).</p>
          <p id="s3p3">The structural and thermodynamic details at the basis of H3K4me3 recognition by PHD fingers are well documented (5,25–31). Conversely, the recognition of non-methylated H3K4me0 peptides by PHD domains is still poorly characterized, as of now, only two crystallographic structures have been reported that describe H3K4me0 interactions with BHC80-PHD and DNMT3L-Cys-rich domain (10,11). To address this question, we have determined the solution structure of AIRE-PHD1 in complex with a peptide corresponding to the first 10 amino acids of histone H3 and performed extensive mutagenesis analysis to verify the interaction specificity. We show that AIRE-PHD1 is exquisitely sensitive to histone H3 post-translational modifications, pointing to a very specific recognition of H3K4me0 by AIRE-PHD1. In particular, R2 dimethylation (H3R2me2) abrogates the binding in vitro and, correspondingly, methylation of H3R2 through overexpression of methyltransferase PRMT6 reduces the in vivo activation of AIRE target genes in HEK293 cells. Our data indicate that H3R2 dimethylation acts as a negative determinant for AIRE binding to its target gene promoters.</p>
        </div>
      </section>
      <section data-type="materials|methods">
        <h2 id="s4h1">MATERIALS AND METHODS</h2>
        <div class="subsections">
          <section data-type="">
            <h3 id="s4s1h1">Sample preparation for NMR and binding assays</h3>
            <div class="content">
              <p id="s4s1p1">The AIRE-PHD1 construct used for structure determination spans wild-type residues Gln293–Glu354. Site-directed mutagenesis was made by standard overlap extension methods. All DNA constructs were sequenced in-house, and the molecular weights of the recombinant proteins were verified by mass spectrometry (MALDI). All the mutants were well folded with the exception of AIRE-PHD1-Arg303Pro as assessed by 1H 1D NMR spectra (Supplementary Figure S1, S4C). The modified pET-24d vectors express proteins with N-terminal His6 tags, removable by cleavage with TEV (tobacco etch virus) protease, enabling use of non-tagged proteins in NMR studies. The protein purification strategy was previously described (32).</p>
              <p id="s4s1p2">Synthetic non-modified H3K4me0 (NH2-ARTKQTARKS-COOH), with corresponding post-translational modifications (H3R2me1K4me0, mono-methylation of R2; H3R2me2sK4me0, symmetric dimethylation of R2; H3R2me2asK4me0, asymmetric di-methylation of R2; H3T3PhK4me0, T3 phosphorylation; H3K9AcK4me0, acetylation of K9; H3S10PhK4me0, phosphorylation of S10) and mutant peptides H3R2KK4me0 (NH2-AKTKQTARKS-COOH), H3R8AK4me0 (NH2-ARTKQTAAKS-COOH) were purchased from CASLO (Lyngby, Denmark), their purity was confirmed by HPLC and mass spectrometry.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s2h1">NMR-binding assays</h3>
            <div class="content">
              <p id="s4s2p1">For NMR titrations the 15N-labelled AIRE-PHD1 domain and the titrant (different peptides) were prepared in identical buffers: 20 mM sodium phosphate pH 6.3, 0.15 M NaCl, 5 mM DTT, 50 µM ZnCl2 and 10% (v/v) D2O. Stepwise additions of the titrant were made up to a ratio of 1:3, AIRE-PHD1:titrant, respectively. Titrations were followed by collecting 1H-15N HSQC spectra after each addition, and the spectra were examined for changing chemical shifts.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s3h1">Fluorescence titration assays</h3>
            <div class="content">
              <p id="s4s3p1">Tryptophan fluorescence spectra were recorded at 20°C on a Varian Eclipse spectrofluorimeter. The samples of 3 μM AIRE-PHD1 finger (wild-type protein or mutants) containing progressively increased concentration (up to 150 μM) of histone H3 tail peptides were excited at 295 nm, in 20 mM phosphate buffer, pH 6.3, 150 mM NaCl, 5 mM DTT and 50 μM ZnCl2. Emission spectra were recorded between 320 and 340 nm with a 0.5 nm step size and a 1 s integration time and averaged over three scans. The peptides were injected sequentially from stock solutions of 1–10 mM. The dissociation constants (KD) were determined by a nonlinear least-squares analysis using the equation: I = (Imax × [L])/(KD + [L]), where L is the concentration of the peptide, I is observed change of signal intensity and Imax is the difference in signal intensity of the free and bound states of the protein. Fluorescence data were corrected for the effect of protein dilution. Data shown are representative of two repeated measurements.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s4h1">Isothermal titration calorimetry thermodynamic analysis</h3>
            <div class="content">
              <p id="s4s4p1">Thermodynamic analyses were performed at 23°C using a VP-ITC isothermal titration calorimeter (MicroCal LLC, Northampton, MA, USA). Histone peptides (0.54–2.5 mM) were injected in AIRE-PHD1 wild type and mutants solutions (40–85 μM). The reaction buffer contained 20 mM phosphate buffer, 150 mM NaCl, 2 mM β-mercaptoethanol, 50 μM ZnCl2 and pH 7.2. All titrations were performed using stirring of 307 rpm, with 28–57 injections, up to 12 μl for each injection and 15 min of spacing between injections into the isothermal titration calorimetry (ITC) sample cell of 1.41 ml. Control experiments were performed under identical conditions to determine the dilution heat of the titrant peptide into buffer and of the buffer into protein samples. The final data were analysed using the software ORIGIN 7.0 (OriginLab Corp., Northampton, MA, USA), integrals of differential heats of injection were corrected by dilution heats and finally fitted by one-site-binding model and nonlinear least-square regression. Measurements were performed in duplicate.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s5h1">NMR spectroscopy and data analysis for structure determination</h3>
            <div class="content">
              <p id="s4s5p1">NMR samples contained typically 0.4 mM uniformly 13C/15N-labelled AIRE-PHD1 and unlabeled H3K4me0 in a 1:0.8 or 1:3 complex, 150 mM NaCl, 20 mM KH2PO4/K2HP04 (pH 6.3) and 50 μM ZnCl2. All NMR experiments were performed at 20°C on Bruker Avance 600 Ultra Shield TM Plus 600 and 800 MHz spectrometers equipped with triple resonance cryoprobes and pulsed field gradients. Spectra were processed with NMRPipe (33) and analysed with CCPNMR (34). Backbone and side-chain resonances of AIRE-PHD1 in complex with H3K4me0 were assigned using 3D HNCA, HNCACB, HNCO, HNHA, H(C)CH-TOCSY, (H)CCH-TOCSY and HCCH-TOCSY (35).</p>
              <p id="s4s5p2">3J(HN, H)-coupling constants were measured to derive restraints for dihedral angles. Additional restraints were obtained from backbone chemical shifts using TALOS (36). Hydrogen bond restraints were defined from slow-exchanging amide protons identified after exchange of the H2O buffer to D2O. 1H–15N residual dipolar couplings were measured in isotropic and anisotropic phases created by the addition of 15 mg/ml Pf1 phage (ASLA Biotech Ltd).</p>
              <p id="s4s5p3">The bound H3K4me0 peptide was assigned analysing 2D 13C/15N-filtered NOESY (mixing time of 150 ms) and TOCSY (mixing time 60 ms) experiments (37). Distance restraints for AIRE-PHD1 structure calculations were obtained from 3D 15N–13C-separated experiments by using mixing times 100 ms. The intermolecular NOEs were obtained from 2D NOESY experiments with 15N/13C filter in F2 (mixing time 200 ms) (37). 15N relaxation rates and heteronuclear NOE experiments were measured using standard two-dimensional methods as previously described (32). The relaxation experiments were analysed with the program Nmrview.5.03 (38).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s6h1">Structure calculation and validation</h3>
            <div class="content">
              <p id="s4s6p1">Structures were calculated using ARIA2.2 (39) in combination with CNS (40) using the experimentally derived restraints (Table 1). All NOEs were assigned manually and calibrated by ARIA2.2. A total of eight iterations (20 structures in the first six iterations) were performed: 200 structures were computed in the last two iterations. The ARIA2.2 default water refinement was performed on the 20 best structures of the final iteration. For the final 20 lowest energy structures, no distance or torsional angle restraints was violated by more than 0.5 Å or 5°, respectively (Table 1). Structural quality was assessed using PROCHECK-NMR (41) and WHATIF (42) (Supplementary Table S1). Molecular images were generated using Pymol, DeLano Scientific (http://pymol.org/). Structural alignment was performed with DaliLite v.3 (43). The structural coordinates of AIRE-PHD1:H3K4me0 complex have been deposited in the Protein Data Base under accession code 2ke1. Table 1.Structural statistics for AIRE-PHD1:H3K4me0 complex&lt;SA&gt;aRestraints informationb,c    Total number of experimental distance restraints1202    AIRE-PHD1 (intraresidual/sequential/medium/long)552/229/103/190    H3K4me0 (intraresidual/sequential/medium/long)42/13/1/0    Intermolecular restraints72    Zn2+ coordination restraints8    Hydrogen bonds16    Dihedral angle restraints86    Residual dipolar couplings52Average RMS deviation from experimental restraints    All experimental distances restraints (Å)0.014 ± 0.001    All dihedral angle restraints (°)0.27 ± 0.05Coordinate RMS deviation (Å)d    Ordered backbone atoms (N, Cα, C′)0.52 ± 0.08    Ordered heavy atoms1.02 ± 0.10Ramachandran quality parameters (%)e    Residues in most favoured regions85.2%    Residues in allowed regions14.4%    Residues in additional allowed regions0.4%    Residues in disallowed regions0.0%aSimulated annealing, statistics refers to the ensemble of 20 structures with the lowest energy.bNo distance restraint in any of the structures included in the ensemble was violated by more then 0.5 Å.cNo dihedral angle restraints in any of the structures included in the ensemble was violated by more than 5°.dRoot mean squared deviation between the ensemble of structures &lt;SA&gt; and the lowest energy structures.eStatistics are given for residues 296–344 of AIRE-PHD1 and for residues 1–7 of H3K4me0.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s7h1">Transfection, plasmids and expression analyses</h3>
            <div class="content">
              <p id="s4s7p1">HEK293 cells were transfected either with pcAIRE (44), myc-tagged pVAX-PRMT6 (kind gift from B. Amati) (45), pd2EYFP-N1 (Clontech Mountain View, CA, USA) or pcDNA3.1B-myc/his (Invitrogen, Carlsbad, CA, USA) using ExGen500 in vitro reagent (Fermentas, Lithuania) according to the manufacturer's protocol. pd2EYFP-N1 was added to compensate the total DNA amount if AIRE and PRMT6 were separately transfected, equal amount of pcAIRE and pVAX-PRMT6 were used in each transfection. RNA purification and expression analysis was essentially performed as previously described (46). Sequences of used PCR primers were previously published (22). To detect transfected protein levels, the following antibodies were used in Western analysis: total H3 (Ab1791 Abcam, Cambridge, UK), anti-human AIRE (6.1) (44), anti-myc (9E10, Sigma), rabbit polyclonal anti-dimethyl-Arg2-Histone H3 (07-585, Upstate, Charlottesville, VA, USA) and rabbit polyclonal anti-mono-methyl Arg2-Histone H3 (Abcam ab15584).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s8h1">Peptide-binding assays</h3>
            <div class="content">
              <p id="s4s8p1">Biotinylated peptides used in in vitro binding assays were (ARTKQTARKSTGGKAPRKQLA-GGK-Biotin with corresponding modifications) H3, H3S10PhK4me0 (Upstate Biotechnology), H3R2me2sK4me0, H3R2me2asK4me0 (Biopeptide Co., Inc., San Diego, CA, USA). The peptide pull-down experiments with GST-AIRE-PHD1 or GST were performed as described earlier (22).</p>
            </div>
          </section>
        </div>
      </section>
      <section data-type="">
        <h2 id="s5h1">RESULTS AND DISCUSSION</h2>
        <div class="subsections">
          <section data-type="">
            <h3 id="s5s1h1">Solution structure of AIRE-PHD1 in complex with H3K4me0</h3>
            <div class="content">
              <p id="s5s1p1">Using classical heteronuclear and isotope-filtered NMR experiments, we determined the three-dimensional solution structure of AIRE-PHD1 in complex with H3K4me0 (Figure 1A–C, Table 1 for structural statistics). Overall, the complex recapitulates the classical recognition mode observed in PHD:H3 structures and reflects in part the interactions that we predicted in a previous three-dimensional model (22): the H3K4me0 peptide docks onto an extensive negatively charged binding area (∼750 Å2) and via an induced fit mechanism it forms a third antiparallel β-strand that pairs with the protein's two-stranded antiparallel β-sheet (Figure 1B). AIRE-PHD1 constitutes a stable pre-formed binding platform as formation of the complex does not induce significant conformational changes in the protein backbone with a RMSD of ∼1 Å between the free (32) and bound AIRE-PHD1 (Supplementary Figure S2A). Substrate recognition results in increased domain rigidity in the ps–ns time scale as assessed by the increased values of heteronuclear NOEs in the binding region comprising residues 295–298 and 332–335 (Supplementary Figure S3). The complex is stabilized by complementary surface interactions, whereby the side chains of the first four H3 tail residues snugly fit inside the pre-formed AIRE-PHD1-binding groove, previously identified by chemical shift perturbation methods (22) (Figure 1C). As observed in other PHD:H3 complexes, peptide residues R2, T3, K4, Q5 adopt an extended conformation and make characteristic β-sheet NOE interactions and inter-strand hydrogen bonds with residues Cys310, Ile309 and Leu308 (we use single-letter code and three-letter code to identify peptide and protein residues, respectively). The amino-group of A1 is tightly bound to the backbone carbonyl atoms of Gly333 and Pro331 (Figure 1B, Supplementary Table S2). Both the methyl groups of A1 and T3 insert into a hydrophobic pocket formed by Pro331, Leu308, Ile330 and Trp335 contributing to complex stabilization (Figure 1B and C). Charge complementarities and hydrophilic interactions appear to be the binding driving force, in particular, R2 and K4 play a major role in substrate recognition by occupying two adjacent negatively charged cavities separated by a non-conserved asparagine group (Asn295) (Figure 1B and C). The guanidinium moiety of R2 orients towards the first cavity forming bifurcated electrostatic interactions with both the backbone carbonyl of Cys311 and Asp312 and with the carboxylate of Asp312 (Figure 1B, Supplementary Table S2). The formation of a salt-bridge between R2 and Asp312 was previously predicted by a three-dimensional model of the complex and by the observation that R2A and Asp312Ala mutants impaired binding (22). We tested whether the multiple interactions between the R2 guanidinium group and AIRE-PHD1 are specifically crucial for substrate recognition. Indeed, mutation of R2 into lysine destabilizes the complex, as assessed by ITC and fluorescence measurements (Table 2). The adjacent hydrophilic pocket is occupied by the ε amino group of K4, which establishes a network of hydrogen bonds with both the backbone and side-chain carbonyls of Asn295 and the conserved carboxyl group of Asp297 (Figure 1B and C; Supplementary Table S2). Importantly, the side chain of K4 is clasped by a narrow hydrophilic cage that mediates steric exclusion of methylated ε amino group, thus accounting for AIRE-PHD1 specificity for non-methylated K4. Electrostatic interactions between non-methylated K4 and Asp297 were previously suggested by alanine mutagenesis experiments, which showed a decrease of the binding affinity by two orders of magnitudes (22,23). Here, we demonstrate that also Asn295 plays a fundamental role in histone H3 recognition, as mutation into alanine has a detrimental effect on complex formation (Table 2). Finally, the amide of T6 makes a stable hydrogen bond contact with the carbonyl of Gly306 and its hydroxyl group interacts with the backbone carbonyl of Glu297 (Figure 1, Supplementary Table S2). Residues 7–10 of H3K4me0 are less structured and present a higher degree of flexibility compared to the first six amino acids, as suggested from the lack of sequential and medium range cross peaks, from the strong intensity of their TOCSY peaks (data not shown) and from the paucity of intermolecular NOEs. Despite the high degree of disorder of this region we often observed the formation of polar contacts between the guanidinium group of R8 and the carboxyl groups of Asp304 and/or Glu307 and between K9 and the side chains of Glu298 and Asp304 (Supplementary Table S2). Indeed, fluorescence-binding experiments and ITC performed on either Glu298Ala, Asp304Ala, Glu307Ala or H3R8A mutants indicated a more than 5-fold reduction in the binding affinity supporting the notion that transient electrostatic interactions occur in the complex (Table 2). Finally, the C-terminal residue S10 is highly disordered and does not contribute with stable interactions to complex formation. Figure 1.Solution structure of AIRE-PHD1 in complex with H3K4me0. (A) Stereoview of the ensemble of 20 complex structures, the backbone traces of AIRE-PHD1 and the side chains of the peptide are represented in pink and blue, respectively. (B) Cartoon representation and (C) surface plot of the lowest energy complex structure. Cyan-dashed lines represent intermolecular hydrogen bonds. Colour coding: pink, AIRE-PHD1; blue, H3K4me0; orange, protein residues forming specific polar contacts with H3K4me0; green, protein residues forming hydrophobic contacts with H3K3me0; grey, Zn2+ ions. Table 2.Values of the dissociation constants between H3 peptides and AIRE–PHD1 wild type (wt) and mutants measured by fluorescence spectroscopy and isothermal titration calorimetryAire-PHD1Fluorescence spectroscopyIsothermal titration calorimetryPeptideKd (μM)ΔH (kcal/mol)−TΔS (kcal/mol)ΔG (kcal/mol)Kd (μM)WtH3K4me0a5.3 ± 1.2−9.3 ± 0.12.2 ± 0.1−7.1 ± 0.16.5 ± 0.2WtH3R2me1K4me0248.0 ± 20.0−4.6 ± 0.70.4 ± 0.1−4.2 ± 0.7198.8 ± 12.4WtH3R2mes2K4me0NDNMNMNMNMWtH3R2me2asK4me0NDNMNMNMNMWtH3T3PhK4me0NDNDNDNDNDWtH3K9me3K4me030.4 ± 2.2−7.6 ± 0.11.5 ± 0.1−6.1 ± 0.129.4 ± 0.5WtH3K9AcK4me039.2 ± 2.3−7.4 ± 0.21.5 ± 0.2−5.9 ± 0.247.6 ± 1.5WtH3S10Ph K4me07.6 ± 1.2−9.6 ± 0.13.3 ± 0.1−6.3 ± 0.124.7 ± 0.2WtH3R2KK4me0NDNDNDNDNDWtH3R8AK4me052.8 ± 4.9−7.3 ± 0.31.6 ± 0.3−5.8 ± 0.355.9Asn295AlaH3K4me0NDNMNMNMNMGlu298AlaH3K4me028.1 ± 2.1−7.2 ± 0.11.3 ± 0.1−5.9 ± 0.143.4 ± 0.5Arg303ProH3K4me0NDNDNDNDNDAsp304AlaH3K4me080.0 ± 7-2−5.3 ± 0.40.1 ± 0.4−5.2 ± 0.4122.8 ± 6.7Glu307AlaH3K4me019.0 ± 2.4−6.7 ± 0.10.7 ± 0.1−5.9 ± 0.140.6 ± 0.5aKd values were previously measured by Org et al. (22).ND, not detectable as binding is too weak to be reliably quantified; NM, not measured.</p>
              <p id="s5s1p2">Overall, both our structural and mutagenesis studies show that recognition of the positively charged histone H3 tail by the negatively charged AIRE-PHD1 surface is driven by a markedly exothermic-binding enthalpy difference, which is a typical thermodynamic signature of PHD:H3 interactions (7,10,11,25). Accordingly, mutations of non-conserved hydrophilic residues located at the binding interface result in a decrease of the enthalpic contribution to the binding (Table 2). In particular, the side chain of Asp304, which engages in multiple interactions with both K9 and R8, seems to play an important role in complex stabilization, as mutation into alanine results in a strong enthalpic penalty, associated to a 20-fold reduction in binding affinity (Table 2).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s2h1">APECED Arg303Pro mutation unfolds AIRE-PHD1</h3>
            <div class="content">
              <p id="s5s2p1">Several pathological point mutations have been mapped to AIRE-PHD1 (47–49). Prompted by the publication of a newly identified pathological point mutation (Arg303Pro) (49), located nearby the Zn2+-binding residue Cys302 (Supplementary Figure S4A, B), we have characterized its effect on AIRE-PHD1 fold. The increased line width of the 1H-1D spectrum and the loss of chemical shift dispersion (Supplementary Figure S4C) clearly indicate that this mutation destroys the domain structure, affecting most probably the first Zn2+-binding site. As a result, fold destabilization drastically compromises binding to H3K4me0 (Supplementary Figure S4D, Table 2). We therefore suggest that disruption of AIRE-PHD1 fold in the context of the full length protein leads to H3-binding failure, with the consequent reduction of AIRE target genes activation, as already observed in the fold destroying mutation Cys311Tyr (23).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s3h1">Structural comparison with other histone H3-binding PHD fingers</h3>
            <div class="content">
              <p id="s5s3p1">Structural comparison with BHC80-PHD and DNMT3L-Cys-rich domain associated to H3K4me0 peptides shows that the overall structural arrangement of AIRE-PHD1:H3K4me0 complex is well conserved with an RMSD of 1.8 Å (2PUY, 60 Cα atoms) and 2.4 Å (2PVC, 57 Cα atoms), respectively (Supplementary Figure S2B, C). At variance to what observed for the BPTF and ING subclass, AIRE-PHD1, BHC80-PHD and DNMT3L-Cys-rich do not present the typical conserved aromatic side chains used to coordinate the tri- or dimethyl ammonium ion of H3K4me3 via Π-cation interactions (24). Specificity towards non-methylated K4 is primarily obtained through a salt-bridge with a conserved aspartate (Asp297; Asp88, Asp489 for AIRE-PHD1, DNMT3L-Cys-rich and BHC80-PHD, respectively) (Figure 2A–C). In the three domains, this interaction is further stabilized by hydrophilic interactions with neighbouring residues: in DNMT3L-Cys-rich K4 recognition occurs through additional polar contacts with the carboxylate of Asp90 and the backbone carbonyl of Gln93 (Figure 2B); in BHC80-PHD K4 makes polar interactions with the backbone carbonyl of Asp488 (Figure 2C); finally, in AIRE-PHD1, both the backbone and the side-chain carbonyls of Asn295 clasp the ε amino group of K4 (Figure 2A). In 15% of the structures, we also observed polar interactions with the carbonyl of Glu296 (Supplementary Table S2). Importantly, as already observed for the H3K4me3 mark, there is no clear correlation between a single epigenetic signature and its functional readout, as the same epigenetic mark can result in opposite effects depending on the cellular context (50). In fact, H3K4me0 recognition can mediate gene activation or gene repression, as observed for AIRE-PHD1 (22) and BHC80-PHD (10), respectively. Figure 2.Structural comparison with other PHD:H3 complexes. Cartoon representations of (A) AIRE-PHD1:H3K4me0, (B) DNMT3L-Cys-rich:H3K4me0 (2PVC), (C) BHC80-PHD:H3K4me0 (2PUY), (D) Taf3-PHD:H3K4me3 (2K17), (E) Pygo-BCL9-PHD:H3R2me2asK4me2 (2VPG) and (F) RAG2-PHD:H3R2me2sK4me3 (2V87). The side chains of R2 and K4 (blue) and their specific interactions with the different PHD domains are explicitly shown. Intermolecular hydrogen bonds are represented in blue with dotted lines.</p>
              <p id="s5s3p2">An important structural feature, which clearly distinguishes AIRE-PHD1 from the other H3K4me0 readers, is linked to its crucial interaction with R2. Both in DNMT3L:H3K4me0 and BHC80:H3K4me0 complexes the side chain of R2 protrudes out from the domain surface into the solvent (Figure 2B and C), whereas, when bound to AIRE-PHD1, the R2 guanidinium group makes specific multiple polar interactions with Asp312 and Cys311 (Figure 2A). Notably, similar electrostatic contacts with R2 have been observed in H3K4me3 binders such as ING2 (7) and Taf3 (51), in which the side chain of R2 is buried in a negatively charged pocket adjacent to the K4me3 cavity (28) (Figure 2D).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s4h1">AIRE-PHD1 binding to histone H3 tail is sensitive to epigenetic marks: cross-talk by R2 dimethylation</h3>
            <div class="content">
              <p id="s5s4p1">Epigenetic information is encoded in histone variants and in histone post-translational modifications (50). Different patterns, relative abundance and combinations of epigenetic marks influence the recruitment, the stabilization or destabilization of macromolecular complexes affecting chromatin dynamics and accessibility. Moreover, different combinations of histone modifications and their cross-talk may have specific meanings and outcomes (52). Recent studies aiming to investigate the effect of histone H3 modifications on AIRE binding, using microarrays with differently modified histone tails, indicate that R2 methylation and T3 phosphorylation impair the interaction with AIRE-PHD1 (23). We applied ITC, fluorescence titrations, NMR spectroscopy and peptide pull-down experiments to provide a quantitative evaluation of AIRE-PHD1 affinity to H3K4me0 peptides containing different covalent modifications, including R2 methylation (mono-methylation, symmetric and asymmetric dimethylation), T3 phosphorylation, K9 acetylation and S10 phosphorylation. (Table 2, Figure 3). Collectively, our results demonstrate that formation of the complex is remarkably sensitive to covalent modifications, as they generally result in reduction of AIRE-PHD1 binding. In agreement with microarrays studies, R2 methylation and T3 phosphorylation are not compatible with complex formation as no binding is observed in fluorescence titrations, ITC and peptide pull-down assays (Figure 3A–C, Table 2). Accordingly, NMR titration experiments show disappearance of several peaks due to line broadening and reduction of chemical shift perturbation as compared to the non-modified peptide (Figure 3D, Supplementary Figure S5). Indeed, the intimate contact between these residues and the domain (Figure 1B and C) strongly suggests that post-translational modifications might antagonize complex formation through steric hindrance and breakage of stabilizing hydrophilic interactions. On one hand, phosphorylated T3 clashes against Ile309 and Ile330 (Figure 1B and C); on the other hand, methylated R2 is too bulky to fit inside the narrow binding pocket. Consequently, mono or dimethylated R2 is expected to reorient its side chain to avoid atomic clash against the binding cleft, hereby causing disruption of intermolecular hydrogen bonds. As already pointed out, both Taf3-PHD and ING2-PHD present a similar R2-binding pocket (Figure 2D) and R2 methylation interferes with H3K4me3 binding (28,53). Interestingly, in Pygo-Bcl9:H3K4me2 complex R2 side-chain protrudes out from the domain (Figure 2E) and is therefore insensitive to R2 methylation (29). Conversely, RAG2-PHD, which presents an aromatic residue in the R2-binding pocket (Figure 2F), can accommodate both K4 and R2 methylation (53). Taken together, these observations point out that also decoding of methylated R2 can vary among the PHD subfamilies thus revealing an enormous complexity in the readout of combinatorial modifications by the same structural scaffold. Figure 3.Effect of H3 post-translational modifications on binding to AIRE-PHD1. (A) Tryptophan ﬂuorescence binding curves of AIRE-PHD1 with H3K4me0, H3R2me2sK4me0 and H3R2me2asK4me0. (B) Interaction between GST-AIRE-PHD1 fusion protein and amino-terminal histone H3 peptides, all detected by anti-GST antibody. (C) ITC-binding curves for H3K4me0, H3R2me1K4me0 and H3T3PhK4me0 to AIRE–PHD1. The upper panels show the sequential heat pulses for peptide–protein binding, and the lower panels show the integrated data, corrected for heat of dilution and fit to a single-site-binding model using a nonlinear least-squares method (line). (D) Distribution of the backbone amide chemical shift changes observed in 15N-labelled AIRE–PHD1 (0.2 mM) upon addition of a threefold excess of H3K4me0, H3R2me1K4me0, H3R2me2sK4me0 and H3R2me2asK4me0. The asterisks indicate residues whose backbone amide signals disappear during titration owing to line broadening. H3R2me1K4me0 (H3R2me2sK4me0, H3R2me2asK4me0), histone H3 monomethylated (respectively, symmetrically and asymmetrically dimethylated) at arginine 2. Tryptophan ﬂuorescence-binding curves, ITC-binding curves and NMR titrations of AIRE-PHD1 with H3K4me0 refer to previous experiments reported in (22).</p>
              <p id="s5s4p2">We next investigated the effect of K9 acetylation on AIRE-PHD1 binding. Previous microarrays experiments suggested that modification of K9 does not affect complex formation (23); however, fluorescence-binding assays, ITC (Table 2) and NMR titrations (data not shown) point out that K9 acetylation reduces binding affinity (Kd ∼40 μM), probably because of steric hindrance and loss of electrostatic interactions with Glu298 and/or Asp304. Finally, in agreement with microarray studies (23), S10 phosphorylation only mildly affects interaction, confirming that this residue is not crucial for in vitro binding, as anticipated by our structural studies (Table 2).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s5h1">R2 methylation antagonizes AIRE ability to activate gene expression</h3>
            <div class="content">
              <p id="s5s5p1">Several evidences indicate that R2 asymmetric dimethylation has a widespread silencing effect on gene transcription and impedes histone H3 tail effector binding (45,54,55). Furthermore, recruitment of the histone H3R2 methyltransferase PRMT6 to gene loci results in enhanced R2 methylation (56). Since our in vitro experiments showed strongly reduced AIRE-PHD1 binding to histone H3R2me2asK4me0, we studied whether the increase in R2 dimethylation level by PRMT6 in HEK293 cells has an influence on activation of AIRE target genes. As seen in Figure 4A, the PRMT6 over-expression strongly increased the dimethylation levels of R2 both when transfected with control plasmid or with AIRE. As we demonstrated earlier (22,57), AIRE efficiently activates the expression of two target genes, involucrin and S100A8, but does not affect AIRE-independent S100A10 gene. However, the activation of AIRE target genes was remarkably reduced in the presence of higher R2 methylation level when PRMT6 was co-transfected with AIRE (Figure 4A and B). Importantly, no reduction of studied mRNA levels was observed when PRMT6 was transfected alone, indicating that R2 methylation antagonizes the AIRE ability to activate gene expression. The highest R2 dimethylation levels have previously been associated with repressed heterochromatin regions (55); therefore, it is conceivable that R2 dimethylation is one of the epigenetic marks, which exclude AIRE binding to heterochromatin areas. Figure 4.R2 methylation hinders AIRE ability to activate target genes. (A) Western blot analysis of transfected HEK293 cells. Transfections were carried out as indicated below, antibodies and/or detected proteins are indicated on the right. (B) Relative expression level of AIRE regulated involucrin (INV) and S100A8 genes and AIRE independent S100A10 gene. Activations, measured with quantitative RT–PCR, are shown as relative quantity of mRNA compared to control transfection (= 1) of each mRNA. The data are the averages of two independent transfections.</p>
            </div>
          </section>
        </div>
      </section>
      <section data-type="conclusions">
        <h2 id="s6h1">CONCLUSIONS</h2>
        <div class="content">
          <p id="s6p1">The recent discovery, that AIRE binds non-methylated histone H3K4me0 through its first PHD finger to activate gene expression has provided a new link between the status of histone modifications and the regulation of tissue-specific antigen expression in thymus (22,23). The solution structure of AIRE-PHD1 in complex with H3K4me0 peptide described here offers a structural rationale for AIRE-PHD1 specificity towards non-methylated histone H3K4me0, showing that AIRE-PHD1 recognizes non-methylated K4 via hydrophilic caging of its ε amino group.</p>
          <p id="s6p2">Moreover, our results show that AIRE-PHD1 is able to specifically decode the modification status of histone H3 tail, as different post-translational modifications on the first 10 residues of histone H3 tail reduce the affinity by up to several orders of magnitudes. In particular, methylation of R2 is a critical epigenetic mark that strongly interferes with H3 tail binding to AIRE-PHD1. Importantly, R2 dimethylation in vivo antagonizes AIRE ability to activate its target genes, suggesting a cross-talk between non-methylated K4, as epigenetic mark for AIRE-mediated gene activation, and R2 methylation as possible signals for regulation of gene expression in thymic medullary epithelial cells.</p>
        </div>
      </section>
      <section data-type="">
        <h2 id="s7h1">SUPPLEMENTARY DATA</h2>
        <div class="content">
          <p id="s7p1">Supplementary Data are available at NAR Online.</p>
        </div>
      </section>
      <section data-type="">
        <h2 id="s8h1">FUNDING</h2>
        <div class="content">
          <p id="s8p1">Fondazione Telethon, Fondazione Cariplo and Compagnia S. Paolo (to G.M.) by Estonian Science Foundation (6663), the European Regional Development Fund and Welcome Trust (to P.P.). Funding for open access charge: Wellcome Trust.</p>
          <p id="s8p2">Conflict of interest statement. None declared.</p>
        </div>
      </section>
      <section data-type="supplementary-material">
        <h2 id="s9h1">Supplementary Material</h2>
      </section>
    </article>
  </body>
</html>
