<!DOCTYPE html >
<html id="aOeQiFpry6NcNrHbZdTH4hd2gET4-PMC3267856" data-origid="PMC3267856" class="anndoc" data-anndoc-version="2.0" lang="" xml:lang="" xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="UTF-8"/>
    <meta name="generator" content="com.jmcejuela.anndoc.v2.bio.NcbiJournalArticleParser_v0_4$"/>
    <meta name="dcterms.source" content="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3267856/"/>
    <title>aOeQiFpry6NcNrHbZdTH4hd2gET4-PMC3267856</title>
  </head>
  <body>
    <article>
      <section data-type="title">
        <h2 id="s1h1">CEP41 is mutated in Joubert syndrome and is required for tubulin glutamylation at the cilium</h2>
      </section>
      <section data-type="abstract">
        <h2 id="s2h1">Abstract</h2>
        <div class="content">
          <p id="s2p1">Tubulin glutamylation is a post-translational modification (PTM) occurring predominantly on ciliary axonemal tubulin and has been suggested to be important for ciliary function 1,2. However, its relationship to disorders of the primary cilium, termed ‘ciliopathies’, has not been explored. Here, in Joubert syndrome (JBTS) 3, we identify the JBTS15 locus and the responsible gene as CEP41, encoding a centrosomal protein of 41 KDa 4. We show that CEP41 is localized to the basal body/primary cilium, and regulates the ciliary entry of TTLL6, an evolutionarily conserved polyglutamylase enzyme 5. Depletion of CEP41 causes ciliopathy-related phenotypes in zebrafish and mouse, and induces cilia axonemal glutamylation defects. Our data identify loss of CEP41 as a cause of JBTS ciliopathy and highlight involvement of tubulin PTM in pathogenesis of the ciliopathy spectrum.</p>
        </div>
      </section>
      <section data-origid="S1" data-type="methods">
        <h2 id="s3h1">METHODS</h2>
        <div class="subsections">
          <section data-origid="S2" data-type="subjects">
            <h3 id="s3s1h1">Research subjects</h3>
            <div class="content">
              <p id="s3s1p1">MTI-429 family and additional families were recruited worldwide based upon the presence of at least one individual with a neuroradiographically proven ‘molar tooth sign’ associated with any JBTS or related disorder (JSRD) phenotype. Whenever possible, patients underwent a full diagnostic protocol as previously reported 38 and a standardized clinical questionnaire was obtained to assess extent of multi-organ involvement. We used standard methods to isolate genomic DNA from peripheral blood of the affected and unaffected family members after obtaining informed consent from all participating. Human subject research was approved by the Ethics Boards of Leeds (East), CASA Sollievo della Sofferenza Hospital/CSS-Mendel Institute, Hôpital Necker-Enfants Malades, Human subjects divisions at the University of Washington, University of Michigan Institutional Review Board and Human Research Protection Program, University of California, San Diego.</p>
            </div>
          </section>
          <section data-origid="S3" data-type="">
            <h3 id="s3s2h1">Genome-wide screen and fine mapping</h3>
            <div class="content">
              <p id="s3s2p1">A 5K whole genome linkage SNP-scan was performed with family MTI-429 using the Illumina Linkage IVb mapping panel 39, and analyzed with easyLinkage-Plus software 40, which runs Allegro version 1.2c in a PC Windows interface to calculate multipoint LOD scores. Parameters were set to autosomal recessive with full penetrance, and disease allele frequency of 0.001. Fine mapping on the pedigree was performed with the Affymetrix 250K Nsp1 SNP array, and data were searched for common shared homozygous intervals from all affected family members using a custom script implemented in Mathematica (B. Merriman, unpublished). This script identifies all homozygous intervals longer than 2 Mbp for which there are no more than 1 % heterozygous calls (to permit potential genotyping errors).</p>
            </div>
          </section>
          <section data-origid="S4" data-type="">
            <h3 id="s3s3h1">Mutation screening</h3>
            <div class="content">
              <p id="s3s3p1">Mutational screening of CEP41 was performed by direct sequencing of the 11 coding exons and the adjacent intronic junctions in patients. PCR products obtained were treated with Exonuclease I (EXO) (Fermentas) and shrimp alkaline phosphatase (SAP) (Promega), and both strands were sequenced using a BigDye terminator cycle sequencing kit with an ABI3100 automated sequencer (Applied Biosystems). The primers and optimized PCR conditions used are depicted (Supplementary Table 3). Segregation of the identified mutations was investigated in all available family members. All identified mutations in CEP41 were not encountered in 96 ethnically matched controls (188 chromosomes) upon direct sequencing.</p>
            </div>
          </section>
          <section data-origid="S5" data-type="">
            <h3 id="s3s4h1">Bioinformatics</h3>
            <div class="content">
              <p id="s3s4p1">Genetic location is according to the March 2006 Human Genome Browser build hg18. The ciliary proteome was searched using web-based tools 41,42. Protein sequence conservation was determined using ClustalW multiple amino acid sequence alignment (see URL section).</p>
            </div>
          </section>
          <section data-origid="S6" data-type="">
            <h3 id="s3s5h1">Cloning</h3>
            <div class="content">
              <p id="s3s5p1">Full-length human CEP41 was cloned into the TOPO blunt vector, and then shuttled into EGFP- and HA- containing vectors. Human and zebrafish cep41 open reading frame were amplified by RT-PCR and cloned into the pCS2+ vector in order to synthesize RNA for injection into zebrafish embryos. Mouse Cep41 open reading frame was amplified and cloned into the GST-, EGFP-, and FLAG-conjugated vectors for biochemical assays.</p>
            </div>
          </section>
          <section data-origid="S7" data-type="">
            <h3 id="s3s6h1">Generation of Cep41 mutant mice</h3>
            <div class="content">
              <p id="s3s6p1">SIGTR ES cell line (AW0157), derived from 129P2/OlaHsd mice carrying a gene trap insertion 43 in the intron 1 of Cep41, was obtained from the European Conditional Mouse Mutagenesis Program (EUCOMM) and injected into C57BL/6 recipient blastocysts at the Mouse Biology Program, UC Davis. High-percentage chimeras (≥ 70 %) were bred to C57BL/6 for germline transmission. Gene-trapped mice were isolated by PCR for β-galactosidase-neomycin fusion gene and genotyped by Southern blot analysis hybridizing a ~10 kb (wild-type allele) and a ~8 kb (gene-trapped allele) product (Supplementary Fig. 5).</p>
            </div>
          </section>
          <section data-origid="S8" data-type="">
            <h3 id="s3s7h1">Zebrafish experiments</h3>
            <div class="content">
              <p id="s3s7p1">AB wild type zebrafish strains were used for in situ hybridization carried out by standard protocols using a DIG-labeled sense and anti-sense RNA probe for cep41. To knockdown zebrafish cep41, a translational blocking morpholino antisense oligonucleotide (MO, Gene Tools Inc) was used: 5′-CATCTTCCAGCAGCAGAGCTTCGGC-3′, diluted to appropriate concentrations in deionized sterile water, and injected into one-two cell stage embryos, obtained from natural spawning of zebrafish lines. To rescue the phenotypes in MO-injected embryos (morphants), RNA transcribed in vitro with the SP6 mMessage mMachine kit (Ambion) was co-injected. For characterization of ciliary defects in zebrafish embryos, the morphological phenotype of either cep41 or ttll6 morphants were observed until 5 days post-fertilization (dpf) and quantified under bright-field microscopy based upon previously established criteria 44. For western blot analysis at 1–2 dpf zebrafish control embryos and cep41 morphants, about 50 embryos with each genotype were deyolked and the embryo lysates were extracted with RIPA buffer. For immunostaining with GT335, Ac-Tub and PolyG Abs in whole-mount zebrafish embryos, 3 dpf control embryos and cep41 morphants were fixed in Dent’s fixative (80 % MeOH: 20 % DMSO) at 4 °C overnight and incubated with GT335 Ab (1:400), Ac-Tub (1:400) and PolyG (1:300) as primary and with anti-goat-mouse 594 (1:1000) as secondary in diluted blocking solution (10 % normal goat serum: 0.5 % Tween 20 in PBS).</p>
            </div>
          </section>
          <section data-origid="S9" data-type="">
            <h3 id="s3s8h1">Cell culture and transfection</h3>
            <div class="content">
              <p id="s3s8p1">IMCD3, hTERT-RPE1, human fibroblast, and human embryonic kidney (HEK293) cells were grown in appropriate DMEM or MEM media supplemented with 10–20 % fetal bovine serum (FBS) at 37 °C in 5 % CO2. Healthy human female and male control fibroblast cells were obtained from ATCC, and patient fibroblasts from skin biopsies were propagated in culture (≤ 5 passage number). Human fibroblast cells were transfected using the Basic Nucleofector Kit for Primary Mammalian Fibroblasts (Lonza). Other cells were transfected at 60–80 % confluency with plasmids or siRNAs using Lipofectamine 2000 (Invitrogen). The transfected cells were incubated for 24–72 hr with FBS or without FBS, dependent on the experimental purpose.</p>
            </div>
          </section>
          <section data-origid="S10" data-type="">
            <h3 id="s3s9h1">Fluorescence microscopy and transmission electron microscopy (TEM)</h3>
            <div class="content">
              <p id="s3s9p1">Images of immunofluorescent stained cells were obtained on a Deltavision RT Deconvolution microscope (Olympus IX70), under the same parameters for each experiment. Taken images were edited and analyzed using Adobe Photoshop CS. For electron microscopy, a standard protocol 45 was used except for one modification that tannic acid was included in the fixative to enhance the final contrast of the images. Formvar-coated slot grids (Electron microscopy Sciences) were used for sections (60–70 nm) to maximize visibility of the tissue and cross section performed to observe cilia axonemal structure.</p>
            </div>
          </section>
          <section data-origid="S11" data-type="">
            <h3 id="s3s10h1">Live imaging of zebrafish embryos</h3>
            <div class="content">
              <p id="s3s10p1">Zebrafish embryos (12 hpf for cilia in the KV and 2.5 dpf for the renal cilia) were transferred with embryo media to glass bottom culture dishes (MatTek), and 2.5 dpf embryos were anesthetized in tricaine solution (~0.016 mg/ml). Images were acquired for 30 sec – 2 min using a Perkin Elmer UltraView Vox Spinning Disk Confocal with EMCCD Hamamatsu 14 bit 1K × 1K camera, and edited with Volocity imaging software (Perkin Elmer).</p>
            </div>
          </section>
          <section data-origid="S12" data-type="">
            <h3 id="s3s11h1">Immunofluorescence and biochemical assay</h3>
            <div class="content">
              <p id="s3s11p1">For immunofluorescence, cells were fixed in 100 % methanol at −20 °C for 10 min. Primary antibodies used for immunofluorescence are: rabbit anti-CEP41, raised in rabbit to a purified bacterially expressed protein of GST-fused CEP41, (PRF&amp;L, PA), mouse anti-acetylated-tubulin (Sigma), mouse GT335 Ab (gift from C. Janke), rabbit PolyE (gift from M. Gorovsky) and rabbit anti-ARL13B (gift from T. Caspary). We used Alexa 488- or Alexa 594-conjugated secondary antibodies (Molecular Probes) and Hoechst 33342 nuclear dye. All antibodies were diluted in 4 % normal donkey serum in PBS, primary antibodies were incubated for overnight at 4 °C and secondary antibodies were applied for 1 hr at room temperature. For immunoblotting, cells were extracted with RIPA lysis buffer 3 days after transfection and boiled with SDS sample buffer. Same concentrations of lysates were loaded for paired experiments. Primary antibodies used for western blotting were: rabbit anti-GFP (Roche), mouse anti-Flag (Sigma), and mouse anti-alpha-Tubulin (Sigma). Bound antibodies were detected using horseradish peroxidase-conjugated secondary antibodies (Pierce). For co-immunoprecipitation (co-IP), whole cell extracts (WCE) were prepared from confluent HEK293 cells transiently transfected with 14 μg plasmid DNA in 10 cm tissue culture dishes. WCE supernatants were processed for IP experiments by using 2 μg primary antibodies and protein A/G mixed agarose beads (Pierce).</p>
            </div>
          </section>
          <section data-origid="S13" data-type="">
            <h3 id="s3s12h1">Statistical analysis</h3>
            <div class="content">
              <p id="s3s12p1">The χ2 staticstic was computed manually with P value assigned for 1 degree of freedom in the characterization of mouse embryonic phenotype (Supplementary Table 2). For other studies, Student’s two-tailed non-paired t-tests were carried out to determine the statistical significance of differences between samples. P &lt; 0.05 was considered statistically significant for all tests.</p>
            </div>
          </section>
          <section data-origid="S14" data-type="">
            <h3 id="s3s13h1">URLs</h3>
            <div class="content">
              <p id="s3s13p1">Human Genome Browser, http://www.genome.ucsc.edu; ClustalW, http://www.ebi.ac.uk/Tools/msa/clustalw2/.</p>
            </div>
          </section>
        </div>
      </section>
      <section data-origid="S15" data-type="supplementary-material">
        <h2 id="s4h1">Supplementary Material</h2>
      </section>
      <section data-type="resources">
        <h2 id="s5h1">Resources</h2>
        <div class="content">
          <figure data-origid="F1">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3267856/#F1">Figure 1</a>
            <figcaption>
              <p id="s5f1p1">Identification of mutations in CEP41 in affected individuals linked to the JBTS15 locus. (a) Pedigree MTI-429 shows double first cousin marriage with five affected offspring. (b) Axial brain MRI images from patients with CEP41 mutations in each MTI-429, MTI-1491 (T1-weighted) and COR-98 (T2-weighted) family, showing the ‘molar tooth sign’ (red arrows). (c) JBTS15 is 5.5 Mbp located at Chr. 7q31.33–32.3 (red box) defined by rs766240 and rs4728251. Further narrowing of the interval to 2.8 Mbp based upon denser SNP scan to encompass rs17165226 and rs2971773 (black arrows). (d) CEP41 genomic organization, depicting locations of identified base changes including homozygous (red) splice mutations and heterozygous (blue) missense or nonsense mutations. Capital letters: exon sequences, small letters: intron sequences, asterisks: point mutations, underline: deletion. (e) RT-PCR confirmation of the splicing defect of CEP41 in MTI-429 patient fibroblasts. Both CEP41 mutant cells failed to produce CEP41 mRNA, compared with WT. GAPDH is control. Green arrows: positions of primers, asterisk: region of splice mutation in MTI-429.</p>
            </figcaption>
          </figure>
          <figure data-origid="F2">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3267856/#F2">Figure 2</a>
            <figcaption>
              <p id="s5f2p1">CEP41 is expressed in ciliated tissues and its loss recapitulates ciliopathy-related phenotypes in zebrafish and mouse. (a) Zebrafish cep41 mRNA is expressed ubiquitously at gastrulation stages (6 hpf), but at later stages, it is specifically expressed in ciliated organs: Kupffer’s vesicle (red box), inner ear (white boxes), brain (brackets), eyes (arrows), pronephric duct (black box), and heart (asterisks). A, anterior; D, dorsal; P, posterior; V, ventral. (b) CEP41 is predominantly localized to the basal body (arrowheads) and primary cilium (arrows) in ciliated IMCD3 and hTERT-RPE1 cells. Insets: co-localization of CEP41 with GT335, a marker of basal bodies and cilia. Scale bar 5 μm. (c) Knockdown of cep41 by injection of morpholino oligonucleotide (MO) causes heart asymmetry defects in Tg (myl7:egfp) zebrafish embryos. The cep41 morphants show either loss of asymmetry (midline) or inversion of V/A asymmetry (reversed) at 72hpf. *P &lt; 0.01, **P &lt; 0.001. Error bars: s.e.m. A, atrium; L, left; R, right; V, ventricle. (d) Murine Cep41 gene-trap shows altered embryonic morphogenesis. Range of phenotypes of Cep41Gt/Gt embryos includes mild malformed hindbrain (arrowheads), exencephaly (brackets), hemorrhage in the head (asterisk), dilated pericardial sac (arrows), failure to rotate and lethality at E10–12. (e) Injection of human CEP41 RNA into cep41 morphants rescued ciliary phenotypes of pericardial edema (arrowhead), hydrocephalus (asterisk) and curved tail (arrow), completely or partially.</p>
            </figcaption>
          </figure>
          <figure data-origid="F3">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3267856/#F3">Figure 3</a>
            <figcaption>
              <p id="s5f3p1">CEP41 is required for tubulin glutamylation at the ciliary axoneme. (a) Absent CEP41 protein in CEP41 mutant patient cells. (b) Ciliary localization of endogenous CEP41 in human fibroblasts and loss of the protein in CEP41 mutant cilia. Scale bar 5 μm. (c) WT cells have both GT335-positive basal bodies (arrowheads) and cilia (arrows, marked by ARL13B), while mutant cells show staining of GT335 only at the basal bodies. Scale bar 5 μm. (d) Depletion of cep41 causes glutamylation defects in zebrafish olfactory placode cilia (red arrows). Images of GT335-stainined WT and cep41 morphant embryos were taken in anterior view (head is up and tail is down, D, dorsal; V, ventral), quantified below. (e) Exogenous CEP41 expression restores ciliary axoneme glutamylation in CEP41 mutant cells. Arrows: primary cilia stained for CEP41 and GT335. Insets: merged images at higher power, quantified below. Scale bar 5 μm. **P &lt; 0.001; error bars = s.e.m. (f) Ultrastructural analysis of the pronephric ciliary axoneme at 72 hpf zebrafish embryos. Compared to WT, cep41 morphants have A-tubule specific defects in the outer doublet microtubules. Arrows: A-tubules and one of nine outer doublet microtubules is magnified in the red box. The numbers of cilia, categorized in normal and abnormal A-tubules according to the schematic, were counted in both WT embryos and cep41 morphants (n = 3 embryos, &gt;20 cilia per animal), quantified below. Scale bar 100 nm. **P &lt; 0.01; error bars = s.e.m.</p>
            </figcaption>
          </figure>
          <figure data-origid="F4">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3267856/#F4">Figure 4</a>
            <figcaption>
              <p id="s5f4p1">CEP41 interacts with TTLL6 and is required for localizing TTLL6 to the cilium. (a) Morpholino knockdown of zebrafish ttll6 associates with ciliary phenotypes such as curved tail (arrows), abnormal number/orientation of ear otolith (boxes), cystic kidney (arrowheads) and peripheral cardiac edema (asterisks) at different dosages indicated and show A-tubule specific defects in the outer doublet microtubules. The numbers of defective cilia were counted in both WT embryos and ttll6 morphants (n = 3 embryos, &gt;20 cilia per animal), quantified below. (b) GFP-CEP41 was immunoprecipitated with anti-Flag antibody recognizing Flag-tagged TTLL6 from whole-cell extract (WCE), compared with GFP-empty vector. In the reciprocal co-IP experiment with anti-GFP antibody, the interaction between CEP41 and TTLL6 was confirmed. (c) Disturbed localization of TTLL6 to the cilium following Cep41 siRNA co-transfection with GFP-TTLL6 into IMCD3 cells and immunostained with either GT335 or ARL13B antibody. Arrows: cilia; Arrowheads: basal bodies. Cells expressing ciliary localized TTLL6 were counted only in siRNA-transfected cells quantified in graph. Scale bar 5 μm. *P&lt; 0.01, **P &lt; 0.001. Error bars = s.e.m.</p>
            </figcaption>
          </figure>
          <figure class="table" data-origid="T1">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3267856/#T1">Table 1</a>
            <figcaption>
              <p id="s5f5p1">Abbreviations. AG: Ambiguous genitalia, B: Bilateral, F: Female, GHD: Growth hormone deficiency, M: Male, MP: Micropenis, N: No, N/A: Not available/Not applicable, NPHP: Nephronophthisis, OMA: oculomotor arpaxia, PDSV: potentially deleterious sequence variant, UCD: Urinary concentration defect, U: Unilateral, Y: Yes</p>
            </figcaption>
          </figure>
        </div>
      </section>
    </article>
  </body>
</html>
