<!DOCTYPE html >
<html id="ar4UP4A1jiw3BIXhT1FdHYzkpq_S-PMC3184204" data-origid="PMC3184204" class="anndoc" data-anndoc-version="2.0" lang="" xml:lang="" xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="UTF-8"/>
    <meta name="generator" content="com.jmcejuela.anndoc.v2.bio.NcbiJournalArticleParser_v0_4$"/>
    <meta name="dcterms.source" content="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3184204/"/>
    <title>ar4UP4A1jiw3BIXhT1FdHYzkpq_S-PMC3184204</title>
  </head>
  <body>
    <article>
      <section data-type="title">
        <h2 id="s1h1">Heritable GATA2 Mutations Associated with Familial Myelodysplastic Syndrome and Acute Myeloid Leukemia</h2>
      </section>
      <section data-type="abstract">
        <h2 id="s2h1">Abstract</h2>
        <div class="content">
          <p id="s2p1">We report the discovery of the GATA2 gene as a new myelodysplastic syndrome (MDS)/acute myeloid leukemia (AML) predisposition gene. We found the same, novel heterozygous c.1061C&gt;T (p.Thr354Met) missense mutation in the GATA2 transcription factor gene segregating with the multigenerational transmission of MDS/AML in three families, and a GATA2 c.1063_1065delACA (p.Thr355del) mutation at an adjacent codon in a fourth MDS/AML family. The mutations reside within the second zinc finger of GATA2 which mediates DNA-binding and protein-protein interactions. We show differential effects of the mutants on transactivation of target genes, cellular differentiation, apoptosis and global gene expression. Identification of such predisposing genes to familial forms of MDS and AML is critical for more effective diagnosis and prognosis, counselling, selection of related bone marrow transplant donors, and development of therapies.</p>
        </div>
      </section>
      <section data-origid="S1" data-type="methods">
        <h2 id="s3h1">ONLINE METHODS</h2>
        <div class="subsections">
          <section data-origid="S2" data-type="subjects">
            <h3 id="s3s1h1">Patients</h3>
            <div class="content">
              <p id="s3s1p1">Families (Supplementary Table 2, Supplementary Note) were recruited and sample use approved through institutional human ethics review board approved protocols from the Australian Familial Haematological Cancer Study (Royal Adelaide Hospital (RAH) #091203 and #100702, and Children, Youth and Women’s Health Service #REC1542/12/12, Adelaide, SA Australia), The Queen Elizabeth Hospital and the University of Washington (Seattle, WA USA).</p>
            </div>
          </section>
          <section data-origid="S3" data-type="">
            <h3 id="s3s2h1">Sequence analysis of candidate genes</h3>
            <div class="content">
              <p id="s3s2p1">To identify germline and somatic mutations in patients with familial AML, a panel of 50 hematopoietic candidate genes, incorporating a total of 638 exons, was assembled (Supplementary Table 1). Primer design, PCR amplification, and dideoxy sequencing of genomic DNA purified from lymphoblastoid cells of probands from 7 MDS/AML pedigrees were performed by the Australian Genome Research Facility (AGRF). Sequences were aligned with NCBI RefSeq sequences using Mutation Surveyor (SoftGenetics) and variants compared to the UCSC and NCBI SNP databases for novelty. Sequence changes were confirmed by re-sequencing in both directions. Primer sequences are available upon request. Screening of control and sporadic AML populations was performed using high resolution melt (HRM) analysis (Supplementary Table 8 and Supplementary Note).</p>
            </div>
          </section>
          <section data-origid="S4" data-type="">
            <h3 id="s3s3h1">Cell culture</h3>
            <div class="content">
              <p id="s3s3p1">HEK293, 293T and Cos-7 cells were cultured in DMEM with 10% fetal bovine serum (FBS) (JRH Biosciences) and transient transfections were performed using Lipofectamine 2000 (Invitrogen). HL-60 promyelocytic cells were cultured in RPMI containing 10% FBS. All cultures contained 50 units/ml penicillin and 50 μg/ml streptomycin (Sigma).</p>
            </div>
          </section>
          <section data-origid="S5" data-type="">
            <h3 id="s3s4h1">Generation of mutant GATA2 plasmid and lentiviral expression constructs</h3>
            <div class="content">
              <p id="s3s4p1">An expression clone (pCMV6-XL6-GATA2) containing a 3.7 kb GATA2 cDNA insert was obtained from OriGene, and p.Thr354Met, p.Thr355del and p.Leu359Val mutants were generated by site directed mutagenesis. The coding regions of wildtype (WT), p.Thr354Met and p.Leu359Val were cloned into a dual lentiviral vector system which was used to generate HL-60 cells expressing GATA2 WT or mutants upon addition of 4-hydroxytamoxifen (4HT) (see Supplementary Note).</p>
            </div>
          </section>
          <section data-origid="S6" data-type="">
            <h3 id="s3s5h1">GATA2-responsive promoter and enhancer studies</h3>
            <div class="content">
              <p id="s3s5p1">The GATA2-responsive promoter (LYL1) and enhancer (RUNX) were PCR amplified and cloned into pGL4.12[luc2CP] (SfiI) and pGL3-Promoter (KpnI/BglII) (Promega), respectively. The CSF1R (M-CSF-R) promoter was PCR amplified and cloned into pGL4.12[luc2CP] (SfiI). See Supplementary Table 9 for PCR primers used. The GATA2-responsive CD34 enhancer-luciferase construct (CD34x2/Luc) and one with the GATA binding sites mutated (mutant CD34x2/Luc)31 were kindly provided by Tariq Enver, Weatherall Institute of Molecular Medicine, Oxford, U.K. HEK293 or Cos-7 cells were transfected at 90% confluence with Lipofectamine 2000. In all experiments, the molar equivalents of EV constructs were used to balance gene expressing constructs to avoid squelching artifacts. After 20 h, cells were harvested and luciferase activity determined with the Dual-Luciferase Reporter Assay System (Promega) using a GloMax®-Multi Detection System (Promega). All assays were performed a minimum of three times in triplicate. All results were analysed using Student’s t-test, and reported as mean ± s.e.m. with significance, p&lt;0.05 (asterisk).</p>
            </div>
          </section>
          <section data-origid="S7" data-type="">
            <h3 id="s3s6h1">Cell differentiation and proliferation assays</h3>
            <div class="content">
              <p id="s3s6p1">HL-60 cells were plated at 1.25 × 104 cells/ml and treated with or without 30 nm 4-hyrdoxy tamoxifen (4-HT) for 24 h and then with or without 2 μM all-trans retinoic acid (ATRA). Cell numbers were determined by manual counting and FACS analysis (Phycoerythrin anti-mouse CD11b and Phycoerythrin rat IgG2b isotype control) (eBioscience) was performed 6 days after addition of ATRA (Sigma). The cells were also stained with hematoxylin and eosin for assessing morphological changes.</p>
            </div>
          </section>
          <section data-origid="S8" data-type="">
            <h3 id="s3s7h1">Haplotyping</h3>
            <div class="content">
              <p id="s3s7p1">Haplotype mapping was performed by PCR amplification and sequencing of amplicons containing 50 single nucleotide polymorphisms (SNP) within and surrounding the position of the p.Thr354Met variant of the GATA2 gene (Supplementary Table 10). All amplicons were generated using AmpliTaq Gold (Applied Biosystems) according to the manufacturer’s protocol using 2 mM MgCl2 and the following cycle strategy; 95°C, 10 min; 95°C, 30 s, 66°C – 58°C, 20 s (touchdown, 0.8°C/cycle for 10 cycles), 72°C, 45 s (total of 40 cycles); 72°C, 3 min.</p>
            </div>
          </section>
          <section data-origid="S9" data-type="">
            <h3 id="s3s8h1">Generation of mutant GATA2 plasmids and lentiviral expression constructs</h3>
            <div class="content">
              <p id="s3s8p1">An expression clone (pCMV6-XL6-GATA2) containing a 3.7 kb GATA2 cDNA insert was obtained from OriGene (Cat. No. SC125368). p.Thr354Met, p.Thr355del and p.Leu359Val mutants were generated by QuikChange mutagenesis (Stratagene) using the primers T354M-F and T354M-R, 355delT-F and 355delT-R, and L359V-F and L359V-R (Supplementary Table 11), respectively. For the generation of lentiviral expression constructs, the regulatable pF 5xUAS W SV40 Puro (5xUAS)32 was used. GATA2 WT or mutants were PCR amplified from the above pCMV6 plasmid vectors using the primers (KOZAK-GATA2-F and either GATA2-FLAG-R or GATA2-R) (Supplementary Table 12) and Pfu Turbo (Stratagene), excised with XbaI and cloned into the unique XbaI site of 5xUAS.</p>
            </div>
          </section>
          <section data-origid="S10" data-type="">
            <h3 id="s3s9h1">Generation of regulatable GATA2 expressing HL-60 cell lines</h3>
            <div class="content">
              <p id="s3s9p1">A dual lentiviral vector system was used to generate HL-60 cells expressing GATA2 WT or mutants upon addition of 4-hydroxytamoxifen (4HT) (Sigma). Infectious third generation lentivirus was made by cotransfecting 293T cells with either 5xUAS-GATA2 (WT or mutants) or pF GEV16 Super PGKHygro (GEV16)33 plasmid and the three packaging plasmids pHCMVwhvgagpolml, pHCMV-G and pHCMVwhvrevml34 (mass ratio 50:5:2.5:1). Supernatants were harvested 24 h later and filtered (Nalgene 45 μm syringe filter) (Nalge Nunc Int.). HL-60 cells were firstly transduced with GEV16 lentiviral supernatant including 4 μg/ml polybrene and 2.5 μg/ml fungizone. After 48 h, HL-60GEV cells were selected in 1 mg/ml hygromycin (Roche). These cells were subsequently transduced with the GATA2 (WT, p.Thr354Met, p.Thr355del and p.Leu359Val) or EV lentiviral supernatant and selected in 3 μM Puromycin (Sigma).</p>
            </div>
          </section>
          <section data-origid="S11" data-type="">
            <h3 id="s3s10h1">Immunofluorescence staining</h3>
            <div class="content">
              <p id="s3s10p1">HL-60 cells carrying stably transduced 4HT-regulatable GATA2 (WT, p.Thr354Met, p.Thr355del and p.Leu359Val) were treated with and without 100nM 4HT. After 24 h, the cells were fixed with 4% of paraformaldehyde for 10 min. The cells were permeabilized with 0.1% Triton/PBS, for 10 min and blocked with 2% BSA for 30 min. The cells were then stained with rabbit α-GATA2 antibody (Santa Cruz Biotechnology, Inc) (1:1000) for 1 h followed by Alexa 594-conjugated goat anti-rabbit secondary antibody (Molecular Probes) (2 μg/ml) for 20 min. The slides were mounted in Vectashield® mounting medium with DAPI (Vector Laboratories, Inc). Cells without primary antibody served as negative controls. All incubations were performed at room temperature.</p>
            </div>
          </section>
          <section data-origid="S12" data-type="">
            <h3 id="s3s11h1">Western blot analysis</h3>
            <div class="content">
              <p id="s3s11p1">HL-60 cells carrying stably transduced 4HT-regulatable GATA2 (WT, p.Thr354Met, p.Thr355del and p.Leu359Val) were treated with and without 100 nM 4HT. After 24 h, the cells were harvested in RIPA buffer (50 mM Tris-Cl pH 7.6, 150 mM NaCl, 1% Triton X-100, 0.5% sodium deoxycholate, and 0.1% sodium dodecyl sulfate with protease inhibitor (cOmplete Mini EDTA free protease inhibitor tablets, Roche Diagnostics)). Samples were loaded onto the 10% acrylamide gels, electrophoresed and transferred onto Hybond-P PVDF membranes (Amersham). Membranes were probed with antibodies using standard techniques and visualised using ECL plus detection reagents (Amersham) on x-ray film (Amersham Hyperfilm™ MP).</p>
            </div>
          </section>
          <section data-origid="S13" data-type="">
            <h3 id="s3s12h1">Apoptosis Assays</h3>
            <div class="content">
              <p id="s3s12p1">HL-60 cells were stained for surface Annexin V and propidium iodide according to the manufacturer’s protocol (#556547, Becton Dickinson).</p>
            </div>
          </section>
          <section data-origid="S14" data-type="">
            <h3 id="s3s13h1">Electromobility shift assay (EMSA) and EMSA-Western Blot</h3>
            <div class="content">
              <p id="s3s13p1">HEK293 cells were transfected with GATA2 WT or mutants using Lipofectamine™ 2000. After 24 h, nuclear extracts were prepared using a NE-PER® Nuclear and Cytoplasmic Extraction kit (Pierce) according to the manufacturer’s protocol. Double stranded DNA oligonucleotides containing two GATA binding sites (Human TCRδ enhancer) or a single GATA binding site (GATA Consensus and Human GM-CSF-153 promoter) were synthesized (Supplementary Table 13). Each single stranded oligomer was labeled using a Biotin 3′ End DNA Labeling kit (Pierce) and annealed according to manufacturer’s protocol. Electrophoretic mobility shift assays were performed using a modified protocol from Kumar et al 200835 and visualized using a Chemiluminescent Nucleic Acid Detection Module (Pierce) according to the manufacturer’s protocol. Double stranded labeled probes (100 fmol) were incubated with 3 μg of nuclear extract for 20 min in 1x binding buffer containing 20 mM HEPES-KOH, pH 7.9, 100 mM KCl, 2 mM MgCl2, 10 μM ZnSO4, 10 mM 2 mercaptoethanol, 0.1% NP-40, 10% glycerol, 0.2 mM EDTA and 5 μg/ml sheared salmon sperm DNA. Polyclonal rabbit α-GATA2 (H-116) antibody (Cat. No. sc-9008; Santa Cruz Biotechnology, Inc) (1:100) was added to nuclear lysates for 20 min prior to addition of probe to demonstrate GATA2 as the binding protein. To assess the specificity of the binding, 200-fold excess of each unlabeled probe was used as competitor. The mixtures were resolved in 6% non-denaturing polyacrylamide gels made in 0.5x TGE buffer (12.5 mM Tris-HCl, pH 8.5, 85 mM glycine and 0.5 mM EDTA) and the electrophoresis was performed at 4°C. For EMSA-Western blots, the experiment was carried as described above, except that the shifted DNA oligonucleotides-protein complexes were transferred onto nitrocellulose membrane, instead of PVDF. The membrane was probed with monoclonal mouse α-GATA2 (CG2-96) antibody (Cat. No. sc-267; Santa Cruz Biotechnology, Inc) and detection was performed as mentioned above.</p>
            </div>
          </section>
          <section data-origid="S15" data-type="">
            <h3 id="s3s14h1">Determination of genes differentially expressed in the presence of GATA2 mutants</h3>
            <div class="content">
              <p id="s3s14p1">HL-60 cell lines were treated with 100 nM 4HT to turn on GATA2 WT and mutant protein expression. After 24 h, gene expression levels were determined by microarray(Supplementary Note).</p>
            </div>
          </section>
        </div>
      </section>
      <section data-origid="S16" data-type="supplementary-material">
        <h2 id="s4h1">Supplementary Material</h2>
      </section>
      <section data-type="resources">
        <h2 id="s5h1">Resources</h2>
        <div class="content">
          <figure data-origid="F1">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3184204/#F1">Figure 1</a>
            <figcaption>
              <h5 id="s5f1h1">Identification of novel germline p.Thr354Met and p.Thr355del variants in the highly conserved zinc finger 2 domain of GATA2 that is associated with MDS-AML</h5>
              <p id="s5f1p1">a. Pedigrees containing the p.Thr354Met and p.Thr355del variants. One family from Australia (Pedigree 1) and two from the USA (Pedigrees 2 and 3) display the p.Thr354Met variant segregating with MDS-AML, and one USA family (Pedigree 4) contains a p.Thr355del variant that segregates with MDS. The genotype of tested individuals is shown; T354, (Thr354/Thr354); T354M, (Thr354/Met354). b. Domain structure of GATA2 showing positions of mutations. The positions of the p.Thr354Met, p.Thr355del, AML-M57 (green) and CML BC6 (black) mutations are shown with respect to zinc finger (ZF) 1 and 2, transactivation domain (TA) and nuclear localization signal (NLS). c. Zinc finger 2 (ZF2) domain of GATA2 and GATA3 contains mutations associated with leukemia and breast cancer. The primary sequence is that of human GATA2 with the two alternative residues in GATA3 ZF2 shown (light grey with black letters). The position of p.Thr354Met and p.Thr355del is highlighted along with mutations found in GATA2 in AML-M57 (green) and CML BC6 (black), and in GATA3 in breast cancer (summarized in 8) (mutated residues in the corresponding GATA3 ZF2; grey with white letters).</p>
            </figcaption>
          </figure>
          <figure data-origid="F2">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3184204/#F2">Figure 2</a>
            <figcaption>
              <h5 id="s5f2h1">Subcellular localisation and DNA binding properties of GATA2 WT and mutants</h5>
              <p id="s5f2p1">HEK293 cells were transiently transfected with EV (pCMV-XL6 empty vector), WT, p.Thr354Met, p.Thr355del or p.Leu359Val and harvested after 24 h. a. Western blot analysis of GATA2 expression in nuclear lysates. Nuclear lysates were prepared and western blots performed, probing for GATA2. b. Cells were stained for GATA2 (red) and DAPI (blue). Scale bars, 10 μm. c. Electromobility shift assay (EMSA) of GATA2 WT and mutants. Nuclear lysates were prepared and bound to the TCRδ enhancer (contains GATA binding site) oligonucleotide in the absence or presence of 200-fold unlabeled competitor oligonucleotide (D, human TCRδ enhancer; C, GATA consensus; G, GM-CSF promoter). The probes were visualised using chemiluminescence (top panel). Note, GATA2 &amp; NS relates to a band that contains both GATA2 and a non-specific (NS) protein. To visualise GATA2 alone, an EMSA-western blot was performed probing with polyclonal α-GATA2 antibody (bottom panel), showing the level of binding of GATA2 WT and mutants. A neutralizing α-GATA2 antibody in the far right lane removes GATA2, but not the non-specific binding protein (NS) (top panel), and the specificity of GATA2 is confirmed in the bottom panel.</p>
            </figcaption>
          </figure>
          <figure data-origid="F3">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3184204/#F3">Figure 3</a>
            <figcaption>
              <h5 id="s5f3h1">p.Thr354Met and p.Thr355del cause altered transactivation via target GATA2 response elements</h5>
              <p id="s5f3p1">p.Thr354Met and p.Thr355del act as a loss-of-function mutations on GATA2 target promoter and enhancer elements. HEK293 cells were cotransfected with 1) GATA2-responsive CD34 (mut – CD34 enhancer with GATA binding sites mutated31) (a) and RUNX1 (b) enhancer elements linked to a LUC reporter, and 2) GATA2 (WT, p.Thr354Met, p.Thr355del or p.Leu359Val) expression constructs or pCMV6-XL6 empty vector (EV). Similarly, Cos-7 cells were cotransfected using LYL1 promoter LUC as reporter (c). After 20 h, cells were harvested and luciferase assays performed and plotted as fold (mean ± s.e.m.) compared to EV control. Pairwise comparisons are shown (*p&lt;0.05, n = 3). d. p.Thr354Met and p.Thr355del act as dominant negative mutations over WT GATA2. HEK293 cells were cotransfected with: 1) CD34 enhancer-LUC reporter, and equivalent mole ratios of 2) WT to 3) p.Thr354Met or p.Thr355del. After 20 h, cells were harvested and luciferase assays performed. Pairwise comparisons are shown (*p&lt;0.05; NS -not significant, n=3). e. p.Thr354Met has reduced ability to co-activate the CSF1R (M-CSF-R) promoter with PU.1. Cos-7 cells were cotransfected with 1) CSF1R promoter-LUC reporter, 2) PU.1 expression construct, and 3) WT, p.Thr354Met, p.Thr355del or p.Leu359Val expression constructs or EV. After 20 h, luciferase assays were performed and plotted as fold compared to EV. Pairwise comparisons are shown (*p&lt;0.05, compared to WT plus PU.1; **p&lt;0.05 compared to WT plus PU.1, but not significant when compared to p.Thr354Met or p.Thr355del plus PU.1, respectively). In all comparisons, a Student’s t-test was used.</p>
            </figcaption>
          </figure>
          <figure data-origid="F4">
            <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3184204/#F4">Figure 4</a>
            <figcaption>
              <h5 id="s5f4h1">p.Thr354Met inhibits differentiation and apoptosis while allowing accumulation of cells in the presence of ATRA-induced differentiation</h5>
              <p id="s5f4p1">HL-60 cells carrying stably transduced 4HT-regulatable GATA2 (WT, p.Thr354Met, p.Thr355del and p.Leu359Val) or EV were treated with or without 30 nM 4HT for 24 h and then with or without 2 μM ATRA for 6 days. a–d. Differentiation of HL-60 cells into granulocytes. Differentiation was measured by FACS analysis for percentage of CD11b positive cells (mean ± s.e.m.) (see also Supplementary Fig. 7b). e–h. Cell numbers following differentiation. Cells were counted after 6 days (mean ± s.e.m.). i–m. Apoptosis following differentiation with ATRA. Cells were FACS analysed following staining with FITC anti-Annexin V and propidium iodide (PI). Annexin V+, PI− (black) or Annexin V+, PI+ (white). Indicative FACS plots (Supplementary Fig. 7c). a,e,i. −4HT, −ATRA; b,f,j. +4HT, −ATRA; c,g,k. −4HT, +ATRA; d,h,m. +4HT, +ATRA.(*p&lt;0.05; **p&lt;0.01, compared to WT). In all comparisons, a Student’s t-test was used.</p>
            </figcaption>
          </figure>
        </div>
      </section>
    </article>
  </body>
</html>
