<!DOCTYPE html >
<html id="aUq41dE.IhLj9a3VTf2rnHQDgwi4-12893989" data-origid="12893989" class="anndoc" data-anndoc-version="2.0" lang="" xml:lang="" xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="UTF-8"/>
    <meta name="generator" content="net.tagtog.anndoc.v2.parsers.bio.PubMedAbstractsParser_v0_1$"/>
    <meta name="dcterms.source" content="http://www.ncbi.nlm.nih.gov/pubmed/12893989"/>
    <title>aUq41dE.IhLj9a3VTf2rnHQDgwi4-12893989</title>
  </head>
  <body>
    <article>
      <section data-type="title">
        <h2 id="s1h1">Mutations in mitochondrial aldehyde dehydrogenase (ALDH2) change cofactor affinity and segregate with voluntary alcohol consumption in rats.</h2>
      </section>
      <section data-type="abstract">
        <h2 id="s2h1">Abstract</h2>
        <div class="content">
          <p id="s2p1">Genetic factors influence alcohol consumption and alcoholism. A number of groups have bred alcohol drinker and non drinker rat strains, but genetic determinants remain unknown. The University of Chile rat lines UChA (low drinkers) and UChB (high drinkers) display differences in the relative K(m) for NAD+ of mitochondrial aldehyde dehydrogenase (ALDH2) but no V(max) differences. The relative K(m) differences may be due to mitochondrial changes or to genetic differences coding for ALDH2. We investigated whether there are differences in the coding regions of ALDH2 cDNA in these lines and whether the Aldh2 genotype predicts the phenotype of alcohol consumption and the K(m) of ALDH2 for NAD+. Liver cDNA was prepared, and the Aldh2 transcript was amplified, cloned and sequenced. Genotyping was conducted by DNA amplification and restriction enzyme digestion. When compared to Aldh21 of Sprague-Dawley, 94% of the UChA (low drinker) rats (n = 61), presented a mutation that changes Gln67 to Arg in the mature enzyme (allele referred to as Aldh22). In UChB (high drinker) rats (n = 69), 58% presented the Aldh21 allele, while 42% presented the Gln67Arg change plus a second mutation that changed Glu479 to Lys (allele Aldh23). The Aldh22 allele was absent in high drinker rats. Rats of different Aldh2 genotypes displayed marked phenotypic differences in both ethanol consumption (g/kg/day; means +/- SE): (Aldh21/Aldh21) = 5.7 +/- 0.2, (Aldh22/Aldh22) = 0.9 +/- 0.2 and (Aldh23/Aldh23) = 4.6 +/- 0.2; and K(m)s for NAD+ of 43 +/- 3 microm, 132 +/- 13 microm and 41 +/- 2 microm, respectively (Aldh22 versus Aldh21 or Aldh23; P &lt; 0.0001 for both phenotypes). Overall, the data show that alleles of Aldh2 strongly segregate with the phenotype of ethanol consumption and the relative K(m) for NAD+ of ALDH2. Bases mutated suggest that non drinker Aldh22 is ancestral with regard to the coding changes in either Aldh21 or Aldh23, variants which would allow ethanol consumption and may provide an evolutionary advantage by promoting calorie intake from fermented products along with carbohydrates.</p>
        </div>
      </section>
    </article>
  </body>
</html>
