<html class="anndoc" data-anndoc-version="2.0" data-origid="12786754" id="12786754" lang="" xml:lang="" xmlns="http://www.w3.org/1999/xhtml"><head><title>12786754</title></head><body><article><section data-type="article"><div class="content"><p id="s1p1">Analysis of splice-site mutations of the alpha-galactosidase A gene in Fabry disease.</p><p id="s1p2">Fabry disease is an X-linked disease caused by a defective lysosomal enzyme, alpha-galactosidase A, and characterized by skin lesions and multiorgan involvement, including kidney, heart, and the central nervous system. Currently more than 200 genotypes have been identified, including several aberrant splicing. However, most of the mutation analyses were performed using genomic sequencing only, and therefore some of the splicing mutations were misclassified as missense mutations. In order to predict the splicing event caused by each mutation, we conducted a literature search for all published mutations located near the splice sites, including exonic point mutations, and performed a splice-site score (SSS) analysis. The literature search identified 13 donor-site mutations, including four exonic mutations (S65T, D183S, K213N, and M267I), located at the end of exons 1, 3, 4, and 5, respectively, six acceptor-site mutations, and one new exon creation. All mutated splice sites, except for the one associated with the new exon creation, had a lower SSS than their respective natural sites. Cryptic or newly created sites were identified with SSS from 0.09 to 1.0. The predictions, based on SSS analysis, are in agreement with all six mutations with known cDNA sequence from the literature, including five mutations with exon skipping and one mutation with creation of a new acceptor site. For the S65T genotype, we performed reverse transcription-polymerase chain reaction (RT-PCR) analysis using RNA isolated from the whole-blood sample. We verified that a weak cryptic site (SSS = 0.09) 14 nucleotides downstream was activated and resulted in an insertion of 14 bp and a frameshift stop at codon 106. This change is more consistent with the clinical presentation of the patient, the classical Fabry disease, than the amino acid substitution (S65T), which does not affect the enzyme function. In conclusion, the SSS analysis is very useful for predicting splicing events and genotype/phenotype correlation in Fabry disease. As different mechanisms may be involved in pre-mRNA splicing, it is important to obtain cDNA sequencing for molecular diagnosis.</p></div></section></article></body></html>